<html><head><meta name=ProgId content=PowerPoint.Slide><meta name=Generator content="Microsoft Macintosh PowerPoint 11"><title>PowerPoint Presentation  -  Complex trait analysis, develop-ment, andgenomics</title><link title="Presentation File" type="application/powerpoint" rel=alternatehref="webqtl_demo2.ppt.ppt"><script>
if ( ! top.PPTPRESENTATION ) {
	window.location.replace( "endshow.htm" );
}
</script><meta name=Descriptioncontent="Jul-18-05: WebQTL searches for upstream controllers"><style media=print><!--.sld	{left:0px !important;	width:6.0in !important;	height:4.5in !important;	font-size:76% !important;}--></style><script language=Javascript src=script.js></script>
<script language=JavaScript>
function font_size() {
	var min = window.innerWidth/20;
	if ( min > window.innerHeight/20 )
		min = window.innerHeight/20;
		
	return min+"pt";
}
var SoundEnabled = navigator.plugins && navigator.plugins["LiveAudio"];
function restore() {
	self.location.reload();
}
function set_size() {
	document.SlideLayer.width = window.innerWidth;
	document.SlideLayer.height = window.innerHeight;
}
document.captureEvents(Event.MOUSEDOWN);
document.captureEvents(Event.KEYPRESS);
document.onkeypress = _KPH ;
document.onmousedown = _KPH ;
 parent.base.highlite(); 
</script><script language=JavaScript><!--
function DoOnLoad() { 
 LoadNavSld("slide0053.htm");
}//-->
</script></head><body onresize="restore()" onload="DoOnLoad()" style='font-size:&{font_size()};'bgcolor="#000000"><script>

g_origHeight = 566;
g_origWidth = 755;
g_width = window.innerWidth - 10;
g_height = window.innerHeight - 10;

g_top = 0;
g_left = 0;

if ( g_width < ( g_origWidth * g_height /g_origHeight ) )
	g_height = g_width * g_origHeight / g_origWidth;
else
	g_width = g_origWidth * g_height / g_origHeight;
	
g_top = (window.innerHeight - g_height) / 2;
g_left = (window.innerWidth - g_width) / 2;

document.write( '<layer id=SlideLayer BGCOLOR=#484848 background="" top=' + g_top + ' left=' + g_left + ' height=' + g_height + ' width=' + g_width + ' >' );
document.SlideLayer.clip.top = 0;
document.SlideLayer.clip.left = 0;
document.SlideLayer.clip.height = g_height;
document.SlideLayer.clip.width = g_width;
function new_font_size() {
	var size = 11 * g_width / g_origWidth;
	if ( size < 1 ) size = 1;
	return size+"pt";
}
</script><div id=SlideObj class=sld style='position:relative;font-size:&{new_font_size()};'><script>
	mytop = 0 * g_height; myleft = 0 * g_width; myheight = 1 * g_height; mywidth = 1 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
ht =   g_height;
wd =   g_width;
document.write(
'<img src="master03_background.png" height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer><script>
	mytop = 0 * g_height; myleft = 0.01 * g_width; myheight = 0.08 * g_height; mywidth = 1.01 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script>
	mytop = 0.01 * g_height; myleft = 0 * g_width; myheight = 0.06 * g_height; mywidth = 1 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div style='text-align:left;font-family:"Gill Sans";font-weight:normal;font-style:normal;text-decoration:none;text-shadow:none;text-effect:none;mso-fareast-hint:no;layout-flow:horizontal;color:#E9EB5D;font-size:293%;mso-text-raise:0%;mso-line-spacing:"-358 0 -1";mso-margin-left-alt:233;mso-text-indent-alt:0;mso-line-spacing:"-358 0 -1";mso-margin-left-alt:233;mso-text-indent-alt:0'><span style='font-family:Verdana;font-size:64%'><i>WebQTLsearches for upstream controllers</i></span><span style='font-family:Verdana;font-size:73%;mso-special-format:lastCR;display:none'><i><br></i></span></div></layer></div></layer><script>
	mytop = 0.71 * g_height; myleft = 0.34 * g_width; myheight = 0.19 * g_height; mywidth = 0.45 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script>
	mytop = 0 * g_height; myleft = 0.01 * g_width; myheight = 0.18 * g_height; mywidth = 0.44 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div style='text-align:left;font-family:Verdana;font-weight:normal;font-style:normal;text-decoration:none;text-shadow:none;text-effect:none;mso-fareast-hint:no;layout-flow:horizontal;color:black;mso-color-index:1;font-size:80%;mso-text-raise:0%'><script>
	mytop = 0 * g_height; myleft = 0 * g_width; myheight = 0.04 * g_height; mywidth = 0.44 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><spanstyle='text-align:left'><span style='font-size:167%;color:#E9EB5D'><i>App mapson Chr 16 (blue </i></span></span></layer><script>
	mytop = 0.04 * g_height; myleft = 0 * g_width; myheight = 0.04 * g_height; mywidth = 0.44 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><spanstyle='text-align:left'><span style='font-size:167%;color:#E9EB5D'><i>arrowpoints to the orange </i></span></span></layer><script>
	mytop = 0.09 * g_height; myleft = 0 * g_width; myheight = 0.04 * g_height; mywidth = 0.38 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><spanstyle='text-align:left'><span style='font-size:167%;color:#E9EB5D'><i>triangle)but the best </i></span></span></layer><script>
	mytop = 0.13 * g_height; myleft = 0 * g_width; myheight = 0.04 * g_height; mywidth = 0.38 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><spanstyle='text-align:left'><span style='font-size:167%;color:#E9EB5D'><i>locus ison Chr 7.</i></span><span style='font-size:233%;color:#E9EB5D;mso-special-format:lastCR;display:none'><i><br></i></span></span></layer></div></layer></div></layer><script>
	mytop = 0.08 * g_height; myleft = 0.01 * g_width; myheight = 0.53 * g_height; mywidth = 0.98 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
xx = 0.01 * g_width;
yy = 0.08 * g_height;
ht = 0.53 * g_height;
wd = 0.98 * g_width;
document.write(
'<img border=0 src="slide0022_image102.png" x=' + xx + ' y=' +yy + ' height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer><script>
	mytop = 0.46 * g_height; myleft = 0.15 * g_width; myheight = 0.53 * g_height; mywidth = 0.15 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
xx = 0.15 * g_width;
yy = 0.46 * g_height;
ht = 0.53 * g_height;
wd = 0.15 * g_width;
document.write(
'<img border=0 src="slide0022_image103.png" x=' + xx + ' y=' +yy + ' height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer><script>
	mytop = 0.46 * g_height; myleft = 0.8 * g_width; myheight = 0.53 * g_height; mywidth = 0.1 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
xx = 0.8 * g_width;
yy = 0.46 * g_height;
ht = 0.53 * g_height;
wd = 0.1 * g_width;
document.write(
'<img border=0 src="slide0022_image104.png" x=' + xx + ' y=' +yy + ' height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer><script>
	mytop = 0.41 * g_height; myleft = 0.76 * g_width; myheight = 0.22 * g_height; mywidth = 0.11 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
xx = 0.76 * g_width;
yy = 0.41 * g_height;
ht = 0.22 * g_height;
wd = 0.11 * g_width;
document.write(
'<img border=0 src="slide0022_image105.png" x=' + xx + ' y=' +yy + ' height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer><script>
	mytop = 0.41 * g_height; myleft = 0.29 * g_width; myheight = 0.24 * g_height; mywidth = 0.12 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
xx = 0.29 * g_width;
yy = 0.41 * g_height;
ht = 0.24 * g_height;
wd = 0.12 * g_width;
document.write(
'<img border=0 src="slide0022_image106.png" x=' + xx + ' y=' +yy + ' height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer></div></LAYER>
<layer> <div id=NotesObj style='display:none'> <table style='color:white' border=0 width="100%">  <tr>   <td width=5 nowrap></td>   <td width="100%"></td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>This is a major output   type: a so-called full-genome interval map.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>The X-axis represents all   19 autosomes and the X chromosome as if they were laid end to end with short   gaps between the telomere of one chromosome and the centromere of the next   chromosome (mouse chromosomes only have a single long arm and the centromere   represents the origin of each chromosome for numerical purpose: 0   centimorgans at almost 0 megabases). The blue labels along the bottom of the   figure list a subset of the 3795 markers that were used in mapping.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>The thick blue wavy line   running across chromosomes summarizes the strength of association between   variation in the phenotype (App expression differences) and the two   genotypes of all markers and the intervals between markers (hence, interval   mapping).<span style="mso-spacerun: yes">&nbsp; </span>The height of the   wave (blue Y-axis to the left) provides the likelihood ratio statistic   (LRS). Divide by 4.61 to convert these values to LOD scores.<span   style="mso-spacerun: yes">&nbsp; </span>Or you can read them as a   chi-square-like statistic.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>The red line and the red   axis to the far right provides an estimate of the effect that a QTL has on   expression of App (this estimate of the so-called additive effect tends to   be too high). If the red line is below the X-axis then this means that the   allele inherited from C57BL/6J (B6 or B) at a particular marker is   associated with higher values. If the red line is above the X-axis then the   DBA/2J allele (D2 or D) is associated with higher trait values. Multiply the   additive effect size by 2 to estimate the difference between the set of   strains that have the B/B genotype and those that have the D/D genotype at a   specific marker. For example, on distal Chr 7 the red line peaks at a value   of about 0.2. That means that this region of chromosome 2 is responsible for   a 0.4 unit expression difference between B/B strains and the D/D strains.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>The yellow histogram   bars: These summarize the results of a whole-genome bootstrap of the trait   that is performed 1000 times. What is a bootstrap? A bootstrap provides a   method to evaluate whether results are robust. If we drop out one strain, do   we still get the same results? When mapping quantitative traits, each strain   normally gets one equally weighted vote. But using the bootstrap procedure,   we give each strain a random weighting factor of between 0 and 1.<span   style="mso-spacerun: yes">&nbsp; </span>We then remap the trait and find THE   SINGLE BEST LRS VALUE per bootstrap. We do this 1000 times. In this example,   most bootstrap results cluster on Chr 3 and Chr 7 under the LRS peaks. That   is somewhat reassuring. But notice that a substantial number of bootstrap   are scattered around on other chromosomes. About 30% of the bootstrap   resamples have a peak on Chr 7. That is pretty good, but does makes us   realize that the sample we are working with is still quite small and   fragile.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>The horizontal dashed   lines at 10.5 and 17.3 are the likelihood ratio statistic (LRS) values   associated with the suggestive and significant genome-wide probabilities   that were established by permutations of phenotypes across genotypes. We   shuffle randomly 2000 times and obtain a distribution of peak LRS scores to   generate a null distribution. Five percent of the time, one of these   permuted data sets will have a peak LRS higher than 17.3. We call that level   the 0.05 significance threshold for a whole genome scan. The p = 0.67 point   is the the suggestive level, and corresponds to the green dashed line.<span   style="mso-spacerun: yes">&nbsp; </span>These thresholds are conservative   for transcripts that have expression variation that is highly heritable. The   putative or suggestive QTL on Chr 3 is probably more than just suggestive.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>One other point: the   mapping procedure we use is computationally very fast, but it is relatively   simple. We are not looking for gene-gene interactions and we are not fitting   multiple QTLs in combinations. Consider this QTL analysis a first pass that   will highlight hot spots and warm spots that are worth following up on using   more sophisticated models.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>CLICKABLE REGIONS:</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>1. If you click on the   Chromosome number then you will generate a new map just for that chromosome.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>2. If you click on the   body of the map, say on the blue line, then you will generate a view on a 10   Mb window of that part of the genome from the UCSC Genome Browser web site.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>3. If you click on a   marker symbol, then you will generate a new Trait data and Analysis window   with the genotypes loaded into the window just like any other trait. More on   this in Section 3.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>4. You can drag these   maps off of the browser window and onto your desktop. They will be saved as   PNG or PDF files. You can import them into Photoshop or other programs.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>5. There is also an   option at the bottom of the page to download a 2X higher resolution image of   this plot for papers and presentations.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>6. You can also download   the results of the analysis in a text format</font><br>   </td>  </tr> </table> </div></layer><script language=JavaScript><!--
//-->
</script><script language=JavaScript><!--
function playList() {

}
//-->
</script></body></html>