<html><head><meta name=ProgId content=PowerPoint.Slide><meta name=Generator content="Microsoft Macintosh PowerPoint 11"><link id=Main-File rel=Main-File href="webqtl_demo2.ppt.htm"><link title="Presentation File" type="application/powerpoint" rel=alternatehref="webqtl_demo2.ppt.ppt"><script>
if ( ! top.PPTPRESENTATION ) {
	window.location.replace( "endshow.htm" );
}
</script></head><body bgcolor=black text=white><table border=0 width="100%"> <tr>  <td width=5 nowrap></td>  <td width="100%"></td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face="Times New Roman" size=4><b>Evaluating  candidate genes (CHECKED BOXES) responsible for variability in APP  expression:</b></font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face="Times New Roman" size=4>A large number  of genes are usually in the QTL interval and are therefore POSITIONAL  CANDIDATES, but they will differ greatly in their biological and  bioinformatic plausibility. Assume that the QTL has been located between 119  and 131 Mb (12 Mb). There will typically be 12 to 15 genes per Mb, so we  might need to evaluate several hundred positional candidates. In this  particular case there are about 100 known genes in this interval. Eight of  these are highlighted in the table above with check marks in the boxes to the  left.<span style="mso-spacerun: yes">&nbsp; </span>We need to highlight and  objectively score the biologically relevant subset of all 100 positional  candidate genes. We could look through gene ontologies and expression levels  to help us shorten the list. An alternate way available using WebQTL is to  generate a list of those genes in this interval that have transcripts that  co-vary in expression with App expression. That is what the table shows.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face="Times New Roman" size=4>Notes:</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face="Times New Roman" size=4>1. To replicate  this table go back to the Trait Data and Analysis Form. Choose to sort  correlations by POSITION and select RETURN = 500. Then scroll down the list  to Chr 7 and review the subset of positional candidates that share expression  with App. You should see a list similar to that shown above. Gtf3c1 is a good  biological candidate and has a high covariation in expression with App.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face="Times New Roman" size=4>2. Caveat:<span  style="mso-spacerun: yes">&nbsp;&nbsp; </span>Of course, the gene or genes  that control App expression may not be in this list. A protein coding  difference might be the ultimate cause of variation in App transcript level  and the expression covariation might be close to zero. Our list may also  simply be missing the right transcript since the microarray is not truly  comprehensive. Furthermore, even if the list contains the QT gene, an  expression difference may only have been expressed early in development or  even in another tissue such as liver. While it is important to recognize  these caveats, it is equally important to devise a rational way to rank  candidates given existing data. Coexpression is one of several criteria used  to evaluate positional candidates. We will see others in the next slide.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face="Times New Roman" size=4>3. We can also  assess the likelihood that candidates contain functional polymorphism in  promoters and enhancers that affect their expression simply by mapping the  transcripts of all candidate genes to see if they “map back” to the location  of gene itself. A transcript that maps to its own location is referred to as  a cis QTL. We essentially ask: Which of the the transcripts listed in the  Correlation Table above (from Gtf3c1 to Zranb1) has variation in expression  that maps to Chr 7 at about 120 Mb?<span style="mso-spacerun: yes">&nbsp;  </span>The logic of this search is that if a gene controls the level of its  own expression it is also much more likely to generate other downstream  effects. The Gtf3c1 transcript is a weak cis QTL with a local LRS maximum of  about 7.0 (D alleles are high). That is just about sufficient to declare it  to be a cis QTL. [No whole genome correction is required and a point-wise  p-value of 0.05 is the appropriate test. A p-value of 0.05 is roughly  equivalent to an LRS of 6.0 (LOD = 1.3).]</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr></table></body></html>