<html><head><meta http-equiv=Content-Type content="text/html; charset=macintosh"><meta name=ProgId content=PowerPoint.Slide><meta name=Generator content="Microsoft Macintosh PowerPoint 11"><link id=Main-File rel=Main-File href="webqtl_demo2.ppt.htm"><title>PowerPoint Presentation  -  Complex trait analysis, develop-ment, andgenomics</title><link title="Presentation File" type="application/powerpoint" rel=alternatehref="webqtl_demo2.ppt.ppt"><script>
if ( ! top.PPTPRESENTATION ) {
	window.location.replace( "endshow.htm" );
}
</script><meta name=Description content="Aug-15-05: Physical map for distal chromosome 7"><link rel=Stylesheet href="master03_stylesheet.css"><style media=print><!--.sld	{left:0px !important;	width:6.0in !important;	height:4.5in !important;	font-size:76% !important;}--></style><script language=JavaScript src=script.js></script><script language=JavaScript><!--
gId="slide0007.htm"
if ( !parent.base.g_done && (parent.base.msie < 0 ) ) {
	parent.base.g_done = 1;
	document.location.replace( parent.base.GetHrefObj(parent.base.g_currentSlide).m_slideHref);
}
if( !IsNts() ) Redirect( "PPTSld", gId );
var g_hasTrans = true, g_autoTrans = false, g_transSecs = 1;
//-->
</script><script for=window event=onload language=JavaScript><!--
if( !IsSldOrNts() ) return;
if( MakeNotesVis() ) return;
LoadSld( gId );
playList();MakeSldVis(1);
//-->
</script></head><body style='margin:0px;background-color:black' onclick="DocumentOnClick(event)"onresize="_RSW()" onkeypress="_KPH(event)"><div id=SlideObj class=sld style='position:absolute;top:0px;left:0px;width:755px;height:566px;font-size:16px;background-color:#484848;clip:rect(0%, 101%, 101%, 0%);visibility:hidden'><img src="master03_background.png" v:shapes="_x0000_s1026"style='position:absolute;top:0%;left:0%;width:100.0%;height:100.0%'><div style='position:absolute;top:-.53%;left:1.72%;width:101.05%;height:8.65%;filter:DropShadow(Color=#000000, OffX=2, OffY=2)'><div class=T style='mso-line-spacing:"-358 0 -1";mso-margin-left-alt:233;mso-text-indent-alt:0;position:absolute;top:18.36%;left:.91%;width:97.9%;height:75.51%'><span style='font-family:Verdana;font-size:64%'>Physical map fordistal chromosome 7</span><span style='font-family:Verdana;font-size:73%;mso-special-format:lastCR;display:none'><i><br></i></span></div></div><div style='position:absolute;top:7.77%;left:5.82%;width:94.03%;height:6.18%;filter:DropShadow(Color=#000000, OffX=2, OffY=2)'><div class=O style='position:absolute;top:11.42%;left:1.12%;width:98.87%;height:74.28%'><span style='font-size:167%;color:#E9EB5D'>Distal Chr 7 from~120 and 132 Mb may modulate</span><span style='font-size:167%;color:#E9EB5D'><i>App</i></span><span style='font-size:200%;color:#E9EB5D;mso-special-format:lastCR;display:none'><i><br></i></span></div></div><img border=0 src="slide0007_image073.png" style='position:absolute;top:93.28%;left:6.62%;width:75.23%;height:3.71%'><img border=0 src="slide0007_image075.png"style='position:absolute;top:15.54%;left:6.62%;width:68.34%;height:75.61%'></div><div id=NotesObj style='display:none'><table style='color:white' border=0 width="100%"> <tr>  <td width=5 nowrap></td>  <td width="100%"></td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>Physical map of variation  in App expression in brain on distal Chr 7 (a blow up of the whole-genome map  on the previous slide).</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>Notes:</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Arial size=3>1.</font><font face=Verdana  size=3> You can now see that the X-axis is on a physical scale of megabases  (Mb). The QTL peak is roughly between 120 and 132 Mb.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>2. The small irregular  colored blocks and marks toward the top of the map mark the locations of  genes superimposed on the physical map. Neighboring genes are offset slightly  in the vertical axis for display purpose. Note one region of very high gene  density from about 120 to 123 Mb.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>3. The orange hash marks  along the X-axis represent the number of single nucleotide polymorphisms that  distinguish the two parental strains (C57BL/6J and DBA/2J) from each other.  We call this the SNP seismograph track (see Glossary for more details). Regions  with low numbers of SNPs have closely matched sequences and are less likely  to contain QTLs.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>4. As before, the thin red  line shows the additive effect size. By convention the positive values  signify the D alleles are associated with higher expression of App in this  region of Chr 7 than the B alleles. The maximum effect size is about +0.20  log2 expression units per D allele. The differences been the BB and DD  genotypes (BB and DD because each strain has two alleles; one per chromosome)  is therefore about 2^0.4 = 1.32 or a 32% increment in DD relative to BB at  this locus.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>5. If you scroll just  under the Physical Map you will see text that reads “DISPLAY from XXX Mb TO  YYY Mb…..”<span style="mso-spacerun: yes">&nbsp; </span>These physical maps  are zoomable, a feature we will exploit to evaluate candidate genes in this  QTL interval.</font><br>  </td> </tr></table></div><script language=JavaScript><!--
function playList() {

}
//-->
</script></body></html>