<html><head><meta http-equiv=Content-Type content="text/html; charset=macintosh"><meta name=ProgId content=PowerPoint.Slide><meta name=Generator content="Microsoft Macintosh PowerPoint 11"><link id=Main-File rel=Main-File href="webqtl_demo2.ppt.htm"><title>PowerPoint Presentation  -  Complex trait analysis, develop-ment, andgenomics</title><link title="Presentation File" type="application/powerpoint" rel=alternatehref="webqtl_demo2.ppt.ppt"><script>
if ( ! top.PPTPRESENTATION ) {
	window.location.replace( "endshow.htm" );
}
</script><meta name=Description content="Aug-15-05: Physical maps are zoomable"><link rel=Stylesheet href="master03_stylesheet.css"><style media=print><!--.sld	{left:0px !important;	width:6.0in !important;	height:4.5in !important;	font-size:76% !important;}--></style><script language=JavaScript src=script.js></script><script language=JavaScript><!--
gId="slide0009.htm"
if ( !parent.base.g_done && (parent.base.msie < 0 ) ) {
	parent.base.g_done = 1;
	document.location.replace( parent.base.GetHrefObj(parent.base.g_currentSlide).m_slideHref);
}
if( !IsNts() ) Redirect( "PPTSld", gId );
var g_hasTrans = true, g_autoTrans = false, g_transSecs = 1;
//-->
</script><script for=window event=onload language=JavaScript><!--
if( !IsSldOrNts() ) return;
if( MakeNotesVis() ) return;
LoadSld( gId );
playList();MakeSldVis(1);
//-->
</script></head><body style='margin:0px;background-color:black' onclick="DocumentOnClick(event)"onresize="_RSW()" onkeypress="_KPH(event)"><div id=SlideObj class=sld style='position:absolute;top:0px;left:0px;width:755px;height:566px;font-size:16px;background-color:#484848;clip:rect(0%, 101%, 101%, 0%);visibility:hidden'><img src="master03_background.png" v:shapes="_x0000_s1026"style='position:absolute;top:0%;left:0%;width:100.0%;height:100.0%'><div style='position:absolute;top:-.53%;left:1.72%;width:101.05%;height:8.65%;filter:DropShadow(Color=#000000, OffX=2, OffY=2)'><div class=T style='mso-line-spacing:"-358 0 -1";mso-margin-left-alt:233;mso-text-indent-alt:0;position:absolute;top:18.36%;left:.91%;width:97.9%;height:75.51%'><span style='font-family:Verdana;font-size:64%'>Physical mapsare zoomable</span><span style='font-family:Verdana;font-size:73%;mso-special-format:lastCR;display:none'><i><br></i></span></div></div><img border=0 src="slide0009_image078.png" style='position:absolute;top:8.83%;left:3.31%;width:89.93%;height:41.69%'><img border=0src="slide0009_image079.png" style='position:absolute;top:44.52%;left:7.54%;width:42.91%;height:48.76%'><img border=0 src="slide0009_image080.png"style='position:absolute;top:94.52%;left:0%;width:97.08%;height:3.71%'></div><div id=NotesObj style='display:none'><table style='color:white' border=0 width="100%"> <tr>  <td width=5 nowrap></td>  <td width="100%"></td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>An even higher blow-up of  part of the Chr 7 physical map of variation in App expression in brain.<span  style="mso-spacerun: yes">&nbsp; </span>The QTL region actually extends from  about 119 to 129.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face="Times New Roman" size=4>Notes:</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face="Times New Roman" size=4>1. As mentioned  in the previous slide another important approach to ranking candidates is  based on the number of sequence variants that distinguish the parental  strains. If we were sure that the sequences of the gene, its promoter, and  its enhancers were identical between the strains then we could discount--but  not eliminate--that gene as a candidate. The Gtf3c1 candidate almost falls  into this category: of 663 known SNPs in and around this gene, only four  differ between C57BL/6J and DBA/2J. Gtf3c1 is essentially  identical-by-descent in these strains and is a less likely candidate. In  contrast, if the two alleles of the gene have dozens of functional variants  in exons, promoters, enhancers, and splice sites, then it becomes a higher  priority candidate.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face="Times New Roman" size=4>Of course it  only takes a single critical sequence variant to generate downstream effects.  The argument above is really about the prior probabilities. Where would you  place your bets given the information at hand?</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>2.<span  style="mso-spacerun: yes">&nbsp; </span>If you scroll down the INTERVAL  ANALYST you will find that Ctbp2 is a particularly interesting candidate that  contains lots of SNPs (n = 75 and a SNP density of 0.55 SNP/Kb). Ctbp2 is  also closer to our QTL peak than was Gtf3c1. Not only does Ctbp2 contain lots  of SNPs but it is also is associated with a powerful cis QTL with an LRS of  24.2 (divide by 4.61 to get the equivalent LOD score of 5.25).</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>3.<span  style="mso-spacerun: yes">&nbsp; </span>At this high magnification,  individual genes are distinct. They are color coded by their density of SNPs.  Bright orange represents those genes that have a high SNP density (C57BL/6J  versus DBA/2J), black represents genes with low SNP density. Roll the cursor  over a gene block and its name will pop up, along with information on exon  number.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>4.<span  style="mso-spacerun: yes">&nbsp; </span>Beneath the physical map you will  find an INTERVAL ANALYST table that lists information on known genes in the  region on which you have zoomed the Physical Map.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Verdana size=3>5.<span  style="mso-spacerun: yes">&nbsp; </span>As always: error-checking is  important. Some genes may be missing from the Interval Analyst (recent  additions or errors of omission). In this case the Zranb1 gene that is  located just proximal to Ctbp2 is not listed in the INTERVAL ANALYST.  Double-check the interval using the Genome Browser links (blue and beige  horizontal bars) at the top of the PHYSICAL MAP.</font><br>  </td> </tr></table></div><script language=JavaScript><!--
function playList() {

}
//-->
</script></body></html>