<html><head><meta name=ProgId content=PowerPoint.Slide><meta name=Generator content="Microsoft Macintosh PowerPoint 11"><title>PowerPoint Presentation  -  Complex trait analysis, develop-ment, andgenomics</title><link title="Presentation File" type="application/powerpoint" rel=alternatehref="webqtl_demo2.ppt.ppt"><script>
if ( ! top.PPTPRESENTATION ) {
	window.location.replace( "endshow.htm" );
}
</script><meta name=Description content="Jul-18-05: Physical maps are zoomable"><style media=print><!--.sld	{left:0px !important;	width:6.0in !important;	height:4.5in !important;	font-size:76% !important;}--></style><script language=Javascript src=script.js></script>
<script language=JavaScript>
function font_size() {
	var min = window.innerWidth/20;
	if ( min > window.innerHeight/20 )
		min = window.innerHeight/20;
		
	return min+"pt";
}
var SoundEnabled = navigator.plugins && navigator.plugins["LiveAudio"];
function restore() {
	self.location.reload();
}
function set_size() {
	document.SlideLayer.width = window.innerWidth;
	document.SlideLayer.height = window.innerHeight;
}
document.captureEvents(Event.MOUSEDOWN);
document.captureEvents(Event.KEYPRESS);
document.onkeypress = _KPH ;
document.onmousedown = _KPH ;
 parent.base.highlite(); 
</script><script language=JavaScript><!--
function DoOnLoad() { 
 LoadNavSld("slide0057.htm");
}//-->
</script></head><body onresize="restore()" onload="DoOnLoad()" style='font-size:&{font_size()};'bgcolor="#000000"><script>

g_origHeight = 566;
g_origWidth = 755;
g_width = window.innerWidth - 10;
g_height = window.innerHeight - 10;

g_top = 0;
g_left = 0;

if ( g_width < ( g_origWidth * g_height /g_origHeight ) )
	g_height = g_width * g_origHeight / g_origWidth;
else
	g_width = g_origWidth * g_height / g_origHeight;
	
g_top = (window.innerHeight - g_height) / 2;
g_left = (window.innerWidth - g_width) / 2;

document.write( '<layer id=SlideLayer BGCOLOR=#484848 background="" top=' + g_top + ' left=' + g_left + ' height=' + g_height + ' width=' + g_width + ' >' );
document.SlideLayer.clip.top = 0;
document.SlideLayer.clip.left = 0;
document.SlideLayer.clip.height = g_height;
document.SlideLayer.clip.width = g_width;
function new_font_size() {
	var size = 11 * g_width / g_origWidth;
	if ( size < 1 ) size = 1;
	return size+"pt";
}
</script><div id=SlideObj class=sld style='position:relative;font-size:&{new_font_size()};'><script>
	mytop = 0 * g_height; myleft = 0 * g_width; myheight = 1 * g_height; mywidth = 1 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
ht =   g_height;
wd =   g_width;
document.write(
'<img src="master03_background.png" height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer><script>
	mytop = 0 * g_height; myleft = 0.01 * g_width; myheight = 0.08 * g_height; mywidth = 1.01 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script>
	mytop = 0.01 * g_height; myleft = 0 * g_width; myheight = 0.06 * g_height; mywidth = 0.98 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div style='text-align:left;font-family:"Gill Sans";font-weight:normal;font-style:normal;text-decoration:none;text-shadow:none;text-effect:none;mso-fareast-hint:no;layout-flow:horizontal;color:#E9EB5D;font-size:293%;mso-text-raise:0%;mso-line-spacing:"-358 0 -1";mso-margin-left-alt:233;mso-text-indent-alt:0;mso-line-spacing:"-358 0 -1";mso-margin-left-alt:233;mso-text-indent-alt:0'><span style='font-family:Verdana;font-size:64%'>Physicalmaps are zoomable</span><span style='font-family:Verdana;font-size:73%;mso-special-format:lastCR;display:none'><i><br></i></span></div></layer></div></layer><script>
	mytop = 0.08 * g_height; myleft = 0.03 * g_width; myheight = 0.41 * g_height; mywidth = 0.89 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
xx = 0.03 * g_width;
yy = 0.08 * g_height;
ht = 0.41 * g_height;
wd = 0.89 * g_width;
document.write(
'<img border=0 src="slide0026_image114.png" x=' + xx + ' y=' +yy + ' height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer><script>
	mytop = 0.44 * g_height; myleft = 0.07 * g_width; myheight = 0.48 * g_height; mywidth = 0.42 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
xx = 0.07 * g_width;
yy = 0.44 * g_height;
ht = 0.48 * g_height;
wd = 0.42 * g_width;
document.write(
'<img border=0 src="slide0026_image115.png" x=' + xx + ' y=' +yy + ' height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer><script>
	mytop = 0.94 * g_height; myleft = 0 * g_width; myheight = 0.03 * g_height; mywidth = 0.97 * g_width;
	document.write( '<layer top=' + mytop + ' left=' + myleft + ' height=' + myheight + ' width=' + mywidth + ' >' );
</script><div><script language=JavaScript><!--
xx = 0 * g_width;
yy = 0.94 * g_height;
ht = 0.03 * g_height;
wd = 0.97 * g_width;
document.write(
'<img border=0 src="slide0026_image116.png" x=' + xx + ' y=' +yy + ' height=' + ht + ' width=' + wd + '>' );
//-->
</script></div></layer></div></LAYER>
<layer> <div id=NotesObj style='display:none'> <table style='color:white' border=0 width="100%">  <tr>   <td width=5 nowrap></td>   <td width="100%"></td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>Even higher blow-up of   part of the Chr 7 physical map of variation in App expression in brain.<span   style="mso-spacerun: yes">&nbsp; </span>The QTL region actually extends from   about 119 to 129.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face="Times New Roman" size=4>Notes:</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face="Times New Roman" size=4>1. As mentioned   in the previous slide another important approach to ranking candidates is   based on the number of sequence variants that distinguish the parental   strains. If we were sure that the sequences of the gene, its promoter, and   its enhancers were identical between the strains then we could discount--but   not eliminate--that gene as a candidate. The Gtf3c1 candidate almost falls   into this category: of 663 known SNPs in and around this gene, only four   differ between C57BL/6J and DBA/2J. Gtf3c1 is essentially   identical-by-descent in these strains and is a less likely candidate. In   contrast, if the two alleles of the gene have dozens of functional variants   in exons, promoters, enhancers, and splice sites, then it becomes a higher   priority candidate.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face="Times New Roman" size=4>Of course it   only takes a single critical sequence variant to generate downstream   effects. The argument above is really about the prior probabilities. Where   would you place your bets given the information at hand?</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>2.<span   style="mso-spacerun: yes">&nbsp; </span>If you scroll down the INTERVAL   ANALYST you will find that Ctbp2 is a particularly interesting candidate   that contains lots of SNPs (n = 75 and a SNP density of 0.55 SNP/Kb). Ctbp2   is also closer to our QTL peak than was Gtf3c1. Not only does Ctbp2 contain   lots of SNPs but it is also is associated with a powerful cis QTL with an   LRS of 24.2 (divide by 4.61 to get the equivalent LOD score of 5.25).</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>3.<span   style="mso-spacerun: yes">&nbsp; </span>At this high magnification,   individual genes are distinct. They are color coded by their density of   SNPs. Bright orange represents those genes that have a high SNP density   (C57BL/6J versus DBA/2J), black represents genes with low SNP density. Roll   the cursor over a gene block and its name will pop up, along with   information on exon number.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>4.<span   style="mso-spacerun: yes">&nbsp; </span>Beneath the physical map you will   find an INTERVAL ANALYST table that lists information on known genes in the   region on which you have zoomed the Physical Map.</font><br>   </td>  </tr>  <tr>   <td colspan=1></td>   <td align=left colspan=1><font face=Verdana size=3>5.<span   style="mso-spacerun: yes">&nbsp; </span>As always: error-checking is   important. Some genes may be missing from the Interval Analyst (recent   additions or errors of omission). In this case the Zranb1 gene that is   located just proximal to Ctbp2 is not listed in the INTERVAL ANALYST.   Double-check the interval using the Genome Browser links (blue and beige   horizontal bars) at the top of the PHYSICAL MAP.</font><br>   </td>  </tr> </table> </div></layer><script language=JavaScript><!--
//-->
</script><script language=JavaScript><!--
function playList() {

}
//-->
</script></body></html>