<html><head><meta http-equiv=Content-Type content="text/html; charset=macintosh"><meta name=ProgId content=PowerPoint.Slide><meta name=Generator content="Microsoft Macintosh PowerPoint 10"><link id=Main-File rel=Main-File href="WebQTLDemo.htm"><title>PowerPoint Presentation  -  Complex trait analysis, develop-ment, andgenomics</title><link title="Presentation File" type="application/powerpoint" rel=alternatehref=WebQTLDemo.ppt><script>
if ( ! top.PPTPRESENTATION ) {
	window.location.replace( "endshow.htm" );
}
</script><meta name=Description content=Jun-19-03><link rel=Stylesheet href="master03_stylesheet.css"><style media=print><!--.sld	{left:0px !important;	width:6.0in !important;	height:4.5in !important;	font-size:76% !important;}--></style><script language=JavaScript src=script.js></script><script language=JavaScript><!--
gId="slide0023.htm"
if ( !parent.base.g_done && (parent.base.msie < 0 ) ) {
	parent.base.g_done = 1;
	document.location.replace( parent.base.GetHrefObj(parent.base.g_currentSlide).m_slideHref);
}
if( !IsNts() ) Redirect( "PPTSld", gId );
var g_hasTrans = true, g_autoTrans = false, g_transSecs = 1;
//-->
</script><script for=window event=onload language=JavaScript><!--
if( !IsSldOrNts() ) return;
if( MakeNotesVis() ) return;
LoadSld( gId );
playList();MakeSldVis(1);
//-->
</script></head><body style='margin:0px;background-color:black' onclick="DocumentOnClick(event)"onresize="_RSW()" onkeypress="_KPH(event)"><div id=SlideObj class=sld style='position:absolute;top:0px;left:0px;width:755px;height:566px;font-size:16px;background-color:#484848;clip:rect(0%, 101%, 101%, 0%);visibility:hidden'><img src="master03_background.gif" v:shapes="_x0000_s1026"style='position:absolute;top:0%;left:0%;width:100.0%;height:100.0%'><div style='position:absolute;top:1.59%;left:3.97%;width:104.1%;height:8.12%;filter:DropShadow(Color=#000000, OffX=4, OffY=4)'><div class=O style='mso-line-spacing:" 0 0";mso-margin-left-alt:16;mso-text-indent-alt:16;position:absolute;top:2.17%;left:1.27%;width:98.85%;height:80.43%'><spanstyle='font-size:233%;color:#E9EB5D'><i>Hars2 probe level analysis: 16 PMsmapped</i></span></div></div><img border=0 src="slide0023_image094.gif" style='position:absolute;top:10.77%;left:13.64%;width:72.58%;height:84.8%'><img border=0src="slide0023_image095.gif" style='position:absolute;top:59.18%;left:2.38%;width:41.05%;height:36.74%'><img border=0 src="slide0023_image096.gif"style='position:absolute;top:60.07%;left:36.82%;width:26.75%;height:19.08%'><imgborder=0 src="slide0023_image097.gif" style='position:absolute;top:33.39%;left:21.19%;width:64.37%;height:2.65%'><img border=0src="slide0023_image098.gif" style='position:absolute;top:35.68%;left:21.19%;width:64.37%;height:2.65%'><img border=0 src="slide0023_image099.gif"style='position:absolute;top:38.33%;left:21.19%;width:64.37%;height:2.65%'><imgborder=0 src="slide0023_image100.gif" style='position:absolute;top:19.78%;left:20.79%;width:28.74%;height:21.73%'><div class=O style='position:absolute;top:28.09%;left:30.06%;width:4.9%;height:3.35%'><span style='font-size:117%;color:#FF0F0F'><i>SNP</i></span></div><div class=O style='mso-line-spacing:"100 50 0";position:absolute;top:29.15%;left:40.13%;width:18.67%;height:3.35%'><span style='font-size:117%;color:#FF0F0F'>Cin B6, T in D2 </span><span style='font-size:150%;color:black;mso-special-format:lastCR;display:none'><b><br></b></span></div><div class=O style='position:absolute;top:36.39%;left:50.99%;width:9.8%;height:3.35%'><span style='font-size:117%;color:#003399;mso-color-index:4'><i>noSNPs</i></span></div></div><div id=NotesObj style='display:none'><table style='color:white' border=0 width="100%"> <tr>  <td width=5 nowrap></td>  <td width="100%"></td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=3>Let’s look at Hars2 in  more detail by mapping all of the perfect match probes (16 of them) that  target this transcript.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=3>Go back to the Trait  Data and Editing window and select Chr 2 (rather than ALL as shown above) and  also select PM Probes. Then click on Interval Mapping button.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=3>You will get the  illustration above, but without the sequence data that we have added.<span  style="mso-spacerun: yes">&nbsp; </span>The 16 perfect match probes are  arranged in sequence (red is 5 prime, blue is the 3 prime end). For example,  the 5 prime-most primer 307387 has the sequence CACTG..... It also has a  polymorphism at the 17 nucleotide of this 25 nt probe sequence.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=3>How do we know that the  5 prime probe is polymorphic? By looking up the sequence in the Celera  Genomics databases which often contqains sequence data for C57BL/6J (B6  above) and for DBA/2J.<span style="mso-spacerun: yes">&nbsp; </span>But two  blue probes (14 and 15) do NOT contain SNPs but still have very large LRS  scores. The other probes do not perform so wel. Highly variable probe  performance is probably a result of the very different stacking energies of  DNA-RNA duplexes.</font><br>  </td> </tr></table></div><script language=JavaScript><!--
function playList() {

}
//-->
</script></body></html>