<html><head><meta http-equiv=Content-Type content="text/html; charset=macintosh"><meta name=ProgId content=PowerPoint.Slide><meta name=Generator content="Microsoft Macintosh PowerPoint 10"><link id=Main-File rel=Main-File href="WebQTLDemo.htm"><title>PowerPoint Presentation  -  Complex trait analysis, develop-ment, andgenomics</title><link title="Presentation File" type="application/powerpoint" rel=alternatehref=WebQTLDemo.ppt><script>
if ( ! top.PPTPRESENTATION ) {
	window.location.replace( "endshow.htm" );
}
</script><meta name=Descriptioncontent="Jun-19-03: Are there experimental results to corroborate a link between App with Hsp84-1?"><link rel=Stylesheet href="master03_stylesheet.css"><style media=print><!--.sld	{left:0px !important;	width:6.0in !important;	height:4.5in !important;	font-size:76% !important;}--></style><script language=JavaScript src=script.js></script><script language=JavaScript><!--
gId="slide0011.htm"
if ( !parent.base.g_done && (parent.base.msie < 0 ) ) {
	parent.base.g_done = 1;
	document.location.replace( parent.base.GetHrefObj(parent.base.g_currentSlide).m_slideHref);
}
if( !IsNts() ) Redirect( "PPTSld", gId );
var g_hasTrans = true, g_autoTrans = false, g_transSecs = 1;
//-->
</script><script for=window event=onload language=JavaScript><!--
if( !IsSldOrNts() ) return;
if( MakeNotesVis() ) return;
LoadSld( gId );
playList();MakeSldVis(1);
//-->
</script></head><body style='margin:0px;background-color:black' onclick="DocumentOnClick(event)"onresize="_RSW()" onkeypress="_KPH(event)"><div id=SlideObj class=sld style='position:absolute;top:0px;left:0px;width:755px;height:566px;font-size:16px;background-color:#484848;clip:rect(0%, 101%, 101%, 0%);visibility:hidden'><img src="master03_background.gif" v:shapes="_x0000_s1026"style='position:absolute;top:0%;left:0%;width:100.0%;height:100.0%'><div style='position:absolute;top:3.53%;left:2.11%;width:97.48%;height:21.02%;filter:DropShadow(Color=#000000, OffX=2, OffY=2)'><div class=T style='mso-line-spacing:" 0 &#1;";mso-margin-left-alt:232;mso-text-indent-alt:0;position:absolute;top:18.48%;left:.95%;width:99.04%;height:61.34%'><span style='position:absolute;top:0%;left:0%;width:100.0%'><spanstyle='font-size:68%'><i>Are there experimental results to corroborate a link </i></span></span><spanstyle='position:absolute;top:49.31%;left:4.25%;width:90.12%'><spanstyle='font-size:68%'><i>between App with Hsp84-1?</i></span></span></div></div><div style='position:absolute;top:82.15%;left:4.1%;width:96.42%;height:18.55%;filter:DropShadow(Color=#000000, OffX=2, OffY=2)'></div><div class=O style='position:absolute;top:29.5%;left:4.37%;width:99.33%;height:13.07%'><span style='position:absolute;top:0%;left:0%;width:100.0%'><spanstyle='font-family:"Gill Sans";font-size:250%;color:#E9EB5D'><i>Yes: Heat shock84-1 induces the expression of App, </i></span></span><span style='position:absolute;top:51.35%;left:0%;width:85.33%'><span style='font-family:"Gill Sans";font-size:250%;color:#E9EB5D'><i>ubiquitin, and pyruvate kinase</i></span></span></div><div class=O style='position:absolute;top:50.88%;left:5.16%;width:95.23%;height:19.78%'><span style='position:absolute;top:0%;left:0%;width:100.0%'><spanstyle='font-family:"Gill Sans";font-size:250%;color:#E9EB5D'><i>Having“confirmed” these known relations, we can </i></span></span><spanstyle='position:absolute;top:33.92%;left:0%;width:99.72%'><spanstyle='font-family:"Gill Sans";font-size:250%;color:#E9EB5D'><i>now add newmembers to this family: Atp6l, Gnas, </i></span></span><span style='position:absolute;top:67.85%;left:0%;width:89.01%'><span style='font-family:"Gill Sans";font-size:250%;color:#E9EB5D'><i>Ndr4. A thin veneer of functional genomics.</i></span></span></div></div><div id=NotesObj style='display:none'><table style='color:white' border=0 width="100%"> <tr>  <td width=5 nowrap></td>  <td width="100%"></td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=3>Having worked with  WebQTL now for 30 minutes, do we know anything new? The hypothesis that we  have generated (but not validated) is that three transcripts: Atp6l, Gnas,  and Ndr4 are part of a family of genes that are coregulated in normal mouse  forebrain with App and Hsp84-1. We need to add functional and mechanistic  significance to this hypothesis to make it biologically vibrant. But from a  statiistical standpoint it is a strong inference.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=3>Please don’t say: But  these are mere correlations. A high correlation in this context has a  biological basis. The real question is are we smart enough to understand the  web (not chain) of causality that produced the correlation. Once we  understand the web of causality, does it have utility? Very often the answer  will be NO. This will often be the case when a high correlation is generated  by linkage disequilibrium of sets of polymorphisms that modulate a set of  mechanistically separated traits. Chromosomal linkage can produce  correlations that are not mechanistic in the conventional sense used by  molecular biologists. For example, clusters<span style="mso-spacerun:  yes">&nbsp; </span>of hox transcription factor genes tend to be close  physically to keratin gene clusters, and one might expect shared patterns of  variance produced by this linkage in a mapping panel, no matter how large.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=3>If Affymetrix designed  probe sets with reasonable care, if we did the experiments correctly, if we  sampled animals appropriately, then a correlation of 0.70 or higher between  transcripts in the brain tells you that these two transcripts are effectively  coupled in this set of animals under this set of conditions. More than 50%  the variance in the expression of one transcript can be predicted from the  other. That is a major piece of information that could be of significant  clinical, economic, and predictive value, whatever its causes. Yes,  correlation coefficients are noisy and have large error terms, but we have  larger n of strains coming to the rescue. Expect more than 50 BXD lines soon.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=3>This is a thin veneer of  functional genomics. It is enough to generate some marvelous hypotheses in a  semi-automated way.</font><br>  </td> </tr></table></div><script language=JavaScript><!--
function playList() {

}
//-->
</script></body></html>