<html><head><meta name=ProgId content=PowerPoint.Slide><meta name=Generator content="Microsoft Macintosh PowerPoint 11"><link id=Main-File rel=Main-File href="webqtl_demo2_part1.ppt.htm"><link title="Presentation File" type="application/powerpoint" rel=alternatehref="webqtl_demo2_part1.ppt.ppt"><script>
if ( ! top.PPTPRESENTATION ) {
	window.location.replace( "endshow.htm" );
}
</script></head><body bgcolor=black text=white><table border=0 width="100%"> <tr>  <td width=5 nowrap></td>  <td width="100%"></td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=2>Please link to the web  site:<span style="mso-spacerun: yes">&nbsp; </span>http://www.genenetwork.org</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=2>To begin a search you  make choices about what species, group, and database to explore.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=2>For this demonstration  enter APP as above and click on the SEARCH button. Make sure that the DEFAULT  SETTINGS are species = Mouse, Group = BXD, Type = Whole Brain, and Database =  INIA BRAIN mRNA M430 (Apr05) PDNN.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=2>Notes:</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=2>1. The GeneNetwork and  WebQTL are often used to work with public data sets. However, it is possible  to enter and analyze your own data for specific genetic reference populations  such as the BXD genetic reference population of mice or the HXB strains of rat.  Entering your own data is a more advanced topic, but if you click on the HOME  pop-down menu (upper left), you will see “Enter Trait Data” that will explain  the process.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Arial size=2>2.</font><font  face=Helvetica size=2> For help on advanced searching methods read the left  side of the page (INTRODUCTION).<span style="mso-spacerun: yes">&nbsp;  </span>If you make a search term too complex, you may get no hits (try  entering “amyloid beta” for example). If you make it too simple, you may also  get too many.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Arial size=2>3.</font><font  face=Helvetica size=2> Use the asterisk * as a wildcard. For example, to find  all <i>Hoxb</i> transcripts, search for Hoxb*.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Arial size=2>4.</font><font  face=Helvetica size=2> In some cases you can also research for transcripts  and genes using special search strings such as “</font><font face=Geneva  color="#222222" size=2>Mb = (Chr1, 100 102)” to find all genes on Chromosome  1 between 98 and 104 megabases (don’t actually use the quotes). Details are  described at </font><font face=Verdana color=black size=2>http://www.genenetwork.org/searchHelp.html</font><font  face=Geneva color="#222222" size=2>.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=2>5.<span  style="mso-spacerun: yes">&nbsp;&nbsp; </span>These INFO buttons provide  links to data about the different data types. Try them.</font><br>  </td> </tr> <tr>  <td colspan=1></td>  <td align=left colspan=1><font face=Helvetica size=2>6.<span  style="mso-spacerun: yes">&nbsp; </span>The SET TO DEFAULT button is used to  change the database default setting to match your typical search categories.</font><br>  </td> </tr></table></body></html>