<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Barley 150 Embryo mRNA (Dec06)</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>
</HEAD><!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>GN INFO on: Barley 150 Embryo mRNA (Apr06)</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		
		<TD vAlign=top width="100%" height=200 bgColor=#eeeeee>

<P class="title">Genetics of mRNA abundance in barley
<BR>Affymetrix RMA data set from SCRI, December 2006 
<A HREF="/webqtl/main.py?FormID=editHtml">
		<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>
		

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote>
<P>PRELIMINARY TEXT: The December 2006 SCRI barley data set was generated to provide estimates of mRNA abundance in doubled haploid recombinant lines of cultivated barley. Embryo-derived tissues at four days after imbibition (150 lines) and seedling leaves at 12 days after imbibition (subset of 34 lines) and three biological replicates of each parental cultivar (Steptoe and Morex) for each tissue were used for the isolation of total RNA and hybridization to the Barley1 22K GeneChip.

<P>ARNIS: Please revise and update this text. I copied the April 2006 data and have NOT made any modifications below.

</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the lines used to generate this set of data:</P>

<Blockquote>
<P>The SM cross was originally made to map barley grain quality traits; Steptoe is high-yielding barley cultivar used for animal feeding, but Morex has good malting barley characteristics (Hayes et al 1993). Many agronomic quality traits have been mapped using this population (for the lists see BeerGenes web-site http://gnome.agrenv.mcgill.ca/bg/).

<P>The sample used in this study consists of 150 Steptoe x Morex doubled haploid recombinant lines (Kleinhofs et al. 1993) was used to obtain embryo-derived tissue. For the seedling leaf tissue a subset of 35 lines was used. This subset was selected based on evenly spaced crossovers along each of seven barley chromosomes. The following are the IDs of the 35 line subset:
<P>
SM004 SM007 SM012 SM013 SM022 SM024 SM027 SM041 SM043 SM044 SM046 SM061 SM063 SM073 SM074 SM079 SM085 SM088 SM089 SM116 SM130 SM135 SM136 SM140 SM141 SM146 SM152 SM155 SM160 SM167 SM169 SM173 SM177 SM184 SM200.

<P>Line SM073 has been removed from the analysis of the leaf tissue because it appeared to be a duplicate of SM074, but the data are available from the ArrayExpress.

<P>The following classical phenotypes have also been deposited in GeneNetwork in the Phenotype file. Full descriptions of the phenotyping procedures are available from Hayes et al. (1993):

<OL>
<LI>Grain yield (MT/ha) 
<LI>Lodging (%) 
<LI>Height (cm) 
<LI>Heading date (days after January 1) 
<LI>Grain protein (%) 
<LI>Alpha amylase (20 Deg units) 
<LI>Diastatic power (Deg) 
<LI>Malt extract (%)
</OL>

<P>Agronomic and malting quality traits were measured in 16 and 9 environments, respectively. The phenotype data files are coded for each environment as follows: 
Environment #

<Blockquote>
<BR>…_01 Crookston, Minnesota
<BR>…_02 Ithaca, New York
<BR>…_03 Guelph, Ontario
<BR>…_04 Pullman, Washington
<BR>…_05 Brandon, Manitoba
<BR>…_06 Outlook, Saskatchewan
<BR>…_07 Goodale, Saskatchewan
<BR>…_08 Saskatoon, Saskatchewan
<BR>…_09 Tetonia, Idaho
<BR>…_10 Bozeman, Montana (irrigated)
<BR>…_11 Bozeman, Montana (dryland)
<BR>…_12 Aberdeen, Idaho
<BR>…_13 Klamath Falls, Oregon
<BR>…_14 Pullman, Washington
<BR>…_15 Bozeman, Montana (irrigated)
<BR>…_16 Bozeman, Montana (dryland)
</Blockquote>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About tissues used to generate this set of data:</P>

<Blockquote>
<P>Plant material according to the current plant ontologies: Embryo-derived tissues: whole plant (PO:0000003) at the development stage 1.05-coleoptile emerged from seed (GRO:0007056);  Seedling leaves: primary shoot (PO:0006341) at the developmental stage 2.02-first leaf unfolded (GRO:0007060) (Druka et al. 2006).

<P>To obtain seedling leaves, three Microclima 1000 growth chambers (Snijders Scientific B.V., Tilburg, Holland) were used for the experiment. Each cabinet accomodated 40 (13x13 cm) pots. Humidity was set to 70%, with light conditions for 16 hours light at 17C and 8 hours dark at 12C. The cycle started at 10 am with lights on. Light intensity was 337-377 mmol m-2 s-1, measured at the beginning of the experiment, 11 cm from the light source. Measurement was done using Sky Quantium light sensor at 15oC. Plants were placed 55 cm from the light source (from the bulb to the surface of the vermiculite). Ten sterilized seeds per pot were planted and 3 pots per genotype / per cabinet were used. After 12 days, leaf blade and sheath from 5-7 the same size plants was cut off, bulked and flash frozen in the liquid nitrogen.

<P>To obtain embryo-derived tissue, growth room#2, AN building, SCRI, with the standard laboratory bench positioned in the middle of the room was used to germinate sterilized seeds. Seeds were placed between three layers of wet 3MM filter paper in the 156 10 mm Petri plates. Thirty to fifty seeds per line (per Petri plate) were used. Germination was in the dark, 16 hours at 17 deg C and 8 hours at 12 deg C. After 96 hours, embryo-derived tissue (mesocotyl, coleoptile, and seminal roots) from three grains was dissected and flash frozen in the liquid nitrogen. Germination and collection was repeated two more times. Complete randomization of the Petri plates was done for each germination event. Tissues from all three germinations (collections) were bulked before RNA isolation. Three replicates of the parental cultivars were germinated for each collection.

</Blockquote>

<Blockquote>
<P><B>RNA Sample Processing:</B> 
<P>Detailed descriptions of these procedures can be found under the ArrayExpress (http://www.ebi.ac.uk/aerep/?) protocol P-MEXP-4631 (Caldo et al. 2004).

<P><B>Replication and Sample Balance:</B> 

<P><B>Experimental Design and Batch Structure:</B> 

</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Downloading all data:</P>
<Blockquote>
<P> The following are ArrayExpress (http://www.ebi.ac.uk/aerep/?) experiment IDs: 
E-TABM-111 (leaf, 41 chips) and E-TABM-112 (embryo derived, 156 chips). Line SM073 was not used in this GeneNetwork data set because it is suspected replicate of SM074.
</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>

<Blockquote>
<P>Affymetrix 22K Barley1 GeneChip probe array (http://www.affymetrix.com/products/arrays/specific/barley.affx ; Affymetrix product #900515 GeneChip Barley Genome Array) representing 21,439 non-redundant Barley1 exemplar sequences was derived from worldwide contribution of 350,000 high-quality ESTs from 84 cDNA libraries, in addition to 1,145 barley gene sequences from the National Center for Biotechnology Information non-redundant database (Close et al 2004). Abbreviated annotations were created based on the exemplar sequence homology by Arnis Druka using data from the Harvest (http://harvest.ucr.edu/) data depository.

</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote>
<P>The CEL files were imported into the GeneSpring GX 7.3 (Agilent Technologies, Palo Alto, CA) software and processed using the RMA algorithm. Per-chip and per-gene normalization was done following the standard GeneSpring procedure (citation of the GeneSpring normalization description):
<OL>
<LI> Values below 0.01 were set to 0.01.
<LI> Each measurement was divided by the 50.0th percentile of all measurements in that sample.
<LI> Each gene was divided by the median of its measurements in all samples. If the median of the raw values was below 10 then each measurement for that gene was divided by 10 if the numerator was above 10, otherwise the measurement was thrown out.
</OL>
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>

<Blockquote>
<P>Plant maintenance, tissue collection, RNA isolation, and data preparation for submission to GeneNetwork was done at SCRI by Arnis Druka with support from BBSRC/SEERAD grant to Prof. Michael Kearsey (University of Birmingham, UK) and Dr. Robbie Waugh (SCRI, UK). Probe synthesis, labeling and hybridization were performed according to manufacturer’s protocols (Affymetrix, Santa Clara, CA) at the Iowa State University GeneChip Core facility (Rico Caldo and Roger Wise). ArrayExpress (EBI, UK) team members Tim Rayner, Helen Parkinson, and Alvis Brazma are acknowledged for excellent help with data submission to Arrayexpress.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Contact address:</P>
<Blockquote>
<BR>Arnis Druka
<BR>Genetics Programme
<BR>Scottish Crop Research Institute
<BR>Invergowrie, Dundee DD2 5DA
<BR>Angus, Scotland, United Kingdom
<BR>Tel +44 01382 562731
<BR>adruka@scri.sari.ac.uk
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;References:</P>
<Blockquote>

<P>Druka A, Muehlbauer G, Druka I, Caldo R, Baumann U, Rostoks N, Schreiber A, Wise R, Close T, Kleinhofs A, Graner A, Schulman A, Langridge P, Sato K, Hayes P, McNicol J, Marshall D, Waugh R. (2006) An atlas of gene expression from seed to seed through barley development. Funct Integr Genomics, in press.

<P>Kleinhofs A, Kilian A, Saghai Maroof M, Biyashev R, Hayes P, Chen F, Lapitan N, Fenwick A, Blake T, Kanazin V, Ananiev E, Dahleen L, Kudrna D, Bollinger J, Knapp SJ, Liu BH, Sorrells M, Heun M, Franckowiak J, Hoffman D, Skadsen R, Steffenson B (1993) A molecular, isozyme, and morphological map of the barley (<I>Hordeum vulgare</I>) genome.  Theor Appl Genet 86:705-712.

<P>Caldo RA, Nettleton D, Wise RP (2004) Interaction-dependent gene expression in Mla-specified response to barley powdery mildew. Plant Cell 16:2514-2528.

<P>Close TJ, Wanamaker SI, Caldo RA, Turner SM, Ashlock DA, Dickerson JA, Wing RA, Muehlbauer GJ, Kleinhofs A, Wise RP. (2004) A new resource for cereal genomics: 22K barley GeneChip comes of age. Plant Physiol 134:960-968.

<P>Hayes PM, Liu BH, Knapp SJ, Chen F, Jones B, Blake T, Franckowiak J, Rasmusson D, Sorrells M, Ullrich SE, Wesenberg D, Kleinhofs A (1993) Quantitative trait locus effects and environmental interaction in a sample of North American barley germplasm. Theor Appl Genet 87:392-401

</Blockquote>


</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About this text file:</P>
<Blockquote><P>
This text file originally generated by Arnis Druka on May 8, 2006. Modified Aug1 by AD. Entered by RWW Aug 4, 2006.
</P></Blockquote>




<P></P>

		</TD>
	</TR>
	<TR>
		<TD colspan=2 align=center bgColor=#ddddff>
	 
	 
	
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->



		</TD>
	</TR>
</TABLE><!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>

<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	// Note where menu initialization block is located in HTML document.
	// Don't try to position menu locating menu initialization block in
	// some table cell or other HTML element. Always put it before </body>

	// each menu gets two parameters (see demo files)
	// 1. items structure
	// 2. geometry structure

	new menu (MENU_ITEMS, MENU_POS);
	// make sure files containing definitions for these variables are linked to the document
	// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
	// error in menu_tpl.js file or that file isn't linked properly.
	
	// also take a look at stylesheets loaded in header in order to set styles
	//-->
</script>
</BODY>
</HTML>
