<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>M430 RMA Liver F2 Aug05 / GeneNetwork</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">

(B6 x BTBR)F<SUB class="fs12">2</SUB>-ob/ob Genotype Database  <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote>
<P>
The Genotype database of August 2005 lists genotypes for 194 MIT microsatellite markers and 110 F2 animals used in combination with the Phenotypes and Liver transcriptome databases for mapping quantitative trait loci. To review a complete list of these genotypes type in the wildcard character *  in the ANY search field. You can also search more selectively for markers using this general syntax <B>Mb=(Chr1 50 150)</B> to find all markers on Chr 1 between 50 and 150 Mb. This marker set includes genotypes for all 60 selected animals whose liver mRNAs were quantified using the Affymetrix M430A and B arrays, as well as an additional 50 F2 ob/ob animals from the same cross.
</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>The 110 F<SUB class="fs12">2</SUB>-ob/ob mice were from a mapping panel that we created to map diabetes related physiological phenotypes (Stoehr et al. 2000). These F<SUB class="fs12">2</SUB>-ob/ob mice were also used to map mRNA abundance traits derived by quantitative real-time RT-PCR (Lan et al. 2003). The sixty F<SUB class="fs12">2</SUB>-ob/ob mice that were used to generate microarray-derived mRNA abundance traits were selected from the 110 mice based on a selective phenotyping algorithm (Jin et al. 2004). The F<SUB class="fs12">2</SUB>-ob/ob mice were housed at weaning at the University of Wisconsin-Madison animal care facility on a 12-h light/dark cycle. Mice were provided Purina Formulab Chow 5008 (6.5% fat) and acidified water ad libitum. Mice were killed at 14 weeks of age by CO<SUB class="fs12">2</SUB> asphyxiation after a 4-hour fast. The livers, along with other tissues, were immediately foil wrapped and frozen in liquid nitrogen, and subsequently transferred to -80 &deg;C freezers for storage. </Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the marker set:</P>
<Blockquote>All 110 mice were genotyped at 194 MIT microsatellite markers separated an average of approximately 10 cM apart across the entire genome (Y chromsome, excepted). The maximum distance between markers wass less than 30 cM. The genotyping error-check routine implemented within R/qtl (Broman et al. 2003) showed no likely errors at p <0.01 probability.

<!--Why is the genotype value for Case438 on marker D1Mit303 listed as 99? -->
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>This project was supported in part by NIH/NIDDK 5803701, NIH/NIDDK 66369-01 and American Diabetes Association 7-03-IG-01 to Alan D. Attie, USDA CSREES grants to the University of Wisconsin-Madison to Brian S. Yandell, and HHMI grant A-53-1200-4 to Christina Kendziorski.
</blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>This text file originally generated by RWW and Alan Attie, August 20, 2005.
</P></Blockquote>

</TD>

<!-- Body End Here -->
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">




<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->




</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
