<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>BXD Genotype / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">

<P class="title">Arabidopsis Bay X Sha Genotypes Database

 <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>

<P>The Bay-0 x Shahdara genetic reference population consists of 420 recombinant inbred lines (RIL). This set was created by Olivier Loudet and Sylvain Chaillou between 1997 and 2000 at the <A HREF=" http://w3.inra.fr/ " target="_blank" class="fs14">INRA</A> in Versailles, France. This plant has a genome consisting of <A HREF="http://www.inra.fr/qtlat/BayxSha/BayxShaMarker.htm" target="_blank" class="fs14">five</A> chromsomes and a total of 125 megabases of DNA, equivalent in length to a single human chromosome. However, the genome is nonetheless rich and contains approximately <A HREF="http://www.tigr.org/tdb/e2k1/ath1/intro.shtml" target="_blank" class="fs14">26,000</A> genes. Full sequence data are available for this species. (The Col-0 accession was sequenced.)
</P>

<P>The 420 RILs were derived from a cross between Bay-0 (accession N954) and Shahdara (accession N929); two accessions obtained from the <A HREF="http://arabidopsis.info/" class="fs14" target="_blank">NASC</A> European Arabidopsis Stock Centre. Bay-0 and Shahdara were chosen because of their well characterized genetic, geographical, and ecological differences. Lines were propagated by single seed descent through the sixth generation (F6) without selection. One plant per line was then used for genotyping (420 RILs x 38 markers) and selfed to obtain F7 seeds. F8 seed stock generated by bulk multiplication of F7 plants are available for analysis for 411 of these RILs. Data sets in WebQTL include up to 415 BXS accessions and the two parental stock.
</P>

</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the genotypes used in these studies:</P>

<Blockquote>
This Bay x Sha Arabidopsis genotype data set is described in Loudet and colleagues (2002). This marker data set consists of 38 markers for 420 strains. It is used to map QTLs for phenotypes listed in the BayxSha Published Phenotypes data set.

<P> <A HREF="./genotypes/BayXSha.geno" class="fs14">Download</A> the BayXSha genotype data set.
</Blockquote>



</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Reference:</P>

<Blockquote>
Loudet O, Chaillou S, Camilleri C, Bouchez D, Daniel-Vedele F (<A HREF="http://www.inra.fr/qtlat/BayxSha/Loudet2002.pdf" target="_blank" class="fs14">2002</A>) Bay-0 x Shahdara recombinant inbred line population: a powerful tool for the genetic dissection of complex traits in Arabidopsis. <U>Theoretical and Applied Genetics</U> 104:1173-1184  (<A HREF="http://www.inra.fr/qtlat/BayxSha/Loudet2002.pdf" target="_blank" class="fs14">pdf</A>)
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Acknowledgments and File History</P>
<Blockquote>

<P>This text file was originally written by Robert Williams and Olivier Loudet (March 8, 2006). Updated March 8, 2006 by OL.
</Blockquote>


<Blockquote><P>

</P><TABLE width="100%"><TR><TD align="left"> <A HREF="http://dbsgap.versailles.inra.fr/vnat/"><IMG SRC="/images/VNATCrop.gif"></A></TD><TD align="right">
<A HREF="http://www.dpw.wau.nl/natural/"><IMG SRC="/images/naturalbig.jpg"></A></TD></TABLE></Blockquote>


</TD>
</TR></TABLE>
</Blockquote>
<Blockquote><P>
<P></P>

</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
