<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>BXD Microarray June03 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">


<P class="title">

UTHSC Brain mRNA U74Av2 (Jun03) MAS5

 <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=2">GN2</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>
This June 2003 data freeze provides estimates of mRNA expression in brains of BXD recombinant inbred mice measured using Affymetrix U74Av2 microarrays. New users are encouraged to use one of the more recent data sets (March 2004). Data were generated at the University of Tennessee Health Science Center (UTHSC). Data were generated at the University of Tennessee Health Science Center (UTHSC). Over 300 brain samples from 34 strains were hybridized in small pools (n=3) to 99 arrays. Data were processed using the Microarray Suite 5 (<a href="www.affymetrix.com/support/technical/whitepapers/sadd_whitepaper.pdf" class="fs14">MAS 5</a>) protocol of Affymetrix. To simplify comparison between transforms, MAS 5 values of each array were adjusted to an average of 8 units and a variance of 2 units. In general, these MAS 5 transforms do not perform as well as RMA, PDNN, or the new heritability weighted transforms (HW1PM).
</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote><P>
This data set includes estimate of gene expression for 34 genetically uniform lines of mice: C57BL/6J (B6, or simply B), DBA/2J (D2 or D), B6D2 F1 intercross progeny, and 32 BXD recombinant inbred (RI) strains derived by crossing female B6 mice with male D2 mice and then inbreeding progeny for more than 21 generations. This set of RI strains is a remarkable resource because many of these strains have been extensively phenotyped for hundreds of interesting traits over a 25-year period. A significant advantage of this RI set is that the two parental strains (B6 and D2) have both been extensively sequenced and are known to differ at approximately 1.8 million SNPs. Coding variants (mostly single nucleotide polymorphisms and insertion-deletions) that may produce interesting phenotypes can be rapidly identified in this particular RI set.</P>

<P>BXD1 through BXD32 were produced by Benjamin A. Taylor starting in the late 1970s. BXD33 through BXD42 were also produced by Taylor, but from a second set of crosses initiated in the early 1990s. These strains are all available from the Jackson Laboratory, Bar Harbor, Maine. BXD43 through BXD99 were produced by Lu Lu, Jeremy Peirce, Lee M. Silver, and Robert W. Williams in the late 1990s and early 2000s using advanced intercross progeny (Peirce et al. <a href="http://www.biomedcentral.com/1471-2156/5/7"  class="fs14">2004</a>). Only two of these incipient strains are included in the current database (BXD67 and BXD68).</P>

<P>In this mRNA expression database we generally used progeny of stock obtained from The Jackson Laboratory between 1999 and 2001. Animals were generated in-house at the University of Alabama by John Mountz and Hui-Chen Hsu and at the University of Tennessee Health Science Center by Lu Lu and Robert Williams.
</P></Blockquote>


<Blockquote>
The table below lists the arrays by strain, sex, and age. Each array was hybridized to a pool of  mRNA from three mice.
</Blockquote>




<TABLE border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="85%" align="Center">
<TR>
<TD>
<TABLE border="0" cellpadding="5" cellspacing="1" width="100%">
<TR bgcolor="royalblue">
<TD rowspan=2 align="Center"> <FONT color="#FFFFFF"> Strain
</FONT> </TD>
<TD colspan=3> <FONT color="#FFFFFF">
<CENTER>
Age
</CENTER>
</FONT> </TD>
<TD rowspan=2 align="Center"> <FONT color="#FFFFFF"> Strain
</FONT> </TD>
<TD colspan=3> <FONT color="#FFFFFF">
<CENTER>
Age
</CENTER>
</FONT> </TD>
</TR>
<TR bgcolor="royalblue">
<TD id="small"> <FONT color="#FFFFFF">
<CENTER>
8 Wks
</CENTER>
</FONT> </TD>
<TD id="small"> <FONT color="#FFFFFF">
<CENTER>
20 Wks
</CENTER>
</FONT> </TD>
<TD id="small"> <FONT color="#FFFFFF">
<CENTER>
52 Wks
</CENTER>
</FONT> </TD>
<TD id="small"> <FONT color="#FFFFFF">
<CENTER>
8 Wks
</CENTER>
</FONT> </TD>
<TD id="small"> <FONT color="#FFFFFF">
<CENTER>
20 Wks
</CENTER>
</FONT> </TD>
<TD id="small"> <FONT color="#FFFFFF">
<CENTER>
52 Wks
</CENTER>
</FONT> </TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">C57BL/6J (B6)</TD>
<TD id="small">&#9794&#9794&#9794</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">DBA/2J (D2)</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9794&#9794&#9792</TD>
<TD id="small">&nbsp; </TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">B6D2F1 (F1)</TD>
<TD id="small">&#9792 &#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">BXD1</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&#9792</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD2</TD>
<TD id="small">&#9794</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">BXD5</TD>
<TD id="small">&#9794&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&nbsp;</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD6</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">BXD8</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9794&#9792</TD>
<TD id="small">&nbsp;</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD9</TD>
<TD id="small">&#9794</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">BXD11</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&#9792</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD12</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&#9794&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">BXD13</TD>
<TD id="small">&#9792 </TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&nbsp; </TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD14</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">BXD15</TD>
<TD id="small">&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&#9792</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD16</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">BXD18</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9794</TD>
<TD id="small">&#9792</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD19</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">BXD21</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9794&#9794</TD>
<TD id="small">&nbsp;</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD22</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">BXD24</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small"> &#9792</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD25</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">BXD27</TD>
<TD id="small">&nbsp; </TD>
<TD id="small">&nbsp; </TD>
<TD id="small">&#9792&#9792</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD28</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">BXD29</TD>
<TD id="small">&#9794</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&#9792</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD31</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&#9792&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">BXD32</TD>
<TD id="small">&#9792</TD>
<TD id="small">&#9794&#9792</TD>
<TD id="small">&#9792</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD33</TD>
<TD id="small">&#9794&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">BXD34</TD>
<TD id="small">&#9794&#9792</TD>
<TD id="small">&#9792</TD>
<TD id="small">&nbsp;</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD38</TD>
<TD id="small">&#9794&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">BXD39</TD>
<TD id="small">&#9794&#9792</TD>
<TD id="small">&#9794</TD>
<TD id="small">&nbsp;</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD id="small">BXD40</TD>
<TD id="small">&#9794&#9794&#9792&#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&nbsp; </TD>
<TD id="small">BXD42</TD>
<TD id="small">&#9794&#9794 &#9792 </TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&nbsp;</TD>
</TR>
<TR bgcolor="#eeeeee">
<TD height="28" id="small">BXD67 (F8)</TD>
<TD id="small">&#9792 </TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&nbsp; </TD>
<TD id="small">BXD68 (F9)</TD>
<TD id="small">&#9792 &#9792</TD>
<TD id="small">&nbsp;</TD>
<TD id="small">&nbsp;</TD>
</TR>
</TABLE>
</TD></TR></TABLE>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the samples used to generate these data:</P>

<Blockquote><P>
Each array was hybridized with labeled cRNA generated from a pool of three brains from adult animals usually of the same age and always of the same sex. The brain region included most of the forebrain and midbrain, bilaterally. However, the sample excluded the olfactory bulbs, retinas, or the posterior pituitary (all formally part of the forebrain). A total of 99 such pooled samples were arrayed: 75 from females and 24 from males. Animals ranged in age from 56 to 441 days, usually with a balanced design: one pool at approximately 8 weeks, one pool at approximately 20 weeks, and one pool at approximately 1 year. Strain averages of mRNA expression level are therefore typically based on three pooled biological replicate arrays. This data set does not incorporate statistical adjustment for possible effects of age and sex.
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>

<Blockquote><P>
<B>Affymetrix U74Av2 GeneChip:</B> The expression data were generated using 99 U74Av2 arrays. The chromosomal locations of U74Av2 probe sets were determined by BLAT analysis of concatenated probe sequences using the Mouse Genome Sequencing Consortium May 2004 (mm5) assembly. This BLAT analysis is performed periodically by Yanhua Qu as each new build of the mouse genome is released (see <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse" class="fs14">http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse</a>). We thank Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis. It is possiible to confirm the BLAT alignment results yourself simply by clicking on the <B>Verify</B> link in the Trait Data and Editing Form (right side of the <B>Location</B> line).
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B> Probe signal intensity estimates in the Affymetrix CEL files are the 75% quantile value taken from a set of <a href="images/AffyU74.pdf" target="_blank" class="fs14">36</a> (6x6) pixels per probe cell in the DAT image file.
<UL>
<LI>Step 1: We added an offset of 1.0 to the CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log2 of each cell signal intensity.

<LI>Step 3: We computed the Z score for each of these log2 cell signal intensity values within a single array.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added a constant of 8 units to the value of the Z score. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8 units, a variance of 4 units, and a standard deviation of 2 units. The advantage of this modified Z score is that a two-fold difference in expression level corresponds roughly to 1 unit.

<LI>Step 6: We computed the arithmetic mean of the values for the set of microarrays for each strain. We have not corrected for variance introduced by sex, age, source of animals, or any possible interaction. We have not corrected for background beyond that implemented by Affymetrix in generating the CEL file.
</UL>


<B>Probe set data from the CHP file: </B>Probe set estimates of expression were initially generated using the standard Affymetrix MAS 5 algorithm. The CHP values were then processed following precisely the same six steps listed above to normalize expression and stabilize the variance of all arrays. The mean expression within each array is therefore 8 units with a standard deviation of 2 units. A 1-unit difference represents roughly a 2-fold difference in expression level. Expression levels below 5 are close to the background noise level. While a value of 8 unit is nominally the average expression, this average includes all those transcripts with negligible expression in the brain that would often be eliminated from subsequent analysis (so-called "absent" and "marginal" calls in the CHP file). </Blockquote>


<Blockquote> <P>
Most probe sets on the U74Av2 array consist of a total of 32 probes, divided into 16 perfect match probes and 16 mismatch controls. Each set of these 25-nucleotide-long probes has an identifier code that includes a unique number, an underscore character, and several suffix characters that highlight design features.  The most common probe set suffix is  <b>at</b>. This code indicates that the probes should hybridize relatively selectively with the complementary anti-sense target (i.e., the complemenary RNA) produced from a single gene. Other codes include:</P>

<li> <b>f_at (sequence family)</b>: Some probes in this probe set will hybridize to identical and/or slightly different sequences of related gene transcripts.</li>

<li><b>s_at (similarity constraint)</b>: All Probes in this probe set target common sequences found in transcripts from several genes. </li>

<li> <b>g_at (common groups)</b>: Some probes in this set target identical sequences in multiple genes and some target unique sequences in the intended target gene.</li>

<li> <b>r_at (rules dropped)</b>: Probe sets for which it was not possible to pick a full set of unique probes using the Affymetrix probe selection rules. Probes were picked after dropping some of the selection rules.</li>

<li> <b>i_at (incomplete)</b>: Designates probe sets for which there are fewer than the standard numbers of unique probes specified in the design (16 perfect match for the U74Av2).</li>

<li> <b>st </b> (sense target): Designates a sense target; almost always generated in error.</li>

<P>Descriptions for the probe set extensions were taken from the Affymetrix<a href="./dbdoc/data_analysis_fundamentals_manual.pdf" class="fs14"> GeneChip Expression Analysis Fundamentals</a>.
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote><P>Data were generated with funds to RWW from the Dunavant Chair of
Excellence, University of Tennessee Health Science Center, Department
of Pediatrics. The majority of arrays were processed at <a href="http://www.genomeexplorations.com" target="_blank" class="fs14">Genome Explorations</A> by Dr. Divyen Patel. We thank Guomin Zhou for generating advanced intercross stock used to produce most of the new BXD RI strains.
</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>
This text file originally generated by RWW, EJC, and YHQ, June 2003. Updated by RWW, October 30, 2004.


</P></Blockquote>

</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
