<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>BXD Microarray May03 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">


<P class="title">UTHSC Brain mRNA U74Av2 (May03) MAS5

 <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=4">GN4</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>
This May 2003 freeze provides estimates of mRNA expression in brains of BXD recombinant inbred mice measured using Affymetrix U74Av2 microarrays. New users are encouraged to use one of the more recent data sets (December 2003 or March 2004). Data were generated at the University of Tennessee Health Science Center (UTHSC). Over 300 brain samples from 33 strains were hybridized in small pools (n=3) to 97 arrays. Data were processed using the Microarray Suite 5 (<a href="www.affymetrix.com/support/technical/whitepapers/sadd_whitepaper.pdf" class="fs14">MAS 5</a>) protocol of Affymetrix. To simplify comparison between transforms, MAS 5 values of each array were adjusted to an average of 8 units and a variance of 2 units. In general, these MAS 5 transforms do not perform as well as RMA, PDNN, or the new heritability weighted transforms (HW1PM).
</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>The set of animals used for mapping (a mapping panel) consists of 30 groups of genetically uniform mice of the BXD type. The parental strains are C57BL/6J (B6 or B) and DBA/2J (D2 or D). The first generation hybrid is labeled F1. The F1 hybrids were made by crossing B6 females to D2 males.

All other lines are recombinant inbred strains derived from C57BL/6J and DBA/2J crosses. BXD2 through BXD32 were produced by Dr. Benjamin Taylor starting in the late 1970s. BXD33 through BXD42 were also produced by Dr. Taylor, but they were generated in the 1990s. Lines BXD67 and BXD68 are two partially inbred advanced recombinant strains (F8 and F9) that are part of a large set of BXD-Advanced strains being produced by Drs. Robert Williams, Lu Lu, Lee Silver, and Jeremy Peirce. There will eventually be ~45 of these strains. For additional background on recombinant inbred strains, please see <a href="http://www.nervenet.org/papers/bxn.html"  class="fs14">http://www.nervenet.org/papers/bxn.html</a>.
</Blockquote>

<Blockquote>The table below lists the arrays by strain, sex, and age. Each array was hybridized to a pool of  mRNA from 3 mice.</Blockquote>

<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="85%" align="Center">
<tr>
<td>
<TABLE border="0" cellpadding="5" cellspacing="1" width="100%">
<TR bgcolor="royalblue">
<td rowspan=2 align="Center"> <font color="#FFFFFF"> Strain
</font> </td>
<td colspan=3> <font color="#FFFFFF">
<center>
Age
</center>
</font> </td>
<td rowspan=2 align="Center"> <font color="#FFFFFF"> Strain
</font> </td>
<td colspan=3> <font color="#FFFFFF">
<center>
Age
</center>
</font> </td>
</tr>
<tr bgcolor="royalblue">
<td id="small"> <font color="#FFFFFF">
<center>
8 Wks
</center>
</font> </td>
<td id="small"> <font color="#FFFFFF">
<center>
20 Wks
</center>
</font> </td>
<td id="small"> <font color="#FFFFFF">
<center>
52 Wks
</center>
</font> </td>
<td id="small"> <font color="#FFFFFF">
<center>
8 Wks
</center>
</font> </td>
<td id="small"> <font color="#FFFFFF">
<center>
20 Wks
</center>
</font> </td>
<td id="small"> <font color="#FFFFFF">
<center>
52 Wks
</center>
</font> </td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">C57BL/6J (B6)</td>
<td id="small">&#9794&#9794&#9794</td>
<td id="small">&#9792</td>
<td id="small">&#9792</td>
<td id="small">DBA/2J (D2)</td>
<td id="small">&#9792</td>
<td id="small">&#9794&#9794&#9792</td>
<td id="small">&nbsp; </td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">B6D2F1 (F1)</td>
<td id="small">&#9792 &#9792</td>
<td id="small">&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">BXD1</td>
<td id="small">&#9792&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">&#9792</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD2</td>
<td id="small">&#9794</td>
<td id="small">&#9792</td>
<td id="small">&#9792</td>
<td id="small">BXD5</td>
<td id="small">&#9794&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">&nbsp;</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD6</td>
<td id="small">&#9792</td>
<td id="small">&#9792</td>
<td id="small">&#9792</td>
<td id="small">BXD8</td>
<td id="small">&#9792</td>
<td id="small">&#9794&#9792</td>
<td id="small">&nbsp;</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD9</td>
<td id="small">&#9794</td>
<td id="small">&#9792</td>
<td id="small">&#9792</td>
<td id="small">BXD11</td>
<td id="small">&#9792&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">&#9792</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD12</td>
<td id="small">&nbsp;</td>
<td id="small">&#9794&#9792</td>
<td id="small">&#9792</td>
<td id="small">BXD13</td>
<td id="small">&#9792 </td>
<td id="small">&nbsp;</td>
<td id="small">&nbsp; </td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD14</td>
<td id="small">&nbsp;</td>
<td id="small">&#9792&#9792</td>
<td id="small">&#9792</td>
<td id="small">BXD15</td>
<td id="small">&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">&#9792</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD16</td>
<td id="small">&#9792</td>
<td id="small">&#9792&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">BXD18</td>
<td id="small">&#9792</td>
<td id="small">&#9794</td>
<td id="small">&#9792</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD19</td>
<td id="small">&#9792</td>
<td id="small">&#9792</td>
<td id="small">&#9792</td>
<td id="small">BXD21</td>
<td id="small">&#9792</td>
<td id="small">&#9794&#9794</td>
<td id="small">&nbsp;</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD22</td>
<td id="small">&#9792</td>
<td id="small">&#9792&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">BXD24</td>
<td id="small">&#9792&#9792</td>
<td id="small">&nbsp;</td>
<td id="small"> &#9792</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD25</td>
<td id="small">&#9792&#9792</td>
<td id="small">&#9792&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">BXD27</td>
<td id="small">&nbsp; </td>
<td id="small">&nbsp; </td>
<td id="small">&#9792&#9792</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD28</td>
<td id="small">&#9792</td>
<td id="small">&#9792</td>
<td id="small">&#9792</td>
<td id="small">BXD29</td>
<td id="small">&#9794</td>
<td id="small">&nbsp;</td>
<td id="small">&#9792</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD31</td>
<td id="small">&#9792&#9792</td>
<td id="small">&#9792&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">BXD32</td>
<td id="small">&#9792</td>
<td id="small">&#9794&#9792</td>
<td id="small">&#9792</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD33</td>
<td id="small">&#9794&#9792</td>
<td id="small">&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">BXD34</td>
<td id="small">&#9794&#9792</td>
<td id="small">&#9792</td>
<td id="small">&nbsp;</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD39</td>
<td id="small">&#9794&#9792</td>
<td id="small">&#9794</td>
<td id="small">&nbsp;</td>
<td id="small">BXD40</td>
<td id="small">&#9794&#9794&#9792&#9792</td>
<td id="small">&nbsp;</td>
<td id="small">&nbsp;</td>
</tr>
<tr bgcolor="#eeeeee">
<td id="small">BXD42</td>
<td id="small">&#9794&#9794 &#9792 </td>
<td id="small">&nbsp;</td>
<td id="small">&nbsp; </td>
<td id="small">BXD67</td>
<td id="small">&#9792 </td>
<td id="small">&nbsp;</td>
<td id="small">&nbsp;</td>
</tr>
<tr bgcolor="#eeeeee">
<td height="28" id="small">BXD68 (F9)</td>
<td id="small">&#9792 &#9792</td>
<td id="small">&nbsp;</td>
<td id="small">&nbsp; </td>
<td id="small">&nbsp;</td>
<td id="small">&nbsp;</td>
<td id="small">&nbsp;</td>
<td id="small">&nbsp;</td>
</tr>
</table></td>
</tr>
</table>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>
<Blockquote><P>Most expression data are averages based on three microarrays (U74Av2). Each individual array experiment involved a pool of brain tissue (forebrain plus the midbrain, but without the olfactory bulb) that was taken from three adult animals usually of the same age. A total of 97 arrays were used: 74 were female pools and 23 were male pools. Animals ranged in age from 56 to 441 days, usually with a balanced design (one pool at 8 weeks, one pool at ~20 weeks, one pool at approximately 1 year).
</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by MAS 5 are the 75% quantiles from a set of <a href="images/AffyU74.pdf" target="_blank" class="fs14">36 pixel </a>values per cell (the pixel with the 12th highest value represents the whole cell).
<UL>
<LI>Step 1: We added an offset of 1.0 to the CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log2 of each cell.

<LI>Step 3: We computed the Z score for each cell.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6: We computed the arithmetic mean of the values for the set of microarrays for each of the individual strains. We have not corrected for variance introduced by sex, age, or a sex-by-age interaction. We have not corrected for background beyond the background correction implemented by Affymetrix in generating the CEL file.
</UL>
<B>Probe set data from the .TXT file: </B>These .TXT files were generated using the MAS 5. The same simple steps described above were also applied to these values. Every microarray data set therefore has a mean expression of 8 with a standard deviation of 2. A 1-unit difference therefor represents roughly a 2-fold difference in expression level. Expression levels below 5 are usually close to background noise levels.</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the chromosome and megabase position values:</P>

<Blockquote>The chromosomal locations of probe sets and gene markers were initially determined by BLAT analysis using the Mouse Genome Sequencing Consortium OCT 2003 Assembly (see <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse" class="fs14">http://genome.ucsc.edu/</a>). We thank Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis.</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array probe set names:</P>

<Blockquote> <P>
Most probe sets on the U74Av2 array consist of a total of 32 probes, divided into 16 perfect match probes and 16 mismatch controls. Each set of these 25-nucleotide-long probes has an identifier code that includes a unique number, an underscore character, and several suffix characters that highlight design features.  The most common probe set suffix is  <b>at</b>. This code indicates that the probes should hybridize relatively selectively with the complementary anti-sense target (i.e., the complemenary RNA) produced from a single gene. Other codes include:</P>

<li> <b>f_at (sequence family)</b>: Some probes in this probe set will hybridize to identical and/or slightly different sequences of related gene transcripts.</li>

<li><b>s_at (similarity constraint)</b>: All Probes in this probe set target common sequences found in transcripts from several genes. </li>

<li> <b>g_at (common groups)</b>: Some probes in this set target identical sequences in multiple genes and some target unique sequences in the intended target gene.</li>

<li> <b>r_at (rules dropped)</b>: Probe sets for which it was not possible to pick a full set of unique probes using the Affymetrix probe selection rules. Probes were picked after dropping some of the selection rules.</li>

<li> <b>i_at (incomplete)</b>: Designates probe sets for which there are fewer than the standard numbers of unique probes specified in the design (16 perfect match for the U74Av2).</li>

<li> <b>st </b> (sense target): Designates a sense target; almost always generated in error.</li>

<P>Descriptions for the probe set extensions were taken from the Affymetrix<a href="./dbdoc/data_analysis_fundamentals_manual.pdf" class="fs14"> GeneChip Expression Analysis Fundamentals</a>.
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote><P>Data were generated with funds to RWW from the Dunavant Chair of
Excellence, University of Tennessee Health Science Center, Department
of Pediatrics. The majority of arrays were processed at <a href="http://www.genomeexplorations.com" target="_blank" class="fs14">Genome Explorations</A> by Divyen Patel. We thank Guomin Zhou for generating advanced intercross stock used to produce most of the new BXD RI strains.
</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>
This text file originally generated by RWW, EJC, and YHQ, May 2003. Updated by RWW, October 30, 2004.


</P></Blockquote>


</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
