<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>OHSU/VA B6D2F2 Brain mRNA M430A (Mar04) RMA / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>


</HEAD>
<BODY  bottommargin="5" topmargin="5" leftmargin="5" rightmargin="5" marginheight="5" marginwidth="5" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=3 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<!-- Body Start from Here -->
<TD vAlign=top width="100%" colspan=2 bgColor=#eeeeee>


<P class="title">OHSU/VA B6D2F2 Brain mRNA M430A RMA Database (March/04 Freeze)  <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=32">GN32</A></P>

<Blockquote><P>
This March 2004 data freeze provides estimate of mRNA expression in adult brains of F2 intercross mice (C57BL/6J x DBA/2J F2) measured using Affymetrix M430A microarrays. Data were generated at The Oregon Health Sciences University (OHSU) in Portland, Oregon, by John Belknap and Robert Hitzemann. Data were processed using the <a  href="http://stat-www.berkeley.edu/users/bolstad/RMAExpress/RMAExpress.html" target="_blank" class="normal">RMA</a> protocol and are presented with secondary normalization to an average expression value of 8 units. To simplify comparison between transforms, RMA values of each array were adjusted to an average of 8 units and a variance of 2 units. This data set was run as a single large batch with effort to balance samples by sex, age, and environment.
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote><P>Fifty-six B6D2F2 samples, each taken from a single brain hemisphere from an individual mouse, were assayed using 56 M430A Affymetrix short oligomer microarrays. [The remaining hemisphere will be used later for an anaysis of specific brain regions.] Each array ID (see table below) includes a three letter code; the first letter usually denotes sex of the case (note that we have made a few corrections and there are therefore several sex-discordant IDs), the second letter denotes the hemisphere (R or L), and the third letter is the mouse number within each cell. The F2 mice were experimentally naive, born within a 3-day period from second litters of each dam, and housed at weaning (20- to 24-days-of-age) in like-sex groups of 3 to 4 mice for females and 2 to 3 mice for males in standard mouse shoebox cages within Thoren racks. All 56 F2 mice were killed at 77 to 79 days-of-age by cervical dislocation on December 17, 2003. The brains were immediately split at the midline and then quickly frozen on dry ice. The brains were stored for about two weeks at -80 degrees C until further use.</P>

<P>The F2 was derived as follows: C57BL/6J (B6) and DBA/2J (D2) breeders were obtained from The Jackson Laboratory, and two generations later their progeny were crossed to produce B6D2F1 and D2B6F1 hybrid at the Portland VA Veterinary Medical Unit (AAALAC approved). The reciprocal F1s were mated to create an F2 population with both progenitor X and Y chromosomes about equally represented.</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>

<Blockquote><P>Brain samples were from 31 male and 25 females and between 28 right and 28 left hemispheres distributed with good balance across the two sexes. The tissue arrayed included the forebrain, midbrain, one olfactory bulb, the cerebellum; and the rostral part of the medulla. The medulla was trimmed transversely at the caudal aspect of the cerebellum. The sagittal cut was made from a dorsal to ventral direction. (Note that several of the other brain transcriptome databases do not include olfactory bulb or cerebellum.) Total RNA was isolated with TRIZOL Reagent (Life Technologies Inc.) using a modification of the single-step acid guanidinium isothiocyanate phenol-chloroform extraction method according to the manufacturer’s protocol. The extracted RNA was then purified using RNeasy (Qiagen, Inc.). RNA samples were evaluated by UV spectroscopy for purity; only samples with an A260/280 ratio greater than 1.8 were used. RNA quality was monitored by visualization on an ethidium bromide-stained denaturing formaldehyde agarose gel. Samples containing at least 10 micrograms of total RNA were sent to the OHSU Gene Microarray Shared Resource facility for analysis. The procedures used at the facility precisely follow the manufacturer’s specifications. Details can be found at <a href="http://www.ohsu.edu/gmsr/amc" target="_blank" class="normal">http://www.ohsu.edu/gmsr/amc</a>. Following labeling, all samples were hybridized to the GeneChip Test3 array for quality control. If target performance did not meet recommended thresholds, the sample would have been discarded. All labeled samples passed the threshold and were hybridized to the 430A array.
</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the arrays:</P>
<Blockquote><P>All 56 430A arrays used in this project were purchased at one time and had the same Affymetrix lot number.

The table below lists the arrays by Case ID, Array ID, Side, Cage ID and Sex.</P></Blockquote>

<TABLE border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="70%" align="Center">
<TR>
<TD><center>
<TABLE border="0" cellpadding="5" cellspacing="1" width="100%">
<TR bgcolor="royalblue">
<td> <font color="#FFFFFF"> <center>Order</center>
</font> </td>
<td Stain> <font color="#FFFFFF">
<center>
CaseID
</center>
</font> </td>
<td> <font color="#FFFFFF"> <center>ArrayID </center>
</font> </td>
<td> <font color="#FFFFFF">
<center>
Side
</center>
</font> </td>

<td> <font color="#FFFFFF">
<center>
CageID
</center>
</font> </td>
<td> <font color="#FFFFFF">
<center>
Sex
</center>
</font> </td>
</TR>
<tr bgcolor="#eeeeee"><td><center>1</td></center>	<td><center>20</td></center>	<td><center>FL10</td></center>	<td><center>L</td></center>	<td><center>H1</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>2</td></center>	<td><center>2</td></center>	<td><center>FL11</td></center>	<td><center>L</td></center>	<td><center>H2</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>3</td></center>	<td><center>5</td></center>	<td><center>FL12</td></center>	<td><center>L</td></center>	<td><center>H3</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>4</td></center>	<td><center>63</td></center>	<td><center>FL13</td></center>	<td><center>L</td></center>	<td><center>H4</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>5</td></center>	<td><center>6</td></center>	<td><center>FL14</td></center>	<td><center>L</td></center>	<td><center>K2</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>6</td></center>	<td><center>10</td></center>	<td><center>FL15</td></center>	<td><center>L</td></center>	<td><center>Q2</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>7</td></center>	<td><center>52</td></center>	<td><center>FL2</td></center>	<td><center>L</td></center>	<td><center>E1</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>8</td></center>	<td><center>53</td></center>	<td><center>FL3</td></center>	<td><center>L</td></center>	<td><center>E2</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>9</td></center>	<td><center>42</td></center>	<td><center>FL4</td></center>	<td><center>L</td></center>	<td><center>E3</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>10</td></center>	<td><center>31</td></center>	<td><center>FL5</td></center>	<td><center>L</td></center>	<td><center>E4</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>11</td></center>	<td><center>14</td></center>	<td><center>FL6</td></center>	<td><center>L</td></center>	<td><center>F1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>12</td></center>	<td><center>48</td></center>	<td><center>FL7</td></center>	<td><center>L</td></center>	<td><center>F2</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>13</td></center>	<td><center>60</td></center>	<td><center>FL8</td></center>	<td><center>L</td></center>	<td><center>F3</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>14</td></center>	<td><center>54</td></center>	<td><center>FL9</td></center>	<td><center>L</td></center>	<td><center>F4</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>15</td></center>	<td><center>35</td></center>	<td><center>FR10</td></center>	<td><center>R</td></center>	<td><center>K3</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>16</td></center>	<td><center>11</td></center>	<td><center>FR11</td></center>	<td><center>R</td></center>	<td><center>O1</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>17</td></center>	<td><center>21</td></center>	<td><center>FR12</td></center>	<td><center>R</td></center>	<td><center>O2</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>18</td></center>	<td><center>23</td></center>	<td><center>FR13</td></center>	<td><center>R</td></center>	<td><center>Q1</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>19</td></center>	<td><center>15</td></center>	<td><center>FR14</td></center>	<td><center>R</td></center>	<td><center>Q3</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>20</td></center>	<td><center>4</td></center>	<td><center>FR15</td></center>	<td><center>R</td></center>	<td><center>Q4</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>21</td></center>	<td><center>41</td></center>	<td><center>FR2</td></center>	<td><center>R</td></center>	<td><center>A2</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>22</td></center>	<td><center>44</td></center>	<td><center>FR3</td></center>	<td><center>R</td></center>	<td><center>A3</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>23</td></center>	<td><center>37</td></center>	<td><center>FR4</td></center>	<td><center>R</td></center>	<td><center>C1</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>24</td></center>	<td><center>8</td></center>	<td><center>FR5</td></center>	<td><center>R</td></center>	<td><center>C2</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>25</td></center>	<td><center>19</td></center>	<td><center>FR6</td></center>	<td><center>R</td></center>	<td><center>C3</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>26</td></center>	<td><center>40</td></center>	<td><center>FR7</td></center>	<td><center>R</td></center>	<td><center>C4</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>27</td></center>	<td><center>62</td></center>	<td><center>FR8</td></center>	<td><center>R</td></center>	<td><center>D2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>28</td></center>	<td><center>39</td></center>	<td><center>FR9</td></center>	<td><center>R</td></center>	<td><center>D3</td></center>	<td><center>F</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>29</td></center>	<td><center>13</td></center>	<td><center>ML1</td></center>	<td><center>L</td></center>	<td><center>B1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>30</td></center>	<td><center>22</td></center>	<td><center>ML10</td></center>	<td><center>L</td></center>	<td><center>L2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>31</td></center>	<td><center>38</td></center>	<td><center>ML11</td></center>	<td><center>L</td></center>	<td><center>L4</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>32</td></center>	<td><center>43</td></center>	<td><center>ML12</td></center>	<td><center>L</td></center>	<td><center>M1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>33</td></center>	<td><center>58</td></center>	<td><center>ML13</td></center>	<td><center>L</td></center>	<td><center>N2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>34</td></center>	<td><center>7</td></center>	<td><center>ML14</td></center>	<td><center>L</td></center>	<td><center>R1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>35</td></center>	<td><center>30</td></center>	<td><center>ML15</td></center>	<td><center>L</td></center>	<td><center>R3</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>36</td></center>	<td><center>46</td></center>	<td><center>ML3</td></center>	<td><center>L</td></center>	<td><center>G1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>37</td></center>	<td><center>57</td></center>	<td><center>ML4</td></center>	<td><center>L</td></center>	<td><center>G2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>38</td></center>	<td><center>51</td></center>	<td><center>ML5</td></center>	<td><center>L</td></center>	<td><center>I1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>39</td></center>	<td><center>27</td></center>	<td><center>ML6</td></center>	<td><center>L</td></center>	<td><center>I2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>40</td></center>	<td><center>50</td></center>	<td><center>ML7</td></center>	<td><center>L</td></center>	<td><center>J2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>41</td></center>	<td><center>16</td></center>	<td><center>FL1</td></center>	<td><center>L</td></center>	<td><center>O2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>42</td></center>	<td><center>3</td></center>	<td><center>ML9</td></center>	<td><center>L</td></center>	<td><center>L1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>43</td></center>	<td><center>47</td></center>	<td><center>MR10</td></center>	<td><center>R</td></center>	<td><center>R2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>44</td></center>	<td><center>56</td></center>	<td><center>MR11</td></center>	<td><center>R</td></center>	<td><center>S1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>45</td></center>	<td><center>1</td></center>	<td><center>MR12</td></center>	<td><center>R</td></center>	<td><center>S2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>46</td></center>	<td><center>55</td></center>	<td><center>MR13</td></center>	<td><center>R</td></center>	<td><center>T1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>47</td></center>	<td><center>34</td></center>	<td><center>MR14</td></center>	<td><center>R</td></center>	<td><center>U1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>48</td></center>	<td><center>25</td></center>	<td><center>MR15</td></center>	<td><center>R</td></center>	<td><center>U2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>49</td></center>	<td><center>59</td></center>	<td><center>MR2</td></center>	<td><center>R</td></center>	<td><center>J1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>50</td></center>	<td><center>32</td></center>	<td><center>MR3</td></center>	<td><center>R</td></center>	<td><center>M2</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>51</td></center>	<td><center>24</td></center>	<td><center>MR4</td></center>	<td><center>R</td></center>	<td><center>M3</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>52</td></center>	<td><center>12</td></center>	<td><center>MR5</td></center>	<td><center>R</td></center>	<td><center>M4</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>53</td></center>	<td><center>9</td></center>	<td><center>MR6</td></center>	<td><center>R</td></center>	<td><center>N1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>54</td></center>	<td><center>36</td></center>	<td><center>MR7</td></center>	<td><center>R</td></center>	<td><center>N3</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>55</td></center>	<td><center>28</td></center>	<td><center>MR8</td></center>	<td><center>R</td></center>	<td><center>P1</td></center>	<td><center>M</td></center></tr>
<tr bgcolor="#eeeeee"><td><center>56</td></center>	<td><center>33</td></center>	<td><center>MR9</td></center>	<td><center>R</td></center>	<td><center>P2</td></center>	<td><center>M</td></center></tr>
</table>
</center></td>
</TR>
</table>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the marker set:</P>

<Blockquote><P>The 56 mice were each genotyped at 309 MIT microsatellite markers distributed across the genome, including the Y chromosome. The genotyping error check routine (Lincoln and Lander, 1992) implemented within <a href="http://biostat.jhsph.edu/~kbroman/qtl" target="_blank" class="normal">R/qtl</a> (Broman et al., 2003) showed no likely errors at p <.01 probability. Initial genotypes were generated at OHSU. Approximately 200 genotypes were generated at UTHSC by Jing Gu and Shuhua Qi.
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a target="_blank" class="normal" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx" target="_blank" class="normal">GCOS</a> are the 75% quantiles from a set of 91 pixel values per cell. Probe values were processed as follows:

<UL>
<LI>Step 1: We added an offset of 1 to the CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log2 of each probe signal.

<LI>Step 3: We computed the Z score for each signal within array.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6: We computed the arithmetic mean of the values for the set of microarrays for each of the individual strains.
</UL>

<Blockquote><P>
<B>Probe set data: </B>The uncorrected, untransformed CEL files were subject to probe (low) level processing using both the RMA (Robust Multiarray Average; Irizarry et al. 2003) and PDNN (Position Dependent Nearest Neighbor; Zhang et al. 2003) methods because these two performed the best of four methods tested in a recent four inbred strain comparison using the M430A chip on whole brain samples (Hitzemann et al, submitted). RMA was implemented by the Affy package (11/24/03 version) within Bioconductor (<a href="http://www.bioconductor.org" class="normal">http://www.bioconductor.org</a>) and PDNN by the PerfectMatch v. 2.1 program from Li Zhang (<a href="http://odin.mdacc.tmc.edu/~zhangli/PerfectMatch" class="normal">PDNN </a>).  For sake of comparison with other data sets, MAS 5 files have also been generated.

<P>To better compare data sets, the same simple steps (1 through 6 above) were applied to PDNN and RMA values. Every microarray data set therefore has a mean expression of 8 units with a standard deviation of 2 units. A 1-unit difference therefore represents roughly a 2-fold difference in expression level. Expression levels below 5 are usually close to background noise levels. </P>


</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the chromosome and megabase position values:</P>

<Blockquote>The chromosomal locations of M430A and M430B probe sets were determined by <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse" target="_blank" class="normal">BLAT</A> analysis of concatenated probe sequences using the Mouse Genome Sequencing Consortium May 2004 (mm5) assembly. This BLAT analysis is performed periodically by Yanhua Qu as each new build of the mouse genome is released. We thank Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis. It is possible to confirm the BLAT alignment results yourself simply by clicking on the <B>Verify</B> link in the Trait Data and Editing Form (right side of the <B>Location</B> line).
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote><P>This project was supported by two Department of Veterans Affairs Merit Review Awards (to JK Belknap and R Hitzemann, respectively), AA10760 (Portland Alcohol Research Center), AA06243, AA13484, AA11034, DA05228 and MH51372.
</P> <P>
Please contact either <A HREF="mailto:belknajo@ohsu.edu">John Belknap </A> or <A HREF="mailto:hitzeman@ohsu.edu">Robert Hitzemann </A> at the Dept. of Behavioral Neuroscience, Oregon Health & Science University (L470), or Research Service (R&D5), Portland VA Medical Ctr., Portland, OR 97239 USA.
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;References:</P>
<Blockquote><P>Hitzemann, R, McWeeney, S, Harrington, S, Malmanger, B, Lawler, M, Belknap, JK (2004) Brain gene expression among four inbred mouse strains: The development of an analysis strategy for the integration of QTL and gene expression data. Submitted.
</P></Blockquote>
<Blockquote><P>Irizarry, RA, Bolstad, BM, Collin, F, Cope, LM, Hobbs, B, Speed, TP (2003) Summaries of Affymetrix GeneChip probe level data. Nuc Acids Res 31:1-15.
</P></Blockquote>
<Blockquote><P>Lincoln, SE, Lander, ES (1992) Systematic detection of errors in genetic linkage data. Genomics 14:604-610.
</P></Blockquote>
<Blockquote><P>Zhang, L, Miles, MF, Aldape, KD (2003) A model of molecular interactions on short oligonucleotide microarrays. Nat Biotech 21:818-821.
</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>This text file was originally generated by John Belknap, March 2004. Updated by RWW, October 31, 2004.
</P></Blockquote>
</TD>
</TR>
<TR>
<TD colspan=2 align=center bgColor=#ddddff>



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>

// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure

new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
