<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>M430 Microarray SetAB January04 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">

<P class="title">SJUT Cerebellum mRNA M430 (Oct04) PDNN

 <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=45">GN45</A></P>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>
The October 2004 freeze provides estimates of mRNA expression in cerebellum of adult BXD recombinant inbred mice measured using Affymetrix M430 short oligomer microarrays. Data were generated by a small consortium of investigators at St. Jude Children's Research Hospital (SJ) and the University of Tennessee Health Science Center (UT). Cerebellar samples from BXD strains were hybridized in small pools (n=3) to 430A and 430B arrays. Data were processed using the Position-Dependent Nearest Neighbor (<a href="http://odin.mdacc.tmc.edu/~zhangli/PerfectMatch/" target="_blank" class="fs14">PDNN</a>) method developed by Zhang and colleagues (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=12794640" target="_blank" class="fs14">2003</A>). To simplify comparison between transforms, PDNN values of each array were adjusted to an average of 8 units and a variance of 2 units.</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>We have exploited a set of BXD recombinant inbred strains. The parental strains from which all BXD lines are derived are C57BL/6J (B6 or B) and DBA/2J (D2 or D). Both B and D strains have been almost fully sequence (8x coverage for B6 by a public consortium and approximately 1.5x coverage for D by <a href="http://www.celeradiscoverysystem.com/index.cfm" target="_blank" class="fs14">Celera Discovery Systems</a>). Chromosomes of the two parental strains have been recombined and fixed randomly in the many different BXD strains. BXD2 through BXD32 were produced by Benjamin A. Taylor starting in the late 1970s. BXD33 through 42 were also produced by Taylor, but they were generated in the 1990s. These strains are all available from The Jackson Laboratory, Bar Harbor, Maine. BXD43 through BXD99 were produced by Lu Lu, Jeremy Peirce, Lee M. Silver, and Robert W. Williams in the late 1990s and early 2000s using advanced intercross progeny (Peirce et al. <a href="http://www.biomedcentral.com/1471-2156/5/7"   target="_blank" class="fs14">2004</a>).</P>

<P>In this mRNA expression data set we generally used progeny of stock obtained from The Jackson Laboratory between 1999 and 2001. Animals were generated in-house at the University of Alabama by John Mountz and Hui-Chen Hsu and at the University of Tennessee Health Science Center by Lu Lu and Robert Williams.
</P></Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>

<Blockquote><P>The October04 data set was processed in two large batches. The first batch (the May 2003 data set) consists of samples from 20 samples and 20 strains run on pairs of Affymetrix 430A and 430B arrays (40 arrays total). The second batch consists of 29 samples, included may biological replicates, 2 technical replicates, and data for 9 new strains. Each individual array experiment involved a pool of whole cerebellum taken from three adult animals of the same age and sex. The age range may look broad, but translated into human terms corresponds to a range from about 20 years to 50 years.</P>

<P>RNA was extracted at UTHSC by Lu Lu, Zhiping Jia, and Hongtao Zhai.</P>

<P>All samples were subsequently processed at the <A HREF="http://www.hartwellcenter.org/bio_services/cact/affy/index.php" target="_blank" class="fs14">Hartwell Center</A> Affymetrix laboratory at SJCRH by Jay Morris.</P>

</Blockquote>

<Blockquote>
The table below summarizes informaton on strain, sex, age, sample name, and batch number.</Blockquote>


<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="50%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="5" cellspacing="1">
<tr bgcolor="royalblue"><td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>Sex</font></td>
<td><font color=#FFFFFF>Age</font></td>
<td><font color=#FFFFFF>SampleID</font></td><td><font color=#FFFFFF>Batch</font></td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>M</td><td>127</td><td>766-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>M</td><td>94</td><td>S347-1C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>F</td><td>116</td><td>773-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>M</td><td>109</td><td>S054-1C2</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>F</td><td>71</td><td>S175-1C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>F</td><td>91</td><td>782-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD1</td><td>F</td><td>57</td><td>813-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD2</td><td>F</td><td>142</td><td>751-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD2</td><td>F</td><td>78</td><td>774-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD5</td><td>F</td><td>56</td><td>802-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD5</td><td>M</td><td>71</td><td>752-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD6</td><td>F</td><td>92</td><td>719-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD8</td><td>F</td><td>72</td><td>S173-1C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD9</td><td>M</td><td>86</td><td>737-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD11</td><td>F</td><td>441</td><td>S200-1C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD11</td><td>M</td><td>92</td><td>790-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD12</td><td>F</td><td>130</td><td>776-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD12</td><td>M</td><td>64</td><td>756-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD14</td><td>F</td><td>190</td><td>794-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD14</td><td>M</td><td>91</td><td>758-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD16</td><td>F</td><td>163</td><td>750-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD19</td><td>F</td><td>61</td><td>772-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD21</td><td>F</td><td>116</td><td>711-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD21</td><td>M</td><td>64</td><td>803-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD22</td><td>F</td><td>65</td><td>S174-1C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD23</td><td>F</td><td>88</td><td>814-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD24</td><td>F</td><td>71</td><td>805-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD24</td><td>M</td><td>71</td><td>759-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD25</td><td>M</td><td>90</td><td>S429-1C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD28</td><td>F</td><td>113</td><td>785-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD28</td><td>F</td><td>427</td><td>S203-1C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD29</td><td>F</td><td>82</td><td>777-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD29</td><td>M</td><td>76</td><td>714-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD29</td><td>M</td><td>76</td><td>714-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD31</td><td>F</td><td>142</td><td>816-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD32</td><td>F</td><td>62</td><td>778-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD32</td><td>M</td><td>218</td><td>786-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD33</td><td>F</td><td>184</td><td>793-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD33</td><td>M</td><td>124</td><td>715-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD34</td><td>F</td><td>56</td><td>725-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD34</td><td>M</td><td>91</td><td>789-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD38</td><td>F</td><td>55</td><td>781-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD38</td><td>M</td><td>65</td><td>761-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD39</td><td>M</td><td>165</td><td>723-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD40</td><td>F</td><td>56</td><td>718-C1</td><td>1</td></tr>
<tr bgcolor="#eeeeee"><td>BXD40</td><td>F</td><td>56</td><td>718-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD40</td><td>M</td><td>73</td><td>812-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD42</td><td>F</td><td>100</td><td>799-C1</td><td>2</td></tr>
<tr bgcolor="#eeeeee"><td>BXD42</td><td>M</td><td>97</td><td>709-C1</td><td>1</td></tr>

</table>
</td>
</tr>
</table>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>

<Blockquote><B><A HREF="http://www.affymetrix.com/products/arrays/specific/mouse430.affx" class="fs14" target="_blank">Affymetrix Mouse Expression 430 GeneChip set</A>:</B> The expression data were generated using 430A and 430B arrays. Chromosomal positions of probe sets were determined by <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse"  target="_blank" class="fs14">BLAT</a> analysis of concatenated probe sequences using the Mouse Genome Sequencing Consortium May 2004 (mm5) assembly. This BLAT analysis is performed periodically by Yanhua Qu as each new build of the mouse genome is released. We thank Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis. It is possiible to confirm the BLAT alignment results yourself simply by clicking on the <B>Verify</B> link in the Trait Data and Editing Form (right side of the <B>Location</B> line).
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a target="_blank" class="fs14" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.
<UL>
<LI>Step 1: We added a constant offset of 1 to the CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log2 of each cell signal level.

<LI>Step 3: We computed the Z scores for each cell within its array.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8 units, a variance of 4 units, and a standard deviation of 2 units.

<LI>Step 6a: The 430A and 430B arrays include a set of 100 shared probe sets that have identical probe sequences. These 100 probe sets and 2200 probes provide a good way to adjust expression of the two GeneChips to a common scale. The absolute mean expression on the 430B array is almost invariably lower than that on the 430A array (the A array contains the more commonly expressed transcripts). To bring the two arrays into numerical alignment, we regressed Z scores of the common set of 2200 probes to obtain linear regression corrections to rescale the 430B arrays to values that match the 430A array. This involved multiplying all 430B Z scores by the slope of the regression and adding a very small offset (the regression intercept). The result of this adjustment is that the mean of the 430A array expression is fixed at a value of 8, whereas that of the 430B chip is typically 7.

<LI>Step 6b: We recentered the combined set of 430A and 430B transcripts to a mean of 8 and a standard deviation of 2. This involves reapplying Steps 3 through 5 above but now using the entire set of probes and probe sets from a merged 430A and B data set.

<LI>Step 7: We corrected for technical variance introduced by running two large batches. Individual probe means for the two batches (n = 20 and 29 samples, respectively) were calcuated separately. Probe values of the smaller batch (1) were then adjusted by multiplying batch 2 probe estimates by the Batch_2/Batch_1 ratio of the averages for that probe.

<LI>Step 8: Finally, we compute the arithmetic mean of the values for the set of microarrays for each strain. In this data set we have only a very modest number of replicates and for this reason we do not yet provide error terms for transcripts or probes. Note, that this data set does not provide any correction for variance introduced by differences in sex, age, tissue source, or any interaction terms. We have not corrected for background beyond the background correction implemented by Affymetrix in generating the CEL file. We expect to add statistical controls and adjustments for these variables in subsequent versions of WebQTL.
</UL>


</B><B>Probe set data: </B>The expression data were generated by PDNN method. The original expression values in CEL files were read into the <a href="http://odin.mdacc.tmc.edu/~zhangli/PerfectMatch/" target="_blank" class="fs14">PerfectMatch</a>. There were normalized using the PDNN method of background correction and normalization. The same simple steps described above were also applied to these values. A 1-unit difference represents roughly a two-fold difference in expression level. Expression levels below 5 are usually close to background noise levels.
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>
Data were generated with funds contributed equally by The UTHSC-SJCRH Cerebellum Transcriptome Profiling Consortium. Our members include:<UL>
<LI>Tom Curran
<LI>Dan Goldowitz
<LI>Kristin Hamre
<LI>Lu Lu
<LI>Peter McKinnon
<LI>Jim Morgan
<LI>Clayton Naeve
<LI>Richard Smeyne
<LI>Robert Williams
<LI>The Center of Genomics and Bioinformatics at UTHSC
<LI>The Hartwell Center at SJCRH
</UL>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>
This text file originally generated by RWW and YHQ, September 2004. Updated by RWW, October 30, 2004.
</P></Blockquote>


</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
