<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>M430 Microarray brain February04 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">INIA M430 brain RMA Database (February/04 Freeze)  <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=43">GN43</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the mice used to map microarray data:</P>

<Blockquote>We have exploited a set of BXD recombinant inbred strains. The parental strains from which all BXD lines are derived are C57BL/6J (B6 or B) and DBA/2J (D2 or D). Both B and D strains have been almost fully sequence (8x coverage for B6 by a public consortium and approximately 1.5x coverage for D by <a href="http://www.celeradiscoverysystem.com/index.cfm" target="_blank" class="fs14">Celera</a>). Chromosomes of the two parental strains have been recombined and fixed randomly in the many different BXD strains. BXD lines 2 through 32 were produced by Dr. Benjamin Taylor starting in the late 1970s. BXD33 through 42 were also produced by Dr. Taylor, but they were generated in the 1990s. All of these strains are available from The <a href="http://www.jax.org" target="_blank" class="fs14">Jackson Laboratory</a>. Lines such as BXD43, BXD67, BXD68, etc. are BXD recombinant inbred strains that are part of a large set produced by Drs. Lu Lu and Jeremy Peirce. There are approximately 45 of these new BXD strains. For additional background on recombinant inbred strains please see Peirce et al. <a href="http://www.biomedcentral.com/1471-2156/5/7"  class="fs14">2004</a>.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>
<Blockquote>The <a href="http://www.iniastress.org" target="_blank" class="fs14">INIA</a> M430 brain Database (February04) consists of 30 Affymetrix MOE 430A and MOE430B GeneChip microarray pairs. Each AB pair of arrays was hybridized in sequence (A array first, B array second) with a pool of brain tissue (forebrain minus olfactory bulb, plus the entire midbrain) taken from three adult animals of closely matched age and the same sex. RNA was extracted at UTHSC by Lu Lu, Zhiping Jia, and Hongtao Zhai. All samples were subsequently processed in the INIA Bioanalytical Core at the W. Harry Feinstone Center of Excellence by Thomas R. Sutter and colleagues at the  University of Memphis. Before running the main batch of 30 pairs of array, we ran four "test" samples (one male and one female pool from each of the two parental strains, C57BL/6J and DBA/2J). The main batch of 30 array pairs includes the same four samples (in other words we have four technical replicates between the test and the main batches), two F1 hybrid sample (each run two times for a within-batch technical replication), and 22 BXD strains. The February04 data set therefore consists of one male and one female pool from C57BL/6J, DBA/2J, the B6D2F1 hybrid, 11 female BXD samples, and 11 male BXD samples. We should note that the four technical replicates between batches were eventually combined with a correction for a highly significant batch effect. This was done at both the probe and probe set levels to "align" the test batch values with the two main batches. (The ratio of the probe average in the four test arrays to the average of the same probe in the four corresponding main batch arrays was used as a correction factor.)  The F1 within-batch technical replicates were simply averaged. In the next batch we will reverse the sex of the BXD samples to achieve a balance with at least 22 BXD strains with one male and one female sample each.
</Blockquote>
<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="5" cellspacing="1">
<tr bgcolor="royalblue"><td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>Sex</font></td>
<td><font color=#FFFFFF>Age</font></td>
<td><font color=#FFFFFF>Sample_name</td>
<td><font color=#FFFFFF>Result date</font></td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>F</td><td>127</td><td>919-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>F</td><td>127</td><td>919-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>M</td><td>127</td><td>920-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>M</td><td>127</td><td>920-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>F</td><td>65</td><td>903-F1</td><td>Nov03</td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>F</td><td>65</td><td>903-F2</td><td>Jan03</b></td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>M</td><td>66</td><td>906-F1</td><td>Nov03</td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>M</td><td>66</td><td>906-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>F</td><td>60</td><td>917-F1</td><td>Nov03</td></b></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>F</td><td>60</td><td>917-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>M</td><td>60</td><td>918-F1</td><td>Nov03</td></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>M</td><td>60</td><td>918-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD1</td><td>F</td><td>95</td><td>895-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD5</td><td>M</td><td>71</td><td>728-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD6</td><td>M</td><td>92</td><td>902-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD8</td><td>F</td><td>72</td><td>S167-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD9</td><td>M</td><td>86</td><td>909-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD12</td><td>M</td><td>64</td><td>897-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD13</td><td>F</td><td>86</td><td>748-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD14</td><td>M</td><td>91</td><td>912-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD18</td><td>F</td><td>108</td><td>771-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD19</td><td>F</td><td>56</td><td>S236-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD21</td><td>F</td><td>67</td><td>740-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD23</td><td>F</td><td>88</td><td>815-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD24</td><td>M</td><td>71</td><td>913-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD25</td><td>F</td><td>74</td><td>S373-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD28</td><td>F</td><td>79</td><td>910-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD29</td><td>F</td><td>76</td><td>693-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD32</td><td>F</td><td>93</td><td>898-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD33</td><td>M</td><td>77</td><td>915-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD34</td><td>M</td><td>72</td><td>916-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD36</td><td>M</td><td>77</td><td>926-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD38</td><td>M</td><td>69</td><td>731-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD42</td><td>M</td><td>97</td><td>936-F1</td><td>Jan04</td></tr>

</table>
</td>
</tr>
</table>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the .CEL file: </B>These .CEL values produced by <a target="_blank" class="fs14" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.
<UL>
<LI>Step 1: We added an offset of 1.0 to the .CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log base 2 of each cell.

<LI>Step 3: We computed the Z scores for each cell.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6a: The 430A and 430B GeneChips include a set of 100 shared probe sets (2200 probes) that have identical sequences. These probes and probe sets provide a way to calibrate expression of the two GeneChips to a common scale. The absolute mean expression on the 430B array is almost invariably lower than that on the 430A array. To bring the two arrays into alignment, we regressed Z scores of the common set of probes to obtain a linear regression corrections to rescale the 430B arrays to the 430A array. In our case this involved multiplying all 430B Z scores by the slope of the regression and adding or subtracting a very small offset. The result of this step is that the mean of the 430A GeneChip expression is fixed at a value of 8, whereas that of the 430B chip is typically 7. Thus average of A and B arrays is approximately 7.5.

<LI>Step 6b: We recenter the whole set of 430A and B transcripts to a mean of 8 and a standard deviation of 2. This involves reapplying Steps 3 through 5 above but now using the entire set of probes and probe sets from a merged 430A and B data set.

<LI>Step 7: We corrected for technical variance introduced by two batches. Means separated by tow batchs for each gene are corrected same with the data of two common strains in these two batches.

<LI>Step 8: Finally, we compute the arithmetic mean of the values for the set of microarrays for each strain. In this data set we have modest numbers of replicates and for this reason we do not yet provide error terms for transcripts or probes. Note, that we have not (yet) corrected for variance introduced by differences in sex, age,  or any interaction terms. We have not corrected for background beyond the background correction implemented by Affymetrix in generating the .CEL file. We expect to add statistical controls and adjustments for these variables in subsequent versions of WebQTL.

</UL>
</B><Blockquote><B>Probe set data: </B>These expression data
were generated by RMA method. The raw expression values in .CEL files were read into the R environment (Ihaka a
nd Gentleman, 1996). These were normalized using the <a target="_blank" href="http://www.bioconductor.org" class="fs14">RMA</a> method of background correction and normalization (Irrizary et al, 2003). The same simple steps described above were also applied to these values. A 1-unit difference represents roughly a two-fold difference in expression level. Expression levels below 5 are usually close to background noise levels.
</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the chromosome and megabase position values:</P>

<Blockquote>The chromosomal locations of probe sets and gene markers on the 430A and 430B microarrays were determined by BLAT analysis using the Mouse Genome Sequencing Consortium Oct 2003 (mm4) Assembly (see <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse" class="fs14">http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse</a>). We thank Dr. Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis.</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>
Data were generated with funds from multiple data sources including NIAAA INIA support to RWW and Thomas Sutter, an NIMH Human Brain Project, and the Dunavant Chair of Excellence, University of Tennessee Health Science Center. All arrays were processed at the University of Memphis by Dr. Thomas Sutter and colleagues with support of the INIA Bioanalytical Core.
</Blockquote>

<P></P>

</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
