<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>HEIMED M430 Microarray Eye RMA November05 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">

Hamilton Eye Institute Mouse Eye M430v2 (November05) RMA Data Set <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=93">GN93</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote>
<P>
SUPERCEDED EYE DATASET. The HEIMED November 2005 data set provides estimates of mRNA expression in whole eyes of 63 lines of mice without significant biological replication. Data were generated at UTHSC with support from a grant from Dr. Barrett Haik, Director of the Hamilton Eye Institute (<a href="http://www.eye.utmem.edu/cvr/hei.html" target="_blank" class="fs14">HEI</a>). Pooled RNA samples were hybridized to Affymetrix M430 2.0 arrays. This particular data set was processed using the <a href="http://stat-www.berkeley.edu/users/bolstad/RMAExpress/RMAExpress.html" target="_blank" class="normal">RMA</a> protocol. To simplify comparison among different transforms, RMA values of each array have been adjusted to an average expression of 8 units and a standard deviation of 2 units.</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>
We have used a set of 14 conventional inbred strains, reciprocal F1s between C57BL/6J (B6 or B)  and DBA/2J D2 (or D), and 47 BXD recombinant inbred strains. The BXD strains were generated by crossing C57BL/6J with DBA/2J. The BXDs are particularly useful for systems genetics because both parental strains have been sequenced (8x coverage of B6 and 1.5x coverage of D). Physical maps in WebQTL incorporate approximately 2 million B vs D SNPs from <a href="http://www.celeradiscoverysystem.com/index.cfm" target="_blank" class="fs14">Celera Genomics</a> and from the Perlegen-NIEHS sequencing effort. BXD1 through BXD32 were bred by Benjamin A. Taylor starting in the late 1970s. BXD33 through 42 were bred by Taylor in the 1990s. These strains are available from The Jackson Laboratory.

<P>Please note that BXD24/TyJ (JAX stock number 000031) used in this study is also known as BXD24b/TyJ and has complete retinal degeneration.  BXD24a/TyJ, a 1988 F80 stock that has now been rederived, does not have retinal degeneration (stock number 005243) and is an ideal coisogenic control.

<P>BXD43 through BXD99 were bred by Lu Lu, Jeremy Peirce, Lee M. Silver, and Robert W. Williams in the late 1990s and early 2000s using advanced intercross progeny (Peirce et al. <a href="http://www.biomedcentral.com/1471-2156/5/7"   target="_blank" class="fs14">2004</a>). Many of the 50 new BXD strains are available from <A HREF="mailto:lulu@nb.utmem.EDU" target="_blank" class="fs14">Lu Lu</A> and colleagues.</P>




<P><B>Mouse Diversity Panel (MDP)</B>. In addition to the BXD strains, we have profiled a MDP consisting 14 inbred strains and a pair of reciprocal F1 hybrids; B6D2F1 and D2B6F1. These strains were selected for several reasons:
<UL>
<LI> genetic and phenotypic diversity, including use by the <A HREF="http://aretha.jax.org/pub-cgi/phenome/mpdcgi?rtn=strains%2Flist&listmode=pri&typemode=all&genaltmode=no&submit1=%A0%A0Go%A0%A0" target="_empty" class="fs14">Phenome Project</A>
<LI> their use in making genetic reference populations including recombinant inbred strains, cosomic strains, congenic and recombinant congenic strains
<LI> their use by the <A HREF="http://www.complextrait.org" target="_empty" class="fs14">Complex Trait Consortium</A> to make the Collaborative Cross (Nairobi/Wellcome, Oak Ridge/DOE, and Perth/UWA)
<LI> genome sequence data from three sources (NHGRI, Celera, and Perlegen-NIEHS)
<LI> availability from The Jackson Laboratory
</UL>

<P>All eight parents of the Collaborative Cross (129, A, C57BL/6J, CAST, NOD, NZO, PWK, and WSB) have been included in the MDP (noted below in the list). Twelve MDP strains have been sequenced, or are currently being resequenced by Perlegen for the NIEHS. This panel will be extremely helpful in systems genetic analysis of a wide variety of traits, and will be a powerful adjunct in fine mapping modulators using what is essentially an association analysis of sequence variants.

<OL>
<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/129.shtml" target="_empty" class="fs14">129S1/SvImJ </A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; background for many knockouts (R1 ES cell line); Phenome Project A list. This strain (JAX No 002448, aka 129S1/Sv-++<Tyr-c>Kitl<Sl-J>/+) also carries hypopigmentation mutations (white bellied chinchilla) of the tyrosinase gene on Chr 7 and a mutant allele of the steel (Kitl) gene.

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/A.shtml" target="_empty" class="fs14">A/J</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by Perlegen/NIEHS; parent of the AXB/BXA panel. A tyrosinase-negative albino (c) mutant


<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/BALB.shtml" target="_empty" class="fs14">BALB/cByJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp; Sequenced by NIEHS; maternal parent of the CXB panel; Phenome Project A list. A tyrosinase-negative albino (c) mutant


<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C3H.shtml" target="_empty" class="fs14">C3H/HeJ</A>      <BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS; paternal parent of the BXH panel; Phenome Project A list. Important to note for this Eye dataset, C3H/HeJ is a <I>Pdeb6</I> mutant with near total photoreceptor loss at maturity.

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C57BL.shtml" target="_empty" class="fs14">C57BL/6J</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by NHGRI; parental strain of AXB/BXA, BXD, and BXH; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/CAST.shtml" target="_empty" class="fs14">CAST/Ei</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/DBA.shtml" target="_empty" class="fs14">DBA/2J</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS and Celera; paternal parent of the BXD panel; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/KK.shtml" target="_empty" class="fs14">KK/HlJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/LG.shtml" target="_empty" class="fs14">LG/J</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Paternal parent of the LGXSM panel

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/NOD.shtml" target="_empty" class="fs14">NOD/LtJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project B list; diabetic

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/NZO.shtml" target="_empty" class="fs14">NZO/HILtJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/PWD.shtml" target="_empty" class="fs14">PWD/PhJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS; parental strain for a consomic set by Forjet and colleagues

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/PWK.shtml" target="_empty" class="fs14">PWK/PhJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain; Phenome Project D list

<LI><A HREF="   " target="_empty" class="fs14">WSB/EiJ</A>   <BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project C list

<LI>B6D2F1 and D2B6F1, aka F1 in some graphs and tables
<BR>F1 hybrids generated by crossing C57BL/6J with DBA/2J
</OL>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate this set of data:</P>

<Blockquote>
<P><B>Tissue preparation protocol</B>. Animal were killed by rapid cervical dislocation. Eyes were removed immediately and placed in RNA<I>later</I> at room temperature. Usually six eyes from animals with a common sex, age, and strain were stored in a single tube.

<P>Each array was hybridized with a pool of RNA from 4 to 8 eyes from 2 to 4 animals. RNA was extracted at UTHSC by Zhiping Jia. If tissue was saved for RNA extraction at a later time, eyes were placed directly in RNA<i>later</i> (Ambion, Inc.) and treated per the manufacturer’s directions.  If eyes were used for immediate RNA extraction then we proceeded immediately to the next steps.<p>

Dissecting and preparing eyes for RNA extraction
<OL>
<LI>Place eyes for RNA extraction in RNA STAT-60 (Tel-Test Inc.) and process per manufacturer’s instructions (in brief form below).
<LI>Store RNA in 75% ethanol at –80 deg. C until use.
</OL>

<P>Total RNA was extracted with RNA STAT-60 (Tel-Test Inc.) according to the manufacturer's instructions. Briefly we:
<OL>
<LI>homogenize tissue samples in the RNA STAT-60 (1 ml/50 to 100 mg tissue)
<LI>allowed the homogenate to stand for 5 min at room temperature
<LI>added 0.2 ml of chloroform per 1 ml RNA STAT-60
<LI>shook the sample vigorously for 15 sec and let the sample sit at room temperature for 3 min
<LI>centrifuged at 12,000 G for 15 min
<LI>transfered the aqueous phase to a fresh tube
<LI>added 0.5 ml of isopropanol per 1 ml RNA STAT-60
<LI>vortexed and allowed sample to stand at room temperature for 5-10 min
<LI>centrifuged at 12,000 G for 10-15 min
<LI>removed the supernatant and washed the RNA pellet with 75% ethanol
<LI>stored the pellet in 75% ethanol at -80 deg C until use
</OL>


<P><B>Sample Processing</B>. Samples were processed in the VA Medical Center, Memphis, Rheumatology Disease Research Core Center lead by Drs. John Stuart and Weikuan Gu. All processing steps were performed by Dr. Yan Jiao. In brief, samples were purified using a standard sodium acetate in alcohol method (recommended by Affymetrix). The RNA quality was checked using a 1% agarose gel. The 18S and 28S bands had to be clear and the 28S band had to be more prominent. RNA concentation was measured using a spectrophotometer. The 260/280 ratios had to be greater than 1.7, and the majority were 1.8 or higher. We used a total of 8 micrograms of RNA as starting amount for cDNA synthesis using a standard Eberwine T7 polymerase method (Superscript II RT, Invitrogen Inc., Affy Part No 900431, GeneChip Expression 3' Amplification One-Cyle cDNA Synthesis Kit). The Affymetrix IVT labeling kit (Affy 900449) was used to generate labeled cRNA. At this point the cRNA was evaluated again using both the 260/280 ratio (values of 2.0 or above were acceptable) and 1% agarose gel inspection of the product (a size range from 200 to 7000 bp is considered suitable for use). We used 45 micrograms of labeled cRNA for fragmentation. Those samples that passed both QC steps (<10% usually fail) were then sheared using a fragmentation buffer included in the Affymetrix GeneChip Sample Cleanup Module (Part No.900371). After fragmentation, samples were either stored at -80 deg. centrigrade until use (one third roughly) or were immediately used for hybridization.

<P><B>Dealing with Ocular Pigmentation</B>: Variable ocular pigmentation is a potential confound in a study of the whole eye transcriptome. Even the most careful RNA preparations of brown and beige colored mice tend to have faint residual pigmentation that does affect hybridization signal. The key determinant of this interesting effect is the <I>Tyrp1</I> (brown) locus on Chr 5 at about 80 Mb. Loci on Chr 4 that map at this location should be considered with skepticism and reviewed carefully. To address this problem Yan Jiao purified total RNA a second time using the Qiagen RNeasy MinElute Cleanup Kit (Cat No. 74204). This was done for 8 colored samples (R2534E2, R2578E2, R1441E2, R2537E2, R2536E2, R2589E2, R2539E2), of which 5 were finally included in this data set (cases in which the strain ID is labeled with asterisks in the table).


<P><B>Replication and Sample Balance:</B> Our goal is to obtain data for independent biological sample pools from at least one of sample from each sex for all BXD strains. The current first batch of array data, represents a balanced sample of males and females, but without within-strain replication. We expect to add roughly 100 additional samples inthe next few months.

<P><B>Batch Structure:</B> This data set consists of a single batch. The great majority of arrays are from a single lot.

<P>The table below summarizes information on strain, age, sex, original CEL filename, several quality control values, and source of mice. Columns labeled "PDNN 2Z outlier" and "RMA 2Z outlier" list the fraction of probe sets with values that deviated more than 2 z units from the mean. Scale factor, background average, present, absent, marginal and 3'/5' ratios for actin and Gapdh were collated from the Affymetrix Report (RPT) files.
</Blockquote>


<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="7" cellspacing="1">
<tr bgcolor="royalblue"><td><font color=#FFFFFF>Id</font></td><td><font color=#FFFFFF>tube ID</font></td><td><font color=#FFFFFF>group type</font></td><td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>age</font></td>
<td><font color=#FFFFFF>sex</font></td>
<td><font color=#FFFFFF>original CEL filename</td>
<td><font color=#FFFFFF>PDNN 2Z outlier</font></td>
<td><font color=#FFFFFF>RMA 2Z outlier</font></td>
<td><font color=#FFFFFF>scale factor</td>
<td><font color=#FFFFFF>background average</font></td>
<td><font color=#FFFFFF>present</font></td>
<td><font color=#FFFFFF>absent</td>
<td><font color=#FFFFFF>marginal</font></td>
<td><font color=#FFFFFF>AFFX-b-ActinMur(3'/5')</font></td>
<td><font color=#FFFFFF>AFFX-GapdhMur(3'/5')</td>
<td><font color=#FFFFFF>source</td></tr>
<tr bgcolor="#eeeeee"><td>1</td><td>R2607E1</td><td>GDP BXD</td><td>C57BL/6J</td><td>67</td><td>F</td><td>R2605E.CEL</td><td>0.005</td><td>0.009</td><td>2.428</td><td>115.12</td><td>0.586</td><td>0.4</td><td>0.014</td><td>1.31</td><td>0.76</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>2</td><td>R0872E2</td><td>GDP BXD</td><td>C57BL/6J</td><td>66</td><td>M</td><td>R0872E.CEL</td><td>0.013</td><td>0.012</td><td>3.128</td><td>88.58</td><td>0.589</td><td>0.396</td><td>0.015</td><td>1.3</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>3</td><td>R2572E1</td><td>GDP BXD</td><td>DBA/2J</td><td>65</td><td>M</td><td>R2572E.CEL</td><td>0.041</td><td>0.051</td><td>2.406</td><td>79.07</td><td>0.555</td><td>0.429</td><td>0.016</td><td>1.37</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>4</td><td>R2601E1</td><td>GDP BXD</td><td>B6D2F1</td><td>73</td><td>F</td><td>R2601E.CEL</td><td>0.003</td><td>0.004</td><td>2.545</td><td>91.96</td><td>0.589</td><td>0.396</td><td>0.015</td><td>1.44</td><td>0.78</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>5</td><td>R2602E1</td><td>GDP BXD</td><td>B6D2F1</td><td>73</td><td>M</td><td>R2602E.CEL</td><td>0.001</td><td>0.004</td><td>2.599</td><td>84.44</td><td>0.597</td><td>0.388</td><td>0.015</td><td>1.37</td><td>0.78</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>6</td><td>R2600E1</td><td>GDP BXD</td><td>D2B6F1</td><td>72</td><td>F</td><td>R2600E.CEL</td><td>0.003</td><td>0.008</td><td>2.47</td><td>94.75</td><td>0.581</td><td>0.402</td><td>0.017</td><td>1.41</td><td>0.78</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>7</td><td>R2604E1</td><td>GDP BXD</td><td>D2B6F1</td><td>69</td><td>M</td><td>R2604E.CEL</td><td>0.003</td><td>0.007</td><td>2.657</td><td>89.63</td><td>0.594</td><td>0.392</td><td>0.015</td><td>1.28</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>8</td><td>R2597E1</td><td>BXD</td><td>BXD2</td><td>61</td><td>M</td><td>R2597E.CEL</td><td>0.003</td><td>0.007</td><td>2.374</td><td>93.56</td><td>0.603</td><td>0.383</td><td>0.015</td><td>1.34</td><td>0.77</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>9</td><td>R2591E1</td><td>BXD</td><td>BXD5</td><td>60</td><td>F</td><td>R2591E.CEL</td><td>0.051</td><td>0.009</td><td>1.7</td><td>136.48</td><td>0.585</td><td>0.4</td><td>0.015</td><td>1.33</td><td>0.78</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>10</td><td>R2570E1</td><td>BXD</td><td>BXD6</td><td>65</td><td>F</td><td>R2570E.CEL</td><td>0.002</td><td>0.006</td><td>1.987</td><td>86.73</td><td>0.585</td><td>0.4</td><td>0.015</td><td>1.46</td><td>0.76</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>11</td><td>R2538E1</td><td>BXD</td><td>BXD8</td><td>77</td><td>F</td><td>R2538E.CEL</td><td>0.037</td><td>0.028</td><td>1.905</td><td>101.98</td><td>0.612</td><td>0.373</td><td>0.015</td><td>1.52</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>12</td><td>R2569E1</td><td>BXD</td><td>BXD9</td><td>67</td><td>M</td><td>R2569E.CEL</td><td>0.014</td><td>0.027</td><td>1.753</td><td>87.36</td><td>0.551</td><td>0.434</td><td>0.015</td><td>2.82</td><td>3.14</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>13</td><td>R2581E1</td><td>BXD</td><td>BXD11</td><td>65</td><td>F</td><td>R2581E.CEL</td><td>0.006</td><td>0.012</td><td>1.941</td><td>88.55</td><td>0.621</td><td>0.364</td><td>0.016</td><td>1.55</td><td>0.81</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>14</td><td>R2543E1</td><td>BXD</td><td>BXD12</td><td>63</td><td>M</td><td>R2543E.CEL</td><td>0.036</td><td>0.007</td><td>1.605</td><td>117.69</td><td>0.586</td><td>0.399</td><td>0.016</td><td>1.43</td><td>0.77</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>15</td><td>R2586E1</td><td>BXD</td><td>BXD13</td><td>60</td><td>F</td><td>R2586E.CEL</td><td>0.020</td><td>0.035</td><td>2.006</td><td>73.61</td><td>0.564</td><td>0.42</td><td>0.016</td><td>2.85</td><td>3.81</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>16</td><td>R2557E1</td><td>BXD</td><td>BXD14</td><td>60</td><td>F</td><td>R2557E.CEL</td><td>0.014</td><td>0.017</td><td>1.83</td><td>98.76</td><td>0.625</td><td>0.361</td><td>0.014</td><td>1.31</td><td>0.78</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>17</td><td>R2567E1</td><td>BXD</td><td>BXD16</td><td>60</td><td>M</td><td>R2567E.CEL</td><td>0.016</td><td>0.025</td><td>2.239</td><td>82.35</td><td>0.567</td><td>0.416</td><td>0.017</td><td>1.37</td><td>0.75</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>18</td><td>R2559E1</td><td>BXD</td><td>BXD18</td><td>59</td><td>M</td><td>R2559E.CEL</td><td>0.035</td><td>0.006</td><td>1.654</td><td>103.68</td><td>0.608</td><td>0.377</td><td>0.015</td><td>1.27</td><td>0.78</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>19</td><td>R2560E1</td><td>BXD</td><td>BXD19</td><td>60</td><td>F</td><td>R2560E.CEL</td><td>0.026</td><td>0.007</td><td>1.792</td><td>98.33</td><td>0.609</td><td>0.375</td><td>0.016</td><td>1.35</td><td>0.8</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>20</td><td>R2584E1</td><td>BXD</td><td>BXD20</td><td>59</td><td>F</td><td>R2584E.CEL</td><td>0.003</td><td>0.007</td><td>2.07</td><td>83.82</td><td>0.593</td><td>0.391</td><td>0.016</td><td>1.4</td><td>0.76</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>21</td><td>R2541E2</td><td>BXD</td><td>BXD21</td><td>61</td><td>M</td><td>R2541E2.CEL</td><td>0.049</td><td>0.036</td><td>2.625</td><td>125.08</td><td>0.56</td><td>0.424</td><td>0.015</td><td>1.29</td><td>0.78</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>22</td><td>R2553E1</td><td>BXD</td><td>BXD22</td><td>58</td><td>F</td><td>R2553E.CEL</td><td>0.003</td><td>0.005</td><td>1.952</td><td>111.3</td><td>0.599</td><td>0.385</td><td>0.015</td><td>1.28</td><td>0.76</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>23</td><td>R2558E1</td><td>BXD</td><td>BXD23</td><td>60</td><td>F</td><td>R2558E2.CEL</td><td>0.013</td><td>0.015</td><td>1.908</td><td>114.53</td><td>0.599</td><td>0.388</td><td>0.014</td><td>1.2</td><td>0.82</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>24</td><td>R2589E2</td><td>BXD</td><td>BXD24-rd*</td><td>59</td><td>M</td><td>R2589E2.CEL</td><td>0.098</td><td>0.098</td><td>2.606</td><td>112.19</td><td>0.575</td><td>0.409</td><td>0.016</td><td>1.24</td><td>0.8</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>25</td><td>R2573E1</td><td>BXD</td><td>BXD25</td><td>67</td><td>F</td><td>R2573E2.CEL</td><td>0.009</td><td>0.018</td><td>3.153</td><td>71.88</td><td>0.579</td><td>0.407</td><td>0.014</td><td>1.77</td><td>0.97</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>26</td><td>R2562E1</td><td>BXD</td><td>BXD28</td><td>60</td><td>F</td><td>R2562E.CEL</td><td>0.003</td><td>0.005</td><td>1.649</td><td>116.35</td><td>0.599</td><td>0.384</td><td>0.017</td><td>1.37</td><td>0.79</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>27</td><td>R2561E1</td><td>BXD</td><td>BXD29</td><td>60</td><td>F</td><td>R2561E.CEL</td><td>0.019</td><td>0.029</td><td>1.952</td><td>93.32</td><td>0.583</td><td>0.402</td><td>0.015</td><td>2.19</td><td>1</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>28</td><td>R2598E1</td><td>BXD</td><td>BXD31</td><td>61</td><td>M</td><td>R2598E.CEL</td><td>0.003</td><td>0.006</td><td>1.989</td><td>106.48</td><td>0.609</td><td>0.376</td><td>0.015</td><td>1.27</td><td>0.78</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>29</td><td>R2563E1</td><td>BXD</td><td>BXD32</td><td>63</td><td>F</td><td>R2563E.CEL</td><td>0.008</td><td>0.011</td><td>1.547</td><td>101.52</td><td>0.619</td><td>0.367</td><td>0.014</td><td>1.5</td><td>0.8</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>30</td><td>R2542E1</td><td>BXD</td><td>BXD33</td><td>67</td><td>F</td><td>R2542E.CEL</td><td>0.010</td><td>0.016</td><td>2.128</td><td>97.08</td><td>0.565</td><td>0.418</td><td>0.016</td><td>1.91</td><td>0.93</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>31</td><td>R2585E1</td><td>BXD</td><td>BXD34</td><td>60</td><td>M</td><td>R2585E.CEL</td><td>0.007</td><td>0.014</td><td>2.64</td><td>75.13</td><td>0.583</td><td>0.4</td><td>0.017</td><td>1.25</td><td>0.77</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>32</td><td>R2532E1</td><td>BXD</td><td>BXD38</td><td>62</td><td>M</td><td>R2532E.CEL</td><td>0.002</td><td>0.003</td><td>2.038</td><td>93.65</td><td>0.598</td><td>0.387</td><td>0.015</td><td>1.37</td><td>0.8</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>33</td><td>R2574E1</td><td>BXD</td><td>BXD39</td><td>70</td><td>F</td><td>R2574E.CEL</td><td>0.001</td><td>0.004</td><td>1.981</td><td>90.64</td><td>0.612</td><td>0.373</td><td>0.015</td><td>1.39</td><td>0.78</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>34</td><td>R2590E1</td><td>BXD</td><td>BXD40</td><td>60</td><td>M</td><td>R2590E.CEL</td><td>0.004</td><td>0.007</td><td>2.708</td><td>77.3</td><td>0.591</td><td>0.393</td><td>0.015</td><td>1.4</td><td>0.77</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>35</td><td>R2596E1</td><td>BXD</td><td>BXD42</td><td>59</td><td>M</td><td>R2596E.CEL</td><td>0.013</td><td>0.017</td><td>2.632</td><td>108.46</td><td>0.59</td><td>0.396</td><td>0.015</td><td>1.24</td><td>0.8</td><td>Glenn</td></tr>
<tr bgcolor="#eeeeee"><td>36</td><td>R2605E1</td><td>BXD</td><td>BXD43</td><td>79</td><td>M</td><td>R2607E.CEL</td><td>0.003</td><td>0.006</td><td>1.817</td><td>131.22</td><td>0.605</td><td>0.382</td><td>0.013</td><td>1.32</td><td>0.8</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>37</td><td>R2594E1</td><td>BXD</td><td>BXD44</td><td>63</td><td>F</td><td>R2594E.CEL</td><td>0.004</td><td>0.009</td><td>1.766</td><td>117.33</td><td>0.598</td><td>0.388</td><td>0.014</td><td>1.35</td><td>0.85</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>38</td><td>R2592E1</td><td>BXD</td><td>BXD45</td><td>62</td><td>M</td><td>R2592E.CEL</td><td>0.002</td><td>0.004</td><td>1.85</td><td>106.16</td><td>0.601</td><td>0.386</td><td>0.013</td><td>1.43</td><td>0.85</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>39</td><td>R2606E1</td><td>BXD</td><td>BXD48</td><td>78</td><td>M</td><td>R2606E.CEL</td><td>0.003</td><td>0.010</td><td>2.556</td><td>106.16</td><td>0.589</td><td>0.397</td><td>0.014</td><td>1.35</td><td>0.83</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>40</td><td>R2603E1</td><td>BXD</td><td>BXD51</td><td>66</td><td>F</td><td>R2603E.CEL</td><td>0.003</td><td>0.009</td><td>2.488</td><td>115.16</td><td>0.577</td><td>0.408</td><td>0.015</td><td>1.24</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>41</td><td>R2534E2</td><td>BXD</td><td>BXD61*</td><td>70</td><td>F</td><td>R2534E2.CEL</td><td>0.030</td><td>0.028</td><td>2.473</td><td>117.76</td><td>0.579</td><td>0.406</td><td>0.015</td><td>1.42</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>42</td><td>R2611E1</td><td>BXD</td><td>BXD64</td><td>68</td><td>M</td><td>R2611E.CEL</td><td>0.013</td><td>0.022</td><td>2.292</td><td>91.99</td><td>0.58</td><td>0.405</td><td>0.015</td><td>1.57</td><td>1.06</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>43</td><td>R2583E1</td><td>BXD</td><td>BXD65</td><td>60</td><td>M</td><td>R2583E.CEL</td><td>0.005</td><td>0.010</td><td>2.492</td><td>70.43</td><td>0.569</td><td>0.415</td><td>0.016</td><td>1.67</td><td>1.01</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>44</td><td>R2536E2</td><td>BXD</td><td>BXD66*</td><td>64</td><td>F</td><td>R2536E2.CEL</td><td>0.039</td><td>0.065</td><td>2.74</td><td>108.62</td><td>0.561</td><td>0.423</td><td>0.017</td><td>1.28</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>45</td><td>R2551E1</td><td>BXD</td><td>BXD68</td><td>67</td><td>F</td><td>R2551E.CEL</td><td>0.037</td><td>0.039</td><td>2.493</td><td>92.38</td><td>0.543</td><td>0.441</td><td>0.016</td><td>2.91</td><td>1.55</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>46</td><td>R2593E1</td><td>BXD</td><td>BXD69</td><td>59</td><td>F</td><td>R2593E.CEL</td><td>0.008</td><td>0.013</td><td>1.672</td><td>127.6</td><td>0.592</td><td>0.395</td><td>0.013</td><td>1.47</td><td>0.92</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>47</td><td>R2537E2</td><td>BXD</td><td>BXD70*</td><td>59</td><td>M</td><td>R2537E2.CEL</td><td>0.046</td><td>0.044</td><td>2.93</td><td>98.66</td><td>0.58</td><td>0.405</td><td>0.016</td><td>1.29</td><td>0.75</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>48</td><td>R2565E1</td><td>BXD</td><td>BXD75</td><td>61</td><td>F</td><td>R2565E.CEL</td><td>0.009</td><td>0.017</td><td>1.79</td><td>101.68</td><td>0.58</td><td>0.405</td><td>0.015</td><td>2.31</td><td>3.47</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>49</td><td>R2579E1</td><td>BXD</td><td>BXD80</td><td>65</td><td>F</td><td>R2579E.CEL</td><td>0.005</td><td>0.010</td><td>2.419</td><td>72.13</td><td>0.592</td><td>0.394</td><td>0.015</td><td>1.73</td><td>0.82</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>50</td><td>R2540E1</td><td>BXD</td><td>BXD87</td><td>63</td><td>M</td><td>R2540E.CEL</td><td>0.013</td><td>0.016</td><td>2.333</td><td>93.15</td><td>0.611</td><td>0.374</td><td>0.014</td><td>1.22</td><td>0.81</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>51</td><td>R2545E1</td><td>BXD</td><td>BXD89</td><td>67</td><td>M</td><td>R2546E.CEL</td><td>0.046</td><td>0.046</td><td>1.667</td><td>104.76</td><td>0.562</td><td>0.423</td><td>0.015</td><td>3.6</td><td>9.84</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>52</td><td>R2578E2</td><td>BXD</td><td>BXD90*</td><td>61</td><td>F</td><td>R2578E2.CEL</td><td>0.033</td><td>0.034</td><td>2.785</td><td>92.27</td><td>0.586</td><td>0.398</td><td>0.016</td><td>1.52</td><td>0.77</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>53</td><td>R2554E1</td><td>BXD</td><td>BXD96</td><td>67</td><td>M</td><td>R2554E.CEL</td><td>0.004</td><td>0.004</td><td>2.177</td><td>93.02</td><td>0.602</td><td>0.383</td><td>0.015</td><td>1.46</td><td>0.77</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>54</td><td>R2577E1</td><td>BXD</td><td>BXD97</td><td>55</td><td>M</td><td>R2577E.CEL</td><td>0.019</td><td>0.016</td><td>2.07</td><td>76.58</td><td>0.595</td><td>0.391</td><td>0.014</td><td>1.87</td><td>1.29</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>55</td><td>R2595E1</td><td>GDP</td><td>129S1/SvImJ</td><td>59</td><td>F</td><td>R2595E.CEL</td><td>0.017</td><td>0.021</td><td>1.792</td><td>115.39</td><td>0.61</td><td>0.375</td><td>0.015</td><td>1.46</td><td>0.77</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>56</td><td>R2533E1</td><td>GDP</td><td>129S1/SvImJ</td><td>60</td><td>M</td><td>R2533E.CEL</td><td>0.021</td><td>0.013</td><td>2.107</td><td>93.55</td><td>0.579</td><td>0.405</td><td>0.016</td><td>1.37</td><td>0.78</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>57</td><td>R2546E1</td><td>GDP</td><td>A/J</td><td>66</td><td>F</td><td>R2545E.CEL</td><td>0.018</td><td>0.014</td><td>1.989</td><td>95.59</td><td>0.586</td><td>0.397</td><td>0.017</td><td>1.47</td><td>0.78</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>58</td><td>R0754E2</td><td>GDP</td><td>A/J</td><td>60</td><td>M</td><td>R0754E.CEL</td><td>0.014</td><td>0.016</td><td>2.718</td><td>85.63</td><td>0.598</td><td>0.387</td><td>0.015</td><td>1.36</td><td>0.76</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>59</td><td>R1676E1</td><td>GDP</td><td>BALB/cByJ</td><td>83</td><td>F</td><td>R1676E.CEL</td><td>0.042</td><td>0.041</td><td>2.685</td><td>98.37</td><td>0.589</td><td>0.396</td><td>0.015</td><td>1.46</td><td>0.74</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>60</td><td>R1672E1</td><td>GDP</td><td>BALB/cByJ</td><td>83</td><td>M</td><td>R1672E.CEL</td><td>0.022</td><td>0.022</td><td>2.216</td><td>110.52</td><td>0.599</td><td>0.386</td><td>0.015</td><td>1.26</td><td>0.8</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>61</td><td>R1700E1</td><td>GDP</td><td>C3H/HeJ</td><td>83</td><td>F</td><td>R1700E.CEL</td><td>0.090</td><td>0.092</td><td>2.978</td><td>68.77</td><td>0.608</td><td>0.379</td><td>0.014</td><td>1.48</td><td>0.78</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>62</td><td>R1704E1</td><td>GDP</td><td>C3H/HeJ</td><td>83</td><td>M</td><td>R1704E.CEL</td><td>0.086</td><td>0.089</td><td>2.581</td><td>88.29</td><td>0.601</td><td>0.386</td><td>0.013</td><td>1.38</td><td>0.84</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>63</td><td>R2564E1</td><td>GDP</td><td>CAST/Ei</td><td>64</td><td>F</td><td>R2564E.CEL</td><td>0.078</td><td>0.064</td><td>1.937</td><td>88.89</td><td>0.585</td><td>0.399</td><td>0.016</td><td>1.6</td><td>0.77</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>64</td><td>R2580E1</td><td>GDP</td><td>CAST/Ei</td><td>64</td><td>M</td><td>R2580E.CEL</td><td>0.076</td><td>0.067</td><td>2.089</td><td>94.64</td><td>0.582</td><td>0.401</td><td>0.017</td><td>1.4</td><td>0.76</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>65</td><td>R2636E1</td><td>GDP</td><td>KK/HIJ</td><td>64</td><td>F</td><td>R2636E.CEL</td><td>0.023</td><td>0.026</td><td>2.61</td><td>93.1</td><td>0.589</td><td>0.395</td><td>0.015</td><td>1.39</td><td>0.76</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>66</td><td>R2637E1</td><td>GDP</td><td>KK/HIJ</td><td>64</td><td>M</td><td>R2637E.CEL</td><td>0.039</td><td>0.020</td><td>2.189</td><td>102.78</td><td>0.594</td><td>0.39</td><td>0.015</td><td>1.3</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>67</td><td>R0999E1</td><td>GDP</td><td>LG/J</td><td>57</td><td>F</td><td>R0999E.CEL</td><td>0.012</td><td>0.012</td><td>2.448</td><td>82.09</td><td>0.594</td><td>0.391</td><td>0.015</td><td>1.38</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>68</td><td>R1004E1</td><td>GDP</td><td>LG/J</td><td>65</td><td>M</td><td>R1004E.CEL</td><td>0.013</td><td>0.015</td><td>2.438</td><td>91.71</td><td>0.587</td><td>0.398</td><td>0.015</td><td>1.38</td><td>0.79</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>69</td><td>R1688E1</td><td>GDP</td><td>NOD/LtJ</td><td>66</td><td>F</td><td>R1688E.CEL</td><td>0.017</td><td>0.019</td><td>2.664</td><td>97.65</td><td>0.586</td><td>0.399</td><td>0.015</td><td>1.26</td><td>0.8</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>70</td><td>R2566E1</td><td>GDP</td><td>NOD/LtJ</td><td>76</td><td>M</td><td>R2566E2.CEL</td><td>0.019</td><td>0.025</td><td>3.031</td><td>69.44</td><td>0.598</td><td>0.388</td><td>0.015</td><td>1.38</td><td>0.75</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>71</td><td>R2550E1</td><td>GDP</td><td>NZO/HlLtJ</td><td>96</td><td>M</td><td>R2550E.CEL</td><td>0.023</td><td>0.015</td><td>1.794</td><td>87.16</td><td>0.607</td><td>0.378</td><td>0.015</td><td>1.52</td><td>0.82</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>72</td><td>R2535E1</td><td>GDP</td><td>NZO/HlLtJ</td><td>62</td><td>F</td><td>R2535E.CEL</td><td>0.046</td><td>0.025</td><td>1.893</td><td>85.67</td><td>0.604</td><td>0.382</td><td>0.014</td><td>1.41</td><td>0.85</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>73</td><td>R2634E1</td><td>GDP</td><td>PWD/PhJ</td><td>62</td><td>F</td><td>R2635E.CEL</td><td>0.077</td><td>0.069</td><td>3.292</td><td>89.8</td><td>0.559</td><td>0.425</td><td>0.016</td><td>1.57</td><td>0.81</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>74</td><td>R2635E1</td><td>GDP</td><td>PWD/PhJ</td><td>62</td><td>M</td><td>R2634E.CEL</td><td>0.088</td><td>0.081</td><td>3.722</td><td>80.05</td><td>0.542</td><td>0.441</td><td>0.017</td><td>1.53</td><td>0.85</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>75</td><td>R2544E1</td><td>GDP</td><td>PWK/PhJ</td><td>63</td><td>F</td><td>R2544E.CEL</td><td>0.106</td><td>0.100</td><td>2.196</td><td>107.51</td><td>0.549</td><td>0.435</td><td>0.017</td><td>1.36</td><td>0.82</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>76</td><td>R2549E1</td><td>GDP</td><td>PWK/PhJ</td><td>83</td><td>M</td><td>R2549E.CEL</td><td>0.065</td><td>0.048</td><td>2.275</td><td>83.8</td><td>0.573</td><td>0.412</td><td>0.015</td><td>1.57</td><td>0.83</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>77</td><td>R2368E1</td><td>GDP</td><td>WSB/EiJ</td><td>67</td><td>F</td><td>R2368E.CEL</td><td>0.025</td><td>0.028</td><td>2.567</td><td>85.7</td><td>0.595</td><td>0.391</td><td>0.014</td><td>1.29</td><td>0.74</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>78</td><td>R2547E1</td><td>GDP</td><td>WSB/EiJ</td><td>67</td><td>M</td><td>R2547E.CEL</td><td>0.032</td><td>0.021</td><td>2.135</td><td>90.04</td><td>0.582</td><td>0.401</td><td>0.016</td><td>1.32</td><td>0.77</td><td>UTM RW</td></tr>            </table>
</td>
</tr>
</table>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platfrom :</P>
<Blockquote>
<P><B>Affymetrix Mouse Genome 430 2.0 arrays: </B>The <A HREF="http://www.affymetrix.com/support/technical/byproduct.affx?product=moe430-20" target="_blank" class="normal">430 2.0</A> array consists of 992936 25-nucleotide probes that estimate the expression of approximately 39,000 transcripts (many are essentially duplicates). The array sequences were selected late in 2002 using Unigene Build 107. The array nominally contain the same probe sequence as the old M430A and 430B  array pair. However, we have found that roughy 75000 probes differ between those on A and B arrays and those on the new 430 2.0.</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a target="_blank" class="fs14" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.
<UL>

<LI>Step 1: We added an offset of 1.0 unit to each cell signal to ensure that all values could be logged without generating negative values. We then computed the log base 2 of each cell.

<LI>Step 2: We performed a quantile normalization of the log base 2 values for the total set of arrays  using the same initial steps used by the RMA transform.

<LI>Step 3: We computed the Z scores for each cell value.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 7: Finally, when appropriate, we computed the arithmetic mean of the values for the set of microarrays for each strain. Technical replicates were averaged before computing the mean for independent biological samples.

</UL>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>
Support for acquisition of microarray data sets was generously provided by Dr. Barrrett Haik, Chair of the Department of Ophthalmology, and director of the Hamilton Eye Institute. Support for the continued development of GeneNetwork was provided by a NIDA/NIMH/NIAAA Human Brain Project grant. All arrays were processed at the VA Medical Center, Memphis by Weikuan Gu.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>This text file originally generated by RWW, Nov 4, 2005. Updated by RWW, Nov 5, 2005. Modified Nov 7 with help of Y. Jiao.
</P></Blockquote>

<P></P>

</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
