<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>RAE230A Microarray Kidney RMA April05 /
WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>

<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">MDC/CAS/ICL RAE230A Peritoneal Fat Database RMA 2ZPlus8 (June/05 freeze) <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=75">GN75</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>
This June 2005 data set provides estimates of mRNA expression in normal peritoneal fat of 32 strains of rats. The set includes the hypertensive SHR strain, the normotensive BN
strain, and 30 HXB/BXH recombinant inbred strains. Each strain was sampled in quadruplicate (6-week-old males). Animals and tissues were generated by Michal Pravenec and
colleagues at the Czech Academy of Sciences (CAS). RNA samples were processed at the Max-Delbr&uuml;ck-Center for Molecular Medicine (MDC), Berlin-Buch, by Norbert Hubner and colleagues. Transcriptome mapping was carried out by Norbert Hubner, Timothy Aitman and colleagues at the MDC and the MRC Clinicial Sciences Centre, Imperial College London (ICL). Samples were hybridized individually to a total of 130 Affymetrix RAE230A array. This particular data set includes 124 arrays processed using the RMA protocol. RMA values of each array were adjusted to an average of 8 units and a standard deviation of 2 units (2ZPlus8).  This data set complements the MAS5 data set exploited by Hubner and colleagues <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15711544" target="_blank" class="fs14">2005</A>.
<A HREF="./array_data/Rat_230A_Fat/Transform/FT_2A_0605_Rz.xls" class="fs14"> Download</A> the particular transform in an Excel work book with both strain means and SEMs.</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>
<Blockquote>We have exploited a set of HXB/BXH recombinant inbred strains of rats generated over the past 20 years in Prague. The parental strains from which all HXB lines are derived are SHR (SHR/OlaIpcv or HSR = H) and Brown Norway (BN.Lx/Cub= B). These parental strains have been used extensively to study cardiovascular system physiology and genetics.</P>

<P>The HXB strains were generated by Michal Pravenec at the Institute of Physiology, Czech Academy of Sciences. The BXH strains were generated by Vladimir Kren (see Pravenec et al. 1989, 2004) at a similar animal facility at the Institute of Biology and Medical Genetics, 1st Medical Faculty, Charles University. These strains are at approximately the 60th generation of continuous inbreeding (F60).</P>

<P>Animals used in the transcriptome analyses of kidney and fat (H&uuml;bner and colleagues, <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15711544" target="_blank" class="fs14">2005</A>) were weaned at 4 weeks. Those born at the Charles University were transferred to the Institute of Physiology. Animals were reared on a commercial rat chow (ST-1 from VELAZ, Czech Republic). Four males were house per cage. Cages were made of polystyrene and have a floor size of 22 x 38 cm and height of 23 cm. The bedding was changed twice a week. Light cycle was 12:12 on-off. Vivarium rooms were maintained at 23 degrees C. Rats were sexually naive.  All males used in the initial transcriptome studies (Hubner et al., 2005) were born between May and August 2002. They were sacrificed unfastged by rapid cervical dislocation between 9 and 10 AM, following an approved animal protocol (Ethics Committee of the Institute of Physiology, Czech Academy of Sciences, Prague; Animal Protection Law of the Czech Republic (311/1997).
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>
<Blockquote>All tissues were collected at the age of 6 weeks. Peritoneal fat pads were rapidly dissected and cleaned extraneous tissue, inserted into a vial, and immersed in liquid nitrogen for storage until RNA extraction.
</Blockquote>


<Blockquote>
The table below lists 130 arrays by strain and sample identifier.  Each array was hybridized with mRNA from a single young male rat. Six arrays marked with asterisks were eventually excluded.
</Blockquote>

<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="50%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="5" cellspacing="1">
<tr bgcolor="royalblue">
<td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>SampleID</font></td>
</tr>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN1</td>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN2</td>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN3</td>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN5</td>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN6</td>
<tr bgcolor="#eeeeee"><td>BXH10</td><td>RI 10c-1</td>
<tr bgcolor="#eeeeee"><td>BXH10</td><td>RI 10c-2</td>
<tr bgcolor="#eeeeee"><td>BXH10</td><td>RI 10c-3</td>
<tr bgcolor="#eeeeee"><td>BXH10</td><td>RI 10c-5</td>
<tr bgcolor="#eeeeee"><td>BXH11</td><td>RI 11c-1</td>
<tr bgcolor="#eeeeee"><td>BXH11</td><td>RI 11c-2</td>
<tr bgcolor="#eeeeee"><td>BXH11</td><td>RI 11c-3</td>
<tr bgcolor="#eeeeee"><td>BXH11</td><td>RI 11c-4</td>
<tr bgcolor="#eeeeee"><td>BXH12</td><td>RI 12c-1</td>
<tr bgcolor="#eeeeee"><td>BXH12</td><td>RI 12c-2</td>
<tr bgcolor="#eeeeee"><td>BXH12</td><td>RI 12c-3</td>
<tr bgcolor="#eeeeee"><td>BXH12</td><td>RI 12c-4</td>
<tr bgcolor="#eeeeee"><td>BXH13</td><td>RI 13c-1</td>
<tr bgcolor="#eeeeee"><td>BXH13</td><td>RI 13c-2</td>
<tr bgcolor="#eeeeee"><td>BXH13</td><td>RI 13c-3</td>
<tr bgcolor="#eeeeee"><td>BXH13</td><td>RI 13c-4</td>
<tr bgcolor="#eeeeee"><td>BXH2</td><td>RI 02c-1</td>
<tr bgcolor="#eeeeee"><td>BXH2</td><td>RI 02c-2</td>
<tr bgcolor="#eeeeee"><td>BXH2</td><td>RI 02c-4</td>
<tr bgcolor="#eeeeee"><td>BXH2</td><td>RI 02c-5</td>
<tr bgcolor="#eeeeee"><td>BXH3</td><td>RI 03c-1</td>
<tr bgcolor="#eeeeee"><td>BXH3</td><td>RI 03c-2</td>
<tr bgcolor="#eeeeee"><td>BXH3</td><td>RI 03c-3</td>
<tr bgcolor="#eeeeee"><td>BXH3</td><td>RI 03c-4</td>
<tr bgcolor="#eeeeee"><td>BXH5</td><td>RI 05c-1</td>
<tr bgcolor="#eeeeee"><td>BXH5</td><td>RI 05c-2</td>
<tr bgcolor="#eeeeee"><td>BXH5</td><td>*RI 05c-3</td>
<tr bgcolor="#eeeeee"><td>BXH5</td><td>RI 05c-5</td>
<tr bgcolor="#eeeeee"><td>BXH6</td><td>RI 06c-1</td>
<tr bgcolor="#eeeeee"><td>BXH6</td><td>RI 06c-4</td>
<tr bgcolor="#eeeeee"><td>BXH6</td><td>RI 06c-5</td>
<tr bgcolor="#eeeeee"><td>BXH6</td><td>RI 06c-6</td>
<tr bgcolor="#eeeeee"><td>BXH8</td><td>RI 08c-2</td>
<tr bgcolor="#eeeeee"><td>BXH8</td><td>RI 08c-3</td>
<tr bgcolor="#eeeeee"><td>BXH8</td><td>RI 08c-4</td>
<tr bgcolor="#eeeeee"><td>BXH8</td><td>RI 08c-5</td>
<tr bgcolor="#eeeeee"><td>BXH9</td><td>RI 09c-1</td>
<tr bgcolor="#eeeeee"><td>BXH9</td><td>RI 09c-2</td>
<tr bgcolor="#eeeeee"><td>BXH9</td><td>RI 09c-4</td>
<tr bgcolor="#eeeeee"><td>BXH9</td><td>RI 09c-5</td>
<tr bgcolor="#eeeeee"><td>HXB1</td><td>RI 01-1</td>
<tr bgcolor="#eeeeee"><td>HXB1</td><td>RI 01-2</td>
<tr bgcolor="#eeeeee"><td>HXB1</td><td>RI 01-4</td>
<tr bgcolor="#eeeeee"><td>HXB1</td><td>RI 01-5</td>
<tr bgcolor="#eeeeee"><td>HXB10</td><td>RI 10-2</td>
<tr bgcolor="#eeeeee"><td>HXB10</td><td>RI 10-3</td>
<tr bgcolor="#eeeeee"><td>HXB10</td><td>RI 10-4</td>
<tr bgcolor="#eeeeee"><td>HXB10</td><td>RI 10-5</td>
<tr bgcolor="#eeeeee"><td>HXB15</td><td>RI 15-1</td>
<tr bgcolor="#eeeeee"><td>HXB15</td><td>RI 15-2</td>
<tr bgcolor="#eeeeee"><td>HXB15</td><td>RI 15-5</td>
<tr bgcolor="#eeeeee"><td>HXB15</td><td>RI 15-6</td>
<tr bgcolor="#eeeeee"><td>HXB17</td><td>RI 17-1</td>
<tr bgcolor="#eeeeee"><td>HXB17</td><td>RI 17-2</td>
<tr bgcolor="#eeeeee"><td>HXB17</td><td>*RI 17-3</td>
<tr bgcolor="#eeeeee"><td>HXB17</td><td>RI 17-4</td>
<tr bgcolor="#eeeeee"><td>HXB18</td><td>RI 18-1</td>
<tr bgcolor="#eeeeee"><td>HXB18</td><td>RI 18-2</td>
<tr bgcolor="#eeeeee"><td>HXB18</td><td>*RI 18-3</td>
<tr bgcolor="#eeeeee"><td>HXB18</td><td>RI 18-4</td>
<tr bgcolor="#eeeeee"><td>HXB2</td><td>RI 02-1</td>
<tr bgcolor="#eeeeee"><td>HXB2</td><td>RI 02-2</td>
<tr bgcolor="#eeeeee"><td>HXB2</td><td>RI 02-3</td>
<tr bgcolor="#eeeeee"><td>HXB2</td><td>RI 02-4</td>
<tr bgcolor="#eeeeee"><td>HXB20</td><td>RI 20-1</td>
<tr bgcolor="#eeeeee"><td>HXB20</td><td>RI 20-2</td>
<tr bgcolor="#eeeeee"><td>HXB20</td><td>*RI 20-3</td>
<tr bgcolor="#eeeeee"><td>HXB20</td><td>RI 20-4</td>
<tr bgcolor="#eeeeee"><td>HXB21</td><td>RI 21-1</td>
<tr bgcolor="#eeeeee"><td>HXB21</td><td>RI 21-2</td>
<tr bgcolor="#eeeeee"><td>HXB21</td><td>RI 21-3</td>
<tr bgcolor="#eeeeee"><td>HXB21</td><td>RI 21-4</td>
<tr bgcolor="#eeeeee"><td>HXB22</td><td>RI 22-1</td>
<tr bgcolor="#eeeeee"><td>HXB22</td><td>RI 22-2</td>
<tr bgcolor="#eeeeee"><td>HXB22</td><td>*RI 22-3</td>
<tr bgcolor="#eeeeee"><td>HXB22</td><td>RI 22-4</td>
<tr bgcolor="#eeeeee"><td>HXB23</td><td>RI 23-1</td>
<tr bgcolor="#eeeeee"><td>HXB23</td><td>RI 23-2</td>
<tr bgcolor="#eeeeee"><td>HXB23</td><td>RI 23-3</td>
<tr bgcolor="#eeeeee"><td>HXB23</td><td>RI 23-4</td>
<tr bgcolor="#eeeeee"><td>HXB24</td><td>RI 24-1</td>
<tr bgcolor="#eeeeee"><td>HXB24</td><td>RI 24-2</td>
<tr bgcolor="#eeeeee"><td>HXB24</td><td>RI 24-3</td>
<tr bgcolor="#eeeeee"><td>HXB24</td><td>RI 24-5</td>
<tr bgcolor="#eeeeee"><td>HXB25</td><td>RI 25-1</td>
<tr bgcolor="#eeeeee"><td>HXB25</td><td>RI 25-3</td>
<tr bgcolor="#eeeeee"><td>HXB25</td><td>RI 25-4</td>
<tr bgcolor="#eeeeee"><td>HXB25</td><td>RI 25-5</td>
<tr bgcolor="#eeeeee"><td>HXB26</td><td>RI 26-1</td>
<tr bgcolor="#eeeeee"><td>HXB26</td><td>RI 26-2</td>
<tr bgcolor="#eeeeee"><td>HXB26</td><td>*RI 26-3</td>
<tr bgcolor="#eeeeee"><td>HXB26</td><td>RI 26-4</td>
<tr bgcolor="#eeeeee"><td>HXB27</td><td>RI 27-1</td>
<tr bgcolor="#eeeeee"><td>HXB27</td><td>RI 27-2</td>
<tr bgcolor="#eeeeee"><td>HXB27</td><td>RI 27-3</td>
<tr bgcolor="#eeeeee"><td>HXB27</td><td>RI 27-4</td>
<tr bgcolor="#eeeeee"><td>HXB29</td><td>RI 29-1</td>
<tr bgcolor="#eeeeee"><td>HXB29</td><td>RI 29-2</td>
<tr bgcolor="#eeeeee"><td>HXB29</td><td>RI 29-4</td>
<tr bgcolor="#eeeeee"><td>HXB29</td><td>RI 29-5</td>
<tr bgcolor="#eeeeee"><td>HXB3</td><td>RI 03-1</td>
<tr bgcolor="#eeeeee"><td>HXB3</td><td>RI 03-2</td>
<tr bgcolor="#eeeeee"><td>HXB3</td><td>RI 03-3</td>
<tr bgcolor="#eeeeee"><td>HXB3</td><td>RI 03-4</td>
<tr bgcolor="#eeeeee"><td>HXB31</td><td>RI 31-1</td>
<tr bgcolor="#eeeeee"><td>HXB31</td><td>RI 31-2</td>
<tr bgcolor="#eeeeee"><td>HXB31</td><td>RI 31-3</td>
<tr bgcolor="#eeeeee"><td>HXB31</td><td>RI 31-4</td>
<tr bgcolor="#eeeeee"><td>HXB4</td><td>RI 04-1</td>
<tr bgcolor="#eeeeee"><td>HXB4</td><td>RI 04-2</td>
<tr bgcolor="#eeeeee"><td>HXB4</td><td>RI 04-3</td>
<tr bgcolor="#eeeeee"><td>HXB4</td><td>RI 04-4</td>
<tr bgcolor="#eeeeee"><td>HXB5</td><td>RI 05-1</td>
<tr bgcolor="#eeeeee"><td>HXB5</td><td>RI 05-2</td>
<tr bgcolor="#eeeeee"><td>HXB5</td><td>RI 05-3</td>
<tr bgcolor="#eeeeee"><td>HXB5</td><td>RI 05-4</td>
<tr bgcolor="#eeeeee"><td>HXB7</td><td>RI 07-1</td>
<tr bgcolor="#eeeeee"><td>HXB7</td><td>RI 07-2</td>
<tr bgcolor="#eeeeee"><td>HXB7</td><td>RI 07-3</td>
<tr bgcolor="#eeeeee"><td>HXB7</td><td>RI 07-4</td>
<tr bgcolor="#eeeeee"><td>HSR</td><td>HSR1</td>
<tr bgcolor="#eeeeee"><td>HSR</td><td>HSR2</td>
<tr bgcolor="#eeeeee"><td>HSR</td><td>HSR6</td>
<tr bgcolor="#eeeeee"><td>HSR</td><td>HSR7</td>
<tr bgcolor="#eeeeee"><td>HSR</td><td>HSR8</td>
</table>
</td>
</tr>
</table>
<P><B>*</B>: These six arrays were excluded in the final strain summary
data. See section of Quality Control for further explanation.

</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>

<Blockquote><P>
<B>Affymetrix 230A GeneChip:</B> Expression data were generated using 230A array. The chromosomal locations of probe sets were determined by <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=rat" target="_blank" class="fs14">BLAT</A> analysis of concatenated probe sequences using the Rat Genome Sequencing Consortium assembly.</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About Quality Control Procedures:</P>

<Blockquote>
<P><B>RNA processing:</B>RNA was extracted using Trizol reagent (Invitrogen) and purified using an RNeasy Mini kit from Qiagen. Double-stranded cDNA was generated without pooling. Fat samples were processed using the Enzo Diagnostics Bioarray High Yield RNA Transcript labeling kit. See Hubner et al. 2005 for additional detail. One-hundred and twenty eight samples passed RNA quality control assays.</P>

<P><B>Probe level QC:</B> All 130 CEL files were collected into a single DataDesk 6.2 analysis file. Probe data from pairs of arrays were plotted and compared after quantile normalization. Six arrays were considered potential outliers (despite having passed RNA quality control) and in the interest of minimizing technical variance, a decision was made to withhold them from the calculation of strain means. The remaining 124 arrays were then quantile normalized again and reexamined in DataDesk to ensure reasonable colinearity of all final array data sets.</P>

<P><B>Strain assignment check:</B> To confirm strain assignment we exploit a set of transcripts with near-Mendelian segregation patterns  (search for "test Mendelian"). Strain means with both intermediate expression values AND unusually high error terms often indicate at a misassignment of a case to a particular strain. This error checking has identified 4 strains with possible errors in this data set.


<!-- NOT DONE YET:
<P><B>Probe set level QC:</B> Probe set level QC involves counting the number of times that a single array data set from a single sample generates outliers at the level of the probe set consensus estimate of expression. With 124 arrays, any single array should generate a comparatively small fraction of the total number of outlier calls. This final step of array QC has NOT been implemented in this data set.</P>
-->

</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote>
<P><B>Probe and Probe set data: </B>The original cell-level files (in text format) were downloaded from <A HREF="http://www.ebi.ac.uk/arrayexpress/">Array Express</A>. These files were then converted to a standard Affymetrix CEL file (old MAS5 style) format using a Perl script written by Senhua Yu.  These files were then processed as a large batch (either all 130 arrays or the final 124 arrays) using a custom quantile normalization program written by KF Manly. The output of this program automatically performs the log normalization and variance stabilization at the probe level.  We then computed the mean and standard error for each strain using these normalized probe data.</P>

<P>Probe set data were generated starting with the raw Affymetrix CEL file described above (prior to any normalization) and were processed using the Robust Multichip Average (<a href="http://www.bioconductor.org" class="fs14" target="_blank" >RMA</a>) method (Irrizary et al. 2003).

<P>This data set include further normalization to produce final estimates of expression that can be compared directly to the other transforms (average of 8 units and stabilized variance of 2 units within each array). Data were further transformed as follows:

<UL>

<LI>Step 1: RMA values were generated as described above.

<LI>Step 2: We computed the Z scores for each probe set value for each array.

<LI>Step 3: We multiplied all Z scores by 2.

<LI>Step 4: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 5: Finally, we computed the arithmetic mean of the values for the set of microarrays for each strain. We have not corrected for background beyond the background correction implemented by Affymetrix.

</UL>

<P>All transformation steps were carried out by Senhua Yu at UTHSC.
</P></Blockquote>

</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>
This work was supported with funds to TJA by the MRC Clinical Sciences Centre, the British Heart Foundation, and the Wellcome Trust Cardiovascular Functional Genomics Intiative; to NH from the German Ministry for Science and Education (National Genome Research Network, NGFN); to MP and Vladimir Kren from the Grant Agency of the Czech Republic; to MP and TJA from the Wellcome Trust Collaborative Research Initiative grant, to Theodore W Kurtz from the NIH, to TWK and MP from a Fogarty International Research Collaboration Award. Microarrays were a generous donation of Affymetrix Inc. MP is an International Research Scholar of the Howard Hughes Medical Institute.


</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>This approved text file originally generated by Robert Williams, Norbert Hubner, Michal Pravenec, Timothy Aitman, April 19, 2005. Updated by RWW, April 20, 2005; April 28, 2005. June 15, 2005 by RWW and SY; June 20 by RWW and NH.</P></Blockquote>

<P></P>

</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
