<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>GE-NIAAA Cerebellum mRNA M430v2 (May05) MAS5 dataset</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">GE-NIAAA Cerebellum mRNA M430v2 (May05) MAS5 dataset <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=71">GN71</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote>
<P>
NOT RECOMMENDED: This May 2005 data freeze provides estimates of mRNA expression in adult cerebellum of 40 lines of mice including 28 BXD recombinant inbred strains, C57BL/6J, DBA/2J, and 10 other common inbred strains of mice. Data were generated by <A HREF="http://www.genomeexplorations.com/" target="_blank" class="fs14">Genome Explorations Inc.</A> (Divyen Patel and colleagues). Cerebellar samples were hybridized in small pools (<I>n</I> = 3) to Affymetrix M430 2.0 arrays. This particular data set was processed using the Microarray Suite 5 (<a href="http://www.affymetrix.com/support/technical/whitepapers/sadd_whitepaper.pdf"_blank" class="fs14">MAS 5</a>) protocol. To simplify comparisons among transforms, MAS5 values of each array were adjusted to an average of 8 units and a standard deviation of 2 units.
</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>
We use a set of BXD recombinant inbred strains and standard inbred strains. The BXD lines are derived crossed between C57BL/6J (B6 or B) and DBA/2J (D2 or D). Both B and D parental strains have been almost fully sequenced (8x coverage for B6 by a public consortium and approximately 1.5x coverage for D by <a href="http://www.celeradiscoverysystem.com/index.cfm" target="_blank" class="fs14">Celera Discovery Systems</a>) and data for 1.75 millioin B vs D SNPs are incorporated into WebQTL's genetic maps for the BXDs.  BXD2 through BXD32 were produced by Benjamin A. Taylor starting in the late 1970s. BXD33 through 42 were also produced by Taylor, but they were generated in the 1990s. These strains are all available from The Jackson Laboratory, Bar Harbor, Maine. BXD43 through BXD99 were produced by Lu Lu, Jeremy Peirce, Lee M. Silver, and Robert W. Williams in the late 1990s and early 2000s using advanced intercross progeny (Peirce et al. <a href="http://www.biomedcentral.com/1471-2156/5/7"   target="_blank" class="fs14">2004</a>).</P>

<P>Most BXD animals were generated in-house at the University of Tennessee Health Science Center by Lu Lu and Robert Williams using stock obtained from The Jackson Laboratory between 1999 and 2004. All BXD strains with numbers above 42 are new advanced intecross type BXDs (Peirce et al. <a href="http://www.biomedcentral.com/1471-2156/5/7"   target="_blank" class="fs14">2004</a>) that are current available from UTHSC. Additional cases were provided by Glenn Rosen, John Mountz, and Hui-Chen Hsu. These cases were bred either at The Jackson Laboratory (GR) or at the University of Alabama (JM and HCH).
</P>





</Blockquote>

</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate this set of data:</P>

<Blockquote><P>The May 2005 data set consists of a total of 61 array (Affymetrix 430 2.0 arrays) from 40 different genotypes. Each sample consists of whole cerebellum taken from three adult animals of the same age and sex. The M430 2.0 arrays were processed in several batches.

<P><B>Replication and Sample Balance:</B> We obtained data independent biological sample pools from both sexes for half of the strain, including most of the standard inbred strains (129S1/SvImJ is the exception and is represented by two female-only arrays). Most BXD strains are represented by single pooled samples. You can determine the sex of a sample from the table below or by reviewing the expression of the <I>Ddx3y</I> and <I>Xist</I> RNA signal.



<P>
<DIR><IMG src="/images/upload/DdyvsXist_GENIAAA.png" valign="top">
</DIR>

<DIR><P><SMALL><B>Legend:</B> Sex balance of the GE-NIAAA data set can be easily evaluated by analysis of this scatterplot of Ddx3y and Xist. Ddx3y (also called Dby) is a transcript with high expression in males whereas Xist is a transcript with high expression in females. Strains that fall in the upper left quadrant are represented only by a single female sample (except in the case of the 129S1/SvImJ data) whereas strains that fall in the lower right quadrant are represented only a a single male sample.  </SMALL></P>
</DIR>



<!--
GET DATA
<P>The age range of samples is relatively narrow. </P>

-->

<P>RNA was extracted at Genome Explorations.</P>

<P>All samples were subsequently processed at the <A HREF="http://www.genomeexplorations.com/" target="_blank" class="fs14">Genome Explorations Inc.</A> by Divyen Patel and colleagues.</P>

</Blockquote>




<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform :</P>
<Blockquote>
<P><B>Affymetrix Mouse Genome 430 2.0: </B>The <A HREF="http://www.affymetrix.com/support/technical/byproduct.affx?product=moe430-20" target="_blank" class="normal">430 2.0</A> array consist of approximately 992936  25-nucleotide probes that collectively estimate the expression of approximately 39,000 transcripts. The array sequences were selected late in 2002 using Unigene Build 107. The arrays nominally contain the same probe sequences as the 430A and 430B series. However, we have found that roughy 75000 probes differ from those on A and B arrays and those on the 430 2.0</P>
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a target="_blank" class="fs14" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.
<UL>

<LI>Step 1: We added an offset of 1.0 unit to each cell signal to ensure that all values could be logged without generating negative values. We then computed the log base 2 of each cell.

<LI>Step 2: We performed a quantile normalization for the log base 2 values for the total set of 104 arrays (all three batches) using the same initial steps used by the RMA transform.

<LI>Step 3: We computed the Z scores for each cell value.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6: No correction for potential batch effect was attempted.

<LI>Step 7: Finally, we computed the arithmetic mean of the values for the set of microarrays for each strain. Technical replciates were averaged before computing the mean for independent biological samples. Note, that we have not (yet) corrected for variance introduced by differences in sex, age, source of animals, or any interaction terms. We have not corrected for background beyond the background correction implemented by Affymetrix in generating the CEL file. We eventually hope to add statistical controls and adjustments for these variables.

</UL>


<B>Probe set data: </B>The expression data were processed by Yanhua Qu (UTHSC). Probe set data were generated from the fully normalized CEL files (quantile and batch corrected) using the standard MAS 5 Tukey biweight procedure.  A 1-unit difference represents roughly a two-fold difference in expression level. Expression levels below 5 are usually close to background noise levels.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the chromosome and megabase position values:</P>

<Blockquote>The chromosomal locations of probe sets included on the microarrays were determined by BLAT analysis using the Mouse Genome Sequencing Consortium March 2005 Assembly (see <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse" class="fs14">http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse</a>). We thank Dr. Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis.</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>
Data were generated with funds to Genome Explorations, Inc., for the NIAAA as part of an SBIR grant to Dr. Divyen Patel. Mouse colony resources and integration of data into GeneNetwork was carried out by Drs. RW Williams and Lu Lu at UTHSC.

</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>
This text file originally generated by RWW and YHQ, March 21, 2005. Updated by RWW, March 23, 2005; RWW April 8.




</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
