<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>GNF Microarray March04 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>


</HEAD>
<BODY  bottommargin="5" topmargin="5" leftmargin="5" rightmargin="5" marginheight="5" marginwidth="5" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=3 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<!-- Body Start from Here -->


<TD vAlign=top width="100%" colspan=2 bgColor=#eeeeee>
<P class="title">Genomics Institute of the Novartis Research Foundation (GNF) and Groningen Hematopoietic Stem Cell mRNA U74Av2 Database (March/04 Freeze)  <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=7">GN7</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>
The original March 2004 data freeze provides estimates of mRNA expression in hematopoietic stem cells (HSC) from adult female BXD recombinant inbred mice measured using Affymetrix U74Av2 microarrays (Bystrykh et al., 2005). Data were generated at the Genomics Institute of the Norvartis Research Foundations (Cooke and colleagues) and at the University of Groningen (de Haan and colleagues). Samples  from 30 strains were hybridized to 60 arrays in two batches (Mar03 includes only the first batch). Data were processed using the <a target="_blank" href="http://stat-www.berkeley.edu/users/bolstad/RMAExpress/RMAExpress.html" class="normal">RMA</a> protocol.
</P>

<B>REFERENCE</B>: Bystrykh L, Weersing E, Dontje B, Sutton S, Pletcher MT, Wiltshire T, Su AI, Vellenga E, Wang J, Manly KF, Lu L, Chesler EJ, Alberts R, Jansen RC, Williams RW, Cooke M, de Haan G (2005) Uncovering regulatory pathways affecting hematopoietic stem cell function using "genetical genomics." Nature Genetics, 37:225-232
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the mice used to map microarray data:</P>
<Blockquote><P>BXD recombinant inbred mice were purchased from the Jackson Laboratory and upon arrival were housed under clean conventional conditions in the Central Animal Facility of the University of Groningen, Netherlands. We used female mice between 3 and 6 months of age.
</P>
<P>Stem cells (described below) were isolated from pooled bone marrow obtained from three BXD animals per strain. Pooled RNA samples were split in two aliquots and each sample was independently amplified and hybridized to the U74Av2 array (3 mice x 2 arrays).</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>
<Blockquote><P>Bone marrow cells were flushed from the femurs and tibiae of three mice and pooled. After standard erythrocyte lysis nucleated cells were incubated with normal rat serum for 15 min at 4 degrees Celsius. Subsequently cells were stained with a panel of biotinylated lineage-specific antibodies (murine progenitor enrichment cocktail, containing anti-CD5, anti-CD45R, anti-CD11b, anti-TER119, anti-Gr-1, and anti-7-4, Stem Cell Technologies, Vancouver, Canada), FITC-anti-Sca-1 and APC-anti-c-kit (Pharmingen). Cells were washed twice, and incubated for 30 minutes with streptavidin-PerCP (Pharmingen). After two washes cells were resuspended in PBS with 1% BSA, and purified using a MoFlo flow cytometer. The lineage-depleted bone marrow cell population was defined as the 5% cells showing least PerCP-fluorescence intensity. Stem cell yield across all BXD samples varied from 16,000 to 118,000 Lin-Sca-1+ c-kit+ cells. A small aliquot of each sample of purified cells was functionally tested for stem cell activity by directly depositing single cells in a cobblestone area forming cell assay. The remainder of the cells was immediately collected in RNA lysis buffer. Total RNA was isolated using StrataPrep Total RNA Microprep kit  (Stratagene) as described by the manufacturer. RNA pellets were resolved in 500 microliters absolute ethanol, and sent on dry ice by courrier to GNF, La Jolla, CA. The March 2004 data set was processed in two batches. The first batch consisted of samples from 22 strains, BXD1, 2, 5, 6, 8, 9, 11, 12, 14, 16,18, 19, 21, 28, 31, 32, 33, 34, 38, 39, 40, 42. The second batch included 8 strains, BXD15, 22, 24, 25, 27, 29, 30, 36.
</P></Blockquote>



<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="50%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="5" cellspacing="1">
<tr bgcolor="royalblue"><td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>Array ID</font></td>
<td><font color=#FFFFFF>Result date</font></td></tr>
<tr bgcolor="#eeeeee"><td>BXD1</td><td>M7SS02080615</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD1</td><td>M7SS02080616</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD2</td><td>M7SS02080611</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD2</td><td>M7SS02080612</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD5</td><td>M7SS02080619</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD5</td><td>M7SS02080620</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD6</td><td>M7SS02042201</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD6</td><td>M7SS02042202</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD8</td><td>M7SS02042203</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD8</td><td>M7SS02042204</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD9</td><td>M7SS02080607</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD9</td><td>M7SS02080608</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD11</td><td>M7SS02042205</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD11</td><td>M7SS02042206</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD12</td><td>M7SS02042207</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD12</td><td>M7SS02042208</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD14</td><td>M7SS02080621</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD14</td><td>M7SS02080622</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD15</td><td>M7SS03082701</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD15</td><td>M7SS03082702</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD16</td><td>M7SS02080613</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD16</td><td>M7SS02080614</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD18</td><td>M7SS02080601</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD18</td><td>M7SS02080602</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD19</td><td>M7SS02080605</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD19</td><td>M7SS02080606</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD21</td><td>M7SS02080603</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD21</td><td>M7SS02080604</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD22</td><td>M7SS03082703</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD22</td><td>M7SS03082704</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD24</td><td>M7SS03082705</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD24</td><td>M7SS03082706</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD25</td><td>M7SS03080104</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD25</td><td>M7SS03082715</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD27</td><td>M7SS03082707</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD27</td><td>M7SS03082708</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD28</td><td>M7SS02080623</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD28</td><td>M7SS02080624</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD29</td><td>M7SS03082709</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD29</td><td>M7SS03082710</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD30</td><td>M7SS03082711</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD30</td><td>M7SS03082712</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD31</td><td>M7SS02042209</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD31</td><td>M7SS02042210</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD32</td><td>M7SS02042211</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD32</td><td>M7SS02042212</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD33</td><td>M7SS02042219</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD33</td><td>M7SS02042220</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD34</td><td>M7SS02080617</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD34</td><td>M7SS02080618</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD36</td><td>M7SS03082713</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD36</td><td>M7SS03082714</td><td>8/27/03</td></tr>
<tr bgcolor="#eeeeee"><td>BXD38</td><td>M7SS02042213</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD38</td><td>M7SS02042214</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD39</td><td>M7SS02080609</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD39</td><td>M7SS02080610</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD40</td><td>M7SS02042215</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD40</td><td>M7SS02042216</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD42</td><td>M7SS02042217</td><td>9/27/02</td></tr>
<tr bgcolor="#eeeeee"><td>BXD42</td><td>M7SS02042218</td><td>9/27/02</td></tr>
</table>
</td>
</tr>
</table>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;How to Download these Data:</P>
<Blockquote><P>Array data files are available on the NCBI <A HREF="http://www.ncbi.nlm.nih.gov/geo/" target="_blank" class="normal">GEO</A> site using the accession identifier <A HREF="http://www.ncbi.nlm.nih.gov/geo/gds/gds_browse.cgi?gds=1077" target="_blank" class="normal">GDS1077</A>. Individual U74Av2 arrays are GEO IDs GSM36673 through GSM36716. The single most appropriate reference is: Bystrykh L, Weersing E, Dontje B, Sutton S, Pletcher MT, Wiltshire T, Su AI, Vellenga E, Wang J, Manly KF, Lu L, Chesler EJ, Alberts R, Jansen RC, Williams RW, Cooke M, de Haan G (2005) Uncovering regulatory pathways affecting hematopoietic stem cell function using "genetical genomics". Nature Genetics 37:225-232.</P>
</BLOCKQUOTE>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About amplification and hybridization:</P>

<Blockquote><P>Total RNA was quantified using RiboGreen and split into equal aliquots of approximately 10 ng, representing RNA from approximately 10,000 cells, and labeled using a total of three rounds of RNA amplification, exactly as described previously (Scherer et al. <a href="http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12661160&dopt=Abstract" target="_blank" class="normal">2003</a>). Labeled cRNA was fractionated and hybridized to the U74Av2 microarray from Affymetrix according to manufacture's protocol.
</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values are the 75% quantiles from a set of <a href="images/AffyU74.pdf" target="_blank" class="normal">36 pixel </a>values per cell.

<UL>
<LI>Step 1: We added an offset of 1 to the CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log2 of each cell.

<LI>Step 3: We computed the Z score for each cell within array.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6: We computed the arithmetic mean of the values for the set of microarrays for each of the individual strains.
</UL>


<P><B>Probe set data: </B>Probe set expression data were processed by Ritsert Jansen. The original CEL files produced by the Affymetrix analysis software were read into the R environment (Ihaka and Gentleman 1996). Data were normalized using the Robust Multichip Average (<a target="_blank" href="http://www.bioconductor.org" class="normal">RMA</a>) method of background correction, quantile normalization, and summarization of signal intensity (Irrizary et al. 2003). Probe set intensities were log2 transformed. Probe set data are averages of two technical replicates after batch correction (see below) and were treated as single samples. Please seee Bolstad and colleagues (<A href="http://www.stat.berkeley.edu/users/bolstad/normalize/normalize.html" class="normal">2003</A>) for a helpful comparison of RMA and two other common methods of processing Affymetrix array data sets.
</P></Blockquote>

<P>Samples were processed in two batches. To adjust for the effect of technical batch processing differences, a linear model was applied to RMA normalized expression data. The following ANOVA model fitting the processing batch was applied for each set of perfect match probes:</P>

<P>PM<sub class="fs12">ij</sub> = M + B<sub class="fs12">i</sub> + e<sub class="fs12">ij</sub></P>

<P>in which PM<sub class="fs12">ij</sub> are the RMA probe intensities for arrays i = 1,...,30 and probe j = 1,...,J. M is the overall mean; B<sub id="normallsmall">i</sub> represents the batch effect, and e<sub class="fs12">ij</sub> is the error term. The batch effect parameter was estimated and subtracted from PM probe expression values. Probe level intensities were averaged for each probe set to produce the batch corrected expression.
</Blockquote>


<Blockquote><P>
<B>Affymetrix U74Av2 GeneChip:</B> The expression data were generated using U74Av2 arrays. The chromosomal locations of U74Av2 probe sets were determined by BLAT analysis of concatenated probe sequences using the Mouse Genome Sequencing Consortium May 2004 (mm5) assembly. This BLAT analysis is performed periodically by Yanhua Qu as each new build of the mouse genome is released (see <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse" class="normal">http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse</a>). We thank Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis. It is possiible to confirm the BLAT alignment results yourself simply by clicking on the <B>Verify</B> link in the Trait Data and Editing Form (right side of the <B>Location</B> line).
</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array probe set names:</P>
<Blockquote> <P>
Most probe sets on the U74Av2 array consist of a total of 32 probes, divided into 16 perfect match probes and 16 mismatch controls. Each set of these 25-nucleotide-long probes has an identifier code that includes a unique number, an underscore character, and several suffix characters that highlight design features.  The most common probe set suffix is  <b>at</b>. This code indicates that the probes should hybridize relatively selectively with the complementary anti-sense target (i.e., the complemenary RNA) produced from a single gene. Other codes include:</P>

<li> <b>f_at (sequence family)</b>: Some probes in this probe set will hybridize to identical and/or slightly different sequences of related gene transcripts.</li>

<li><b>s_at (similarity constraint)</b>: All Probes in this probe set target common sequences found in transcripts from several genes. </li>

<li> <b>g_at (common groups)</b>: Some probes in this set target identical sequences in multiple genes and some target unique sequences in the intended target gene.</li>

<li> <b>r_at (rules dropped)</b>: Probe sets for which it was not possible to pick a full set of unique probes using the Affymetrix probe selection rules. Probes were picked after dropping some of the selection rules.</li>

<li> <b>i_at (incomplete)</b>: Designates probe sets for which there are fewer than the standard numbers of unique probes specified in the design (16 perfect match for the U74Av2).</li>

<li> <b>st (sense target)</b>: Designates a sense target; almost always generated in error.</li>

<P>Descriptions for the probe set extensions were taken from the Affymetrix<a href="./dbdoc/data_analysis_fundamentals_manual.pdf" class="normal"> GeneChip Expression Analysis Fundamentals</a>.
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>

<Blockquote><P>Cell and samples were generated by Leonid V. Bystrykh, Ellen Weersing, Bert Dontje, Gerald de Haan, Department of Stem Cell Biology, University of Groningen, the Netherlands.
RNA amplification and array processing were carried out by Michael Cooke, John Hogenesch, Andrew Su and colleagues at GNF. </p>

<p>
The batch correction of this March 2004 data set was carried out by Ritsert Jansen  and his student Rudy Albert in the Department of Bioinformatics (University of Groningen). Conversion for WebQTL was carried out by Robert W. Williams, Kenneth Manly, Jintao Wang, and Yanhua Qu at UTHSC. </p>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;References:</P>

<Blockquote><P>Bystrykh L, Weersing E, Dontje B, Sutton S, Pletcher MT, Wiltshire T, Su AI, Vellenga E, Wang J, Manly KF, Lu L, Chesler EJ, Alberts R, Jansen RC, Williams RW, Cooke M, de Haan G (2005) Uncovering regulatory pathways affecting hematopoietic stem cell function using “genetical genomics�? Nature Genetics, in press.</P></Blockquote>

<Blockquote><P>Ihaka R, Gentleman R (1996) R: A Language for Data Analysis and Graphics. Journal of Computational and Graphical Statistics 5:299-314.</P></Blockquote>
<Blockquote><P>Irizarry R, Hobbs B, Collin F, Beazer-Barclay Y, Antonellis K, Scherf U, Speed T (2003) Exploration, normalization, and summaries of high density oligonucleotide array probe level data. Biostatistics 4:249-264. </P></Blockquote>
<Blockquote><P>Gautier L, Cope L, Bolstad B, Irizarry R (2004) affy -- analysis of Affymetrix GeneChip data at the probe level. Bioinformatics 20:307-315. </P></Blockquote>
<Blockquote><P>Scherer A, Krause A, Walker JR, Sutton SE, Seron D, Raulf F, Cooke MP (2003) Optimized protocol for linear RNA amplification and application to gene expression profiling of human renal biopsies. Biotechniques 34:546-550, 552-554, 556.
</P></Blockquote>
<Blockquote><P>de Haan G, Bystrykh LV, Weersing E, Dontje B, Geiger H, Ivanova N, Lemischka IR, Vellenga E, Van Zant G (2002) A genetic and genomic analysis identifies a cluster of genes associated with hematopoietic cell turnover. Blood 100:2056-2062.
</P></Blockquote>
<Blockquote><P>
Wang J, Williams RW, Manly KF (2003) WebQTL: Web-based complex trait analysis. Neuroinformatics 1:299-308.
</P></Blockquote>
<Blockquote><P>
Williams RW, Manly KF, Shou S, Chesler E, Hsu HC, Mountz J, Wang J, Threadgill DW, Lu L (2002) Massively parallel complex trait analysis of transcriptional activity in mouse brain. International Mouse Genome Conference 16:46.
</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>
This text file originally generated by GdH and RWW, March 2004. Updated by RWW, Oct 30, 2004, Dec 6, 2004. EJC Apr 25, 2005.
</P></Blockquote>

<P></P>

</TD>
</TR>
<TR>
<TD colspan=2 align=center bgColor=#ddddff>



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>

// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure

new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
