
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>HEI BXD ONC vs Control Retina Illumina V6.2 (Feb11) RankInv **</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">

<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<link rel="stylesheet" media="all" type="text/css" href="/css/tabbed_pages.css" />
<script type="text/javascript" src="/javascript/jquery.js"></script>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>
<SCRIPT SRC="/javascript/dhtml.js"></SCRIPT>
<SCRIPT SRC="/javascript/jqueryFunction.js"></SCRIPT>
<script src="/javascript/tabbed_pages.js" type="text/javascript"></script>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff >

<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD  bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5>
		<TR>

		<!-- split from Here -->
		<!-- Body Start from Here -->
		
	<P class="title">HEI BXD ONC vs Control Retina Illumina V6.2 (Feb11) RankInv **<A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>
	
	Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=305">GN305</A></P>
<P class="subtitle">Summary:</P>
<Blockquote>
<P> HEI Optic Nerve Crush (ONC) – Control Retina Illumina V6.2 (April 2010) RankInv **  was normalized and scaled by William E. Orr and uploaded by Arthur Centeno and Xiaodong Zhou on February 1, 2011. This data set consists of  BXD strains, C57BL/6J, DBA/2J, both reciprocal F1s, and BALB/cByJ. The data base was constructed by subtacting the normal expression values from the values from the same strain that were generated from retinas 2 days after optic nerve crush. 
 
<P> This is rank invariant data with 2z+8 stabilization, but without special correction for batch effects. The data includes the mean of four samples per strain. Values in expression range from 0000 to 0000 (0000 units), a nominal range of 0000-fold.
 
<P>The lowest level of expression is 0000 for ILMN_000 (0000) from HEI Retina Illumina V6.2 (April 2010) RankInv **. Lowest single data about 5.842.
 
<P>The highest level of expression is 0000 for ILMN_0000 (000). Highest single value is about 18.934.
<p>
</blockquote>
<P class="subtitle">Relevant Publications</P>
<blockquote>
<P>
<OL>
<LI>Geisert EE, Lu L, Freeman-Anderson NE, Templeton JP, Nassr M, Wang X, Gu W, Jiao Y, Williams RW.:Gene expression in the mouse eye: an online resource for genetics using 103 strains of mice. Molecular Vision 2009 Aug 31;15:1730-63, (<A HREF="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2736153/?tool=pubmed">Link</A>)
<LI>Geisert EE, Jr., Williams RW: The Mouse Eye Transcriptome: Cellular Signatures, Molecular Networks, and Candidate Genes for Human Disease. In Eye, Retina, and Visual System of the Mouse. Edited by Chalupa LM, Williams RW. Cambridge: The MIT Press; 2008:659-674
<LI>Peirce JL, Lu L, Gu J, Silver LM, Williams RW: A new set of BXD recombinant inbred lines from advanced intercross populations in mice. BMC Genet 2004, 5:7. (<A HREF="http://www.biomedcentral.com/1471-2156/5/7" target="_blank" class="fs14">Link</A>)
<LI>Templeton JP, Nassr M, Vazquez-Chona F, Freeman-Anderson NE, Orr WE, Williams RW, Geisert EE: Differential response of C57BL/6J mouse and DBA/2J mouse to optic nerve crush. BMC Neurosci. 2009, July 30;10:90.(<A HREF="http://www.biomedcentral.com/1471-2202/10/90" target="_blank" class="fs14">Link</A>)
 
 
 
<P><P>
</BLOCKQUOTE><blockquote>
<B>Other Data Sets</B>
Users of these mouse retina data may also find the following complementary resources useful:
<OL>
<LI> <A HREF="http://neibank.nei.nih.gov/index.shtml" target="_blank" class="fs14">NEIBank</A> collection of ESTs and SAGE data.
<LI><A HREF="http://www.sph.uth.tmc.edu/Retnet/" target="_blank" class="fs14">RetNet</A>: the Retinal Information Network--tables of genes and loci causing inherited retinal diseases
<LI><A HREF="http://cepko.med.harvard.edu/" target="_blank" class="fs14">Mouse Retina SAGE Library</A> from the Cepko laboratory. This site provides extensive developmental data from as early as embryonic day E12.5.
<LI><A HREF="http://dro.hs.columbia.edu/" target="_blank" class="fs14">Digital reference of ophthalmology </A> from Columbia provides high quality photographs of human ocular diseases, case studies, and short explanations. This reference does not have a molecular focus.
 
<LI><A HREF="http://www.scripps.edu/cb/friedlander/gene_expression/search.html" target="_blank" class="fs14">Mouse Retinal Developmental Gene Expression</A> data sets from the Friedlander laboratory. This site provides extensive developmental data using the Affymetrix U74 v 2 array (predecessor of the M430).
 
 
<LI><A HREF="http://microarray-pubs.stanford.edu/eyecompartments/" target="_blank" class="fs14">Data sets on differential gene expression in anatomical compartments of the human eye</A> from Pat Brown's lab. View expression signatures for different ocular tissues using the <B>geneXplorer 2.0</B>.
</OL>
</Blockquote>
 
 
<P class="subtitle">About the cases used to generate this set of data:</P>
 
<Blockquote>
 
<P>Almost all animals are young adults between 60 and 90 days of age (Table 1, minimum age is 48 and maximum age is 118 days). We measured expression in conventional inbred strains, BXD recombinant inbred (RI) strains, and reciprocal F1s between C57BL/6J and DBA/2J.
 
 
<Blockquote>
<b>BXD strains:</b> <P><LI>The first 32 of these strains  are from the Taylor series of BXD strains generated at the Jackson Laboratory by Benjamin A. Taylor. BXD1 through BXD32 were started in the late 1970s, whereas BXD33 through 42 were started in the 1990s. 
<p>
<LI>In 2004, BXD24/TyJ developed a spontaneous mutation, rd16 which resulted in retinal degeneration and was renamed BXD24b/TyJ (BXD24 in this database). The strain, BXD24a, was cryo-recovered in 2004 from 1988 embryo stocks (F80) and does not exhibit retinal degeneration. In 2009, BXD24b was renamed BXD24/TyJ-Cep290rd16/J by JAX Labs to reflect the discovery of the genetic basis of the mutation. At the same time BXD24a was then referred to just as BXD24/TyJ by Jax Labs, but still called BXD24a in this dataset.
<p>
<LI>The other 36 BXD strains (BXD43 and higher) were bred by Lu Lu, Jeremy Peirce, Lee M. Silver, and Robert W. Williams starting in 1997 using B6D2 generation 10 advanced intercross progeny. This modified breeding protocol doubles the number of recombinations per BXD strain and improves mapping resolution (<a href="http://www.biomedcentral.com/1471-2156/5/7"  target="_blank" class="fs14">Peirce et al. 2004</a>). All of the Taylor series of BXD strains and many of the new BXD strains are available from the Jackson Laboratory. All of the new BXD strains (BXD43 and higher) are also available directly from <A HREF="mailto:lulu@nb.uthsc.edu" target="_blank" class="fs14">Lu Lu</A> and colleagues at the University of Tennessee Health Science Center in Memphis, TN, USA. BXD24/TyJ  is now known as BXD24b/TyJ and has nearly complete retinal degeneration. BXD24a/TyJ, a 1988 F80 inbred stock that has been rederived from cryogenic storage, does not have retinal degeneration (stock number 005243) and is an ideal coisogenic control, but is not included in the HEI data set.
</Blockquote>
</Blockquote>
 
<P class="subtitle">About the tissue used to generate this set of data:</P>
 
<Blockquote>
<P><B>Tissue preparation protocol</B>. Animal were killed by rapid cervical dislocation. Retinas were removed immediately and placed in RNA<I>later</I> at room temperature. Two retinas from one mouse were stored in a single tube.
 
<p><b>Dissecting and preparing eyes for RNA extraction</b>
<P>
<p>Retinas for RNA extraction were placed in RNA STAT-60 (Tel-Test Inc.) and processed per manufacturer’s instructions (in brief form below).
Total RNA was extracted with RNA STAT-60 (Tel-Test Inc.) according to the manufacturer's instructions. Briefly we:
<P><ul>
<li>Homogenize tissue samples in the RNA STAT-60 (1 ml/50 to 100 mg tissue via syringe)
<li>Allow the homogenate to stand for 5-10 min at room temperature
<li>Add 0.2 ml of chloroform per 1 ml RNA STAT-60
<li>Mix the sample vigorously for 15 sec and let the sample incubate at room temperature for 5-10 min
<li>Centrifuge at 12,000 g for 1 hr at 4°C
<li>Transfer the aqueous phase to a clean centrifuge tube
<li>Add 0.5 ml of isopropanol per 1 ml RNA STAT-60
<li>Vortex and incubate the sample at -20°C for 1 hr or overnight
<li>Centrifuge at 12,000 g for 1 hr
<li>Remove the supernatant and wash the RNA pellet with 75% ethanol
<li>Remove ethanol, let air dry (5-10 min)
<li>Dissolve the pellet in 50 μl of nuclease free water.
<p>
</ul>
<p><b>Sample Processing:</b> Drs. Natalie E. Freeman-Anderson and Justin P. Templeton extracted the retinas from the mice and Dr. Natalie Freeman-Anderson processed all samples in the HEI Vision Core Facility. The tissue was homogenized and extracted according to the RNA-Stat-60 protocol as described by the manufacturer (Tel-Test, Friendswood, TX) listed above. The quality and purity of RNA was assessed using an Agilent Bioanalyzer 2100 system. The RNA from each sample was processed with the Illumina TotalPrep RNA Amplification Kit (Ambion, Austin, TX) to produce labeled cRNA. The cRNA for each sample was then hybridized to an Illumina Sentrix® Mouse-6-V2 BeadChip (Illumina, San Diego, CA)
<p>
<P>                                     
<p>Quality control analysis of the raw image data was performed using the Illumina BeadStudio software.  MIAME standards were used for all microarray data. Rank invariant normalization with BeadStudio software was used to calculate the data.  Once this data was collected, the data was globally normalized across all samples using the formula 2 (z-score of log2 [intensity]) + 8. 
 
<P><B>Replication, sex, and sample balance:</B> Our goal was to obtain data for independent biological sample pools from both sexes for most lines of mice. The four batches of arrays included in this final data set, collectively represent a reasonably well-balanced sample of males and females, in general without within-strain-by-sex replication.
</Blockquote>
	
		</TR></TABLE>
		</TD>
	</TR>
	<TR>
		<TD align=center bgColor=#ddddff class="solidBorder">
		<!--Start of footer-->
		<TABLE width="100%">
			<script language='JavaScript' src='/javascript/footer.js'></script>
			<TR>
				<TD colspan=4 id="smallsize" align="left">
					&nbsp;&nbsp;&nbsp;&nbsp;
				</TD>
			</TR>
		</TABLE>
		<!--End of footer-->
		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	// Note where menu initialization block is located in HTML document.
	// Don't try to position menu locating menu initialization block in
	// some table cell or other HTML element. Always put it before </body>
	// each menu gets two parameters (see demo files)
	// 1. items structure
	// 2. geometry structure
	new menu (MENU_ITEMS, MENU_POS);
	// make sure files containing definitions for these variables are linked to the document
	// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
	// error in menu_tpl.js file or that file isn't linked properly.
	
	// also take a look at stylesheets loaded in header in order to set styles
	//-->

</script>
</BODY>
</HTML>
