<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>CEPH Immortalized B Cells, Agilent, Monks et al. (Oct08)</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>




<P class="title">CEPH Immortalized B Cells, Agilent, Monks et al. (Oct08) <A HREF="/webqtl/main.py?FormID=editHtml">

<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=215">GN215</A></P>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<blockquote>
PUBLISHED DATA SET:  This is the first human data set entered into GeneNetwork and not all features have been implemented. You can currently explore and use the data for expression analysis and correlations among transcripts. However, mapping functions have not been implemented. These data were provided by Stephanie Santorico and are taken from her paper (Monks et al., <A HREF="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=15514893">2004</A>). 

<P>Probes are mapped to UCSC Genome Browser hg18. Please update array annotation to hg19 (RWW to Arthur C. Sept 2009).


<P>GEO Accession at <A HREF="http://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GSE1726">GEO data</A>
<P>GEO Platform information at <A HREF="http://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?acc=GPL564">GEO Rosetta (Merck) custom-commercial array, GPL564</A>




<P>Expression values for each transcript have been centered to a mean of zero. It is not possible to compare the absolute expression levels of transcripts with each other.


</blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases and families used to generate this set of data:</P>


<blockquote>

<P>The text below is taken from Monks et al. (2004). We will add additional new annotation over the next several months.

<P><B>Families</B>: Fifteen families from the CEPH/Utah family collection were selected for profiling. The family identifiers were 1334, 1340, 1345, 1346, 1349, 1350, 1358, 1362, 1375, 1377, 1408, 1418, 1421, 1424, and 1477. These families were selected because of the availability of genotypes and lymphoblastoid cell lines for all three generations and because of their large numbers of children. In total, the families represent 210 individuals. Of these, 167 individuals provided adequate quantity and quality of RNA for expression profiling.

<P><B>Tissue Growth, Processing, and Profiling</B>: Lymphoblastoid cell lines were obtained from Coriell Repositories and propagated. All cell lines were grown in media and supplements purchased from the Invitrogen Corporation. The culture media consisted of RPMI supplemented with 15% fetal bovine serum, 1% penicillin/streptomycin, and 0.5% sodium pyruvate. To minimize variability between experiments, all fetal bovine serum used was from lot number 10082147 1129480. The cell lines were grown at 37°C in humidified incubators, in an atmosphere of 5% CO2.

<P>Experiment series were set up by seeding 25-ml cultures in T25 flasks at a density of 2.5×105 cells/ml. Each culture was grown for 48 h or until the cell density was at least 780,000 cells/ml. To harvest the cells, the cultures were centrifuged, the media was decanted, and 500 μl of guanidine isothiocynate cell lysis buffer (Buffer RLT, Qiagen) was added. Cell lysates were then transferred to 96-well block format and stored at −80°C.

<P>Total RNA was isolated using RNeasy 96 kits (Qiagen) with the following protocol modifications. Harvesting of cells was performed in 500 μl, instead of in the 150 μl specified by the protocol. To eliminate DNA contamination, the appended DNase protocol was used in concert with the isolation protocol. DNase was added to the membrane after the first 350-μl RW1 wash (guanidinium thiocyanate and ethanol) and was allowed to sit on an RNeasy membrane for 30 min. An additional 350-μl RW1 buffer wash and an additional 500-μl RPE buffer wash were performed.

<P>To quantitate and perform quality control on the experiments, the A260/A280 ratio was taken through use of a Spectramax spectrophotometer (Molecular Devices). Samples whose A260/A280 ratio deviated ±0.2 from the accepted ratio value of 2.0 were excluded. Formaldehyde gels (1.2%) were run on each sample to ensure that ribosomal RNA bands were intact and that significant degradation had not occurred. Samples that met the minimal mass requirement of 13 μg (for two replicates) and whose ribosomal bands were visible in the QC gel were transferred from the 96-well block and aliquoted into microcentrifuge tubes by use of a Multiprobe II EX (Packard BioScience Company). For samples of individuals that were to be used in the pool, 46 μg of RNA was allocated by use of the same procedure. In total, 167 individuals in 15 pedigrees provided adequate quantity and quality of RNA for expression profiling.

<P>The microcentrifuge tubes were vacuum dried and stored at −80°C before processing. Dried total RNA samples were reconstituted, and 3 μg of total RNA was used from each sample for subsequent RT-PCR–in vitro transcription amplification using the T7 promoter, which produced allyl-UTP–labeled single-stranded complementary RNA (sscRNA) (Hughes et al. 2001). Amplified cRNA was purified using the RNeasy purification kit (Qiagen) and was coupled with either cy3 or cy5 (Hughes et al. 2001). Purified cy3/cy5-labeled cRNA was fragmented using a ZnOAc/EDTA addition and was hybridized to at least two DNA microarray slides with fluor reversal for 24 h in a hybridization chamber, washed, and scanned using a laser confocal scanner (Hughes et al. 2001). Arrays were quantified on the basis of the intensity of each spot relative to background, by use of the Qhyb program (Rosetta Inpharmatics) (Marton et al. 1998).

<P>Expression profiling of lymphoblastoid cell lines was performed using a 25K human gene oligonucleotide microarray. All individuals were compared with a common pool created from equal portions of RNA from all samples that passed quality control and were from founders within the 15 pedigrees (Gene Expression Omnibus Web site). Sequences for the microarray were selected from the RefSeq database (NCBI Reference Sequence Web site; see the Electronic-Database Information section for genes and accession numbers) and EST contigs (van’t Veer et al. 2002).

<P><B>Genotype Data and Genetic Maps</B>: GENOTYPE DATA HAVE NOT YET BEEN INTEGRATED INTO GENENETWORK. Genotype data for 346 autosomal genetic markers for 210 of the pedigree members were obtained from the CEPH genotype database, version 9.0 (CEPH Genotype Database Web site). Genetic markers were selected from the 14,404 markers represented in the full database, so that at least 75% of the pedigrees had genotypes available for at least 75% of the families. The median intermarker distance was 11 cM, on the basis of the deCODE genetic map (Kong et al. 2002). Marker-allele frequencies available from the CEPH genotype database were used for estimating identity-by-descent probabilities.

<P><B>Statistical Methods </B>:  MORE TO COME:  For each profile, genes were tested to assess differential expression relative to the pool, by use of procedures described elsewhere (Hughes et al. 2000).  For each transcript/probe, the value is measured as the gene expression for an individual compared with that of the pool.

<P>Data provided by Stephanie Monks Santorico, University of Colorado, Denver (Oct 8, 2008). Annotation files to follow late Oct 2008. Mapping functions will not be implemented until 2009.

<P>Data entry by Arthur Centeno, Oct 30, 2008.
<P>Agilent annotation entry by Hongqiang Li and Xusheng Wang, Oct 31, 2008.

<P>This annotation file started by Robert W. Williams, Oct 15, 2008. Most text taken from Monks et al., 2004. Last update, Nov 17, 2008 by RWW.






</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">
<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
<TR>
<TD colspan=3 class="fs12">
<UL>

</UL>
</TD>
</TR>
</TABLE>
<!--End of footer-->
</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->

</script>
</BODY>
</HTML>
