<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>M430 Microarray brain PDNN April05 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">

INIA Brain mRNA M430 (April05) PDNN  <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=58">GN58</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote>
<P>
This April 2005 data freeze provides estimates of mRNA expression in adult forebrain and midbrain from 45 lines of mice including C57BL/6J, DBA/2J, their F1 hybrids, and 42 BXD recombinant inbred strains. Data were generated at UTHSC and the University of Memphis with support from grants from the NIAAA Integrative Neuroscience Initiative on Alcoholism (<a href="http://www.iniastress.org" target="_blank" class="fs14">INIA</a>). Samples were hybridized in small pools (<I>n</I> = 3) to a total of 105 Affymetrix M430A and B array pairs. This particular data set was processed using the <a href="http://odin.mdacc.tmc.edu/~zhangli/PerfectMatch/" target="_blank" class="fs14">PDNN</a> method of Zhang. To simplify comparison among transforms, PDNN values of each array were adjusted to an average of 8 units and a standard deviation of 2 units.
</P>
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>
We have used a set of BXD recombinant inbred strains generated by crossing C57BL/6J (B6 or B) with DBA/2J (D2 or D). The BXDs are particularly useful for systems genetics because both parental strains have been sequenced (8x coverage of B6 and 1.5x coverage of D). Physical maps in WebQTL incorporate approximately 2 million B vs D SNPs from <a href="http://www.celeradiscoverysystem.com/index.cfm" target="_blank" class="fs14">Celera Genomics</a> and from the Perlegen-NIEHS sequencing effort. BXD2 through BXD32 were bred by Benjamin A. Taylor starting in the late 1970s. BXD33 through 42 were bred by Taylor in the 1990s. These strains are available from The Jackson Laboratory. BXD43 through BXD99 were bred by Lu Lu, Jeremy Peirce, Lee M. Silver, and Robert W. Williams in the late 1990s and early 2000s using advanced intercross progeny (Peirce et al. <a href="http://www.biomedcentral.com/1471-2156/5/7"   target="_blank" class="fs14">2004</a>). Many of the 50 new BXD strains are available from <A HREF="mailto:lulu@nb.utmem.EDU" target="_blank" class="fs14">Lu Lu</A> and colleagues.</P>

<P> All stock was obtained originally from The Jackson Laboratory between 1998 and 2003. Most BXD animals were born and housed at the University of Tennessee Health Science Center. Some cases were bred at the University of Memphis (Douglas Matthews) or the University of Alabama (John Mountz and Hui-Chen Hsu).
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate this set of data:</P>

<Blockquote>The <a href="http://www.iniastress.org" target="_blank" class="fs14">INIA</a> M430 brain Database (April05) consists of 105 Affymetrix 430A and 430B microarray pairs. Each pair was hybridized in sequence (A array first, B array second) with a pool of brain tissue (forebrain minus olfactory bulb, plus the entire midbrain) taken from three adult animals of closely matched age and the same sex. RNA was extracted at UTHSC by Lu Lu, Zhiping Jia, and Hongtao Zhai at UTHSC.

<P><B>Tissue preparation protocol</B>. Animal were killed by rapid cervical dislocation. Eyes were removed immediately and placed in RNA<I>later</I> at room temperature. Usually six eyes from animals with a common sex, age, and strain were stored in a single tube. The body was sprayed lightly with 70% ethanol to wet the hair. the following standard approach was used to extract the brain:

<OL>
<LI>Using small surgical scissors make an incision under the skin on the dorsal side of the neck. Cut the skin overlying the skull close to the midsagittal plane towards the nose. Pull and reflect the skin to expose the entire dorsal skull.
<LI>Slip the points of the scissors through into the cisterna magna just caudal to the cerebellum and gently enlarge this opening until is it possible to cut through the skull overlying the cerebellum.
<LI>Cut rostrally through the skull along the midsagittal line almost all the way to the nasal opening, taking care not to damage the dorsal surface of the brain.
<LI>Approximately midway along this incision, make a lateral cut. Repeat along the incision and peel back the resulting strips of skull.
<LI>Using small forceps, free the olfactory bulbs rostrally and ventrally, taking care to retain their connection to the rest of the forebrain.
<LI>Gently lift the brain from the base the skull starting from the olfactory bulbs, pulling the brain toward a nearly vertical position. Cut the optic and trigeminal nerves. Separate the brain from the spinal cord about 2 mm distal to the medulla.
<LI>Spread the hemispheres of the forebrain gently with forceps and then cut from dorsal to ventral using a straight scalpel, separating the hemispheres from each other (but not from the cerebellum). Take care to retain both paraflocculi.
</OL>

At this point the protocol divides. If tissue is to be saved for RNA extraction at a later time, the whole brain is placed directly in RNA<i>later</i> (Ambion, Inc.) and treated per the manufacturer’s directions.  Step 7 is still very important because RNA<i>later</i> may not fully penetrate the forebrain if the lobes are not separated. If tissue is to be used for immediate RNA extraction, one lobe of the forebrain is removed for processing and the rest of the brain is stored in RNA<i>later</i>.<p>

Dissecting and preparing forebrain and midbrain for RNA extraction
<OL>
<LI>Remove the left or right hemisphere of the forebrain and midbrain (referred to here as the forebrain for simplicity), either fresh or preserved in RNA<i>later</i> by cutting from the caudal border of the inferior colliculus on the dorsal side and extending the cut ventrally to the basis pedunculi and the pons (cut just rostral of the pons) on the ventral side. See steps 7 and 8 <a href="http://www.mbl.org/anatomy_images/fixed/mba_1.html">here</a>
<LI>Place tissue for RNA extraction in RNA STAT-60 (Tel-Test Inc.) and process per manufacturer’s instructions (in brief form below).
<LI>Store RNA in 75% ethanol at –80 deg. C until use.
</OL>

<!--
Hippocampus: Rule of Use:  In general, if we had three animals in a pool, we would dissect one hippocampus per mouse and these three hippocampii from three animals would be used togehter. If we had only two animals, then we would dissect all four hippocampii and use them together.
We extracted some hippocampus samples immediately from fresh tissue without RNAlater explosure. However, most samples were dissected immediately. Instead, the hippocampus was dissected from whole hemispheres that had been exposed to RNAlater and that had been stored at -80 deg C. Hemispheres were thawed, and the hippocampus was dissected.
-->

<P>Total RNA was extracted with RNA STAT-60 (Tel-Test) according to the manufacturer’s instructions. Briefly we:
<OL>
<LI>homogenize tissue samples in the RNA STAT-60 (1 ml/50 to 100 mg tissue)
<LI>allowed the homogenate to stand for 5 min at room temperature
<LI>added 0.2 ml of chloroform per 1 ml RNA STAT-60
<LI>shook the sample vigorously for 15 sec and let the sample sit at room temperature for 3 min
<LI>centrifuged at 12,000 G for 15 min
<LI>transfered the aqueous phase to a fresh tube
<LI>added 0.5 ml of isopropanol per 1 ml RNA STAT-60
<LI>vortexed and allowed sample to stand at room temperature for 5-10 min
<LI>centrifugeed at 12,000 G for 10-15 min
<LI>removed the supernatant and washed the RNA pellet with 75% ethanol
<LI>stored the pellet in 75% ethanol at -80 deg C until use
</OL>


<P><B>Sample Processing</B>. Samples were processed in the INIA Bioanalytical Core at the W. Harry Feinstone Center of Excellence, The University of Memphis, lead by Thomas R. Sutter. All processing steps were performed by Shirlean Goodwin. In brief, samples were quality control checked for RNA purity using 260/280 ratios (samples had to be greater than 1.8, but the majority were 1.9 or higher). RNA integrity was assessed using the Agilent Bioanalyzer 2100. We required an RNA integrity number (RIN) of greater than 8, based on the intensity ratio and amplitude of 18S and 28S rRNA signals. The standard Eberwine T7 polymerase method was used to catalyze the synthesis of cDNA template from polyA-tailed RNA using Superscript II RT (Invitrogen Inc.).  The Enzo LIfe Sciences, Inc., BioArray High Yield RNA Transcript Labeling Kit (T7, Part No. 42655) was used to synthesize labeled cRNA. At this point the cRNA was evaluated again using both the 260/280 ratio (values of 2.0 or above are acceptable) and the Bioanalyzer output (a dark cRNA smear on the 2100 output centered roughly between 600 and 2000 nt is required). Those samples that passed both QC steps (10% usually fail) were then sheared using a fragment buffer included in the Affymetrix GeneChip Sample Cleanup Module (Part No.900371). After fragmentation, samples were either stored at -80 deg. centrigrade until use or were immediately injected onto the array.

<P><B>Replication and Sample Balance:</B> Our goal is to obtain data for independent biological sample pools from at least one of sample from each sex for all BXD strains. We have not yet achieved this goal. Ten of 45 strains are still represented by single sex samples: BXD2 (F), BXD8 (F), BXD15 (F), BXD18 (F), BXD25 (F), BXD29 (F), BXD33 (M), BXD45 (F), BXD77 (M), and BXD90 (M). Eleven strains are represented by three independent samples with the following breakdown by sex: C57BL/6J (1F 2M), DBA/2J (2F 2M), B6D2F1 (2F 2M) + D2B6F1 (1F 1M), BXD6 (2F 1M), BXD13 (2F 1M), BXD14 (1F 2M), BXD28 (2F 1M), BXD34 (1F 2M), BXD36 (1F 2M), BXD38 (1F 2M), BXD42 (1F 2M).

<P><B>Batch Structure:</B> Before running the first batch of 30 pairs of array (dated Jan04), we ran four test samples (Nov03). The main batch of 30 includes the four test samples (four technical replicates). The Nov03 data was combined with the Jan04 data and was treated as a single batch that consists of one male and one female pool from C57BL/6J, DBA/2J, the B6D2F1 hybrid, 11 female BXD samples, and 11 male BXD samples. The second large batch was run February 2005 (Feb05) and consists of 71 pairs of arrays. Batch effects were corrected at the individual probe level as described below.


<P>The table below summarizes information on strain, sex, age, sample name, batch result date, and source of mice.

</Blockquote>
<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="7" cellspacing="1">
<tr bgcolor="royalblue"><td><font color=#FFFFFF>Id</font></td><td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>Sex</font></td>
<td><font color=#FFFFFF>Age</font></td>
<td><font color=#FFFFFF>Sample_name</td>
<td><font color=#FFFFFF>Result date</font></td><td><font color=#FFFFFF>Sourec</font></td></tr>
<tr bgcolor="#eeeeee"><td>1</td><td>C57BL/6J</td><td>F</td><td>65</td><td>R0903F1</td><td>Nov03</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>2</td><td>C57BL/6J</td><td>F</td><td>65</td><td>R0903F1</td><td>Jan04</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>3</td><td>C57BL/6J</td><td>M</td><td>66</td><td>R0906F1</td><td>Nov03</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>4</td><td>C57BL/6J</td><td>M</td><td>66</td><td>R0906F1</td><td>Jan04</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>5</td><td>C57BL/6J</td><td>M</td><td>66</td><td>R0906F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>6</td><td>C57BL/6J</td><td>M</td><td>76</td><td>R0997F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>7</td><td>D2B6F1</td><td>F</td><td>57</td><td>R1066F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>8</td><td>D2B6F1</td><td>M</td><td>59</td><td>R1381F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>9</td><td>DBA/2J</td><td>F</td><td>60</td><td>R0917F1</td><td>Nov03</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>10</td><td>DBA/2J</td><td>F</td><td>60</td><td>R0917F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>11</td><td>DBA/2J</td><td>F</td><td>60</td><td>R0917F2</td><td>Jan04</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>12</td><td>DBA/2J</td><td>F</td><td>64</td><td>R1123F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>13</td><td>DBA/2J</td><td>M</td><td>60</td><td>R0918F1</td><td>Nov03</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>14</td><td>DBA/2J</td><td>M</td><td>60</td><td>R0918F1</td><td>Jan04</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>15</td><td>DBA/2J</td><td>M</td><td>73</td><td>R1009F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>16</td><td>B6D2F1</td><td>F</td><td>127</td><td>R0919F1</td><td>Jan04</td><td>UTM JB</td></tr>
<tr bgcolor="#eeeeee"><td>17</td><td>B6D2F1</td><td>F</td><td>127</td><td>R0919F2</td><td>Jan04</td><td>UTM JB</td></tr>
<tr bgcolor="#eeeeee"><td>18</td><td>B6D2F1</td><td>F</td><td>64</td><td>R1053F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>19</td><td>B6D2F1</td><td>F</td><td>64</td><td>R1053F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>20</td><td>B6D2F1</td><td>M</td><td>127</td><td>R0920F1</td><td>Jan04</td><td>UTM JB</td></tr>
<tr bgcolor="#eeeeee"><td>21</td><td>B6D2F1</td><td>M</td><td>127</td><td>R0920F2</td><td>Jan04</td><td>UTM JB</td></tr>
<tr bgcolor="#eeeeee"><td>22</td><td>B6D2F1</td><td>M</td><td>66</td><td>R1057F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>23</td><td>BXD1</td><td>M</td><td>181</td><td>R0956F1</td><td>Feb05</td><td>UTM JB</td></tr>
<tr bgcolor="#eeeeee"><td>24</td><td>BXD1</td><td>F</td><td>95</td><td>R0895F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>25</td><td>BXD2</td><td>F</td><td>142</td><td>R0907F1</td><td>Feb05</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>26</td><td>BXD5</td><td>F</td><td>56</td><td>R0744F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>27</td><td>BXD5</td><td>M</td><td>71</td><td>R0728F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>28</td><td>BXD6</td><td>F</td><td>57</td><td>R1711F1</td><td>Feb05</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>29</td><td>BXD6</td><td>F</td><td>92</td><td>R0901F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>30</td><td>BXD6</td><td>M</td><td>92</td><td>R0902F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>31</td><td>BXD8</td><td>F</td><td>72</td><td>R0167F1</td><td>Jan04</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>32</td><td>BXD9</td><td>F</td><td>86</td><td>R0908F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>33</td><td>BXD9</td><td>M</td><td>86</td><td>R0909F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>34</td><td>BXD11</td><td>F</td><td>97</td><td>R0745F1</td><td>Feb05</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>35</td><td>BXD11</td><td>M</td><td>92</td><td>R0666F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>36</td><td>BXD12</td><td>F</td><td>64</td><td>R0896F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>37</td><td>BXD12</td><td>M</td><td>64</td><td>R0897F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>38</td><td>BXD13</td><td>F</td><td>86</td><td>R0730F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>39</td><td>BXD13</td><td>F</td><td>86</td><td>R0748F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>40</td><td>BXD13</td><td>M</td><td>76</td><td>R0929F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>41</td><td>BXD14</td><td>M</td><td>91</td><td>R0912F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>42</td><td>BXD14</td><td>M</td><td>68</td><td>R1051F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>43</td><td>BXD15</td><td>F</td><td>80</td><td>R0928F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>44</td><td>BXD18</td><td>F</td><td>108</td><td>R0771F1</td><td>Jan04</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>45</td><td>BXD19</td><td>M</td><td>157</td><td>R1229F1</td><td>Feb05</td><td>UTM JB</td></tr>
<tr bgcolor="#eeeeee"><td>46</td><td>BXD19</td><td>F</td><td>56</td><td>R0236F1</td><td>Jan04</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>47</td><td>BXD21</td><td>F</td><td>67</td><td>R0740F1</td><td>Jan04</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>48</td><td>BXD21</td><td>F</td><td>67</td><td>R0740F1</td><td>Feb05</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>49</td><td>BXD23</td><td>F</td><td>66</td><td>R1035F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>50</td><td>BXD23</td><td>M</td><td>66</td><td>R1037F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>51</td><td>BXD23</td><td>F</td><td>88</td><td>R0815F1</td><td>Jan04</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>52</td><td>BXD23</td><td>F</td><td>88</td><td>R0815F1</td><td>Feb05</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>53</td><td>BXD24</td><td>F</td><td>71</td><td>R0914F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>54</td><td>BXD24</td><td>M</td><td>71</td><td>R0913F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>55</td><td>BXD25</td><td>F</td><td>74</td><td>R0373F1</td><td>Jan04</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>56</td><td>BXD28</td><td>F</td><td>79</td><td>R0910F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>57</td><td>BXD28</td><td>M</td><td>79</td><td>R0911F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>58</td><td>BXD28</td><td>F</td><td>113</td><td>R0892F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>59</td><td>BXD29</td><td>F</td><td>76</td><td>R0693F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>60</td><td>BXD31</td><td>F</td><td>61</td><td>R1199F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>61</td><td>BXD31</td><td>M</td><td>61</td><td>R1141F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>62</td><td>BXD32</td><td>F</td><td>93</td><td>R0898F1</td><td>Jan04</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>63</td><td>BXD32</td><td>F</td><td>76</td><td>R1214F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>64</td><td>BXD32</td><td>M</td><td>65</td><td>R1478F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>65</td><td>BXD33</td><td>M</td><td>77</td><td>R0915F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>66</td><td>BXD34</td><td>F</td><td>92</td><td>R0900F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>67</td><td>BXD34</td><td>M</td><td>56</td><td>R0617F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>68</td><td>BXD34</td><td>M</td><td>72</td><td>R0916F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>69</td><td>BXD36</td><td>F</td><td>61</td><td>R1145F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>70</td><td>BXD36</td><td>M</td><td>77</td><td>R0926F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>71</td><td>BXD36</td><td>M</td><td>61</td><td>R1211F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>72</td><td>BXD38</td><td>M</td><td>83</td><td>R1208F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>73</td><td>BXD38</td><td>F</td><td>69</td><td>R0729F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>74</td><td>BXD38</td><td>M</td><td>69</td><td>R0731F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>75</td><td>BXD39</td><td>F</td><td>76</td><td>R1712F1</td><td>Feb05</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>76</td><td>BXD39</td><td>M</td><td>71</td><td>R0602F1</td><td>Feb05</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>77</td><td>BXD40</td><td>F</td><td>184</td><td>R0741F1</td><td>Feb05</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>78</td><td>BXD40</td><td>M</td><td>56</td><td>R0894F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>79</td><td>BXD42</td><td>F</td><td>100</td><td>R0742F1</td><td>Feb05</td><td>UAB</td></tr>
<tr bgcolor="#eeeeee"><td>80</td><td>BXD42</td><td>M</td><td>97</td><td>R0936F1</td><td>Jan04</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>81</td><td>BXD42</td><td>M</td><td>105</td><td>R0937F1</td><td>Feb05</td><td>UMemphis</td></tr>
<tr bgcolor="#eeeeee"><td>82</td><td>BXD43</td><td>M</td><td>63</td><td>R1047F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>83</td><td>BXD44</td><td>F</td><td>57</td><td>R1069F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>84</td><td>BXD44</td><td>M</td><td>58</td><td>R1072F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>85</td><td>BXD45</td><td>F</td><td>58</td><td>R1398F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>86</td><td>BXD48</td><td>F</td><td>59</td><td>R0946F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>87</td><td>BXD48</td><td>M</td><td>64</td><td>R0970F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>88</td><td>BXD51</td><td>F</td><td>63</td><td>R1430F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>89</td><td>BXD51</td><td>M</td><td>65</td><td>R1001F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>90</td><td>BXD60</td><td>F</td><td>64</td><td>R0976F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>91</td><td>BXD60</td><td>M</td><td>59</td><td>R1075F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>92</td><td>BXD62</td><td>F</td><td>59</td><td>R1033F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>93</td><td>BXD62</td><td>M</td><td>58</td><td>R1027F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>94</td><td>BXD69</td><td>F</td><td>60</td><td>R1438F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>95</td><td>BXD69</td><td>M</td><td>64</td><td>R1193F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>96</td><td>BXD73</td><td>F</td><td>60</td><td>R1275F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>97</td><td>BXD73</td><td>M</td><td>76</td><td>R1442F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>98</td><td>BXD77</td><td>M</td><td>61</td><td>R1426F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>99</td><td>BXD86</td><td>F</td><td>77</td><td>R1414F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>100</td><td>BXD86</td><td>M</td><td>77</td><td>R1418F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>101</td><td>BXD87</td><td>F</td><td>89</td><td>R1713F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>102</td><td>BXD87</td><td>M</td><td>84</td><td>R1709F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>103</td><td>BXD90</td><td>M</td><td>61</td><td>R1452F</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>104</td><td>BXD92</td><td>F</td><td>58</td><td>R1299F1</td><td>Feb05</td><td>UTM RW</td></tr>
<tr bgcolor="#eeeeee"><td>105</td><td>BXD92</td><td>M</td><td>59</td><td>R1307F1</td><td>Feb05</td><td>UTM RW</td></tr>              </table>
</td>
</tr>
</table>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platfrom :</P>
<Blockquote>
<P><B>Affymetrix Mouse Genome 430A and 430B array pairs: </B>The <A HREF="http://www.affymetrix.com/support/technical/byproduct.affx?product=moe430-20" target="_blank" class="normal">430A and B</A> array pairs collectively consist of 992936 25-nucleotide probes that estimate the expression of approximately 39,000 transcripts (many are essentially duplicates). The array sequences were selected late in 2002 using Unigene Build 107. The arrays nominally contain the same probe sequence as the 430 2.0 series. However, we have found that roughy 75000 probes differ between those on A and B arrays and those on the 430 2.0.</P>
</Blockquote>




<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a target="_blank" class="fs14" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.
<UL>

<LI>Step 1: We added an offset of 1.0 unit to each cell signal to ensure that all values could be logged without generating negative values. We then computed the log base 2 of each cell.

<LI>Step 2: We performed a quantile normalization of the log base 2 values for the total set of 105 arrays (processed as two batches) using the same initial steps used by the RMA transform.

<LI>Step 3: We computed the Z scores for each cell value.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6: We eliminated much of the systematic technical variance introduced by the two batches (n = 34 and n = 71 array pairs) at the probe level. To do this we calculated the ratio of each batch mean to the mean of both batches and used this as a single multiplicative probe-specific batch correction factor. The consequence of this simple correction is that the mean probe signal value for each batch is the same.

<LI>Step 7a: The 430A and 430B arrays include a set of 100 shared probe sets (a total of 2200 probes) that have identical sequences. These probes and probe sets provide a way to calibrate expression of the 430A and 430B arrays to a common scale. To bring the two arrays into alignment, we regressed Z scores of the common set of probes to obtain a linear regression correction to rescale the 430B arrays to the 430A array. In our case this involved multiplying all 430B Z scores by the slope of the regression and adding or subtracting a small offset. The result of this step is that the mean of the 430A expression is fixed at a value of 8, whereas that of the 430B chip is typically reduced to 7. The average of the merged 430A and 430B array data set is approximately 7.5.

<LI>Step 7b: We recentered the merged 430A and 430B data sets to a mean of 8 and a standard deviation of 2. This involved reapplying Steps 3 through 5.

<LI>Step 8: Finally, we computed the arithmetic mean of the values for the set of microarrays for each strain. Technical replicates were averaged before computing the mean for independent biological samples. Note, that we have not (yet) corrected for variance introduced by differences in sex, age, source of animals, or any interaction terms. We have not corrected for background beyond the background correction implemented by Affymetrix in generating the CEL file. We eventually hope to add statistical controls and adjustments for some of these variables.

</UL>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>
Support for acquisition of microarray data were generously provided by the NIAAA and its INIA grant program to RWW, Thomas Sutter, and Daniel Goldowitz (U01AA013515, U01AA013499-03S1, U01AA013488, U01AA013503-03S1). Support for the continued development of the GeneNetwork and WebQTL was provided by a NIMH Human Brain Project grant (P20MH062009). All arrays were processed at the University of Memphis by Thomas Sutter and colleagues with support of the INIA Bioanalytical Core.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>This text file originally generated by RWW, YHQ, and EJC, Oct 2004. Updated by RWW, Nov 5, 2004; April 7, 2005; RNA/tissue preparation protocol updatedby JLP, Sept 2, 2005; Sept 26, 2005.
</P></Blockquote>

<P></P>

</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
