<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>M430 Microarray brain February04 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">

<P class="title">
INIA Brain mRNA M430 (Feb04) MAS5

 <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=11">GN11</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>
The February 2004 freeze provides estimates of mRNA expression in brains of BXD recombinant inbred mice measured using the Affymetrix MOE430 microarrays that replaced the U74 series of arrays in 2003. Data were generated at the University of Tennessee Health Science Center (UTHSC) as part of an research project funded by the <a href="http://www.iniastress.org/" target="_blank" class="fs14">NIAAA</A>. Brain samples from BXD strains were hybridized in small pools (n=3) to M430A and M430B arrays. Data were processed using the Microarray Suite 5 (<a href="www.affymetrix.com/support/technical/whitepapers/sadd_whitepaper.pdf" class="fs14">MAS 5</a>) protocol of Affymetrix. To simplify comparison between transforms, MAS 5 values of each array were adjusted to an average of 8 units and a variance of 2 units. This data set was essentially run as a single large batch with careful consideration to balancing samples by sex and age.
</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>We have exploited a set of BXD recombinant inbred strains. The parental strains from which all BXD lines are derived are C57BL/6J (B6 or B) and DBA/2J (D2 or D). Both B and D strains have been almost fully sequence (8x coverage for B6 by a public consortium and approximately 1.5x coverage for D by <a href="http://www.celeradiscoverysystem.com/index.cfm" target="_blank" class="fs14">Celera</a>). Chromosomes of the two parental strains have been recombined and fixed randomly in the many different BXD strains. BXD2 through BXD32 were produced by Benjamin A. Taylor starting in the late 1970s. BXD33 through 42 were also produced by Taylor, but they were generated in the 1990s. These strains are all available from the Jackson Laboratory, Bar Harbor, Maine. BXD43 through BXD99 were produced by Lu Lu, Jeremy Peirce, Lee M. Silver, and Robert W. Williams in the late 1990s and early 2000s using advanced intercross progeny (Peirce et al. <a href="http://www.biomedcentral.com/1471-2156/5/7"  class="fs14">2004</a>).</P>

<P>In this mRNA expression data set we generally used progeny of stock obtained from The Jackson Laboratory between 1999 and 2001. Animals were generated in-house at the University of Alabama by John Mountz and Hui-Chen Hsu and at the University of Tennessee Health Science Center by Lu Lu and Robert Williams.
</P></Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>

<Blockquote>This <a href="http://www.iniastress.org" target="_blank" class="fs14">INIA</a> M430 brain Database (February04) consists of 30 pairs of Affymetrix 430A and 430B arrays. Each pair was hybridized in succession (A then B) with cRNA generated from a pool of three brains from adult mice of the same age and sex. The brain region included most of the forebrain and midbrain, bilaterally. This sample excluded the olfactory bulbs, retinas, or the posterior pituitary (all formally part of the forebrain). </P>

<P>RNA was extracted at UTHSC by Lu Lu, Zhiping Jia, and Hongtao Zhai.</P>

<P>All samples were subsequently processed in the INIA Bioanalytical Core at the W. Harry Feinstone Center of Excellence by Thomas R. Sutter and colleagues at the University of Memphis. Before running the main batch of 30 pairs of array, we ran four test samples (one male and one female pool from each of the two parental strains, C57BL/6J and DBA/2J). The main batch of 30 array pairs includes the same four samples (in other words we have four technical replicates shared between the test and a single main batch), two F1 hybrid sample (each run two times for within-batch technical replication), and 22 BXD strains. The February04 data set therefore consists of one male and one female pool from C57BL/6J, DBA/2J, the B6D2F1 hybrid, 11 female BXD samples, and 11 male BXD samples. We should note that the four technical replicates were eventually combined with a correction for a highly significant batch effect. This was done at both the probe and probe set levels to numerically align the test batch values with the two main batches. (The ratio of the probe average in the four test arrays to the average of the same probe in the four corresponding main batch arrays was used as a correction factor.)  The F1 within-batch technical replicates were simply averaged. In the next batch we will reverse the sex of the BXD samples to achieve a balance with at least 22 BXD strains with one male and one female sample each.
</Blockquote>
<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="5" cellspacing="1">
<tr bgcolor="royalblue"><td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>Sex</font></td>
<td><font color=#FFFFFF>Age</font></td>
<td><font color=#FFFFFF>Sample_name</td>
<td><font color=#FFFFFF>Result date</font></td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>F</td><td>127</td><td>919-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>F</td><td>127</td><td>919-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>M</td><td>127</td><td>920-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>B6D2F1</td><td>M</td><td>127</td><td>920-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>F</td><td>65</td><td>903-F1</td><td>Nov03</td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>F</td><td>65</td><td>903-F2</td><td>Jan03</b></td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>M</td><td>66</td><td>906-F1</td><td>Nov03</td></tr>
<tr bgcolor="#eeeeee"><td>C57BL/6J</td><td>M</td><td>66</td><td>906-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>F</td><td>60</td><td>917-F1</td><td>Nov03</td></b></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>F</td><td>60</td><td>917-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>M</td><td>60</td><td>918-F1</td><td>Nov03</td></tr>
<tr bgcolor="#eeeeee"><td>DBA/2J</td><td>M</td><td>60</td><td>918-F2</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD1</td><td>F</td><td>95</td><td>895-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD5</td><td>M</td><td>71</td><td>728-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD6</td><td>M</td><td>92</td><td>902-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD8</td><td>F</td><td>72</td><td>S167-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD9</td><td>M</td><td>86</td><td>909-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD12</td><td>M</td><td>64</td><td>897-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD13</td><td>F</td><td>86</td><td>748-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD14</td><td>M</td><td>91</td><td>912-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD18</td><td>F</td><td>108</td><td>771-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD19</td><td>F</td><td>56</td><td>S236-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD21</td><td>F</td><td>67</td><td>740-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD23</td><td>F</td><td>88</td><td>815-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD24</td><td>M</td><td>71</td><td>913-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD25</td><td>F</td><td>74</td><td>S373-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD28</td><td>F</td><td>79</td><td>910-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD29</td><td>F</td><td>76</td><td>693-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD32</td><td>F</td><td>93</td><td>898-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD33</td><td>M</td><td>77</td><td>915-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD34</td><td>M</td><td>72</td><td>916-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD36</td><td>M</td><td>77</td><td>926-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD38</td><td>M</td><td>69</td><td>731-F1</td><td>Jan04</td></tr>
<tr bgcolor="#eeeeee"><td>BXD42</td><td>M</td><td>97</td><td>936-F1</td><td>Jan04</td></tr>

</table>
</td>
</tr>
</table>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>

<Blockquote><B>Affymetrix MOE430 GeneChip Set:</B> The expression data were generated using MOE430A and MOE430B arrays. Chromosomal positions of probe sets were determined by BLAT analysis of concatenated probe sequences using the Mouse Genome Sequencing Consortium May 2004 (mm5) assembly. This BLAT analysis is performed periodically by Yanhua Qu as each new build of the mouse genome is released (see <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse" class="fs14">http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse</a>). We thank Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis. It is possiible to confirm the BLAT alignment results yourself simply by clicking on the <B>Verify</B> link in the Trait Data and Editing Form (right side of the <B>Location</B> line).
</P></Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a target="_blank" class="fs14" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.
<UL>
<LI>Step 1: We added a constant offset of 1 to the CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log2 of each cell signal level.

<LI>Step 3: We computed the Z scores for each cell.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8 units, a variance of 4 units, and a standard deviation of 2 units. The advantage of this modified Z score is that a 2-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6a: The 430A and 430B GeneChips include a set of 100 shared probe sets (2200 probes) that have identical sequences. These 2200 probes and 100 probe sets provide a way to calibrate expression of the two GeneChips to a common scale. The absolute mean expression on the 430B array is almost invariably lower than that on the 430A array (the A array contains the more commonly expressed transcripts). To bring the two arrays into numerical alignment, we regressed Z scores of the common set of 2200 probes to obtain a linear regression corrections to rescale the 430B arrays to values that match the 430A array. In our case this involved multiplying all 430B Z scores by the slope of the regression and adding or subtracting a very small offset (the regression intercept). The result of this adjustment is that the mean of the 430A GeneChip expression is fixed at a value of 8, whereas that of the 430B chip is typically 7. Thus average of A and B arrays is approximately 7.5.

<LI>Step 6b: We recentered the entire combined set of 430A and B transcripts to a mean of 8 and a standard deviation of 2. This involves reapplying Steps 3 through 5 above but now using the entire set of probes and probe sets from a merged 430A and B data set.

<LI>Step 7: When necessary, we correct for technical variance introduced by running multiple batches. However, this data set is essentially a single batch with a few technical replicates in a first test batch.

<LI>Step 8: Finally, we compute the arithmetic mean of the values for the set of microarrays for each strain. In this data set we have only a very modest number of replicates and for this reason we do not yet provide error terms for transcripts or probes. Note, that this data set does not provide any correction for variance introduced by differences in sex, age, tissue source, or any interaction terms. We have not corrected for background beyond the background correction implemented by Affymetrix in generating the CEL file. We expect to add statistical controls and adjustments for these variables in subsequent versions of WebQTL.

</UL>
<B>Probe set data from the CHP file: </B>These CHP files were generated using MAS 5. The same simple steps described above were also applied to these values. A 1-unit difference represents roughly a 2-fold difference in expression level. Expression levels below 5 are close to the noise level.
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>
Array data were generated with funds from the NIAAA INIA to RWW and Thomas Sutter. Informatics resources are supported primarily by an NIMH/NIDA Human Brain Project. All arrays were processed at the University of Memphis by Thomas Sutter and colleagues with support of the INIA Bioanalytical Core.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>
This text file originally generated by RWW, YHQ, and EJC, March 2004. Updated by RWW, October 30, 2004.


</P></Blockquote>




<P></P>

</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
