<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>RAE230A Microarray Kidney MAS5 April05 /
WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">MDC/CAS/ICL RAE230A Kidney Database MAS5 (April/05 freeze)   <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=70">GN70</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>
This April 2005 data set provides estimates of mRNA expression in normal kidneys of 32 strains of rats. The set includes the hypertensive SHR strain, the normotensive BN strain, and 30 HXB/BXH recombinant inbred strains. Each strain was sampled in quadruplicate (6-week-old males). Animals and tissues were generated by Michal Pravenec and colleagues at the Czech Academy of Sciences (CAS). RNA samples were processed at the Max-Delbr&uuml;ck-Center (MDC), Berlin Buch by <A HREF="http://www.mdc-berlin.de/englisch/research/e_index.htm" target="_blank" class="fs14">Nobert H&uuml;bner</A> and colleagues. Transcriptome mapping was carried out by Timothy Aitman and colleagues at the Imperial College, London (ICL). Samples were hybridized individually to a total of 128 Affymetrix RAE230A array. This particular data set includes 120 arrays processed using a quantile normalized variant of the Affymetrix MAS5 protocol. The expression values of each array have been logged and adjusted to a mean of 8 and a stardard deviation of 2 (mean and variance stabilized).  This data set complements the original MAS5 data set exploited by H&uuml;bner and colleagues <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15711544" target="_blank" class="fs14">2005</A>.
<A HREF="./array_data/Rat_230A_Kidney/Transform/KI_2A_0405_M.xls" class="fs14">Download</A> the particular transform in an Excel work book with both strain means and SEMs.</P>

<P>These data can also be viewed using the <A HREF="http://web.bioinformatics.ic.ac.uk/eqtlexplorer/" target="_blank" class="fs14">eQTL Explorer</A> Java application
by John Mangion, Tim Aitman, and colleagues (Mueller et al. <A HREF="http://bioinformatics.oxfordjournals.org/cgi/reprint/btk007v1.pdf
" target="_blank" class="fs14">2006</A>).




</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>
<Blockquote>Data were generated using the HXB/BXH recombinant inbred strains of rats generated over the past 20 years in Prague. The parental strains from which all HXB lines are derived are SHR (SHR/OlaIpcv or HSR = H) and Brown Norway (BN.Lx/Cub= B). These strains have been used extensively to study cardiovascular system physiology and genetics.</P>

<P>The HXB strains were bred by Michal Pravenec at the Institute of Physiology, Czech Academy of Sciences. The BXH strains were bred by Vladimir Kren (see Pravenec et al. 1989, 2004) at a similar animal facility at the Institute of Biology and Medical Genetics, 1st Medical Faculty, Charles University. These strains are at approximately the 6oth geenration of continuous inbreeding (F60).</P>

<P>Animals used in the transcriptome analyses of kidney and fat (H&uuml;bner and colleagues, <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15711544" target="_blank" class="fs14">2005</A>) were weaned at 4 weeks. Those born at the Charles University were transferred to the Institute of Physiology. Animals were reared on a commerical rat chow (ST-1 from VELAZ, Czech Republic). Four males were house per cage. Cages were made of polystyrene and have a floor size of 22 x 38 cm and height of 23 cm. The bedding was changed twice a week. Light cycle was 12:12 on-off. Vivarium rooms were maintained at 23 deg. C. Rats were sexually naive.  All males used in the initial transcriptome studies (Hubner et al., 2005) were born between May and August 2002. They were sacrificed unfastged by rapid cervical dislocation between 9 and 10 AM, following an approved animal protocol (Ethics Committee of the Institute of Physiology, Czech Academy of Sciences, Prague; Animal Protectiion Law of the Czech Republic (311/1997).
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>
<Blockquote>All tissues were collected at the age of 6 weeks. Kidneys and other organs were rapidly dissected and cleaned of fat, inserted into a vial, and immersed in liquid nitrogen for storage until RNA extraction.
</Blockquote>


<Blockquote>
The table below lists the arrays by strain and sample identifier.  Each array was hybridized with mRNA from a single young male rat.
</Blockquote>

<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="50%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="5" cellspacing="1">
<tr bgcolor="royalblue">
<td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>SampleID</font></td>
</tr>
<tr bgcolor="#eeeeee"><td>HSR</td><td>HSR1</td>
<tr bgcolor="#eeeeee"><td>HSR</td><td>HSR2</td>
<tr bgcolor="#eeeeee"><td>HSR</td><td>HSR3</td>
<tr bgcolor="#eeeeee"><td>HSR</td><td>HSR4</td>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN1</td>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN2</td>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN3</td>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN4</td>
<tr bgcolor="#eeeeee"><td>BN</td><td>BN5</td>
<tr bgcolor="#eeeeee"><td>HXB1</td><td>RI 01-1</td>
<tr bgcolor="#eeeeee"><td>HXB1</td><td>RI 01-2</td>
<tr bgcolor="#eeeeee"><td>HXB1</td><td>RI 01-3</td>
<tr bgcolor="#eeeeee"><td>HXB1</td><td>RI 01-4</td>
<tr bgcolor="#eeeeee"><td>HXB2</td><td>RI 02-1</td>
<tr bgcolor="#eeeeee"><td>HXB2</td><td>RI 02-2</td>
<tr bgcolor="#eeeeee"><td>HXB2</td><td>RI 02-3</td>
<tr bgcolor="#eeeeee"><td>HXB2</td><td>*RI 02-4</td>
<tr bgcolor="#eeeeee"><td>HXB3</td><td>RI 03-1</td>
<tr bgcolor="#eeeeee"><td>HXB3</td><td>RI 03-2</td>
<tr bgcolor="#eeeeee"><td>HXB3</td><td>RI 03-3</td>
<tr bgcolor="#eeeeee"><td>HXB3</td><td>RI 03-4</td>
<tr bgcolor="#eeeeee"><td>HXB4</td><td>RI 04-1</td>
<tr bgcolor="#eeeeee"><td>HXB4</td><td>RI 04-2</td>
<tr bgcolor="#eeeeee"><td>HXB4</td><td>RI 04-3</td>
<tr bgcolor="#eeeeee"><td>HXB4</td><td>RI 04-4</td>
<tr bgcolor="#eeeeee"><td>HXB5</td><td>RI 05-1</td>
<tr bgcolor="#eeeeee"><td>HXB5</td><td>RI 05-2</td>
<tr bgcolor="#eeeeee"><td>HXB5</td><td>RI 05-3</td>
<tr bgcolor="#eeeeee"><td>HXB5</td><td>*RI 05-4</td>
<tr bgcolor="#eeeeee"><td>HXB7</td><td>RI 07-1</td>
<tr bgcolor="#eeeeee"><td>HXB7</td><td>RI 07-2</td>
<tr bgcolor="#eeeeee"><td>HXB7</td><td>RI 07-3</td>
<tr bgcolor="#eeeeee"><td>HXB7</td><td>RI 07-4</td>
<tr bgcolor="#eeeeee"><td>HXB10</td><td>RI 10-1</td>
<tr bgcolor="#eeeeee"><td>HXB10</td><td>RI 10-2</td>
<tr bgcolor="#eeeeee"><td>HXB10</td><td>RI 10-3</td>
<tr bgcolor="#eeeeee"><td>HXB10</td><td>RI 10-4</td>
<tr bgcolor="#eeeeee"><td>HXB15</td><td>RI 15-1</td>
<tr bgcolor="#eeeeee"><td>HXB15</td><td>RI 15-2</td>
<tr bgcolor="#eeeeee"><td>HXB15</td><td>RI 15-3</td>
<tr bgcolor="#eeeeee"><td>HXB15</td><td>RI 15-4</td>
<tr bgcolor="#eeeeee"><td>HXB17</td><td>RI 17-1</td>
<tr bgcolor="#eeeeee"><td>HXB17</td><td>RI 17-2</td>
<tr bgcolor="#eeeeee"><td>HXB17</td><td>RI 17-3</td>
<tr bgcolor="#eeeeee"><td>HXB17</td><td>RI 17-4</td>
<tr bgcolor="#eeeeee"><td>HXB18</td><td>RI 18-1</td>
<tr bgcolor="#eeeeee"><td>HXB18</td><td>RI 18-2</td>
<tr bgcolor="#eeeeee"><td>HXB18</td><td>RI 18-3</td>
<tr bgcolor="#eeeeee"><td>HXB18</td><td>RI 18-4</td>
<tr bgcolor="#eeeeee"><td>HXB20</td><td>RI 20-1</td>
<tr bgcolor="#eeeeee"><td>HXB20</td><td>RI 20-2</td>
<tr bgcolor="#eeeeee"><td>HXB20</td><td>RI 20-3</td>
<tr bgcolor="#eeeeee"><td>HXB20</td><td>RI 20-4</td>
<tr bgcolor="#eeeeee"><td>HXB21</td><td>RI 21-1</td>
<tr bgcolor="#eeeeee"><td>HXB21</td><td>RI 21-2</td>
<tr bgcolor="#eeeeee"><td>HXB21</td><td>RI 21-3</td>
<tr bgcolor="#eeeeee"><td>HXB21</td><td>RI 21-4</td>
<tr bgcolor="#eeeeee"><td>HXB22</td><td>RI 22-1</td>
<tr bgcolor="#eeeeee"><td>HXB22</td><td>RI 22-2</td>
<tr bgcolor="#eeeeee"><td>HXB22</td><td>RI 22-3</td>
<tr bgcolor="#eeeeee"><td>HXB22</td><td>RI 22-4</td>
<tr bgcolor="#eeeeee"><td>HXB23</td><td>RI 23-1</td>
<tr bgcolor="#eeeeee"><td>HXB23</td><td>RI 23-2</td>
<tr bgcolor="#eeeeee"><td>HXB23</td><td>RI 23-3</td>
<tr bgcolor="#eeeeee"><td>HXB23</td><td>RI 23-4</td>
<tr bgcolor="#eeeeee"><td>HXB24</td><td>RI 24-1</td>
<tr bgcolor="#eeeeee"><td>HXB24</td><td>RI 24-2</td>
<tr bgcolor="#eeeeee"><td>HXB24</td><td>RI 24-3</td>
<tr bgcolor="#eeeeee"><td>HXB24</td><td>*RI 24-4</td>
<tr bgcolor="#eeeeee"><td>HXB25</td><td>RI 25-1</td>
<tr bgcolor="#eeeeee"><td>HXB25</td><td>RI 25-2</td>
<tr bgcolor="#eeeeee"><td>HXB25</td><td>RI 25-3</td>
<tr bgcolor="#eeeeee"><td>HXB25</td><td>*RI 25-4</td>
<tr bgcolor="#eeeeee"><td>HXB26</td><td>RI 26-1</td>
<tr bgcolor="#eeeeee"><td>HXB26</td><td>RI 26-2</td>
<tr bgcolor="#eeeeee"><td>HXB26</td><td>RI 26-3</td>
<tr bgcolor="#eeeeee"><td>HXB26</td><td>RI 26-4</td>
<tr bgcolor="#eeeeee"><td>HXB27</td><td>RI 27-1</td>
<tr bgcolor="#eeeeee"><td>HXB27</td><td>RI 27-2</td>
<tr bgcolor="#eeeeee"><td>HXB27</td><td>RI 27-3</td>
<tr bgcolor="#eeeeee"><td>HXB27</td><td>RI 27-4</td>
<tr bgcolor="#eeeeee"><td>HXB29</td><td>RI 29-1</td>
<tr bgcolor="#eeeeee"><td>HXB29</td><td>RI 29-2</td>
<tr bgcolor="#eeeeee"><td>HXB29</td><td>RI 29-3</td>
<tr bgcolor="#eeeeee"><td>HXB29</td><td>*RI 29-4</td>
<tr bgcolor="#eeeeee"><td>HXB31</td><td>RI 31-1</td>
<tr bgcolor="#eeeeee"><td>HXB31</td><td>RI 31-2</td>
<tr bgcolor="#eeeeee"><td>HXB31</td><td>RI 31-3</td>
<tr bgcolor="#eeeeee"><td>HXB31</td><td>RI 31-4</td>

<tr bgcolor="#eeeeee"><td>BXH2</td><td>RI 02c-1</td>
<tr bgcolor="#eeeeee"><td>BXH2</td><td>RI 02c-2</td>
<tr bgcolor="#eeeeee"><td>BXH2</td><td>RI 02c-3</td>
<tr bgcolor="#eeeeee"><td>BXH2</td><td>RI 02c-4</td>
<tr bgcolor="#eeeeee"><td>BXH3</td><td>RI 03c-1</td>
<tr bgcolor="#eeeeee"><td>BXH3</td><td>RI 03c-2</td>
<tr bgcolor="#eeeeee"><td>BXH3</td><td>RI 03c-3</td>
<tr bgcolor="#eeeeee"><td>BXH3</td><td>RI 03c-4</td>
<tr bgcolor="#eeeeee"><td>BXH5</td><td>RI 05c-1</td>
<tr bgcolor="#eeeeee"><td>BXH5</td><td>RI 05c-2</td>
<tr bgcolor="#eeeeee"><td>BXH5</td><td>RI 05c-3</td>
<tr bgcolor="#eeeeee"><td>BXH5</td><td>*RI 05c-4</td>
<tr bgcolor="#eeeeee"><td>BXH6</td><td>RI 06c-1</td>
<tr bgcolor="#eeeeee"><td>BXH6</td><td>RI 06c-2</td>
<tr bgcolor="#eeeeee"><td>BXH6</td><td>RI 06c-3</td>
<tr bgcolor="#eeeeee"><td>BXH6</td><td>*RI 06c-4</td>
<tr bgcolor="#eeeeee"><td>BXH8</td><td>RI 08c-1</td>
<tr bgcolor="#eeeeee"><td>BXH8</td><td>RI 08c-2</td>
<tr bgcolor="#eeeeee"><td>BXH8</td><td>RI 08c-3</td>
<tr bgcolor="#eeeeee"><td>BXH8</td><td>RI 08c-4</td>
<tr bgcolor="#eeeeee"><td>BXH9</td><td>RI 09c-1</td>
<tr bgcolor="#eeeeee"><td>BXH9</td><td>RI 09c-2</td>
<tr bgcolor="#eeeeee"><td>BXH9</td><td>RI 09c-3</td>
<tr bgcolor="#eeeeee"><td>BXH9</td><td>RI 09c-4</td>
<tr bgcolor="#eeeeee"><td>BXH10</td><td>RI 10c-1</td>
<tr bgcolor="#eeeeee"><td>BXH10</td><td>RI 10c-2</td>
<tr bgcolor="#eeeeee"><td>BXH10</td><td>RI 10c-3</td>
<tr bgcolor="#eeeeee"><td>BXH11</td><td>RI 11c-1</td>
<tr bgcolor="#eeeeee"><td>BXH11</td><td>RI 11c-2</td>
<tr bgcolor="#eeeeee"><td>BXH11</td><td>RI 11c-3</td>
<tr bgcolor="#eeeeee"><td>BXH11</td><td>*RI 11c-4</td>
<tr bgcolor="#eeeeee"><td>BXH12</td><td>RI 12c-1</td>
<tr bgcolor="#eeeeee"><td>BXH12</td><td>RI 12c-2</td>
<tr bgcolor="#eeeeee"><td>BXH12</td><td>RI 12c-3</td>
<tr bgcolor="#eeeeee"><td>BXH12</td><td>RI 12c-4</td>
<tr bgcolor="#eeeeee"><td>BXH13</td><td>RI 13c-1</td>
<tr bgcolor="#eeeeee"><td>BXH13</td><td>RI 13c-2</td>
<tr bgcolor="#eeeeee"><td>BXH13</td><td>RI 13c-3</td>
<tr bgcolor="#eeeeee"><td>BXH13</td><td>RI 13c-4</td>

</table>
</td>
</tr>
</table>
<P><B>*</B>: These eight arrays were excluded in the final strain summary data. See section of Quality Control for further explanation.

</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>

<Blockquote><P>
<B>Affymetrix 230A GeneChip:</B> Expression data were generated using the Affymetrix 230A array. The chromosomal locations of probe sets were determined by <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=rat" target="_blank" class="fs14">BLAT</A> analysis of concatenated probe sequences using the Rat Genome Sequencing Consortium assembly.</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About Quality Control Procedures:</P>

<Blockquote>
<P><B>RNA processing:</B>RNA was extracted using Trizol reagent (Invitrogen) and purified using an RNeasy Mini kit from Qiagen. Double-stranded cDNA was generated without pooling. The Ambion MEGAscript T7 kit from Ambion was used to generate biotinylated cRNA for kidney. Fat samples were processed at this step using the Enzo Diagnostics Bioarray High Yield RNA Transcript labeling kit. See H&uuml;bner et al. 2005 for additional detail. One-hundred and twenty eight samples passed RNA quality control steps.</P>

<P><B>Probe level QC:</B> All 128 CEL files were collected into a single DataDesk 6.2 file. Probe data from pairs of arrays were plotted and compared. Eight arrays were considered potential outliers (despite having passed RNA quality control) and in the interest of minimizing technical variance, a decision was made to withhold them from the calculation of strain means used in WebQTL. The remaining 120 arrays were quantile normalized and reexamined in DataDesk to ensure reasonble colinearity of all array data sets.</P>

<P><B>Probe set level QC:</B> Probe set level QC involves counting the number of times that a single array data set from a single sample generates outliers at the level of the probe set consensus estimates of expression. With 120 arrays, any single array should generate a comparatively small fraction of the total number of outlier calls. This final step of array QC has NOT been implemented yet in this data set.</P>
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a target="_blank" class="fs14" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx">GCOS</a> are 75% quantiles from a set of pixel measured in each cell.
</Blockquote>


<Blockquote>
<P><B>Probe set data: </B>The original CEL values were log2 transformed and quantile normalized. We then took the antilog values of these quantile adjusted CEL values as input to the standard MAS5 algorithm. Probe set values listed in WebQTL pages are typically the averages of four biological replicates within strain. </P>


</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>
This work was supported with funds to TJA by the MRC Clinical Sciences Centre, the British Heart Foundation, and the Wellcome Trust Cardiovascular Functional Genomics Intiative; to NH from the German Ministry for Science and Education (National Genome Research Network); to MP and Vladimir Kren from the Grant Agency of the Czech Republic; to MP and TJA from the Wellcome Trust Collaborative Research Initiative grant, to Theodore W Kurtz from the NIH, to TWK and MP from a Fogarty International Research Collaboration Award. Microarrays were a generous donation of Affymetrix Inc. Michal Pravenec thanks the  Howard Hughes Medical Institute for its support to him as an international research scholar.




</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>This text file originally generated by Robert Williams, Norbert H&uuml;bner, Michal Pravnec, Timothy Aitman, April 19, 2005. Updated by RWW, May 13, 2005.
</P></Blockquote>

<P></P>

</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
