<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>M430 RMA Liver F2 Aug05 / GeneNetwork</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">

(B6 x BTBR)F<SUB class="fs12">2</SUB>-ob/ob Liver mRNA M430 RMA Database (Aug 2005 Freeze)  <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=39">GN39</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote>
<P>
This August 2005 data freeze provides estimates of mRNA expression in adult liver from a selected set of 60 F2 animals generated by crossing strain C57BL/6J-ob/+ with BTBR and then intercrossing the F1-ob/+ progeny. The F2 progeny included, in a total of 350 progeny, 110 ob/ob progeny homozygous for the obese (ob) allele of leptin (<I>Lep</I>) on Chr 6. Sixty of the ob/ob progeny were selected for expression assays. This selection means that the data set is not useful for defining QTLs on Chr 6. Array data were generated at the University of Wisconsin by Alan Attie and colleagues. This data release accompanies the paper of Lan and colleagues (in submission, 2005). A set of 24 complementary phenotypes such as body weight, blood chemistry, and rtPCR results, are also available for these animals and an additional set of 50 F2s (see <B>Phenotypes</B> database. Samples were hybridized to 60 pairs of Affymetrix M430A and B arrays. This particular data set was processed using the <a href="http://stat-www.berkeley.edu/users/bolstad/RMAExpress/RMAExpress.html" target="_blank" class="fs14">RMA</a> normalization method. To simplify comparison among transforms, RMA values of each array were adjusted to an average of 8 units and a standard deviation of two units.
</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>The F<SUB class="fs12">2</SUB>-ob/ob mice were chosen from a mapping panel that we created to map diabetes related physiological phenotypes (Stoehr et al. 2000). About 110 of these F<SUB class="fs12">2</SUB>-ob/ob mice were also used to map mRNA abundance traits derived by quantitative real-time RT-PCR (Lan et al. 2003). The sixty F<SUB class="fs12">2</SUB>-ob/ob mice that were used to generate microarray-derived mRNA abundance traits were selected from the 110 mice based on a selective phenotyping algorithm (Jin et al. 2004). The F<SUB class="fs12">2</SUB>-ob/ob mice were housed at weaning at the University of Wisconsin-Madison animal care facility on a 12-h light/dark cycle. Mice were provided Purina Formulab Chow 5008 (6.5% fat) and acidified water ad libitum. Mice were killed at 14 weeks of age by CO<SUB class="fs12">2</SUB> asphyxiation after a 4-hour fast. The livers, along with other tissues, were immediately foil wrapped and frozen in liquid nitrogen, and subsequently transferred to -80 &deg;C freezers for storage. </Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate this set of data:</P>

<Blockquote><P>Liver samples were taken from 29 male and 31 females. Total RNA was isolated with RNAzol Reagent (Tel-Test, Inc.) using a modification of the single-step acid guanidinium isothiocyanate phenol-chloroform extraction method according to the manufacturer's protocol. The extracted RNA was purified using RNeasy (Qiagen, Inc.). RNA samples were evaluated by UV spectroscopy for concentration. RNA quality was monitored by visualization on an ethidium bromide-stained denaturing formaldehyde agarose gel. RNA samples were converted to cDNA, and then biotin-labeled cRNA according to Affymetrix Expression Analysis Technical Manual. The labeled samples were hybridized to the M430A, and subsequently the M430B array. The hybridization, washing and scanning steps were carried out by Hong Lan using the Affymetrix core facility at the Gene Expression Center of University of Wisconsin-Madison.
</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array</P>

<Blockquote>Liver samples were assayed individually using 60 M430A and B Affymetrix oligonucleotide microarray pairs. Each array ID is denoted by a 10-letter code: the first three letters represent the F<SUB class="fs12">2</SUB>-ob/ob mouse ID number, the fourth letter (either A or B) denotes M430A or M430B arrays, and the last six letters represent the date the array was scanned (MMDDYY).</Blockquote>

<Blockquote>All 120 M430A and B arrays used in this project were purchased at one time and had the same Affymetrix lot number. The table below lists the arrays by Animal ID, sex, and ArrayID.
</Blockquote>
<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center">
<tr>
<td><center>
<table width="100%" border="0" cellpadding="5" cellspacing="1">
<tr bgcolor="royalblue"><td><center><font color=#FFFFFF>Animal ID</font></center></td>
<td><center><font color=#FFFFFF>sex</font></center></td>
<td><center><font color=#FFFFFF>MOE430A ArrayID</font></center></td>
<td><center><font color=#FFFFFF>MOE430B ArrayID</font></center></td></tr>
<tr bgcolor="#eeeeee"><td><center>2</center></td><td><center>M</center></td><td><center>002A100203</center></td><td><center>002B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>12</center></td><td><center>M</center></td><td><center>012A100203</center></td><td><center>012B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>22</center></td><td><center>M</center></td><td><center>022A100203</center></td><td><center>022B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>44</center></td><td><center>M</center></td><td><center>044A100203</center></td><td><center>044B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>46</center></td><td><center>M</center></td><td><center>046A100203</center></td><td><center>046B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>61</center></td><td><center>M</center></td><td><center>061A100203</center></td><td><center>061B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>100</center></td><td><center>M</center></td><td><center>100A100303</center></td><td><center>100B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>105</center></td><td><center>F</center></td><td><center>105A100303</center></td><td><center>105B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>111</center></td><td><center>F</center></td><td><center>111A100303</center></td><td><center>111B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>123</center></td><td><center>M</center></td><td><center>123A100303</center></td><td><center>123B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>156</center></td><td><center>F</center></td><td><center>156A100303</center></td><td><center>156B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>165</center></td><td><center>M</center></td><td><center>165A100303</center></td><td><center>165B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>167</center></td><td><center>M</center></td><td><center>167A100303</center></td><td><center>167B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>173</center></td><td><center>M</center></td><td><center>173A100303</center></td><td><center>173B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>186</center></td><td><center>F</center></td><td><center>186A100203</center></td><td><center>186B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>190</center></td><td><center>F</center></td><td><center>190A100303</center></td><td><center>190B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>194</center></td><td><center>M</center></td><td><center>194A100303</center></td><td><center>194B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>200</center></td><td><center>F</center></td><td><center>200A100303</center></td><td><center>200B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>207</center></td><td><center>F</center></td><td><center>207A100303</center></td><td><center>207B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>209</center></td><td><center>F</center></td><td><center>209A100203</center></td><td><center>209B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>212</center></td><td><center>F</center></td><td><center>212A100303</center></td><td><center>212B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>223</center></td><td><center>M</center></td><td><center>223A100303</center></td><td><center>223B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>224</center></td><td><center>M</center></td><td><center>224A100303</center></td><td><center>224B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>253</center></td><td><center>F</center></td><td><center>253A100303</center></td><td><center>253B100503</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>254</center></td><td><center>F</center></td><td><center>254A100603</center></td><td><center>254B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>260</center></td><td><center>F</center></td><td><center>260A100603</center></td><td><center>260B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>264</center></td><td><center>F</center></td><td><center>264A100603</center></td><td><center>264B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>310</center></td><td><center>F</center></td><td><center>310A100603</center></td><td><center>310B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>317</center></td><td><center>M</center></td><td><center>317A100603</center></td><td><center>317B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>318</center></td><td><center>F</center></td><td><center>318A100603</center></td><td><center>318B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>324</center></td><td><center>F</center></td><td><center>324A100603</center></td><td><center>324B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>327</center></td><td><center>F</center></td><td><center>327A100603</center></td><td><center>327B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>343</center></td><td><center>M</center></td><td><center>343A100603</center></td><td><center>343B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>416</center></td><td><center>M</center></td><td><center>416A100603</center></td><td><center>416B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>419</center></td><td><center>F</center></td><td><center>419A100603</center></td><td><center>419B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>438</center></td><td><center>M</center></td><td><center>438A100603</center></td><td><center>438B100703</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>440</center></td><td><center>M</center></td><td><center>440A100603</center></td><td><center>440B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>455</center></td><td><center>M</center></td><td><center>455A100603</center></td><td><center>455B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>458</center></td><td><center>F</center></td><td><center>458A100603</center></td><td><center>458B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>472</center></td><td><center>M</center></td><td><center>472A100603</center></td><td><center>472B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>474</center></td><td><center>F</center></td><td><center>474A100603</center></td><td><center>474B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>479</center></td><td><center>F</center></td><td><center>479A100603</center></td><td><center>479B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>484</center></td><td><center>F</center></td><td><center>484A100603</center></td><td><center>484B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>486</center></td><td><center>F</center></td><td><center>486A100603</center></td><td><center>486B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>489</center></td><td><center>F</center></td><td><center>489A100603</center></td><td><center>489B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>493</center></td><td><center>F</center></td><td><center>493A100603</center></td><td><center>493B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>499</center></td><td><center>M</center></td><td><center>499A100603</center></td><td><center>499B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>513</center></td><td><center>M</center></td><td><center>513A100603</center></td><td><center>513B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>517</center></td><td><center>M</center></td><td><center>517A100703</center></td><td><center>517B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>523</center></td><td><center>M</center></td><td><center>523A100703</center></td><td><center>523B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>549</center></td><td><center>M</center></td><td><center>549A100703</center></td><td><center>549B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>553</center></td><td><center>F</center></td><td><center>553A100703</center></td><td><center>553B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>554</center></td><td><center>F</center></td><td><center>554A100703</center></td><td><center>554B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>559</center></td><td><center>F</center></td><td><center>559A100703</center></td><td><center>559B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>560</center></td><td><center>F</center></td><td><center>560A100703</center></td><td><center>560B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>566</center></td><td><center>M</center></td><td><center>566A100703</center></td><td><center>566B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>608</center></td><td><center>F</center></td><td><center>608A100703</center></td><td><center>608B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>615</center></td><td><center>F</center></td><td><center>615A100703</center></td><td><center>615B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>617</center></td><td><center>M</center></td><td><center>617A100703</center></td><td><center>617B100803</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>620</center></td><td><center>M</center></td><td><center>620A100703</center></td><td><center>620B100803</center></td></tr>
</table>
</center></td>
</tr>
</table>
</Blockquote>




<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About Data Access:</P>
<Blockquote>The F2 data set used in the manuscript is available at <A HREF="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi">GEO</A> under the accession number "GSE3330".
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the marker set:</P>
<Blockquote>The 60 mice were each genotyped at 194 MIT microsatellite markers an average of approximately 10 cM (and always < 30 cM) apart across the entire genome (Y chromsome, excepted). The genotyping error-check routine implemented within R/qtl (Broman et al. 2003) showed no likely errors at p <0.01 probability.
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platfrom :</P>
<Blockquote>
<P><B>Affymetrix Mouse Genome 430A and 430B array pairs: </B>The <A HREF="http://www.affymetrix.com/support/technical/byproduct.affx?product=moe430-20" target="_blank" class="fs14">430A and B</A> array pairs collectively consist of 992936 25-nucleotide probes that estimate the expression of approximately 39,000 transcripts (some are variant transcipts and many are duplicates). The array sequences were selected late in 2002 using Unigene Build 107. The arrays nominally contain the same probe sequence as the 430 2.0 series. However, roughy 75000 probes differ between those on A and B arrays and those on the 430 2.0.</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the data processing:</P>

<Blockquote>
<B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx" target="_blank" class="fs14">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.

<UL>
<LI>Step 1: We added an offset of 1.0 to the CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log base 2 of each cell.

<LI>Step 3: We computed the Z scores for each cell.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6a: The 430A and 430B GeneChips include a set of 100 shared probe sets (2200 probes) that have identical sequences. These probes and probe sets provide a way to calibrate expression of the two GeneChips to a common scale. The absolute mean expression on the 430B array is almost invariably lower than that on the 430A array. To bring the two arrays into alignment, we regressed Z scores of the common set of probes to obtain a linear regression corrections to rescale the 430B arrays to the 430A array. In our case this involved multiplying all 430B Z scores by the slope of the regression and adding or subtracting a very small offset. The result of this step is that the mean of the 430A GeneChip expression is fixed at a value of 8, whereas that of the 430B chip is typically 7. Thus average of A and B arrays is approximately 7.5.

<LI>Step 6b: We recenter the whole set of 430A and B transcripts to a mean of 8 and a standard deviation of 2. This involves reapplying Steps 3 through 5 above but now using the entire set of probes and probe sets from a merged 430A and B data set.

</UL>
<B>Probe set data from the TXT file: </B>These TXT files were generated using the RMA (Robust Multiarray Average; (IRIZARRY et al. 2003)). RMA is implemented in the affy package (11/24/03 version) within <A  href="http://www.bioconductor.org" target="_blank" class="normal">Bioconductor</A>. RMA functions provide options for background correction and normalization resulting in a single summary score of expression for every transcript in every condition. The same simple steps described above were also applied to these values. A 1-unit difference represents roughly a two-fold difference in expression level. Expression levels below 5 are usually close to background noise levels.
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>This project was supported in part by NIH/NIDDK 5803701, NIH/NIDDK 66369-01 and American Diabetes Association 7-03-IG-01 to Alan D. Attie, USDA CSREES grants to the University of Wisconsin-Madison to Brian S. Yandell, and HHMI grant A-53-1200-4 to Christina Kendziorski.
</blockquote>

<Blockquote>B6BTBRF<SUB class="fs12">2</SUB> Liver Database.  All of the original (B6 x BTBR)F<SUB class="fs12">2</SUB>-ob/ob liver mRNA M430AB array data were generated by Hong Lan and Alan Attie at The University of Wisconsin-Madison. For contact and citations and other information on these data sets, please review the INFO pages and contact Drs.<A HREF="mailto:attie@biochem.wisc.edu" class="normal"> Alan Attie</A>, <A HREF="mailto:kendzior@biostat.wisc.edu" class="normal">Christina Kendziorski</A>, and <A HREF="mailto:yandell@stat.wisc.edu"   class="normal">Brian Yandell</A> regarding use of this data set in publications or projects.</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;References:</P>


<Blockquote><P>Lan H, Chen M, Byers JE, Yandell BS, Stapleton DS, Mata CM, Mui ET, Flowers MT, Schueler KL, Malnly KF, Williams RW, Kendziorski CM, Attie AD (2005) Combined expression trait correlations and expression quantitative trait locus mapping. <!--PLoS Genetics,--> Submitted, Aug. 2005.
</P></Blockquote>

<Blockquote><P>Broman KW, Wu H, Sen S, Churchill GA (2003) R/qtl: QTL mapping in experimental crosses. Bioinformatics 19: 889-890.
</P></Blockquote>
<Blockquote><P>Irizarry RA, Bolstad BM, Collin F, Cope LM, Hobbs B, Speed TP (2003) Summaries of Affymetrix GeneChip probe level data. Nucleic Acids Res 31: e15.
</P></Blockquote>
<Blockquote><P>Jin C, Lan H, Attie AD, Churchill GA, Bulutuglo D, Yanell BS (<A HREF="http://www.genetics.org/cgi/content/full/168/4/2285" target="_blank" class="fs14">2004</A>) Selective phenotyping for increased efficiency in genetic mapping studies. Genetics 168:2285-2293.
</P></Blockquote>
<Blockquote><P>Lan H, Stoehr JP, Nadler ST, Schueler KL, Yandel BS, Attie AD  (2003) Dimension reduction for mapping mRNA abundance as quantitative traits. Genetics 164: 1607-1614.
</P></Blockquote>
<Blockquote><P>Stoehr JP, Nadler ST, Schueler KL, Rabaglia ME, Yandell BS, Metz SA, Attie AD (2000) Genetic obesity unmasks nonlinear interactions between murine type 2 diabetes susceptibility loci. Diabetes 49: 1946-1954.</P></Blockquote>
<Blockquote><P>Zhang L, Miles MF, Aldape KD (2003) A model of molecular interactions on short oligonucleotide microarrays. Nat Biotechnol 21: 818-821.</P></Blockquote>

<Blockquote></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>This text file originally generated by RWW and Alan Attie, July 2, 2004. Updated by RWW, Aug 20, 5, 2004; April 7, 2005; August 20, 2005.
</P></Blockquote>



<P></P>

</center></td>
</TR>
<TR>
<TD colspan=2 align=center bgColor=#ddddff>



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</center></td>
</TR>
</TABLE>
<!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>

// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure

new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>

