<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>M430 Microarray May03 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
			<P class="title">SJUT M430 Cerebellum Database (May/03) <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>
			
			<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the mice used to map microarray data:</P>
	
			<Blockquote>The set of mouse strains used for mapping (a mapping panel) consists of groups of genetically unique BXD recombinant inbred strains. The ancestral strains from which all BXD lines are derived are C57BL/6J (B6 or B) and DBA/2J (D2 or D). Both B and D strains have been almost fully sequence (8x coverage for B6, and 1.5x coverage for D by Celera Genomics). Chromosomes of the two parental strains are recombined randomly in the many different BXD strains. BXD lines 2 through 32 were produced by Dr. Benjamin Taylor starting in the late 1970s. BXD33 through 42 were also produced by Dr. Taylor, but they were generated in the 1990s. All of these strains are available from <a href="http://www.jax.org" target="_blank" class="fs14">The Jackson Laboratory</a>. Lines such as BXDA12, BXDA20, etc. are BXD Advanced recombinant inbred strains that are part of a large set now being produced by Drs. Lu Lu, Guomin Zhou, Lee Silver, Jeremy Peirce, and Robert Williams. There will eventually be ~45 of these BXDA strains. For additional background on recombinant inbred strains, 
 please see <a href="http://www.nervenet.org/papers/bxn.html"  class="fs14">http://www.nervenet.org/papers/bxn.html</a>.
			</Blockquote>

			<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>
			<Blockquote>The May03 data were run as a single batch. Each individual array experiment involved a pool of brain tissue (intact whole cerebellum) taken from three adult animals usually of the same age. RNA was extracted at UTHSC. Twenty samples passed quality control at SJCRH and were run on Affymetrix MOE 430A and MOE430B GeneChip pairs (40 arrays total).
</Blockquote><Blockquote>
The May03 data set includes a single Affymetrix GeneChip pair (abbreviated 430AB) processed with labeled messenger RNA taken from 20 strains. Please note that the variation of sex and age is intentional and this data set is only the first of many batches that will be required to obtain a fully balanced design by sex and age. However, we note that there is still only quite modest evidence of sex difference in cerebellar transcriptional profiles (beyond such obvious transcripts such as Xist and Dby). The age range may look very broad, but translated into human terms corresponds to a range from about 20 years to 50 years.
</Blockquote>
<Blockquote>
    <table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center">
		<tr> 
            <td> 
			<table width="100%" border="0" cellpadding="5" cellspacing="1">
				
<tr bgcolor="royalblue"> 
				<td><font color=#FFFFFF>SampleID</font></td>
				<td><font color=#FFFFFF>Strain</font></td>
				<td><font color=#FFFFFF>Case_ID_UT</font></td>
				<td><font color=#FFFFFF>Age</font></td>
				<td><font color=#FFFFFF>Sex</font></td>
				</tr>
<tr bgcolor="#eeeeee"> <td>
				S347-1</td><td>B6D2F1/J</td><td>021202.01</td><td>94</td><td>M</td></tr>
<tr bgcolor="#eeeeee"> <td>
				S054-1</td><td>C57BL/6J</td><td>101201.01</td><td>109</td><td>M</td></tr>
<tr bgcolor="#eeeeee"> <td>
				S175-1</td><td>DBA/2J</td><td>011402.07</td><td>71</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				751-C</td><td>BXD2</td><td>022003.02</td><td>142</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				752-C</td><td>BXD5</td><td>031103.01</td><td>71</td><td>M</td></tr>
<tr bgcolor="#eeeeee"> <td>
				719-C</td><td>BXD6</td><td>010803.01</td><td>92</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				S173-1</td><td>BXD8</td><td>011402.01</td><td>72</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				737-C</td><td>BXD9</td><td>031903.04</td><td>86</td><td>M</td></tr>
<tr bgcolor="#eeeeee"> <td>
				S200-1</td><td>BXD11</td><td>011602.04</td><td>441</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				750-C</td><td>BXD16</td><td>021402.04</td><td>163</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				711-C</td><td>BXD21</td><td>121102.01</td><td>116</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				S174-1</td><td>BXD22</td><td>011402.04</td><td>65</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				S429-1</td><td>BXD25</td><td>030702.01</td><td>90</td><td>M</td></tr>
<tr bgcolor="#eeeeee"> <td>
				S203-1</td><td>BXD28</td><td>011602.13</td><td>427</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				714-C</td><td>BXD29</td><td>020503.04</td><td>76</td><td>M</td></tr>
<tr bgcolor="#eeeeee"> <td>
				715-C</td><td>BXD33</td><td>121002.01</td><td>124</td><td>M</td></tr>
<tr bgcolor="#eeeeee"> <td>
				725-C</td><td>BXD34</td><td>111902.07</td><td>56</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				723-C</td><td>BXD39</td><td>120902.01</td><td>165</td><td>M</td></tr>
<tr bgcolor="#eeeeee"> <td>
				718-C</td><td>BXD40</td><td>111902.04</td><td>56</td><td>F</td></tr>
<tr bgcolor="#eeeeee"> <td>
				709-C</td><td>BXD42</td><td>011303.01</td><td>97</td><td>M</td></tr>
              </table>
			  </td>
          </tr>
        </table>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the .CEL file: </B>These .CEL values produced by MAS 5.0 are ~ 75% quantiles from a set of 22 pixel values per cell (6th-ranked pixel).
<UL>
<LI>Step 1: We added an offset of 1.0 to the .CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log base 2 of each cell.

<LI>Step 3: We computed the Z-score for each cell.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z-scores. The consequence of this simple set of transformations is to produce a set of Z-scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z-score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6: The 430A and 430B GeneChips include a set of 100 shared probe sets (2200 probes) that have identical sequences. These probes and probe set provide a way to calibrate expression of the two GeneChips to a common scale. The absolute mean expression on the 430B array is almost invariably lower than that on the 430A array. To bring the two arrays into alignment, we regressed Z scores of the common set of probes to obtain a linear regression corrections to rescale the 430B arrays to the 430A array. In our case this involved multiplying all 430B Z values by approximately 0.87 and subtracting a very small offset. The result of this step is that the mean of the 430A GeneChip expression is fixed at a value of 8, whereas that of the 430B chip is typically 7. Thus average of A and B arrays is approximately 7.5.

<LI>Step 7: This first batch of data intentionally includes no technical or biological replicates. Those are all included in September03 data set and will also be included in all subsequent large batches. For this particular data set we therefore did not need to compute the arithmetic mean of the values for the set of microarrays for each strain. We have not (yet) corrected for variance introduced by sex, age, or a sex-by-age interaction. We have not corrected for background beyond the background correction implemented by Affymetrix in generating the .CEL file.

</UL>
<B>Probe set data from the .TXT file: </B>These .TXT files were generated using the MAS 5.0. The same simple steps described above were also applied to these values. A 1-unit difference represents roughly a two-fold difference in expression level. Expression levels below 5 are usually close to background noise levels.</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the chromosome and megabase position values:</P>

<Blockquote>The chromosomal locations of probe sets and gene markers on the 430A and 430B microarrays were determined by BLAT analysis using the Mouse Genome Sequencing Consortium Oct 2003 Assembly (see <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse" class="fs14">http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse</a>). We thank Dr. Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis.</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P> 
<Blockquote>
Data were generated with funds contributed equally by The UTHSC-SJCRH Cerebellum Transcriptome Profiling Consortium. Our members include:<UL>
<LI>Tom Curran
<LI>Dan Goldowitz
<LI>Kristin Hamre
<LI>Lu Lu
<LI>Peter McKinnon
<LI>Jim Morgan
<LI>Clayton Naeve
<LI>Richard Smeyne
<LI>Robert Williams
<LI>The Center of Genomics and Bioinformatics at UTHSC
<LI>The Hartwell Center at SJCRH
</UL>
</Blockquote>


		</TD>
		</TR></TABLE>
		</TD>
	</TR>
	<TR>
		<TD align=center bgColor=#ddddff class="solidBorder">
	 
	 
	
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->



		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	// Note where menu initialization block is located in HTML document.
	// Don't try to position menu locating menu initialization block in
	// some table cell or other HTML element. Always put it before </body>
	// each menu gets two parameters (see demo files)
	// 1. items structure
	// 2. geometry structure
	new menu (MENU_ITEMS, MENU_POS);
	// make sure files containing definitions for these variables are linked to the document
	// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
	// error in menu_tpl.js file or that file isn't linked properly.
	
	// also take a look at stylesheets loaded in header in order to set styles
	//-->
</script>
</BODY>
</HTML>
