<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>M430 RMA Liver F2 July04 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>


</HEAD>
<BODY  bottommargin="5" topmargin="5" leftmargin="5" rightmargin="5" marginheight="5" marginwidth="5" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=3 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<!-- Body Start from Here -->
<TD vAlign=top width="100%" colspan=2 bgColor=#eeeeee>
<P class="title">NCI Mammary mRNA M430 RMA Database RMA (July/04 Freeze)  <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=37">GN37</A></P>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Reference</P>

<Blockquote>Yang H, Crawford N, Lukes L, Finney R, Lancaster M, Hunter KW (<A HREF="http://www.ncbi.nlm.nih.gov/pubmed/16475030">2006</A>) Metatasis predictive signature profiles pre-exist in normal tissue. Clinical and Experimental Metastasis  22: 593â€“603. <A HREF="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2048974/?tool=pubmed">Full text</A>

</BLOCKQUOTE>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Synopsis</P>

<Blockquote>Used the Affymetrix M430A and M430B pair of arrays (total of 45,137 probe sets). Data available as CEL files from GeneNetwork upon request.

</BLOCKQUOTE>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the mice used to map microarray data:</P>


<Blockquote> The lines of mice used in this NCI-sponsored project consist of 18 groups of isogenic F1 progeny made by crossing females from each of 18 AKXD recombinant inbred strains (AKXD2, 3, 7, 9, 10, 11, 13, 14,
16, 18, 20, 21, 22, 23, 24, 25, 27, and 28) to male FVB/N mice that carry a transgene that consistently leads to the development of mammary tumors in females (e.g. Le Voyer et al., <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?
cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=11414753">2001</a>). The formal nomenclature of the male transgenic line is FVB/N-TgN(MMTV-PyMT)<SUP class="fs12">634Mul</SUP>. The genomes of each AKXD x FVB F1 consist of one set of FVB chromosomes (including the transgene) and one set of chromosomes inherited from one
of the 18 AKXD RI strain mothers. Only the AKXD chromosomes are "recombinant" across this panel of F1 progeny, and the set therefore has a genetic architecture similar to backcross progeny. It is possible to map modifiers that influence tumor characteristics and expression patterns. It is also possible to study covariance of transcript expression levels in tumor tissue. For further background on this special mapping design please see Hunter and Williams (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=11917159">2002</a>).</Blockquote>
<Blockquote>The ancestral strains from which all AKXD strains are derived are AKR/J (AKR) and DBA/2J (D2 or D). DBA/2J has been partially sequenced (approximately 1.5x coverage by D by Celera Genomics). Significant genomic sequence data for AKR is not currently available. Chromosomes of the two parental strains have recombined in the different AKXD strains. All of these strains are available from The Jackson Laboratory as cryopreserved stocks. For additional background on recombinant inbred strains, please see <a target="_blank" href="http://www.nervenet.org/papers/bxn.html">http://www.nervenet.org/papers/bxn.html</a>.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate these data:</P>
<Blockquote>Mammary tumors used in this array experiment were derived from 18 sets of AKXD x FVB/N F1 females as described above. After the primary tumor was diagnosed, the animals were aged an additional 40 days to permit metastatic progression. Females were sacrificed and mammary tumors were harvested. Samples were processed and arrayed on Affymetrix M430A and M430B arrays. The majority of the samples were assayed on arrays obtained from the same lot number.

</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array</P>

<Blockquote>All samples were processed and arrayed in the Laboratory of Population Genetics at the NCI. The table below lists the arrays by Samples, AKXD strain and Age.
</Blockquote>
<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="5" cellspacing="1">
<tr bgcolor="royalblue"><td width="30%"><font color=#FFFFFF><center>Sample</center></font></td>
<td width="15%"><font color=#FFFFFF><center>AKXD strain</center></font></td>

<td width="15%"><font color=#FFFFFF><center>Age</center></font></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2957</center></td><td><center>2</center></td><td><center>98</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2959</center></td><td><center>2</center></td><td><center>96</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1747</center></td><td><center>3</center></td><td><center>84</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD3446</center></td><td><center>3</center></td><td><center>91</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD4225</center></td><td><center>3</center></td><td><center>83</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2543</center></td><td><center>7</center></td><td><center>82</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2967</center></td><td><center>7</center></td><td><center>88</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD3336</center></td><td><center>7</center></td><td><center>95</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2685</center></td><td><center>9</center></td><td><center>113</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2710</center></td><td><center>9</center></td><td><center>109</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2949</center></td><td><center>9</center></td><td><center>115</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2618</center></td><td><center>10</center></td><td><center>99</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2620</center></td><td><center>10</center></td><td><center>99</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD3023</center></td><td><center>10</center></td><td><center>94</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1910</center></td><td><center>11</center></td><td><center>87</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2824</center></td><td><center>11</center></td><td><center>92</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2825</center></td><td><center>11</center></td><td><center>103</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2635</center></td><td><center>13</center></td><td><center>83</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2718</center></td><td><center>13</center></td><td><center>100</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2721</center></td><td><center>13</center></td><td><center>91</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2632</center></td><td><center>14</center></td><td><center>99</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2640</center></td><td><center>14</center></td><td><center>100</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD3444</center></td><td><center>14</center></td><td><center>96</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1636</center></td><td><center>16</center></td><td><center>112</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD3688</center></td><td><center>16</center></td><td><center>80</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD4152</center></td><td><center>16</center></td><td><center>91</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1647</center></td><td><center>18</center></td><td><center>91</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2616</center></td><td><center>18</center></td><td><center>91</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2804</center></td><td><center>18</center></td><td><center>80</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2456</center></td><td><center>20</center></td><td><center>100</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2554</center></td><td><center>20</center></td><td><center>107</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2829</center></td><td><center>20</center></td><td><center>105</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1610</center></td><td><center>21</center></td><td><center>98</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2611</center></td><td><center>21</center></td><td><center>88</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2918</center></td><td><center>21</center></td><td><center>98</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2460</center></td><td><center>22</center></td><td><center>107</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2461</center></td><td><center>22</center></td><td><center>94</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2463</center></td><td><center>22</center></td><td><center>110</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2975</center></td><td><center>23</center></td><td><center>82</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2976</center></td><td><center>23</center></td><td><center>86</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD3955</center></td><td><center>23</center></td><td><center>90</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1494</center></td><td><center>24</center></td><td><center>103</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1880</center></td><td><center>24</center></td><td><center>104</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD3030</center></td><td><center>24</center></td><td><center>89</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1607</center></td><td><center>25</center></td><td><center>110</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2326</center></td><td><center>25</center></td><td><center>92</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2328</center></td><td><center>25</center></td><td><center>90</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2629</center></td><td><center>25</center></td><td><center>96</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1756</center></td><td><center>27</center></td><td><center>100</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1757</center></td><td><center>27</center></td><td><center>98</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1948</center></td><td><center>27</center></td><td><center>99</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD1950</center></td><td><center>27</center></td><td><center>97</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2968</center></td><td><center>27</center></td><td><center>94</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2809</center></td><td><center>28</center></td><td><center>88</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD2815</center></td><td><center>28</center></td><td><center>90</center></td></tr>
<tr bgcolor="#eeeeee"><td><center>AKXD3432</center></td><td><center>28</center></td><td><center>91</center></td></tr>
</table>
</td>
</tr>
</table>
</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the data processing:</P>
<Blockquote>
<B>Probe (cell) level data from the .CEL file: </B>These .CEL values produced by <a target="_blank" class="normal" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.
<UL>
<LI>Step 1: We added an offset of 1.0 to the .CEL expression values for each cell to ensure that all values could be logged without generating negative values.

<LI>Step 2: We took the log base 2 of each cell.

<LI>Step 3: We computed the Z scores for each cell.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 6a: The 430A and 430B GeneChips include a set of 100 shared probe sets (2200 probes) that have identical sequences. These probes and probe sets provide a way to calibrate expression of the two GeneChips to a common scale. The absolute mean expression on the 430B array is almost invariably lower than that on the 430A array. To bring the two arrays into alignment, we regressed Z scores of the common set of probes to obtain a linear regression corrections to rescale the 430B arrays to the 430A array. In our case this involved multiplying all 430B Z scores by the slope of the regression and adding or subtracting a very small offset. The result of this step is that the mean of the 430A GeneChip expression is fixed at a value of 8, whereas that of the 430B chip is typically 7. Thus average of A and B arrays is approximately 7.5.

<LI>Step 6b: We recenter the whole set of 430A and B transcripts to a mean of 8 and a standard deviation of 2. This involves reapplying Steps 3 through 5 above but now using the entire set of probes and probe sets from a merged 430A and B data set.

</UL>
<B>Probe set data: </B>The expression data were generated using the RMA (Robust Multiarray Average; (IRIZARRY et al. 2003)). RMA is implemented in the affy package (11/24/03 version) within <A target="_blank" href="http://www.bioconductor.org">Bioconductor</A>. RMA functions provide options for background correction and normalization resulting in a single summary score of expression for every transcript in every condition. The same simple steps described above were also applied to these values. A 1-unit difference represents roughly a two-fold difference in expression level. Expression levels below 5 are usually close to background noise levels.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the chromosome and megabase position values:</P>

<Blockquote>The chromosomal locations of probe sets and gene markers on the 430A and 430B microarrays were determined by BLAT analysis using the Mouse Genome Sequencing Consortium Oct 2003 (mm4) Assembly (see <a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse" class="normal">http://genome.ucsc.edu/cgi-bin/hgBlat?command=start&org=mouse</a>). We thank Dr. Yan Cui (UTHSC) for allowing us to use his Linux cluster to perform this analysis.</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>All of the NCI mammary mRNA M430A and M430B data sets have been generated by <A target="_blank" href="http://ccr.cancer.gov/staff/staff.asp?profileid=5541">Kent Hunter</A> at the Laboratory of Population Genetics at the National Cancer Institute in Bethesda. For contact and citations and other information on these data sets please review the INFO pages and contact <A HREF="mailto:hunterk@mail.nih.gov">Dr. Hunter</A> regarding use of this data set in publications or projects..
</blockquote>
<P></P>

</TD>
</TR>
<TR>
<TD colspan=2 align=center bgColor=#ddddff>
<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->
</TD>
</TR>
</TABLE>
<!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>

// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure

new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>

