<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>M430 Microarray brain PDNN April05 / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">



HBP/Rosen Striatum M430v2 (April05) RMA Orig <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=66">GN66</A></P>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<blockquote>
This April 2005 data freeze provides estimates of mRNA expression in the <A HREf="http://www.nervenet.org/NetPapers/Rosen/Striatum2001/StriatumMain.html" target="_blank" class ="normal">striatum</A> (caudate nucleus of the forebrain) of 33 lines of mice including C57BL/6J, DBA/2J, and 31 BXD recombinant inbred strains. Data were generated using Affymetrix Mouse Genome 430 2.0 short oligomer microarrays at Beth Israel Deaconess Medical Center (<A HREF="http://www.bidmc.harvard.edu/sites/bidmc/home.aspBIDMC" target="_blank" class ="normal">BIDMC</A>, Boston MA) by Glenn D. Rosen with the support of a Human Brain Project (HBP) grant. Approximately 250 brain samples (males and females) from 33 strains were used in this experiment. Samples were hybridized to a total of 59 arrays. This particular data set was processed using the <a href="http://stat-www.berkeley.edu/users/bolstad/RMAExpress/RMAExpress.html" target="_blank" class="normal">RMA</a> protocol. To simplify comparison among different transforms, RMA values of each array have been adjusted to an average expression of 8 units and a standard deviation of 2 units.
</blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>
<P>We have used a set of BXD recombinant inbred strains generated by crossing C57BL/6J (B6 or B) with DBA/2J (D2 or D). The BXDs are particularly useful for systems genetics because both parental strains have been sequenced (8x coverage of B6 and 1.5x coverage for D). Physical maps in WebQTL incorporate approximately 1.75 million B vs D SNPs from <a href="http://www.celeradiscoverysystem.com/index.cfm" target="_blank" class ="normal">Celera</a>. BXD2 through BXD32 were bred by Benjamin A. Taylor starting in the late 1970s. BXD33 through 42 were bred by Taylor in the 1990s. These strains are available from The Jackson Laboratory.</P>
</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate this set of data:</P>

<Blockquote><P>Animals were obtained from The Jackson Laboratory and housed for several weeks at BIDMC until they reached ~2 months of age (range from 55 to 62 days). Mice were killed by cervical dislocation and brains were removed and placed in RNAlater for 20 to 25 minutes prior to dissection. Cerebella and olfactory bulbs were removed; brains were hemisected, and both striata were dissected using a medial approach by GD Rosen that typically yields 5 to 7 mg of tissue per side. The purity of this dissection has been validated by an analysis of acetylcholinestase activity. A pool of dissected tissue from 3 or 4 adults  (approximately 25 to 30 mg of tissue) of the same strain, sex, and age was collected in one session and used to generate cRNA samples.

<!--Roughly 90 to 95% of all cells in the striatum are medium spiny neurons (Gerfen, <A HREF="http://www.nervenet.org/netpapers/gerfen/striatum92.html" target="_blank" class="normal">1992</A>, for a review of the structure and function of the neostriatum).-->
</Blockquote>

<Blockquote>RNA was extracted by Rosen and colleagues and was then processed by the <a href="https://www.bidmcgenomics.org/" target="_blank" class="normal">BIDMC Genomics Core</a>. Labeled cRNA was generated using the Amersham Biosciences cRNA synthesis kit protocol.

<P><B>Replication and Sample Balance:</B> Our goal is to obtain data for independent biological sample pools from at least one sample from each sex for all BXD strains. We have not yet achieved this goal. Twenty-three of 33 strains are represented by male and female samples. The remaining 10 strains are still represented by single sex samples: BXD11 (F), BXD13 (F), BXD19 (F), BXD20 (F), BXD22 (M), BXD23 (M), BXD24 (M), BXD32 (M), C57BL/6J (M), and DBA/2J (M).

<P><B>Batch Structure:</B> This data set consists of arrays processed in three batches with several "reruns" for the first batch. All arrays were run using a single protocol. All data have been corrected for batch effects as described below.

</Blockquote>


<Blockquote>
The table below lists the arrays by strain, sex, sample name, and batch ID.  Each array was hybridized to a pool of  mRNA from 3 to 4 mice. All mice were between 55 and 62 days.
</Blockquote>

<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center">
<tr>
<td>
<table width="100%" border="0" cellpadding="5" cellspacing="1">
<tr bgcolor="royalblue"><td><font color=#FFFFFF>Id</font></td><td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>Sex</font></td>
<td><font color=#FFFFFF>Sample_name</td>
<td><font color=#FFFFFF>BatchId</font></td></tr>
<tr bgcolor="#eeeeee"><td>1</td><td>C57BL/6J</td><td>M</td><td>Chip41_Batch02_B6_M_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>2</td><td>C57BL/6J</td><td>M</td><td>Chip11_Batch03_B6_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>3</td><td>BXD1</td><td>F</td><td>Chip03_Batch03_BXD1_F_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>4</td><td>BXD1</td><td>M</td><td>Chip04_Batch03_BXD1_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>5</td><td>BXD2</td><td>F</td><td>Chip20_Rerun01_BXD2_F_Str</td><td>Rerun01</td></tr>
<tr bgcolor="#eeeeee"><td>6</td><td>BXD2</td><td>M</td><td>Chip05_Batch01_BXD2_M_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>7</td><td>BXD5</td><td>F</td><td>Chip10_Batch03_BXD5_F_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>8</td><td>BXD5</td><td>M</td><td>Chip12_Batch03_BXD5_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>9</td><td>BXD6</td><td>F</td><td>Chip38_Batch02_BXD6_F_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>10</td><td>BXD6</td><td>M</td><td>Chip39_Batch02_BXD6_M_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>11</td><td>BXD8</td><td>F</td><td>Chip07_Batch03_BXD8_F_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>12</td><td>BXD8</td><td>M</td><td>Chip02_Batch03_BXD8_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>13</td><td>BXD9</td><td>F</td><td>Chip16_Batch01_BXD9_F_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>14</td><td>BXD9</td><td>M</td><td>Chip10_Batch01_BXD9_M_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>15</td><td>BXD11</td><td>F</td><td>Chip31_Batch02_BXD11_F_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>16</td><td>BXD12</td><td>F</td><td>Chip11_Batch01_BXD12_F_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>17</td><td>BXD12</td><td>M</td><td>Chip18_Batch03_BXD12_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>18</td><td>BXD13</td><td>F</td><td>Chip33_Batch02_BXD13_F_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>19</td><td>BXD14</td><td>F</td><td>Chip48_Batch02_BXD14_F_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>20</td><td>BXD14</td><td>M</td><td>Chip47_Rerun01_BXD14_M_Str</td><td>Rerun01</td></tr>
<tr bgcolor="#eeeeee"><td>21</td><td>BXD15</td><td>F</td><td>Chip21_Batch01_BXD15_F_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>22</td><td>BXD15</td><td>M</td><td>Chip13_Batch01_BXD15_M_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>23</td><td>BXD16</td><td>F</td><td>Chip36_Batch02_BXD16_F_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>24</td><td>BXD16</td><td>M</td><td>Chip44_Rerun01_BXD16_M_Str</td><td>Rerun01</td></tr>
<tr bgcolor="#eeeeee"><td>25</td><td>BXD18</td><td>F</td><td>Chip15_Batch03_BXD18_F_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>26</td><td>BXD18</td><td>M</td><td>Chip19_Batch03_BXD18_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>27</td><td>BXD19</td><td>F</td><td>Chip19_Batch01_BXD19_F_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>28</td><td>BXD20</td><td>F</td><td>Chip14_Batch03_BXD20_F_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>29</td><td>BXD21</td><td>F</td><td>Chip18_Batch01_BXD21_F_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>30</td><td>BXD21</td><td>M</td><td>Chip09_Batch01_BXD21_M_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>31</td><td>BXD22</td><td>M</td><td>Chip13_Batch03_BXD22_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>32</td><td>BXD23</td><td>M</td><td>Chip01_Batch03_BXD23_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>33</td><td>BXD24</td><td>M</td><td>Chip17_Batch03_BXD24_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>34</td><td>BXD27</td><td>F</td><td>Chip29_Batch02_BXD27_F_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>35</td><td>BXD27</td><td>M</td><td>Chip40_Batch02_BXD27_M_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>36</td><td>BXD28</td><td>F</td><td>Chip06_Batch01_BXD28_F_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>37</td><td>BXD28</td><td>M</td><td>Chip23_Batch01_BXD28_M_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>38</td><td>BXD29</td><td>F</td><td>Chip45_Batch02_BXD29_F_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>39</td><td>BXD29</td><td>M</td><td>Chip42_Batch02_BXD29_M_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>40</td><td>BXD31</td><td>F</td><td>Chip14_Batch01_BXD31_F_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>41</td><td>BXD31</td><td>M</td><td>Chip09_Batch03_BXD31_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>42</td><td>BXD32</td><td>M</td><td>Chip30_Batch02_BXD32_M_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>43</td><td>BXD33</td><td>F</td><td>Chip27_Rerun01_BXD33_F_Str</td><td>Rerun01</td></tr>
<tr bgcolor="#eeeeee"><td>44</td><td>BXD33</td><td>M</td><td>Chip34_Batch02_BXD33_M_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>45</td><td>BXD34</td><td>F</td><td>Chip03_Batch01_BXD34_F_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>46</td><td>BXD34</td><td>M</td><td>Chip07_Batch01_BXD34_M_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>47</td><td>BXD36</td><td>F</td><td>Chip22_Batch03_BXD36_F_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>48</td><td>BXD36</td><td>M</td><td>Chip24_Batch03_BXD36_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>49</td><td>BXD38</td><td>F</td><td>Chip17_Batch01_BXD38_F_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>50</td><td>BXD38</td><td>M</td><td>Chip24_Batch01_BXD38_M_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>51</td><td>BXD39</td><td>M</td><td>Chip20_Batch03_BXD39_M_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>52</td><td>BXD39</td><td>F</td><td>Chip23_Batch03_BXD39_F_Str</td><td>Batch03</td></tr>
<tr bgcolor="#eeeeee"><td>53</td><td>BXD39</td><td>M</td><td>Chip43_Rerun01_BXD39_M_Str</td><td>Rerun01</td></tr>
<tr bgcolor="#eeeeee"><td>54</td><td>BXD40</td><td>F</td><td>Chip08_Rerun01_BXD40_F_Str</td><td>Rerun01</td></tr>
<tr bgcolor="#eeeeee"><td>55</td><td>BXD40</td><td>M</td><td>Chip22_Batch01_BXD40_M_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>56</td><td>BXD42</td><td>F</td><td>Chip35_Batch02_BXD42_F_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>57</td><td>BXD42</td><td>M</td><td>Chip32_Batch02_BXD42_M_Str</td><td>Batch02</td></tr>
<tr bgcolor="#eeeeee"><td>58</td><td>DBA/2J</td><td>M</td><td>Chip02_Batch01_D2_M_Str</td><td>Batch01</td></tr>
<tr bgcolor="#eeeeee"><td>59</td><td>DBA/2J</td><td>M</td><td>Chip05_Batch03_D2_M_Str</td><td>Batch03</td></tr>
</table>
</td>
</tr>
</table>
</Blockquote>

<!--The following array data sets did not pass quality control at the probe level: BXD19_M_Str_Batch03, BXD23_M_Str_Batch03, and BXD24_M_Str_Batch03. They are not included above.-->

</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform :</P>
<Blockquote>
<P><B>Affymetrix Mouse Genome 430 2.0 array: </B>The <A HREF="http://www.affymetrix.com/support/technical/byproduct.affx?product=moe430-20" target="_blank" class="normal">430v2</A> array consists of 992936 useful 25-nucleotide probes that estimate the expression of approximately 39,000 transcripts (many are near duplicates). The array sequences were selected late in 2002 using Unigene Build 107. The array nominally contains the same probe sequence as the 430A and B series. However, we have found that roughy 75000 probes differ from those on A and B arrays.</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a target="_blank" class="fs14" href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.
<UL>

<LI>Step 1: We added an offset of 1.0 unit to each cell signal to ensure that all values could be logged without generating negative values. We then computed the log base 2 of each cell.

<!-- <LI>Step 2: We performed a quantile normalization of the log base 2 values for the total set of 105 arrays (processed as two batches) using the same initial steps used by the RMA transform. -->

<LI>Step 2: We computed the Z scores for each cell value.

<LI>Step 3: We multiplied all Z scores by 2.

<LI>Step 4: We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level corresponds approximately to a 1 unit difference.

<LI>Step 5: We eliminated much of the systematic technical variance introduced by the batches at the probe level. To do this we calculated the ratio of each batch mean to the mean of all batches and used this as a single multiplicative probe-specific batch correction factor. The consequence of this simple correction is that the mean probe signal for each batch is the same.


<LI>Step 6: Finally, we computed the arithmetic mean of the values for the set of microarrays for each strain. Technical replicates were averaged before computing the mean for independent biological samples. Note, that we have not (yet) corrected for variance introduced by differences in sex or any interaction terms. We have not corrected for background beyond the background correction implemented by Affymetrix in generating the CEL file. We eventually hope to add statistical controls and adjustments for some of these variables.

</UL>
<B>Probe set data from the CHP file: </B>The expression values were generated using RMA. The same simple steps described above were also applied to these values. Every microarray data set therefore has a mean expression of 8 with a standard deviation of 2. A 1 unit difference represents roughly a two-fold difference in expression level. Expression levels below 5 are usually close to background noise levels.</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote><P>Data were generated with funds to <a
href="mailto:grosen@bidmc.harvard.edu" class="normal">Glenn Rosen</a> from P20
MH62009 (see below for specifics). Samples and arrays were processed by the
<a href="https://www.bidmcgenomics.org/" target="_blank" class="normal">Genomics Core</a> at Beth Israel Deaconess Medical Center by Towia Libermann and colleagues.</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About this text file:</P>
<Blockquote><P>
This text file originally generated by GDR, RWW, and YHQ on Nov 2004. Updated by RWW Nov 17, 2004; GDR and RWW, Dec 23, 2004; RWW and GDR April 8, 2005.
</P></Blockquote>




</TD>
</TR>
<TR>
<TD colspan=2 align=center bgColor=#ddddff>



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE><!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>

<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>

// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure

new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
