<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<span><p><HEAD><TITLE>HBP/Rosen Striatum M430V2 (Novmber04 Freeze) MAS5/ WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>



</HEAD>
<span  bottommargin="5" topmargin="5" leftmargin="5" rightmargin="5"
marginheight="5" marginwidth="5" text=#000000 bgColor=#ffffff><p>
<TABLE cellSpacing=3 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<!-- Body Start from Here -->
<TD vAlign=top width="100%" colspan=2 bgColor=#eeeeee>
<P class="title">HBP/Rosen Striatum M430v2 (Nov04) GCRMA  <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=53">GN53</A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<blockquote>
This November 2004 data freeze provides estimates of mRNA expression
in the <A HREf="http://www.nervenet.org/NetPapers/Rosen/Striatum2001/StriatumMain.html" target="_blank" class ="normal">striatum</A> (caudate nucleus of the forebrain) of BXD recombinant inbred mice measured using
Affymetrix Mouse Genome 430 2.0 short oligomer microarrays. Data were generated at Beth Israel Deaconess Medical Center (BIDMC, Boston MA) by Glenn D. Rosen with the support of a Human Brain Project (HBP) grant. Approximately 125 brain samples from 24 strains were used in this initial experiment. Data were processed using the GC-RMA transform of Wu et al. (2004). To simplify comparison among Nov04 data sets, values of each array have been log2 transformed and adjusted to an average expression of 8 units.
</blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>
<P>This data set includes estimate of gene expression for 24 genetically uniform lines of mice: C57BL/6J (B6, or simply B), DBA/2J (D2 or D), and 22 BXD recombinant inbred (RI) strains derived by crossing female B6 mice with male D2 mice and then inbreeding progeny for over 21 generations. This set of RI strains is a remarkable resource because these strains have been extensively phenotyped for hundreds of interesting traits over a 25-year period (see the WebQTL BXD Published Phenotypes database). A significant advantage of this RI set is that both parental strains (B6 and D2) have been extensively sequenced and are known to differ at approximately 1.8 million SNPs. Coding variants (mostly single nucleotide polymorphisms and insertion-deletions) that may produce interesting phenotypes can be rapidly identified in this particular RI set.</P>

<P>BXD1 through BXD32 were produced by Benjamin A. Taylor starting in the late 1970s. BXD33 through BXD42 were also produced by Taylor, but from a second set of crosses initiated in the early 1990s. These strains are all available from the Jackson Laboratory, Bar Harbor, Maine.
</P>

</Blockquote>

<Blockquote>The table below lists the arrays
by strain, sex, and age. Each array was hybridized to a pool of  mRNA
from 3 to 4 mice.</Blockquote>

<TABLE border="0" cellpadding="0" cellspacing="0"
bgcolor="#000000" width="85%" align="Center">
<TR>
<TD>
<TABLE border="0" cellpadding="5"
cellspacing="1" width="100%">
<TR bgcolor="royalblue">
<td rowspan=1 align="Center"> <font color="#FFFFFF"> Strain
</font> </td>
<td> <font color="#FFFFFF">
<center>
Sex
</center>
</font> </td>
<td rowspan=1 align="Center"> <font color="#FFFFFF"> Strain
</font> </td>
<td> <font color="#FFFFFF">
<center>
Sex
</center>
</font> </td>
</tr>
<!--
<tr bgcolor="royalblue">

<td id="small">
<center>

<font color="#FFFFFF">55-62
days</font></center>

</td>

<td id="small">

<center>

<font color="#FFFFFF">55-62
days</font></center>

</td>

</tr>
-->
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=C57BL/6J"
TARGET="_blank" class="normal">C57BL/6J (B6)</A></td>

<td id="small"><!--<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=DBA/2J"
TARGET="_blank" class="normal">DBA/2J (D2)</A></td>

<td id="small"><!--<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=B6D2F1"
TARGET="_blank" class="normal">B6D2F1 (F1)</A></td>

<td id="small"></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD1"
TARGET="_blank" class="normal">BXD1</A></td>

<td id="small"></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD2"
TARGET="_blank" class="normal">BXD2</A></td>

<td id="small"><!--<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD5"
TARGET="_blank" class="normal">BXD5</A></td>

<td id="small"></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD6"
TARGET="_blank"  class="normal">BXD6</A></td>

<td id="small"><!--<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9792<!--</a>--></td>


<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD8"
TARGET="_blank" class="normal">BXD8</A></td>

<td id="small"></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD9"
TARGET="_blank" class="normal">BXD9</A></td>

<td id="small"><!--A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD11"
TARGET="_blank" class="normal">BXD11</A></td>

<td id="small"><!--<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9792<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD12"
TARGET="_blank" class="normal">BXD12</A></td>

<td id="small"><!--<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9792<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD13"
TARGET="_blank" class="normal">BXD13</A></td>

<td id="small"><!--<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9792<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD14"
TARGET="_blank" class="normal">BXD14</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD15"
TARGET="_blank" class="normal">BXD15</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD16"
TARGET="_blank" class="normal">BXD16</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9792<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD18"
TARGET="_blank" class="normal">BXD18</A></td>

<td id="small"></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD19"
TARGET="_blank" class="normal">BXD19</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9792<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD21"
TARGET="_blank" class="normal">BXD21</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD22"
TARGET="_blank" class="normal">BXD22</A></td>

<td id="small"></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD23"
TARGET="_blank" class="normal">BXD23</A></td>

<td id="small"></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD24"
TARGET="_blank" class="normal">BXD24</A></td>

<td id="small"></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD25"
TARGET="_blank" class="normal">BXD25</A></td>

<td id="small"></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD27"
TARGET="_blank" class="normal">BXD27</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD28"
TARGET="_blank" class="normal">BXD28</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD29"
TARGET="_blank" class="normal">BXD29</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD31"
TARGET="_blank" class="normal">BXD31</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9792<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD32"
TARGET="_blank" class="normal">BXD32</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD33"
TARGET="_blank" class="normal">BXD33</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD34"
TARGET="_blank" class="normal">BXD34</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD38"
TARGET="_blank" class="normal">BXD38</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD39"
TARGET="_blank" class="normal">BXD39</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD40"
TARGET="_blank" class="normal">BXD40</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

</tr>
<tr bgcolor="#eeeeee">
<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD42"
TARGET="_blank" class="normal">BXD42</A></td>

<td id="small"><!--<<A
HREF="/webqtl/main.py?FormID=showImage&name=S092-1F1&image=S092-1F1-U74Av2.png"
target="_blank" class="normal">-->&#9794&#9792<!--</a>--></td>

<td id="small"><A
HREF="/webqtl/main.py?FormID=showStrainInfo&database=Striatum_M430_V2.0_MAS5_November04&strain=BXD67"
TARGET="_blank" class="normal"></A></td>

<td id="small"></td>


</table></td>
</tr>
</table>

<Blockquote>
<P>Select the strain name in the table above to review details about the specific cases and to view the array quality control image processed using the <A Href="http://odin.mdacc.tmc.edu/~zhangli/PerfectMatch/" target="_blank" class="normal">PerfectMatch</A> program by Li Zhang.</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the samples used to generate these data:</P>

<Blockquote><P>Animals were obtained from The Jackson Laboratory and housed for several weeks at BIDMC until they reached ~2 months of age (range from 55 to 62 days). Mice were killed by cervical dislocation and brains were removed and placed in RNAlater for 5 to 10 minutes prior to dissection. Cerebella and olfactory bulbs were removed; brains were hemisected, and both striata were dissected using a medial approach by Rosen that typically yields 5 to 7 mg of tissue per side. The purity of this dissection has been validated by an analysis of acetylcholinestase activity. A pool of dissected tissue from 3 or 4 adults  (approximately 25-30 mg of tissue from 6 striata) of the same strain, sex, and age was collected in one session and used to generate cRNA samples. Rought 90 to 95% of all cells in the striatum are medium spiny neurons (Gerfen, <A HREF="http://www.nervenet.org/netpapers/gerfen/striatum92.html" target="_blank" class="normal">1992</A>, for a review of the structure and function of the neostriatum).

<P><B>mRNA processing:</B> We used the Amersham Biosciences cRNA synthesis kit protocol.</P>

</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platfrom :</P>
<Blockquote>
<P><B>Affymetrix Mouse Genome 430 2.0 array: </B>The <A HREF="http://www.affymetrix.com/support/technical/byproduct.affx?product=moe430-20" target="_blank" class="normal">430v2</A> array consists of 992936 useful 25-nucleotide probes that estimate the expression of approximately 39,000 transcripts (many are actually duplicates). The array sequences were selected late in 2002 using Unigene Build 107. The array nominally contains the same probe sequence as the 430A and B series. However, we have found that roughy 75000 probes differ from those on A and B arrays.</P>
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote>Affymetrix CEL files obtained from the BIDMC <a href="https://www.bidmcgenomics.org/" target="_blank" class="normal">Genomics Core</a>  were processed as follows.
<UL>
<LI>Step 1: We added an offset of 1 to the CEL expression values
for each cell to ensure that all values could be logged without
generating negative values.

<LI>Step 2: We took the log base 2 of each cell.

<LI>Step 3: We computed the Z score for each cell.

<LI>Step 4: We multiplied all Z scores by 2.

<LI>Step 5: We added 8 to the value of all Z scores. The consequence
of this simple set of transformations is to produce a set of Z scores
that have a mean of 8, a variance of 4, and a standard deviation of
2. The advantage of this modified Z score is that a two-fold
difference in expression level corresponds approximately to a 1 unit
difference.

<LI>Step 6: We plotted these modified Z score probe level expression estimates in DataDesk. Male-female scatterplots of the probe signals were compared strain by strain to highlight poor array data sets. A total of 36 arrays passed this stringent quality control step.

<LI>Step 7: We computed the arithmetic mean of the values for the set
of microarrays for each of the individual strains. We have not corrected for background beyond the
background correction implemented by Affymetrix in generating the
CEL file.
</UL>

<B>Probe set data from the CHP file: </B>The expression values were
generated using GCRMA; a still experimental new version of RMA. The <A HREF="http://www.bepress.com/jhubiostat/paper1/" target="_empty" class="fs14">GC-RMA</A> method of Wu et al. (2004) is similar to  to RMA, but incorporate the mismatch signal to estimate background and noise levels. The same simple steps described above
were also applied to these values. Every microarray data set therefore has a mean expression of 8 with a standard deviation of 2. A 1 unit difference therefor represents roughly a two-fold difference in expression level. Expression levels below 5 are usually close to background noise levels.</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote><P>Data were generated with funds to <a
href="mailto:grosen@bidmc.harvard.edu" class="normal">Glenn Rosen</a> from P20
MH62009 (see below for specifics). Samples and arrays were processed by the
<a href="https://www.bidmcgenomics.org/" target="_blank" class="normal">Genomics Core</a> at Beth Israel Deaconess Medical Center by Towia Libermann and colleagues.</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About this test file:</P>
<Blockquote><P>
This text file originally generated by GDR, RWW, and YHQ Nov 2004. Updated by RWW Nov 17, 2004</P></Blockquote>
<P></P>

</TD>
</TR>
<TR>
<TD colspan=2 align=center bgColor=#ddddff>



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE><!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>

<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>

// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure

new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
