<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Information on the HQF Striatum Exon Array data of July 2007</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>
</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->


		
		<TD vAlign=top width="100%" height=200 bgColor=#eeeeee>
		<P class="title">The High Q Foundation Striatum Exon 1.0 Array Expression Dataset of July 2007<A HREF="/webqtl/main.py?FormID=editHtml">
		<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<blockquote>
EXPERIMENTAL EXON ST TEST DATA SET (preliminary text, not error checked).  The July 2007 data freeze provides estimates of mRNA expression in the <A HREf="http://www.nervenet.org/NetPapers/Rosen/Striatum2001/StriatumMain.html" target="_blank" class ="normalsize">striatum</A> (caudate nucleus of the forebrain) of 50 lines of mice, including the C57BL/6J and DBA/2J parental strains, their F1 hybrid (B6D2F1), 30 BXD recombinant inbred strains, and 17 more common inbred strains of mice. Data were generated using the new Affymetrix <A HREF="http://www.affymetrix.com/products/arrays/specific/mouse_exon.affx" target="_blank" class ="normalsize">Mouse Exon 1.0 ST</A> short oligomer microarrays by Weikuan Gu, Yan Jiao, David Kulp, and Lu Lu, Glenn D. Rosen, and Robert W. Williams with the support of a grant from the High Q Foundation. This is the first "all exons" array that we have entered into GeneNetwork and the data are still experimental. Approximately 300 brain samples (males and females) from 50 strains were used in this experiment. This data set includes 97 arrays that passed very stringent quality control procedures. Data were processed using the <A HREF="http://bmbolstad.com/misc/ComputeRMAFAQ/ComputeRMAFAQ.html" target="_blank" class ="normalsize">RMA method</A> of Irizarry, Bolstad, Speed, and colleagues. To simplify comparison among transforms, RMA values of each array were adjusted to an average expression of 8 units and a standard deviation of 2 units.
</blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the strains and cases used to generate this set of data:</P>

<Blockquote> 
<P>We have used a set of 30 BXD recombinant inbred strains generated by crossing C57BL/6J (B6 or B) with DBA/2J (D2 or D). The BXDs are particularly useful for systems genetics because both parental strains have been sequenced (8x coverage of B6 and 1.5x coverage of D). Physical maps in WebQTL maps incorporate approximately 1.75 million B vs D SNPs from <a href="http://www.celeradiscoverysystem.com/index.cfm" target="_blank" class ="normalsize">Celera</a>. BXD2 through BXD32 were bred by Benjamin A. Taylor starting in the late 1970s. BXD33 through 42 were bred by Taylor in the 1990s. All of these strains are available from The Jackson Laboratory.</P>
</P>
</Blockquote>

<blockquote>
<P><B>Mouse Diversity Panel (MDP)</B>. We have also profiled a MDP consisting at total of 19 inbred strains (this number includes the C57BL/6J and DBA/2J strains) and one F1 hybrid (B6D2F1 only; not D2B6F1 yet). Strains were selected for several reasons: 
<UL>
<LI> genetic and phenotypic diversity, including use by the <A HREF="http://aretha.jax.org/pub-cgi/phenome/mpdcgi?rtn=strains%2Flist&listmode=pri&typemode=all&genaltmode=no&submit1=%A0%A0Go%A0%A0" target="_empty" class="fs14">Phenome Project</A>
<LI> their use in making genetic reference populations including recombinant inbred strains, cosomic strains, congenic and recombinant congenic strains
<LI> their use by the <A HREF="http://www.complextrait.org" target="_empty" class="fs14">Complex Trait Consortium</A> to make the Collaborative Cross (Nairobi/Wellcome, Oak Ridge/DOE, and Perth/UWA)
<LI> genome sequence data from three sources (NHGRI, Celera, and <A HREF="http://mouse.perlegen.com/mouse/summary_reports.html" target="_empty" class="fs14">Perlegen-NIEHS</A>)
<LI> availability from The Jackson Laboratory 
</UL>

<P>Seven of the eight parents of the Collaborative Cross (129, A, C57BL/6J, NOD, NZO, PWK, and WSB) have been included. CAST/Ei is the member of the Collaborative Cross that is currently missing from this data set. Thirteen of the MDP strains have been sequenced by Celera, NIH, or by Perlegen for the NIEHS. This panel will be extremely helpful in systems genetic analysis of a wide variety of traits, and will be a powerful adjunct in fine mapping modulators using what is essentially an association analysis of sequence variants.

<OL>
<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/129.shtml" target="_empty" class="fs14">129S1/SvImJ </A> 
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; background for many knockouts; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/A.shtml" target="_empty" class="fs14">A/J</A>    
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by Perlegen/NIEHS; parent of the AXB/BXA panel

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/AKR.shtml" target="_empty" class="fs14">AKR/J</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by NIEHS; Phenome Project B list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/BALB.shtml" target="_empty" class="fs14">BALB/cByJ</A>     
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by NIEHS; maternal parent of the CXB panel; Phenome Project A list



<LI><A HREF="http://jaxmice.jax.org/strain/002282.html" target="_empty" class="fs14">BTBR T<+> tf/J</A>     
<BR>&nbsp;&nbsp;&nbsp;&nbsp; Phenome Project group D strain. Used in mutagenesis studies. This black and tan strain carries the recessive tufted allele and is wildtype at the T locus (brachyury).



<LI><A HREF="http://jaxmice.jax.org/strain/000740.html" target="_empty" class="fs14">BXSB/MpJ</A>     
<BR>&nbsp;&nbsp;&nbsp;&nbsp;An isolated recombinant inbred strain generated by crossing C57BL/6J and SB/Le that is used to study autoimmune disease. Males are deficient in pre-B cells.


<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C3H.shtml" target="_empty" class="fs14">C3H/HeJ</A>      <BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS; paternal parent of the BXH panel; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C57BL.shtml" target="_empty" class="fs14">C57BL/6J</A>      
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by NHGRI; parental strain of AXB/BXA, BXD, and BXH; Phenome Project A list


<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/DBA.shtml" target="_empty" class="fs14">DBA/2J</A>     
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS and Celera; paternal parent of the BXD panel; Phenome Project A list

<LI><A HREF="http://jaxmice.jax.org/strain/001800.html" target="_empty" class="fs14">FVB/NJ</A>     
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS. Phenome Project group A strain.


<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/KK.shtml" target="_empty" class="fs14">KK/HlJ</A>     
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS


<LI><A HREF="http://jaxmice.jax.org/strain/000550.html" target="_empty" class="fs14">MOLF/EiJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS. Phenome Project B strain.


<LI><A HREF="http://jaxmice.jax.org/strain/000684.html" target="_empty" class="fs14">NZB/BlNJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp; Phenome Project B list. Please note that the substrain is B-el-J not B-eye-NJ.

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/NOD.shtml" target="_empty" class="fs14">NOD/LtJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project B list; diabetic

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/NZO.shtml" target="_empty" class="fs14">NZO/HlLtJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain


<LI><A HREF="http://jaxmice.jax.org/strain/001058.html" target="_empty" class="fs14">NZW/LacJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Phenome Project D strain

<LI><A HREF="http://jaxmice.jax.org/strain/004660.html" target="_empty" class="fs14">PWD/PhJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS; parental strain for a consomic set by Forjet and colleagues. Not part of the Phenome Project.

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/PWK.shtml" target="_empty" class="fs14">PWK/PhJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain; Phenome Project D list

<LI><A HREF="http://jaxmice.jax.org/strain/001145.html" target="_empty" class="fs14">WSB/EiJ</A>   <BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project C list

<LI><A HREF="http://jaxmice.jax.org/strain/100006.html" target="_empty" class="fs14">B6D2F1</A>
<BR>This F1 hybrid was generated by crossing C57BL/6J with DBA/2J at the Jackson Laboratory. They are also be designated (incorrectly) as B6D2F1/J. 
</OL> 


<P>All of these strains are available from The Jackson Laboratory.</P>

</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the tissue used to generate this set of data:</P>

<Blockquote><P>Many of the tissue samples used in this exon array study were also used in our previous M430 analysis of the striatum, providing a partially matched Exon-M430 pair of data sets. However, the previous study included fewer samples (47) and fewer strains (31 total). Animals were obtained from The Jackson Laboratory and housed for several weeks at BIDMC until they reached ~2 months of age (range from 55 to 62 days). Mice were killed by cervical dislocation and brains were removed and placed in RNAlater for 20 to 25 minutes prior to dissection. Cerebella and olfactory bulbs were removed; brains were hemisected, and both striata were dissected using a medial approach by GD Rosen that typically yields 5 to 7 mg of tissue per side.

<P>All striatal dissections were performed by one person (GD Rosen) using a midsagittal approach that minimizes the likelihood of contamination across tissues. This dissection recovers most, but not all, of neostraitum. We have histologically examined dissected tissue and have found that no evidence of inclusion of cortical or thalamic tissue at the margins. We have further confirmed the dissections by comparative assays for acetylcholinesterase (AChE) protein levels using Western blots. The concentration of AChE in the striatum is far higher than that in cortex or cerebellum. A pool of dissected tissue from 3 or 4 adults  (approximately 25 to 30 mg of tissue) of the same strain, sex, and age was collected in one session and used to generate cRNA samples. 

<P>Roughly 90 to 95% of all cells in the striatum are medium spiny neurons (Gerfen, <A HREF="http://www.nervenet.org/netpapers/gerfen/striatum92.html" target="_blank" class="fs14">1992</A>, for a review of the structure and function of the neostriatum).
</Blockquote>

<Blockquote>
<P><B>RNA Extraction:</B> RNA was extracted by Rosen and colleagues between June 2, 2004 and March 8, 2006. In brief, we used the RNA STAT-60 <A HREF="http://www.tel-test.com/prod02.htm" target="_empty" class="fs14">protocol</A> (TEL-TEST "B" Bulletin No. 1), steps 5.1A (homogenization of tissue), 5.2 (RNA extraction), 5.3 (RNA precipitation), and 5.4 (RNA wash). In Step 5.4 we stopped after adding 75% ethanol (1 ml per 1 ml RNA STAT-60) and stored the mix at -80 deg C until further use. Before RNA labeling we thawed samples and proceeded with the remainder of Step 5.4; pelleting, drying, and redissovling the pellet in RNAase-free water. 


<!--FULL TEL-TEST text on RNA-60 Stat protocol


<P>5. PROTOCOL:  RNA/mRNA isolation by the RNA STAT-60 method includes the 
following steps:

<br>1. Homogenization RNA STAT-60TM&nbsp;(1 ml per 50-100 mg tissue, or 5-10 x 10-6 cells)
<br>2. RNA Extraction 1 vol. of homogenate +0.2 vol. of chloroform
<br>3. RNA Precipitation 0.5 vol. of isopropanol
<br>4. RNA Wash 75% ethanol

<P>Unless stated otherwise the procedure is carried out at room temperature. 
            
<P>5.1 HOMOGENIZATION

<P>A. TISSUES: Homogenize tissues samples in the RNA STAT-60(1 ml/50-100mg tissue)</B> in a glass-Teflon or Polytron homogenizer. Sample volume should not  exceed 10% of the volume of the RNA STAT-60 used for homogenization.

<P>B. CELLS: Cells grown in mono layer are lysed directly in a culture dish by adding the RNA STAT-60TM (1 ml/3.5 cm petri dish) and passing the cell lysate several times through a pipette. Cells grown in suspension are sediment then lysed in the RNA STAT-60TM (1 ml per 5-10 x 106 cells) by repetitive pipetting. Washing calls before addition of the RNA STAT-60TM should be avoided as this increases the possibility of mRNA degradation. <br>

<P>5.2 RNA EXTRACTION: Following homogenization, store the homogenate for 5 min at room temp to permit the complete dissociation of nucleoprotein complexes. Next, add 0.2 ml of chloroform per 1 ml of the RNA STAT-60, cover the sample tightly, shake vigorously for 15 seconds and let it stay at room temperature for 2-3minutes. Centrifuge the homogenate at 12,000g (max) for 15 minutes at 4 deg C. Following centrifugation, the homogenate separates into two phases: a lower red phenol chloroform phase and the colorless upper aqueous phase. RNA remains exclusively in the aqueous phase whereas DNA and proteins are in the interferes and organic phase. The volume of the aqueous phase is about 60%  of the volume of RNA STAT-60 used for homogenization. 

<P>5.3 RNA PRECIPITATION: Transfer the aqueous phase to a fresh tube and mix with isopropanol. Add 0.5 ml of isopropanol per 1 ml of the RNA STAT-60 used for homogenization. Store samples at room temp for 5-10 minutes and centrifuge at 12,000g (max.) for 10 min at 4 deg C</B>. RNA precipitate (often visible before  centrifugation) forms a white pellet at the bottom of the tube. 

<P>5.4 RNA WASH: Remove supernatant and wash the RNA pellet once with 75% ethanol by vortexing and subsequent centrifugation at 7,500g (max.) for 5 min at 4<SUP>o</SUP>C. Add at least <B>1 ml of 75% ethanol per 1 ml of the RNA STAT-60 used  for the initial homogenization. 

<P>At the end of the procedure, dry the RNA pellet briefly by air-drying or in a vacuum (5-10 min.). It is important not to let the RNA pellet dry completly as it will greatly decrease its solubility. Do not use the Speed-Vac for drying. Dissolve the  RNA pellet in water or in 1 mm EDTA, pH 7, or 0.5% SDS solution. Vortex or pass the pellet a few times through a pipette  tip. An incubation for <B>10-15 minutes at  55-60<SUP>o</SUP>C</B> may be required to dissolve RNA samples. Diethylpyrocarbonate  (DEPC) treated RNase-free solutions<SUP>1</SUP> should be used for solubilization of RNA. 

END OF HIDDEN PROTOCOL TEXT-->

<P>RNA samples were then processed by the array core at the VA Medical Center by Drs. Yan Jiao and Weikuan Gu (Director of the the DNA Discovery Core of the UTHSC Center of Genomics and Bioinformatics). Labeled cRNA was generated using the standard <A HREF="http://www.affymetrix.com/products/reagents/specific/exon_wta_assays.affx" target="_empty" class="fs14">Affymetrix whole transcript sense target labeling protocol</A>.

<P>
<IMG src="/images/upload/wt_cdna_synthesis_amplification.jpg" valign="top">


<P><SMALL><B>Legend: </B>Summary of protocol from http://www.affymetrix.com/products/reagents/wt_cdna_synthesis_amp_chart.jsp) as carried out by Dr. Yan Jiao.</SMALL></P>




<P><B>Replication and Sample Balance:</B> The aim of our standard operating procedure is to obtain data for independent biological sample pools from each sex for all strains. We have succeeded for 44 of 50 strains. Several strains are represented by only a single sex or a single sample pool. This sex imbalance can lead to bias with respect to transcripts that have genuine sex differences. One way to handle this issue is to study the correlation between a proxy variable for this bias, as represented by the <I>Xist</I> probe set 5153684, and a data set of interest. 

</DIR>
</DIR>
<P>
<IMG src="/images/upload/XistStriatumExonJul07.gif" valign="top">


<P><SMALL><B>Legend: </B>Sex balance in this data set is illustrated using the sex-specific <I>Xist</I> gene and one of its probe sets (Affy Exon ST probe set: 5153684). Most samples include one male sample pool with very low Xist expression (6 or 7) and one female sample pool with high Xist expression (10 to 12). As a result 43 of the 50 strains have both intermediate values and high variance. The B6D2F1 sample has no error bar due to an early data entry error. Strains for which samples are only male or only female are at the extreme left and right sides of this bar chart, respectively.</SMALL></P>


<UL>
<LI>Strains with two male samples: KK/HlJ, BTBRT<+>tf/J
<LI>Strains with two female samples:BXD5, BXD22
<LI>Only a single female sample:BXD29
<LI>The status of BXD23 is not clear and may represent a single male sample or a possible mixed sex pool.
</UL>



<P><B>Batch Structure:</B> This data set consists of 97 arrays processed in 8 batches. All arrays were processed by a single skilled operator (Dr. Yan Jiao) between and October 20 and Nov 29, 2006 (scan dates from Oct 26 to Nov 29). In general, the male and female samples from a single strain were run within a single batch.
</Blockquote>


</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data Table 1:</P>
<Blockquote>


<Blockquote>
Mouse Exon 1.0 ST data: The table below lists arrays by strain, age, sex, case id, and batch ID.  Each array was hybridized to a pool of  mRNA from 3 to 4 mice. All mice were between 48 and 71 days. 
</Blockquote>

<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center">
		<tr> 
            <td> 
			<table width="100%" border="0" cellpadding="7" cellspacing="1">
<tr bgcolor="royalblue">
<td><font color=#FFFFFF>RNA ID</font></td>
<td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>Age</font></td>
<td><font color=#FFFFFF>Sex</font></td>
<td><font color=#FFFFFF>Case ID</font></td>
<td><font color=#FFFFFF>Batch<br>ID</font></td>
<td><font color=#FFFFFF>Source</font></td></tr>

<tr bgcolor=#eeeeee><td>R3101SA</td><td>C57BL/6J</td><td>58</td><td>F</td><td>073106.70</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3102SA</td><td>C57BL/6J</td><td>59</td><td>M</td><td>073106.01</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3105SA</td><td>DBA/2J</td><td>58</td><td>F</td><td>073106.65</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3106SA</td><td>DBA/2J</td><td>59</td><td>M</td><td>073106.02</td><td>7</td><td>GDRosen</td></tr>

<tr 
bgcolor=#eeeeee><td>R3031SA</td><td>B6D2F1/J</td><td>59</td><td>F</td><td>073106.69</td><td>2</td><td>GDRosen</td></tr>


<tr bgcolor=#eeeeee><td>R3032SA</td><td>B6D2F1/J</td><td>59</td><td>M</td><td>073106.67</td><td>2</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3037SA</td><td>BXD1</td><td>59</td><td>F</td><td>073106.04</td><td>2</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3038SA</td><td>BXD1</td><td>59</td><td>M</td><td>073106.38</td><td>2</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3055SA</td><td>BXD2</td><td>61</td><td>M</td><td>073106.06</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3056SA</td><td>BXD2</td><td>61</td><td>F</td><td>073106.05</td><td>3</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3089SA</td><td>BXD5</td><td>58</td><td>F</td><td>073106.42</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3090SA</td><td>BXD5</td><td>58</td><td>F</td><td>073106.41</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3091SA</td><td>BXD6</td><td>59</td><td>F</td><td>073106.09</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3092SA</td><td>BXD6</td><td>59</td><td>M</td><td>073106.08</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3093SA</td><td>BXD8</td><td>61</td><td>F</td><td>073106.21</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3094SA</td><td>BXD8</td><td>61</td><td>M</td><td>073106.20</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3095SA</td><td>BXD9</td><td>60</td><td>F</td><td>073106.15</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3096SA</td><td>BXD9</td><td>60</td><td>M</td><td>073106.14</td><td>6</td><td>GDRosen</td></tr>



<tr bgcolor=#eeeeee><td>R3039SA</td><td>BXD11</td><td>59</td><td>F</td><td>073106.07</td><td>2</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3040SA</td><td>BXD11</td><td>59</td><td>M</td><td>073106.24</td><td>2</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3041SA</td><td>BXD12</td><td>62</td><td>F</td><td>073106.27</td><td>2</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3042SA</td><td>BXD12</td><td>59</td><td>M</td><td>073106.26</td><td>2</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3044SA</td><td>BXD13</td><td>60</td><td>M</td><td>073106.32</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3043SA</td><td>BXD13</td><td>60</td><td>F</td><td>073106.33</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3045SA</td><td>BXD14</td><td>59</td><td>F</td><td>073106.51</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3144SA</td><td>BXD14</td><td>59</td><td>M</td><td>073106.52</td><td>3</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3047SA</td><td>BXD15</td><td>60</td><td>F</td><td>073106.50</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3048SA</td><td>BXD15</td><td>60</td><td>M</td><td>073106.49</td><td>3</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3049SA</td><td>BXD16</td><td>61</td><td>F</td><td>073106.22</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3050SA</td><td>BXD16</td><td>61</td><td>M</td><td>073106.23</td><td>3</td><td>GDRosen</td></tr>






<tr bgcolor=#eeeeee><td>R3051SA</td><td>BXD18</td><td>59</td><td>F</td><td>073106.60</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3052SA</td><td>BXD18</td><td>59</td><td>M</td><td>073106.59</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3053SA</td><td>BXD19</td><td>60</td><td>F</td><td>073106.40</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3054SA</td><td>BXD19</td><td>60</td><td>M</td><td>073106.39</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3057SA</td><td>BXD20</td><td>60</td><td>F</td><td>073106.75</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3058SA</td><td>BXD20</td><td>60</td><td>M</td><td>073106.74</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3059SA</td><td>BXD21</td><td>48</td><td>F</td><td>073106.62</td><td>3</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3060SA</td><td>BXD21</td><td>48</td><td>M</td><td>073106.61</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3061SA</td><td>BXD22</td><td>58</td><td>F</td><td>073106.73</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3062SA</td><td>BXD22</td><td>60</td><td>M</td><td>073106.77</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3064SA</td><td>BXD23</td><td>60</td><td>M</td><td>073106.57</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3063SA</td><td>BXD23</td><td>60</td><td>F</td><td>073106.58</td><td>8</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3065SA</td><td>BXD24</td><td>59</td><td>F</td><td>073106.03</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3066SA</td><td>BXD24</td><td>60</td><td>M</td><td>073106.12</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3067SA</td><td>BXD27</td><td>60</td><td>F</td><td>073106.55</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3068SA</td><td>BXD27</td><td>60</td><td>M</td><td>073106.56</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3069SA</td><td>BXD28</td><td>60</td><td>F</td><td>073106.47</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3070SA</td><td>BXD28</td><td>60</td><td>M</td><td>073106.48</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3071SA</td><td>BXD29</td><td>58</td><td>F</td><td>073106.45</td><td>4</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3072SA</td><td>BXD29</td><td>58</td><td>M</td><td>073106.46</td><td>5</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3074SA</td><td>BXD31</td><td>60</td><td>M</td><td>073106.43</td><td>8</td><td>GDRosen</td></tr>



<tr bgcolor=#eeeeee><td>R3073SA</td><td>BXD31</td><td>60</td><td>F</td><td>073106.44</td><td>5</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3075SA</td><td>BXD32</td><td>57</td><td>F</td><td>073106.37</td><td>8</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3076SA</td><td>BXD32</td><td>57</td><td>M</td><td>073106.36</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3077SA</td><td>BXD33</td><td>59</td><td>F</td><td>073106.35</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3078SA</td><td>BXD33</td><td>59</td><td>M</td><td>073106.34</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3079SA</td><td>BXD34</td><td>60</td><td>F</td><td>073106.29</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3080SA</td><td>BXD34</td><td>60</td><td>M</td><td>073106.28</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3081SA</td><td>BXD36</td><td>57</td><td>F</td><td>073106.31</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3082SA</td><td>BXD36</td><td>57</td><td>M</td><td>073106.30</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3083SA</td><td>BXD38</td><td>60</td><td>F</td><td>073106.19</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3084SA</td><td>BXD38</td><td>60</td><td>M</td><td>073106.18</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3085SA</td><td>BXD40</td><td>60</td><td>F</td><td>073106.17</td><td>5</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3086SA</td><td>BXD40</td><td>60</td><td>M</td><td>073106.16</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3087SA</td><td>BXD42</td><td>58</td><td>F</td><td>073106.11</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3088SA</td><td>BXD42</td><td>58</td><td>M</td><td>073106.10</td><td>6</td><td>GDRosen</td></tr>


<tr bgcolor=#eeeeee><td>R3025SA</td><td>129S1/SvImJ</td><td>60</td><td>F</td><td>073106.12</td><td>1</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3026SA</td><td>129S1/SvImJ</td><td>59</td><td>M</td><td>073106.87</td><td>1</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3027SA</td><td>A/J</td><td>59</td><td>F</td><td>073106.93</td><td>1</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3028SA</td><td>A/J</td><td>59</td><td>M</td><td>073106.95</td><td>1</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3029SA</td><td>AKR/J</td><td>59</td><td>F</td><td>073106.89</td><td>1</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3030SA</td><td>AKR/J</td><td>59</td><td>M</td><td>073106.91</td><td>1</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3033SA</td><td>BALB/cByJ</td><td>59</td><td>M</td><td>073106.99</td><td>2</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3036SA</td><td>BTBR/T+tf/J</td><td>60</td><td>M</td><td>073106.10</td><td>2</td><td>GDRosen</td></tr>

<tr bgcolor=#eeeeee><td>R3034SA</td><td>BTBR/T+tf/J</td><td>59</td><td>F</td><td>073106.97</td><td>2</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3035SA</td><td>BTBRT+tf/J</td><td>59</td><td>M</td><td>073106.11</td><td>2</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3097SA</td><td>BXSB/MpJ</td><td>61</td><td>F</td><td>073106.15</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3098SA</td><td>BXSB/MpJ</td><td>61</td><td>M</td><td>073106.14</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3099SA</td><td>C3H/HeJ</td><td>60</td><td>F</td><td>073106.11</td><td>6</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3100SA</td><td>C3H/HeJ</td><td>60</td><td>M</td><td>073106.11</td><td>6</td><td>GDRosen</td></tr>


<tr bgcolor=#eeeeee><td>R3107SA</td><td>FVB/NJ</td><td>60</td><td>F</td><td>073106.11</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3108SA</td><td>FVB/NJ</td><td>60</td><td>M</td><td>073106.11</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3109SA</td><td>KK/HlJ</td><td>61</td><td>M</td><td>073106.13</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3110SA</td><td>KK/HlJ</td><td>61</td><td>M</td><td>073106.13</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3111SA</td><td>MOLF/EiJ</td><td>60</td><td>F</td><td>073106.13</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3112SA</td><td>MOLF/EiJ</td><td>60</td><td>M</td><td>073106.13</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3113SA</td><td>NOD/LtJ</td><td>58</td><td>F</td><td>073106.79</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3114SA</td><td>NOD/LtJ</td><td>58</td><td>M</td><td>073106.81</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3115SA</td><td>NZB/BinJ</td><td>61</td><td>F</td><td>073106.16</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3116SA</td><td>NZB/BinJ</td><td>58</td><td>M</td><td>073106.10</td><td>7</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3117SA</td><td>NZO/HlLtJ</td><td>61</td><td>F</td><td>073106.12</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3118SA</td><td>NZO/HlLtJ</td><td>61</td><td>M</td><td>073106.12</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3119SA</td><td>NZW/LacJ</td><td>65</td><td>F</td><td>073106.13</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3120SA</td><td>NZW/LacJ</td><td>70</td><td>M</td><td>073106.12</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3121SA</td><td>PWD/PhJ</td><td>70</td><td>F</td><td>073106.14</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3122SA</td><td>PWD/PhJ</td><td>70</td><td>M</td><td>073106.14</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3123SA</td><td>PWK/PhJ</td><td>59</td><td>F</td><td>073106.12</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3124SA</td><td>PWK/PhJ</td><td>60</td><td>M</td><td>073106.13</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3125SA</td><td>WSB/EiJ</td><td>71</td><td>F</td><td>073106.13</td><td>8</td><td>GDRosen</td></tr>
<tr bgcolor=#eeeeee><td>R3126SA</td><td>WSB/EiJ</td><td>71</td><td>M</td><td>073106.11</td><td>8</td><td>GDRosen</td></tr>

  </table>
			  </td>
          </tr>
        </table>
</Blockquote>


</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platfrom :</P>
<Blockquote>
<P><B>Affymetrix Mouse Exon ST 1.0 array: </B>The <A HREF="http://www.affymetrix.com/products/arrays/specific/mouse_exon.affx" target="_blank" class="fs14">Exon 1.0 ST</A> (sense target) array consists of approximately 4.5 million useful 25-nucleotide probes that estimate the expression of approximately 1 million exon clusters. The array sequences were selected in 2006 using Unigene Build XXX. </P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote><B>Probe (cell) level and Probe set data from the CEL file: </B>

1. Probes overlapping SNPs were removed from the design file
2. Affymetrix Power Tools(APT) package was used extract CEL values and perform RMA normalization
3. Probe set values were normalized to mean=8 and sd=2 (per chip)
4. Strain average was calculated by averaging over chips that belong to same strain

<UL>

<LI>Step 1: Probes overlapping SNPs were removed from the design file

<LI>Step 2: Affymetrix Power Tools(APT) package was used extract CEL values and perform RMA normalization 

<LI>Step 3: Probe set values were normalized to mean=8 and sd=2 (per array)

<LI>Step 4: Strain averages were calculated by averaging over all arrays that belong to same strain (3 maximum in this data set)



</UL>

<B>Probe set data from the CHP file: </B>The expression values were 
generated by Manjunatha in David Kulp's group at the University of Massachusetts Amherst using <A HREF="http://bmbolstad.com/misc/ComputeRMAFAQ/ComputeRMAFAQ.html" target="_blank" class ="normalsize">RMA</A>. The same simple steps described above 
were also applied to these values. Every microarray data set 
therefore has a mean expression of 8 with a standard deviation of 2. 
A 1 unit difference represents roughly a two-fold difference 
in expression level. Expression levels below 5 are usually close to 
background noise levels. </Blockquote>

<Blockquote><B>Data quality control: </B>A total of 97 samples passed RNA quality control. 

<P>Part1: Testing if replicates come from the same strain

<OL>
<LI> RMA normalized values were used in this analysis
<LI>Pair-wise correlations were calculated between all the arrays using the probesets with high variance and high median
<LI>Probability density of correlations between non-replicate pairs and replicate-pairs were calculated
<LI>Threshold of 0.85 using Maximum likelihood estimate
<LI>In total 5 set of replicates might not have come from the same strains. (They are marked as 0 in Manju_Quality Score column)
</OL>

<P>Part 2: Testing if strain labeling is correct
<OL>
<LI>RMA normalized values were used in this analysis
<LI>Only BXD strains were tested
<LI>A set of strongly cis-linked probesets were identified (using linkage to nearest marker)
<LI>The expression of these probesets was used to re-estimate the genotype of nearest marker
<LI>The values of all re-estimated marker genotypes were compared to genotypes of all the BXD strains and optimal match was identified
<LI>In total four set of replicates were found to be mislabeled.
</OL>

<P>Probe set level QC: The final normalized array data were evaluated for outliers. XXX arrays were considered outliers.  These XXX suspect arrays were elimated from this data set. The following arrays were eliminated: XXX, YYY, ZZZ.</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote><P>Data were generated with funds to Weikuan Gu, <a 
href="mailto:rwilliam@nb.utmem.edu.edu" class="fs14">Rob Williams</a>, <a 
href="mailto:grosen@bidmc.harvard.edu" class="fs14">Glenn Rosen</a> from the High Q Foundation. Samples and arrays were processed by Dr. Yan Jiao 
<a href="XXXX" target="_blank" class="fs14">Array Core</a> at the University of Tennessee Health Science Center and VA Medical Center, Memphis.</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About this text file:</P>
<Blockquote><P>
This text file originally generated by RWW on July 24, 2007 using a template from a previous M430 Striatum data set. Updated by RWW July 26, 2007; MJ and RWW, Aug 7, 2007.
</P></Blockquote>




<P></P>

		</TD>
	</TR>
	<TR>
		<TD colspan=2 align=center bgColor=#ddddff>
	 
	 
	
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->



		</TD>
	</TR>
</TABLE><!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>

<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	// Note where menu initialization block is located in HTML document.
	// Don't try to position menu locating menu initialization block in
	// some table cell or other HTML element. Always put it before </body>

	// each menu gets two parameters (see demo files)
	// 1. items structure
	// 2. geometry structure

	new menu (MENU_ITEMS, MENU_POS);
	// make sure files containing definitions for these variables are linked to the document
	// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
	// error in menu_tpl.js file or that file isn't linked properly.
	
	// also take a look at stylesheets loaded in header in order to set styles
	//-->
</script>
</BODY>
</HTML>
