<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>UCLA CXBF2 Brain (2005) mlratio</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">

<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>

<SCRIPT SRC="/javascript/dhtml.js"></SCRIPT>


</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff >

<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD  bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5>
<TR>

<!-- split from Here -->
<!-- Body Start from Here -->
<P class="title">UCLA CAST/Ei X C57BL/6J reciprocal F2 Intercross Adipose Tissue (2005) mlratio <A HREF="/webqtl/main.py?FormID=editHtml">
<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=170">GN170</A></P>


<P>General information and background for all UCLA/Rosetta data sets:
<OL>
<LI>BH/HB F2 UCLA data sets
<LI>BHF2 (Apoe Null) UCLA data sets
<LI>CastB6/B6CastF2 UCLA data sets

<P>[This paragraph applies to BH/HB F2 UCLA data set in GeneNetwork]. C57BL/6J (B6) mice were intercrossed of C3H/HeJ (C3H) mice to generate 321 F2 progeny (161 females, 160 males) for the BXH wild type (BXH/wt, also known as BHF2, but not the same as wildtype BXH recombinant inbred strains). These F2 animals were generated by reciprocal intercrosses of either B6xC3H (BH) F1 parents or C3HxB6 (HB) F1 parents. As a result the Y chromosome and mitochondrial genomes of these two parts of the reciprocal cross with be different (BH will have B6-type mitochondria and males will have a C3H-type Y chromosomes).

<P>[This paragraph applies to CastB6/B6Cast F2 UCLA data set in GeneNetwork]. C57BL/6J (B6) mice were intercrossed with inbred derivatives of Mus castaneus (CAST/EiJ) mice to generate 442 F2 progeny (276 females, 166 males) for the BXC cross. These F2 animals were generated by reciprocal intercrosses of either CAST/EiJ x C57BL/6J (CB) F1 parents or C57BL/6J  x CAST/EiJ  (BC) F1 parents. As a result the Y chromosome and mitochondrial genomes of these two parts of the reciprocal cross with be different (CB will have CAST-type mitochondria and males will have an B6-type Y chromosomes).

All mice were maintained on a 12 h lightâ€“12 h dark cycle and fed ad libitum. BH/HB F2 mice were fed Purina Chow (Ralston-Purina) containing 4% fat until 8 weeks of age. From that time until the mice were killed at 20 weeks, mice were fed a high fat Western diet (Teklad 88137, Harlan Teklad) containing 42% fat and 0.15% cholesterol. CastB6/B6Cast F2 mice were fed Purina Chow until 10 weeks of age, and then fed the same high fat Western diet (Teklad 88137, Harlan Teklad) for the subsequent 8 weeks. Mice were fasted overnight before they were killed. Their liver, white adipose tissue, and whole brains were collected, flash frozen in liquid nitrogen, and stored in -80 deg C prior to RNA isolation.

<P>[This paragraph applies to the BHF2 (Apoe Null) UCLA database in GeneNetwork.] The BHF2 cross on an ApoE null background has been described previously by Yang and colleagues (<A HREF="http://genome.cshlp.org/cgi/content/abstract/16/8/995">2006</A>). (These animals are referred to as BXH/apoE mice in the original publication but in GeneNetwork these mice are referred to as the <B>BHF2 (Apoe Null) UCLA</B> group to avoid confusion with BXH recombinant inbred strains of mice). To generate this animals,  C57BL/6J carrying a knock allele of the <I>Apoe</I> gene (B6.ApoE -/-) were purchased from the Jackson Laboratory. C3H/HeJ Apoe null (C3H.Apoe -/-) were generated by backcrossing B6.Apoe -/- to C3H for 10 generations. F1 mice were generated from reciprocal intercrossing between B6.ApoE -/- and C3H.Apoe -/-, and F2 mice were subsequently bred by intercrossing F1 mice. A total of 334 (169 female, 165 male) were bred, and all were fed Purina Chow containing 4% fat until 8 weeks of age, and then transferred to a high fat Western diet containing a 42% fat and 0.15% cholesterol for 16 weeks. Mice were killed at 24 weeks, and liver, white adipose tissue, and whole brains were immediately collected and flash-frozen in liquid nitrogen.

<P>All procedures of housing and treatment of animals were performed in accordance with Institutional Animal Care and Use Committee regulations (UCLA).

<P><B>Array design and preparation of labeled cDNA and hybridizations to microarrays for the mouse tissue samples.</B> RNA preparation and array hybridizations were performed at Rosetta Inpharmatics. The custom ink-jet microarrays used in the BH/HBF2, BHF2-Apoe Null, and CastB6/B6CastF2 crosses were manufactured by Agilent Technologies. The array used for the BHF2-Apoe Null and BH/HBF2 samples consisted of 2,186 control probes and 23,574 noncontrol oligonucleotides extracted from mouse Unigene clusters and combined with RefSeq sequences and RIKEN full-length cDNA clones (see Table <A HREF="http://biology.plosjournals.org/archive/1545-7885/6/5/supinfo/10.1371_journal.pbio.0060107.st005.xls">S5</A>). The array used for the CastB6/B6CastF2 cross consisted of 39,280 noncontrol oligonuceotides again extracted from the mouse Unigene clusters and combined with RefSeq sequences and RIKEN full-length cDNA clones (see Table <A HREF="http://biology.plosjournals.org/archive/1545-7885/6/5/supinfo/10.1371_journal.pbio.0060107.st006.xls">S6</A>).

<P>Mouse adipose, liver, brain, and muscle from all crosses were homogenized, and total RNA was extracted using Trizol (Invitrogen) according to manufacturer's protocol. Approximately three micrograms of total RNA was reverse transcribed and labeled with either Cy3 or Cy5 fluorochrome. Labeled complementary RNA from each F2 animal was hybridized against a cross-specific pool of labeled cRNAs constructed from equal aliquots of RNA from F2 animals and parental mouse strains for each of the tissues for each cross. The hybridizations for the BHF2-Apoe null animals were performed in fluor reversal for 24 h in a hybridization chamber, washed, and scanned using a confocal laser scanner. The hybridizations for the BH/HB F2 wildtype and CastB6/B6Cast F2 crosses were performed to single arrays (individuals F2 samples labeled with Cy5 and reference pools labeled with Cy3 fluorochromes) for 24 h in a hybridization chamber, washed, and again scanned using a confocal laser scanner.

<P>Arrays were quantified on the basis of spot intensity relative to background, adjusted for experimental variation between arrays using average intensity over multiple channels, and fitted to a previously described error model to determine significance (type I error) as described by  He and colleagues (<A HREF="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/19/8/956">2003<A>. Gene expression measures are reported as the ratio of the mean log10 intensity (mlratio).

<P>This description of the experiment was adapted from the following reference by RW Williams (Sept 4, 2008):

<BR> Schadt EE, Molony C, Chudin E, Hao K, Yang X, Lum PY, Kasarskis A, Zhang B, Wang S, Suver C, Zhu J, Millstein J, Sieberts S, Lamb J, GuhaThakurta D, Derry J, Storey JD, Avila-Campillo I, Kruger MJ, Johnson JM, Rohl CA, van Nas A, Mehrabian M, Drake TA, Lusis AJ, Smith RC, Guengerich FP, Strom SC, Schuetz E, Rushmore TH, Ulrich R (<A HREF="http://biology.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pbio.0060107">2008</A>) Mapping the genetic architecture of gene expression in human liver. PLoS Biology 6:e107.





</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">
<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
<TR>
<TD colspan=3 class="fs12">
<UL>

</UL>
</TD>
</TR>
</TABLE>
<!--End of footer-->
</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->

</script>
</BODY>
</HTML>

