<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>UTK Illumina Spleen (Jan10) VST **</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">

<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<link rel="stylesheet" media="all" type="text/css" href="/css/tabbed_pages.css" />
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>

<SCRIPT SRC="/javascript/dhtml.js"></SCRIPT>

<script src="/javascript/tabbed_pages.js" type="text/javascript"></script>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff >

<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD  bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5>
		<TR>

		<!-- split from Here -->
		<!-- Body Start from Here -->
		<P class="title">UTK Illumina Spleen (Jan10) VST ** (accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=260">GN260</A>) <A HREF="/webqtl/main.py?FormID=editHtml">

<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>


<P><b>Status of Data</B>: Open data; please cite Lynch RM, Naswa S, Rogers GL Jr, Kania SA, Das S, Chesler EJ, Saxton AM, Langston MA, Voy BH (2010) Identifying genetic loci and spleen gene coexpression networks underlying immunophenotypes in BXD recombinant inbred mice. <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/20179155">Physiological Genomics (2010)</A></b></p>


<P><b>Summary: (Taken verbatim from the GEO record )</b></P>
<P>The immune system plays a pivotal role in susceptibility to and progression of a variety of diseases. Due to its strong genetic basis, heritable differences in immune function may contribute to differential disease susceptibility between individuals. Genetic reference populations, such as the BXD (C57BL/6J X DBA/2J) panel of recombinant inbred (RI) mouse strains, provide a unique model through which to integrate baseline phenotypes in healthy individuals with heritable risk for disease because of the ability to combine data collected from these populations across multiple studies and time. We performed basic immunophenotyping (e.g. percentage of circulating B and T lymphocytes and CD4+ and CD8+ T cell subpopulations) in peripheral blood of healthy mice from 41 BXD RI strains to define the phenotypic variation in this model system and to characterize the genetic architecture that unlerlies these traits. Significant QTL models that explained the majority (50-77%) of phenotypic variance were derived for each trait and for the T:B cell and CD4+:CD8+ ratios. Combining QTL mapping with spleen gene expression data uncovered two quantitative trait transcripts (QTTs), Ptprk and Acp1, that which are candidates for heritable differences in the relative abundance of helper and cytotoxic T cells. These data will be valuable in extracting genetic correlates of the immune system in the BXD panel. In addition, they will be a useful resource in prospective, phenotype-driven model selection to test hypotheses about differential disease or environmental susceptibility between individuals with baseline differences in the composition of the immune system.

<p>Reference Abstract: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20179155" target="blank">Identifying genetic loci and spleen gene coexpression networks underlying immunophenotypes in BXD recombinant inbred mice.</a></P>

</P><b>Overall design:</b></P>
</P>Spleen gene expression was analyzed from 38 BXD strains. Adult mice (8-12 weeks) were euthanized by cervical dislocation and spleens were harvested and stabilized in RNAlater. Total RNA was extracted and gene expression profiling was performed on the Illumina Sentrix mouse-6 gene expression arrays v1.1. Each BXD sample profiled consisted of a pool of equal amounts of RNA from two individuals of the same sex and strain (approximately 15ug per strain). In addition, flow cytometry was used for the immunophenotyping of male and female mice (average of four mice/sex/strain) from 41 BXD strains (spleen expression profiling was performed on 34 of these strains) and the parental strains. Lymphoctes were identified as CD3+, CD79+, CD4+, or CD8+ to identify T cells, B cells, T helper cells, and cytotoxic T cells, respectively. These data are presented as percentage of lymphoctes with those cell surface markers (e.g. CD3%, CD79%, CD4%, CD8%). Lymphocyte subpopulations are also represented as natural log-transformed ratios (e.g. LN T:B, LN CD4:CD8). In addition, the median expression of MHCII on B cells is reported (LN MHC Med). The immunophenotype data is available in the supplementary file.</P>

<P>This data set generates eQTLs with a maximum LOD of 42.7 and a maximum LRS of 196.7 using a <I>Bat5</I> probe (Trait ID: ILM5720687). A total of 676 probe sets out of 34,491 probes have LOD scores above 10 and LRS scores above 46. These values are remarkably high and indicative of excellent data quality.


				
		</TR></TABLE>


<table align="left" width=500px>
<tr><td>Contributor(s)</td><td>Lynch RM, Voy BH</td></tr>
<tr><td>Submission date</td><td>Jan 19, 2010</td></tr>
<tr><td>Contact name</td><td>Rachel Lynch</td></tr>
<tr><td>E-mail(s)</td><td>rlynch@utk.edu</td></tr>
<tr><td>Organization name</td><td>University of Tennessee</td></tr>
<tr><td>Department</td><td>Life Sciences</td></tr>
<tr><td>Lab</td><td>Brynn Voy</td></tr>
<tr><td>Street address</td><td>2640 Morgan Circle Drive, 51f McCord Hall</td></tr>
<tr><td>City</td><td>Knoxville</td></tr>
<tr><td>State/province</td><td>TN</td></tr>
<tr><td>ZIP/Postal code</td><td>37996</td></tr>
<tr><td>Country</td><td>USA</td></tr>
</table>


		</TD>
	</TR>
	<TR>
		<TD align=center bgColor=#ddddff class="solidBorder">
		<!--Start of footer-->
		<TABLE width="90%">
			<script language='JavaScript' src='/javascript/footer.js'></script>
			<TR>
				<TD colspan=3 class="fs12">
					<UL>
					
					</UL>
				</TD>
			</TR>
		</TABLE>
		<!--End of footer-->
		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	// Note where menu initialization block is located in HTML document.
	// Don't try to position menu locating menu initialization block in
	// some table cell or other HTML element. Always put it before </body>
	// each menu gets two parameters (see demo files)
	// 1. items structure
	// 2. geometry structure
	new menu (MENU_ITEMS, MENU_POS);
	// make sure files containing definitions for these variables are linked to the document
	// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
	// error in menu_tpl.js file or that file isn't linked properly.
	
	// also take a look at stylesheets loaded in header in order to set styles
	//-->

</script>
</BODY>
</HTML>
