<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>VCU BXD PFC Et vs Sal M430 2.0 (Dec06) Sscore</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>
</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->

<TD vAlign=top width="100%" height=200 bgColor=#eeeeee>
<P class="title">VCU BXD PFC Et vs Sal M430 2.0 (Dec06) Sscore <A HREF="/webqtl/main.py?FormID=editHtml">
<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=137">GN137</A></P>
<p>
<b>Summary:</b>
<p>
This BXD data set provides estimates of ethanol-responsive mRNA expression in the prefrontal cortex across 27 BXD recombinant inbred strains and their B6 and D2 progenitor strains.  All samples are from a total of 468 adult male animals obtained from Jackson Laboratory and raised in a standard laboratory environment.  An average of 8 males per strain was used to measure the anxiety-like behavior in response to restraint and treatment with either saline or 1.8g/kg ethanol in the light-dark transition model of anxiety.  Four hours after treatment, animals were rapidly sacrificed by cervical dislocation, brains were removed, cooled and microdissected as previously described (Kerns et al., J. Neurosci. 25:2255, 2005).
<p>
All RNA isolation and subsequent probe generation and hybridization to microarrays were completed using a supervised randomization procedure to minimize batch effects. Affymetrix M430 type 2.0 microarrays were used for hybridization using standard procedures.  Expression analysis was conducted by estimating the relative abundance of over 45,000 transcripts in the prefrontal cortex in response to ethanol or saline and transforming these data using the S-score method to compare ethanol vs. saline expression from pairs of arrays for each strain (Kerns et al., Methods 31:274, 2003). The S-score is a method developed for Affymetrix oligonucleotide arrays that is particularly suited to comparing expression on 2 or a small number of chips (Zhang et al., 2002).  The S-score output for each probeset is not an indication of expression magnitude but rather, the change in expression between compared arrays.  The S-score is essentially a z-score centered around zero with positive S-scores reflecting increased gene expression with ethanol and negative scores reflecting ethanol-induced decreases in expression.  Larger magnitude S-scores show more significant changes in expression and are generally correlated with the "fold-change".

<P>Zhang, L., et al., 2002. A new algorithm for analysis of oligonucleotide arrays: application to expression profiling in mouse brain regions. J Mol Biol. 317, 225-35.
<p>
<b>Animals and Tissue Used to Generate This Set of Data:</b>
<p>
All animals were obtained at 8-9 weeks of age from the Jackson Laboratory (Bar Harbor, ME) and were treated, behaviorally tested and brains dissected by Alex Putman and colleagues at VCU.  Following an hour acclimation period to the behavioral room, animals were restrained for 15 minutes, immediately injected (I.P.) with either 0.9% saline or 1.8g/kg ethanol, and 5 minutes later placed in the light-dark box for a 10-minute test session.  All behavioral testing occurred between 10 AM and 1 PM during the light phase over a 12 month period beginning August 2005.  Four hours after treatment, animals were rapidly sacrificed by cervical dislocation, brains were removed, cooled and microdissected.  Prefrontal cortex tissue was isolated by microdissection using a wedge-shaped slice taken from a 4 mm thick brain slice extending rostrally from the optic chiasm. The wedge was centered on the inter-hemispheric fissure and extending 2 mm laterally on each side and ventrally to just above the corpus callosum. This tissue and all other brain regions were dissected in less than 5 minutes per mouse and were immediately frozen in liquid nitrogen followed by storage at -80 oC prior to RNA isolation.  A pool of dissected tissue from 3 mice of the same strain/treatment group was used to generate RNA samples. All RNA samples were extracted at VCU by Alex Putman during October 2006 and the order of RNA isolation was randomized across all strains and treatment groups.
<p>
<b>Sample Processing:</b>
<p>
All samples were processed by Paul Vorster and Alex Putman at VCU between October and November 2006.  The BioRad Experion RNA analyzer and used to assess total RNA integrity and verify equal molar ratios of 18S and 28S ribosomal RNA. Standard Affymetrix reagents and protocols were used for generation of cDNA and biotinylated cRNA from total RNA samples. Integrity of cRNA was checked by Experion analysis prior to microarray hybridizations. All probes exceeded a maximum size of 3000 nt for the upper border of the cRNA size distribution.
<p>
<b>Replication and Sample Balance:</b>
<p>
At present, this ethanol-responsive prefrontal cortex mRNA expression BXD data set is represented by a total of 2 microarrays for each BXD strain, one each for saline- and ethanol-treated animals, and 6 microarrays for each progenitor strain, three each for saline and ethanol-treated animals.
<p>
<b>Experimental Design and Batch Structure:</b>
<p>
This data set was generated by comparing saline vs. ethanol prefrontal cortex BXD microarray data, as described in Summary, and consisted of 64 microarrays processed in 5 groups of 8 to 16 microarrays during the month of September 2006.  All RNA extractions, cRNA synthesis, and hybridizations were randomized across strain and treatment groups to minimize batch effects.

<P>References:</B>
<P>Many of the techniques used to generate this data set are described in a recent publication in the <A HREF="http://www.jneurosci.org/cgi/content/full/25/9/2255" class="fs14"  target="_empty">Journal of Neuroscience</A>.


<p>
<b>Data Source Acknowledgments:</b>
<p>
Data were generated with funds to Mike Miles from the NIAAA.
</p>
</TD>

<!-- Body End Here -->
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">




<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->




</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
