<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Introduction</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='css/menu.css'>
<SCRIPT SRC="javascript/webqtl.js"></SCRIPT>


</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		<TD vAlign=top width="55%" bgColor=#eeeeee>
			<P class="title">What is GeneNetwork?<A HREF="/webqtl/main.py?FormID=editHtml">
			<img src="images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>
			<BLOCKQUOTE>
				
<P><A HREF="http://en.wikipedia.org/wiki/Genenetwork">GeneNetwork</A> is a group of linked data sets and tools used to study complex networks of genes, molecules, and higher order gene function and phenotypes. GeneNetwork combines more than 25 years of legacy data generated by hundreds of scientists together with sequence data (SNPs) and massive transcriptome data sets (expression genetic or eQTL data sets). The quantitative trait locus (QTL) mapping module that is built into GN is optimized for fast on-line  analysis of traits that are controlled by combinations of gene variants and environmental factors. GeneNetwork can be used to study humans, mice (BXD, AXB, LXS, etc.), rats (HXB), Drosophila, and plant species (barley and Arabidopsis). Most of these population data sets are linked with dense genetic maps  (genotypes) that can be used to locate the genetic modifiers that cause differences in expression and phenotypes, including disease susceptibility.</P>

<P>
Users are welcome to enter their own private data directly into GeneNetwork to exploit the full range of analytic tools and to map modulators in a powerful environment. This combination of data and fast analytic functions enable users to study relations between sequence variants, molecular networks, and function.</P>
</BLOCKQUOTE>

<P class="title">What can I do with GeneNetwork?</P>

<BLOCKQUOTE>

<P><B>QTL Mapping:</B> </P>

<P><U>Interval Mapping</U>: Statistical tests of association between trait values and the genotypes of marker loci through the genome. A significant association is interpreted as indicating the presence of a QTL linked to the marker that shows the association. </P>

<P><U>Simple interval mapping</U>: This method evaluates the association between the trait values and the expected genotype of a hypothetical QTL (the target QTL) at multiple analysis points between each pair of adjacent 
marker loci. The analysis point that yields the most significant associations may be taken as the location of a putative QTL. Bootstrap methods may be performed for estimating confidence intervals on QTL location. </P>

<P><U>Composite interval mapping</U>: Like simple interval mapping, this method evaluates the possibility of a target QTL at multiple analysis points across each interlocus interval. However, at each point it also includes in the analysis the effect of one or more markers elsewhere in the genome. These markers, also called background markers, have previously been shown to be associated with the trait and therefore are each presumably close to another QTL (a background QTL).  </P>

<P><U>Pair-scan</U>: This method evaluates all marker pairs in two-locus models including main effects of each locus and their interaction. These allow discovery of multiple QTL models for complex phenotypes. For all mapping methods Permutation tests may also be selected to establish empirical significance thresholds.</P>
</BLOCKQUOTE>

<BLOCKQUOTE>
<P><B>Multiple Types of Correlation Analysis:</B> Enables you to study the correlation between traits using a variety of methods including Pearson and Spearman correlations, partial correlations, literature correlations (based on Semantic Gene Organizer data), and tissue correlations. Trait values entered by users or retrieved from the databases can also be correlated "in bulk" against any other database of phenotype from the population of cases. </P>
				
<P><U>Correlation Matrix / Principal Components Analysis</U>: Enables you to compare the values of up to 100 traits in a Trait Collection using correlation matrices. You can export correlations matrices and you can generate novel synthetic phenotypes by using the Principal Component derivatives of your group of traits..  </P>
				
<P><U>QTL Heatmaps</U>: Enables you to simultaneously map and analyze up to 100 traits using the QTL heatmap feature. The traits can be ordered by similarity of correlation (hierarchical clustering) or by their order in the genome. The QTL Heatmaps make is easy to identify common and unique genetic determinants of large sets of  phenotypes.  </P>
				
<P><U>Compare Correlates</U>: Enables you to find shared genetic correlates  among a group of traits by correlating them with all records from any database. </P>
				
<P><U>Network Graph</U>: Enables you to examine the network of associations among large groups of phenotypes. Most graphs are interactive and allow users to define interesting trait sets which can be temporarily stored for further analysis in GeneNetwork.</P>
</BLOCKQUOTE>
			
<BLOCKQUOTE>
				<P><B>Systems Genetics and Complex Trait Analysis:</B></P> 
				<P>GeneNetwork pages are extensively connected to external 
				resources. Numerous links to the UCSC and Ensembl Genome Browsers, PubMed, Entrez Gene, GNF Expression
				Atlas, ABI Panther, and WebGestalt provide users with
				rapid interpretive information about genomic regions, published phenotypes and
				genes highlighted in WebQTL.</P>
			</BLOCKQUOTE>
		</TD>
		<TD vAlign=top width="45%" bgColor=#eeeeee>
			<P class="title">How to Use WebQTL?</P>
			<P>&nbsp;&nbsp;<STRONG>1. Choose RI set and data source:</STRONG></P>
			<CENTER>
			<TABLE cellSpacing=2 cellPadding=0 width="100%" border=0>
				<TBODY>
				<TR vAlign=top>
					<TD width=58 valign=middle>
						<IMG src="images/step1.gif" align=left border=0>
					</TD>
					<TD>
					<P>First select the genetic reference population from the menu. Then you have 
					the options to import the trait data from a file or simply enter trait data
					by pasting or typing multiple values into the text box assigned. You can 
					also leave both blank and input the values during next step. Check the 
					trait variance checkbox if you want to use your trait variance data in 
					WebQTL.</p>
					</TD>
				</TR>
				<TR>
					<TD colspan=2>
						<p><FONT COLOR=RED><B><CENTER>OR</CENTER></B></FONT></P>
					</TD>
				</TR>
				<TR>
					<TD width=58></TD>
					<TD>
					<P>You can map loci controlling traits for phenotypes in 
					recombinant inbred sets by searching our database. </P>
					</TD>
				</TR>
				<TR><TD colspan=2 height=20></TD></TR>
				</TBODY>
			</TABLE>
			<img src="images/arrowdown.gif">
			</CENTER>
			<P>&nbsp;&nbsp;<STRONG>2. Check data and set thresholds</STRONG>:</P>
			<CENTER>
			<TABLE cellSpacing=2 cellPadding=0 width="100%" border=0>
				<TBODY>
				<TR vAlign=top>
					<TD width=58 valign=middle>
						<IMG src="images/step2.gif" align=left border=0>
					</TD>
					<TD>
					<P>During this step, you may check your data for accuracy and edit it, 
					if necessary, before analysis. If you haven't entered data, you can now
					input data into corresponding boxes individually. You can manually set 
					the minimal LRS for display and for significance, otherwise default values
					will be assigned if both are left blank. If you want your result to be 
					returned in an email, enter your email address in the assigned box. WebQTL
					will repeat step one and two to let user enter trait variance data if you 
					select that option.</P>
					</TD>
				</TR>
				<TR><TD colspan=2 height=20></TD></TR>
				</TBODY>
			</TABLE>
			<img src="images/arrowdown.gif">
			</CENTER>
			<P>&nbsp;&nbsp;<STRONG>3. Mapping:</STRONG></P>
			<CENTER>
			<TABLE cellSpacing=2 cellPadding=0 width="100%" border=0>
				<TBODY>
				<TR vAlign=top>
					<TD width=58 valign=middle>
						<IMG src="images/step3.gif" align=left border=0>
					</TD>
					<TD><P>Once all you data have been entered and checked, you now can do 
					various mapping analyses using your data against the genotypes of the 
					cross or recombinant inbred set you have chosen. The result of each 
					analysis will be returned in a separate window.</P>
					</TD>
				</TR>
				<TR><TD height=40 colspan=2></TD></TR>
				<TR><TD align=center colspan=2>
						<FORM>
							<INPUT TYPE=Button onClick=Javascript:location="/webqtl/main.py" VALUE="Start WebQTL" CLASS="button">
						</FORM>
					</TD>
				</TR>
				<TR><TD height=40 colspan=2></TD></TR>
				</TBODY>
			</TABLE>
			</CENTER>
		</TD>
		</TR></TABLE>
		</TD>
	</TR>
	<TR>
		<TD align=center bgColor=#ddddff class="solidBorder">
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->
		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	new menu (MENU_ITEMS, MENU_POS);
	//-->
</script>
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-3782271-1";
urchinTracker();
</script>
</BODY>
</HTML>
