<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Literature Correlation</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='css/menu.css'>
<SCRIPT SRC="javascript/webqtl.js"></SCRIPT>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		<TD vAlign=top width="100%" height=200 bgColor=#eeeeee>
		<P class="title">Literature Correlation <A HREF="/webqtl/main.py?FormID=editHtml"><img src="images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>
		

<P>The literature correlations are calculated using the <A HREF="http://shad.cs.utk.edu/sgo/sgo.html" target="_blank"  class="fs14"> Semantic Gene Organizer (SGO)</A>. The SGO software uses a concept-based vector space model called  latent semantic indexing (LSI) to automatically extract gene-gene relations from titles and abstracts in MEDLINE citations (Homayouni et al. <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15308538" target="_blank"  class="fs14">2005</A>). 

<P>These LSI literature correlations are all positive and range from 0 to 1. They were computed in mid 2005 using the complete PubMed collection.

<P>Vector space modeling is a classical information retrieval technique used to identify conceptually related documents, whereby the semantic structure of a document is represented as a vector in word space and the degree of similarity between documents is calculated by the angle between document vectors. LSI improves retrieval by using a singular value decomposition (or principal component analysis) to create a subspace of concepts in which text documents are represented as vectors. 

<P>Each gene is represented as a vector in word or concept space. The cosine of the angle between the query gene vector and all other gene vectors is used to rank related genes. The distribution of cosine values ranges between 1 and -1, where a value of 1 denotes the highest similarity. 

<P>An important advantage of LSI over other vector-based retrieval methods is that relations can be derived even if a direct link between genes has not been established in the literature. The fewer factors that are used for query matching, the more conceptual the relations, and vice versa. Therefore, genes may be conceptually related even if they have not been studied together directly. This utility of LSI makes it ideal for investigating the functional significance of gene associations identified in discovery oriented genomic studies. 

<P>SGO literature correlation values may be used to rapidly identify known relations between co-regulated genes and the latent relations between co-regulated genes based on current literature.

<P><B>Methods</B>

<P>Gene abstract documents are first compiled using titles and abstracts in MEDLINE citations cross-referenced for each mouse gene and its human and rat homologs. These gene documents were assembled and parsed into a dictionary of terms (tokens) and weighted frequencies that are required for the term-by-gene document (sparse) matrix. In effect, each gene document is viewed as a bag of words upon which operations can be performed. There are a number of different word weighting schemes that can be used in vector space modeling (Baeza-Yates and Ribeiro-Neto, 1999). The aim of any scheme is to measure similarity within a document while at the same time measuring the dissimilarity of a gene document from the other gene documents. In SGO, we use a log entropy weighting scheme to decrease the weight of high frequency words, while giving distinguishing words higher weights (Berry and Browne, 1999). 

<P>Term and document vectors for the LSI model deployed by SGO were generated by truncating the singular value decompisition (SVD) of the term-by-gene document matrix to s factors (i.e., only s columns of the orthogonal matrices U and V are used). LSI therefore produces a rank-reduced space in which to compare two gene documents at different conceptual levels. In practice, the maximum number of factors is limited by the number of documents in the collection. Fewer factors may be used for broad (more conceptual) comparisons, whereas a larger number of factors may be used for specific (more literal) comparisons. Other studies have demonstrated that for large documents collections the optimal number of factors is approximately 300 (Landauer et al., 2004).  

<P>For more information on SGO please refer to <a href="http://shad.cs.utk.edu/sgo" target="_top">http://shad.cs.utk.edu/sgo</a>


		
		</TD>
		</TR></TABLE>
		</TD>
	</TR>
	<TR>
		<TD align=center bgColor=#ddddff class="solidBorder">
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->
		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	new menu (MENU_ITEMS, MENU_POS);
	//-->
</script>
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-3782271-1";
urchinTracker();
</script>
</BODY>
</HTML>
