<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Information</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='css/menu.css'>
<SCRIPT SRC="javascript/webqtl.js"></SCRIPT>


</HEAD>
<BODY  bottommargin="5" topmargin="5" leftmargin="5" rightmargin="5" marginheight="5" marginwidth="5" text=#000000 bgColor=#ffffff onload=init() >
<TABLE cellSpacing=3 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD valign="top" colspan="2" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">

Glossary of Terms and Features

<A HREF="/webqtl/main.py?FormID=editHtml"><img src="images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>
<A NAME="freqOfPeakLRS"></A>		
<UL>
<LI>
<B>Frequency of Peak LRS:</B> The height of the yellow bars provide a measure of the confidence with which a trait maps to a particular chromosomal region. WebQTL runs a total of 2000 bootstrap samples of the original data. (A bootstrap sample is a "sample with replacement" of the same size as the original data set in which some samples will by chance be represented one of more times  and others will not be represented at all.) For each of these 2000 bootstraps, WebQTL remaps each  and keeps track of the location of the single locus with the highest LRS score. These accumulated locations are used produce the yellow histogram of "best locations." A frequency of 10% means that 200 of 2000 bootstraps had a peak score at this location. It the mapping data are robust (for example, insensitive to the exclusion of an particular case), then the bootstrap bars should be confined to a short chromosomal interval. Bootstrap results will vary slightly between runs due to the random generation of the bootstrap samples.
<BR>

<A NAME="LRS"></A>
<BR><LI>
<B>LRS:</B> The likelihood ratio statistic provides a measure of the linkage between variation in the phenotype and genetic differences at a particular genetic locus. LRS values can be converted to LOD scores (logarithm of the odds ratio) by dividing by 4.6. The LRS itself is not a precise measurement of the probability of linkage, but in general for F2 crosses and RI strains, values above 15 will usually be worth attention for simple interval maps (so-called "main" scans).
<BR>

<A NAME="additive"></A>
<BR>
<LI>
<B>Additive Effect:</B> The additive effect is an estimate of the change in the average phenotype that is brought about by substituting a single allele of one type with that of another type (<I>A</I> vs <I>a</I>). There are usually two alleles at every locus, and the additive effect is therefore half of the difference between the mean of all cases that are homozygous for one parental allele (<I>AA</I>) compared to the mean of all cases that are homozygous for the other parental allele (<I>aa</I>): 
<BR><BR>
[(mean of <I>AA</I> cases)-(mean of <I>aa</I> cases)]/2
<BR><BR>
The values on the far right of these plots are given in whatever units of measurement are used in the <B>Trait Data and Editing</B> window. For mRNA estimates these units are usually log2 expression estimates. For this reason an additive effect of 0.5 units indicates that the <I>AA</I> and <I>aa</I> genotypes at that particular locus or marker differ by 1 unit (twice the effect of swapping a single <I>A</I> allele for an <I>a</I> allele). On this log2 scale this is equivalent to a 2-fold difference (2 raised to the power of 1). 
<BR>

<A NAME="significant"></A>
<BR>
<LI>
<B>Significant threshold:</B> This threshold represents the approximate LRS value that corresponds to a genome-wide p-value of 0.05, or a 5% probability of falsely rejecting the null hypothesis that there is no linkage anywhere in the genome. This threshold is computed by evaluating the distribution of highest LRS scores generated by a set of 2000 random permutation of strain means. For example, a random permutation of the correctly ordered data may give a peak LRS score of 10 somewhere across the genome. The set of 2000 of these highest LRS scores is then compared to the actual LRS obtained for the correctly ordered (real) data at any location in the genome.  If fewer than 100 (5%) of the 2000 permutations have peak LRS scores anywhere in the genome that exceed that obtained at a particular locus using the correctly ordered data, then one can usually claim that a QTL has been defined at a genome-wide p-value of .05. The threshold will vary slightly each time it is recomputed due to the random generation of the permutations. You can view the actual histogram of the permutation results by selecting the "Marker Regression" function in the <B>Analysis Tools</B> area of the <B>Trait Data and Editing Form</B>. WebQTL does make it possible to search through hundreds of traits for those that may have Significant linkage somewhere in the genome. Keep in mind that this introduces a second tier of multiple testing problems for which the permutation test will not usually provide adequate protection. If you anticipate mapping many independent traits, then you will need to correct for the number of traits you have tested.

<BR>
<A NAME="suggestive"></A>
<BR>

<LI>
<B>Suggestive threshold:</B> This threshold represents the approximate LRS value that corresponds to a genome-wide p-value of 0.63, or a 63% probability of falsely rejecting the null hypothesis that there is no linkage anywhere in the genome.  This is not a typographical error. The Suggestive LRS threshold is defined as that which yields, on average, one false positive per genome scan. That is, roughly one-third of scans at this threshold will yield no false positive, one-third will yield one false positive, and one-third will yield two or more false positives. This is a very permissive threshold, but it is useful because it calls attention to loci that may be worth follow-up. Regions of the genome in which the LRS exceeds the Suggestive threshold are often worth tracking and screening. They are particularly useful in combined multicross metaanalysis of traits. If two crosses pick up the same Suggestive locus, then that locus may be significant when the joint probability is computed. The Suggestive threshold may vary slightly each time it is recomputed due to the random generation of permutations. You can view the actual histogram of the permutation results by selecting the "Marker Regression" function in the <B>Analysis Tools</B> area of the <B>Trait Data and Editing Form</B>. 


<BR>
<A NAME="transcriptLocation"></A>
<BR>

<LI><B>Transcript Location:</B>The small orange triangle on the x-axis indicates the approximate position of the gene that corresponds to the transcript.

<BR>
<A NAME="intmap"></A>
<BR>
<LI><B>Interval Mapping:</B>For interval mapping, the significance of a hypothetical QTL is evaluated at regular intervals across the genome. The significance is evaluated by regression of trait values on expected genotypes, where expected genotypes are estimated from the genotypes of flanking markers and the genetic distance between the analysis point and the flanking markers.

<BR>
<A NAME="snpSeismograph"></A>
<BR>

<LI><B>SNP Seismograph Track:</B>When possible we have computed the number of single nucleotide polymorphisms (SNPs) that distinguish the two parental strains of certain crosses (C57BL/6J vs DBA/2J, and C57BL/6J vs A/J). Regions with high numbers of SNPs are characterised by wider excursions of the yellow traces that extends along the x axis.


<BR>
<A NAME="intmapOptions"></A>
<BR>

<LI><B>Interval Mapping Options:</B>

<BR><U>Permutation Test</U>: Select this option to determine the approximate LRS value that matches a genome-wide p-value of .05.
<BR><U>Bootstrap Test</U>: Select this option to evaluate the consistency with which peak LRS scores cluster around a putative QTL. Deselect this option if it obscures the SNP track or the additive effect track.
<BR><U>Additive Effect</U>: The additive effect (shown by the red lines in these plots) provide an estimate of the change in the average phenotype that is brought about by substituting a single allele of one type with that of another type.
<BR><U>SNP Track</U>: The SNP Seismograph Track provides information on the regional density of segregating variants in the cross that may generate trait variants. It is plotted along the X axis. If a locus spans a region with both high and low SNP density, then the causal variant has a higher prior probability to be located in the region with high density than in the region with low density.
<BR><U>Gene Track</U>: This track overlays the positions of known genes on the physical Interval Map Viewer. If you hover the cursor over genes on this track, minimal information (symbol, position, and exon number) will appear.
<BR><U>Display from <I>X</I> Mb to <I>Y</I> Mb</U>: Enter values in megabases to regenerate a smaller or large map view.
<BR><U>Graph width (in pixels)</U>: Adjust this value to obtain larger or smaller map views (x axis only).

<BR>
</UL>

<BR>
<BR>
		</TD>
	</TR>
	<TR>
		<TD colspan=2 align=center bgColor=#ddddff>
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->
		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	new menu (MENU_ITEMS, MENU_POS);
	//-->
</script>
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-3782271-1";
urchinTracker();
</script>
</BODY>
</HTML>
