<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>SNP Browser INFO</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='css/menu.css'>
<SCRIPT SRC="javascript/webqtl.js"></SCRIPT>
<SCRIPT SRC="javascript/dhtml.js"></SCRIPT>

</HEAD>

<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff >
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<p class="title">Information on the Variant Browser <a href="/webqtl/main.py?FormID=editHtml"><img src="images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>

<p> There will eventually be four different variant types in the variant browser: SNPs, insertions and deletions (indels), copy number variants (CNVs), and inversions. At present it is only possible to search for SNPs and indels (EGW, August 2009). 

<p><B><font size=+1>SNPs</font></B>

<P>Known problems
<OL>
<LI>Searching for SNPs of the Function type "Mis/non-sense" does not work (try Tpmt)
<LI>It is not yet possible to only show that SNPs that differ among the selected subset of strains
<LI>Information on the strand used to call the SNP is ambiguous
<LI>Function text reads either "synonymous" or "Silent". All should read synonymous
<LI>Selecting InDel should reset Function settings
</OL>


<br>You can browse SNPs either by submitting a gene symbol or SNP Id (with higher priority) or by defining a viewing range (<U>currently a maximum of 5000 SNPs or 50Mb</U>).

<P >Genotypes are from <A HREF="http://www.celera.com/" TARGET="_blank" class="fs13 fwn">Celera Genomics</A>, the <A HREF="http://mouse.perlegen.com/mouse/" TARGET="_blank" class="fs13 fwn">Perlegen/NIEHS resequencing project</A>, the <A HREF="http://www.well.ox.ac.uk/mouse/INBREDS/" TARGET="_blank" class="fs13 fwn">Wellcome-CTC SNP Project</A>, dbSNP, the  Center for INtegratie and Translational Genomics (CITG) at the University of Tennessee Health Science Center, and the <A HREF="http://aretha.jax.org/pub-cgi/phenome/mpdcgi?rtn=docs/home" TARGET="_blank" class="fs13 fwn">MPD</A>. 
<p>
In brief, the column headers are as follows:
<br>
<B>[Blank]</B>: Incremental, temporary ID of the SNPs found in the search.
<br>
<B>ID</B>: Either the official NCBI  reference SNP (rs) number  or the identifier given by the institution where the SNP was found. A large number of SNPs have multiple local IDs (and thus duplicate records). mCV records are from Celera, NES numbers are from Perlegen, and MRS numbers are from UTHSC (Memphis reference SNP). The MRS SNPS are a high quality subset of about 2.8 million SNPs generated by sequencing DBA/2J using SOLiD short sequence reads (about 25 x shotgun) performed by Williams and colleagues. These MRS SNPs were generated by Dr. Xusheng Wang and entered into the Variant Browser by Evan Williams (August 2009). The IDs are linked to small tables that will bring up additional information.
<br>
<br>
<B>Chr</B>: The chromosome on which the SNP is located.
<br>
<B>Mb</B>: The location of the SNP in megabases. Position data are currently set to the NCBI Mouse Genome Build <A HREF="http://www.ncbi.nlm.nih.gov/genome/guide/mouse/">37.1</A> (UCSC mm9, July 2007). The link uses the sequence flanking the SNP (if available) to verify the location using the UCSC BLAT alignment  to the UCSC Genome Browser.
<br>
<B>Domain</B>: If applicable, the region on a gene where the SNP is found.
<br>
<B>Gap</B>: The distance from one SNP to the next; SNPs with gaps of zero are duplicates and (should) contain the same data. Checking "non-redundant" will ensure that all SNPs are unique, but some allele data may be hidden.
<br>
<B>Gene</B>: The gene on which the SNP is found, if applicable. The link goes to the gene's information on NCBI.
<br>
<B>Conservation</B>: How conserved the SNP is across species (note: mammals only: Rat, Rabbit, Human, Chimp, Rhesus Monkey, Dog, Cow, Armadillo, Elephant, Tenrec, Opossum). A high conservation score means the SNP is highly conserved across species; nearly every one will have the same allele. A low score means that the allele is evenly distributed between both. Thsi score is downloaded from the Vertebrate Multiz Alignment Conservation scores from <A HREF="http://genomebiology.com/2007/8/6/R124">UCSC </A> 




in mid 2009, using the following configuration parameters and species: 
<br>
<B>Alleles</B>: The Major/Minor alleles of the SNP. This is based on frequency, and currently counts imputed SNPs and known SNPs separately, so you may see t/T, if imputed t is the most common allele, and known T the least common allele. This is unusual, however; in most cases, it should be the same.
<br>
<B>Source</B>: The source for the SNP. The vast majority (99%) are from either Celera, Perlegen, or UTHSC. Many of the Perlegen SNPs are labeled "Perl Impute," which contain imputed SNP data from Jackson Laboratories, but are otherwise the same as the Perl/NIEHS SNPs.
<br>
<B>129S1/SvImJ</B>: The first mouse alphabetically in the list of 74 strains and their corresponding SNPs.
<p>

For a more detailed explanation of the symbols used, see the <A HREF="http://aretha.jax.org/pub-cgi/phenome/mpdcgi?rtn=snps/help#resultfields" TARGET="_blank" class="fs13 fwn">field descriptions</A> for a similarly structured SNP browser from Jackson Labs. </p>

<p><B><font size=+1>InDels</font></B>
<br>The InDel data currently consists of data from the comparison of the genome of the C57BL/6J strain of mouse relative to the DBA/2J strain of mouse. C57BL/6J is considered the reference, and plus and minus symbols in the <B>Size</B> field indicate a loss or gain of sequence in other strains (currently DBA/2J). This data is from UTHSC SOLiD sequencing and was analyzed by Dr. Xusheng Wang. As this feature is further implemented, InDel data from the Sanger Institute and other sources will be included.

<p><B><font size=+2, color=gray>CNVs</font></B>
<br>
Copy Number Variant data are currently not available in the Variant Browser, but the data should be available within the next few months (EGW: August 2009).

<p><B><font size=+2, color=gray>Transposons</font></B>
<br>
Tranposon data are currently not available in the Variant Browser, but the data should be available within the next few months (EGW: August 2009).

		</TD>
		</TR></TABLE>
		</TD>
	</TR>
<TR>
   <TD align=center bgColor=#ddddff class="solidBorder">
      <!--Start of footer-->
	<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
	</TABLE>
		<!--End of footer-->
		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	new menu (MENU_ITEMS, MENU_POS);
	//-->
</script>
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-3782271-1";
urchinTracker();
</script>
</BODY>
</HTML>
